ID065	Sex linked traits are traits that lie on the sex chromosome. Sex influenced traits are when the sex of the organism affects the genes expressed. Sex limited traits are when traits are limited to strictly one sex.
ID065	Conditional mutants are valuable because they can be crossed to determine more information about a specific gene or allele.
ID065	Crossing over and recombination occurs during prophase when the homologous chromosomes exchange genetic information by swapping parts of the chromosome.
ID038	Traits and genes that are inherited due to the sex chromosomes. Only applicable to diploid organisms.
ID038	some mutations only appear as phenotype if the external conditions are right. These are conditional mutants.
ID038	This is the basis of variation in population. Homologous chromosomes exchange part of their DNA leading to newer phenotypes.
ID118	Sex-linked is traits that are on x and y chromosome. sex influences and sex limited traits are on autosomal chromosomes but are influences by sex of the organism.
ID118	Conditional of mutant are expressed by environmental influences. example in hot area of body some protein denatures and the gene is not expressed, but cold area in body express the gene.
ID118	It gives variation to individual genetic makeup. It occurs during prophase I of meiosis which occurs in germ cell producing sperm or egg and which is yield for reproduction of gamete.
ID081	The sex of the organims helps determine and influence characteristics in the organisms.
ID081	Environmental influences, such as temperature and pH, cause proteins to be turned on and off in the organism which causes the mutant characteristic to be present.
ID081	When chromosomes close together cross over one another and combine or switch genes-- producing more variability in offspring.
ID136	Traits and characteristics that are passed down from parents to their children.
ID136	The value that conditional mutants bring are the diversity it brings to a population.
ID136	Crossing over and recombination are very valuable process that occur. They are the cause behind genetic variation.
ID134	sex-influenced traits are caused by autosomal genes and are influences by hormones and sex chromosomes.Sex limited traits are genes involved in secondary sex characteristics, and trigger section on y-chromosome. Sex linked traits are traits linked to specific sex like color-blindness
ID134	Wild type genes and mutations can be influenced by the environment, pH, position, or heterochromatic.
ID134	It happens at 4-chromatid stage.Crossing over occurs in prophase and can only be detected if thee are clear cut differences in the crossed chromatids.On of the sources of genetic variability. Crossing of chromatids from one allele to the other.
ID035	Gene located on the chromosome that's only located on the X/Y chromosome
ID035	Mutation that is a wild type allele
ID035	An exchange and Mix of information that happens during stages of Mitoisis
ID055	Sex linked is a gene that is carried in the sex chromosome by one sex or the other. Sex influenced is a gene where the expression is influenced dependent on the sex of the individual. Sex limited traits are when the the gene's phenotype is specific to one sex.
ID055	An example of one of these mutations is temperature-sensitive mutations. These help species to activate the gene based on the temperature (environment) around them. This could be useful for survival.
ID055	This allows gene exchange between homologous chromatids. Recombination shuffles alleles up making for different combinations in linkage groups.
ID109	Some genes can be sex linked, like baldness in males.
ID069	Sex-linked traits are located on the sex chromosomes; sex-influenced traits produce different phenotypic effects for each gender; and sex-limited traits are traits specific to either gender.
ID069	Temperature-sensitive mutants can be easily studied and provide great insight into viral genetics.
ID069	Crossing over occurs during meiosis when homologs exchange genetic information. Recombination is the new order of the alleles.
ID027	Sex linked only occur on the x chromosome. Sex influenced occur on the autosomal chromosomes, but their outcome is affected by the sex of the organism. Sex limited traits are autosomal, but they only occur in a certain sex.
ID027	Some mutations will only occur in certain conditions, which means they can be controlled.
ID027	This enhances variation in a population much more than just mutations alone.
ID079	Sex linked traits refer to genes located on sex chromosomes, rather than autosomes. The gene is usually found on the x-chromosome Sex linked traits are usually passed from a grandfather to mother to son. Sex influenced traits refer to genes on autosomes, but the sex of an individual determines how the gene is expressed. Sex limited traits are only expressed in either a male or a female, depending on which trait.
ID079	In conditional mutants, the environment determines if wild-type or mutation is present.
ID079	Crossing over is the exchange of DNA segments between chromatids.  After crossing over occurs, recombination determines which chromosomes are in the daughter cell.
ID048	Traits that are only gender specific. These genes are carried by sex chromosomes.
ID048	There are numerous alleles that have not been discovered and traits. We do not what a specific gene codes for until a mutation occurs
ID048	Both parts of meiosis that occur. Crossing over happens in prophase 1 one of meiosis. It is the exchange of genetic material between homologous chromosome that result in recombinant chromosomes.
ID039	These are genes that on found on the x chromosome and have certain pattern of inheritance
ID039	conditional mutants allows for certain phenotypes to be express under specific conditions
ID039	Allows for more variation in genes
ID094	Some genes are expressed only in certain situations regarding gender, either expressed only in the male or female sex or x-linked genes.
ID094	Conditional mutants are expressed only under certain circumstances.
ID094	Crossing over is more rare than non-crossing but still is seen to cause mixing of traits.
ID007	Some traits are expressed differently in male and females due to hormones' effects on expression (influenced), some are transmitted on the X (few on the Y) which due to the XX/XY difference causes differences in patterns of inheritance (linked), and some are expressed only in males or only in females (limited).
ID007	Conditional mutants are useful for studying gene function because their expression of a trait depends on the environment (e.g. temperature).
ID007	Recombination occurs when chromosomes exchange genetic material while forming the synaptonemal complex, and so a mixture of the grandfather and grandmother's genes may appear on on chromosome passed down to a grandchild.
ID105	Sex linked refers to genes linked to the X chromosome such as X liked dominant or recessive. Sex influenced refers to recessive autosomal genes that both males and females can have but will only be expressed for a particular sex. An example of this would be cock feathering which we saw in class. Sex limited traits are genes involved in secondary sex characteristics.
ID105	Conditional mutants are things like temperature sensitive mutations or pH sensitive mutations. These are mutations caused my alleles responses to there environment.
ID105	This the shuffling of alleles caused by the breaking and linked of different chromosomes.
ID033	There are different genes on the X and Y chromosomes. Males get an XY while Females have XX.
ID033	A mutation may be on a gene but not as bad as if there was another mutation on the same gene.
ID033	Crossing over results in new genetic combinations which makes the species more diverse.
ID026	Some genes are passed down to offspring either on the X or Y chromosome, which can make a specific sex have a higher likelihood of inheriting the trait.
ID026	Mutants are important because the help diversity within a population and can help species adapt.
ID026	Crossing over and recombination mixes the genes of the two parent cells to make a "blended" type of chromosome with genes from both parents.
ID070	Both sex influenced traits and sex limited traits are autosomal which mean genes are not carried on the sex chromosomes. Sex-influence traits are expressed in both sexes, but more frequently in one more than in the other sex. While sex limited traits whose expression is possible only in of the sexes.
ID070	In the class of conditional mutations, a mutant allele causes a mutant phenotype in only a certain environment, called the restrictive condition, but causes a wild-type phenotype in some different environment, called the permissive condition. Geneticists have studied many temperature-conditional mutations.
ID070	During crossing-over, homologous chromosomes line up along their length, intersect a section of their chromatids, and exchange corresponding parts of that chromatid with each other. So in the end, one of the homologous chromosomes has a segment of the other homologous chromosome's DNA, and vice versa.
ID057	Traits carried on the sex chromosomes X and Y. Sex linked traits are more common in males because they only have one X chromosome. Sex influenced are autosomal traits that are influenced by sex. Sex limited traits can determine parental carriers.
ID057	A phenotype is influenced by its environment. Temperature is a type of conditional mutant.
ID057	The cause of variation seen in generation to generation. Chromosomes exchanging genetic material.
ID050	Different traits are influenced to various degrees of severity based on the X chromosome
ID050	Conditional mutants are useful because they can be grown under limited conditions and then used for study in experiments with strict parameters.
ID050	The process that occurs in meiosis in which chromosomes exchange genetic material.
ID089	Some traits exist only on an X chromosome. Some traits exist on autosomes but are expressed differently based on the sex of the organism.
ID089	Conditional mutants are valuable because they can be grown in permissive conditions and then moved to restrictive conditions for study.
ID089	Crossing over occurs during the tetrad phase of meiosis. Nonsister chromatids physically break and reconnect, recombining with others, and mixing genes.
ID150	Because males have only one X chromosome as opposed to two in females, traits linked specifically to this chromosome have different effects on the two sexes, like different inheritance probabilities.
ID150	Certain environmental conditions can prevent a dominant phenotype and replace it with the recessive phenotype.
ID150	Crossing over is the exchange of genetic information between homologous chromosomes during sexual reproduction. Recombination is the more general term to describe the movement of genetic material along a single chromosome or between two chromosomes in the process of crossing over. These processes can produce offspring that would not be expected given parental genotypes.
ID024	Some chromosomes are x linked and or sexually linked. Thus they are passed down in the sex chromosomes
ID024	Conditional mutants phenotype depends on its environment
ID024	This is the main source of variation in a population
ID077	These are carried on sex chromosomes. Girls are XX boys are XY. Depending on the gene it can affect girls and boys differently.
ID077	Mutations that are only expressed at a certain temperature.
ID077	Crossing over and recombination can lead to new phenotypes. This occurs in Prophase I in Meiosis.
ID013	Sex-linked traits occur on the X chromosome. As a result, these traits are usually passed down from mother to son. Sex-influence traits and sex-limited traits both depend on hormones in the different sexes. Sex-influenced traits can occur in both sexes but is more common in one sex while sex-limited traits is only shown in one sex.
ID013	Conditional mutants depend on the environment. For example, temperature-sensitive alleles are mutants that might function at higher temperatures but not at lower temperatures. A seimese cat displays these alleles in that they are black where their fur is cold and white where there fur is warm.
ID013	Crossing over and recombination occurs in prophase I of meiosis during the tetrad stage. Because of crossing over, individuals inherit a variety of alleles from their father and mother. In other words, their allele from their mother will be a combination of their maternal grandmother and grandfather's alleles. Crossing over is a main source for genetic variation.
ID056	All of these are different than the autosomal traits because they are inherited on the X and Y chromosomes. Sex limited and sex influenced traits are different from autosomal because they are dependent on the hormone constitution of the individual.
ID056	Conditional mutants can influence the phenotype of an individual. The influence originates from the environmental surroundings. For example, there are temperature-sensitive mutations that can affect phenotype.
ID056	Recombination occurs in Prophase I of meiosis. It creates genetic differences in genes. It is a combination of the two parental gametes.
ID014	This is referring to characteristics that are either caused by the hormones present in the individuals or the traits that are caused by the sex of the individual.
ID014	These are mutants that are expressed at certain pH's and temperatures. It can have either a benign or malignant affect of the individual.
ID014	Crossing over allows for genetic diversity and is common in meiosis.
ID138	There are different kinds of traits that are linked to different chromosomes. X and Y chromosomes are the sex chromosomes because they determine our gender. There are traits that are passed on by these chromosomes or partially related to these chromosomes. This means that there are things that can/cannot be passed to the next generation depending on gender.
ID138	Conditioned mutations are mutations that occur in certain conditions. An example would be a mutation that only occurs in high temperatures. I suppose the value of this would be that it would help organisms adapt to their environments if they needed to with conditional mutations.
ID138	Crossing over is one of the major factors that result in genetic variability. It is when a sister chromatid from each chromosome (once they aligned at the metaphase plate) cross and transfer some genetic material from one to the other.
ID095	A trait that is sex linked depends on the gene lying on the x or y chromosome of the parent. This affects whether the progeny will receive the trait or not, depending on whether they're female or male.  Males receive an x chromosome from the mother, females receive an x from both parents.  If the father has a dominant trait on his x chromosome, this is surely to show in the female progeny.  The male progeny will not receive this x chromosome from the father.
ID095	Mutants are a large part in the variability of genetics.  Aside from traditional recombination, and probability of receiving certain genes, mutations allow for the advancement of a species to eventually become another species.  Natural selection in mutants allows for this trait to carry on, if it is a desirable trait.
ID095	Crossing over and recombination severely intensifies the variability of genes passed on.  These parts of genetics can actually change gamete formation from heterozygous to homozygous, resulting in even more unique types of individuals within a species.
ID068	Sex-linked genes are genes located on the sex chromosomes (X&Y). Females are XX and males are XY. An X-linked recessive condition usually has a higher affect in males because they only need one recessive allele on their one X chromosome to have the condition. The pattern of inheritance for these conditions begins in the grandfather, the mother is a carrier, then it is seen again in the grandson. Sex limited means both sexes may have the genotype for a certain phenotype on an autosomal chromosome but only one of the sexes actually expresses the phenotype depending on their sex. Sex influenced is a genotype may be expressed different in males than in females.
ID068	Conditional mutants is when a individual has a genotype but the conditions/environment they or their genes are in changes the expression of the gene. An example would be the pH environment or temperature affects on the genes.
ID068	Crossing over and recombination results in variability of traits. This phenomenon occurs less often than not. It is the breaking and recombining of replicated chromosomes producing offspring phenotype that is different than expected.
ID062	traits can be determined by x or y chromosomes.
ID062	conditional mutants add variety to the gene pool, they are due to conditional factors.
ID062	crossing over and recombination, add genetic variation to chromatids
ID103	Sex linked traits are traits that are located on the x chromosome.
ID103	Conditional mutants allow a species to adapt more rapidly to changing environmental conditions
ID103	Crossing over and recombination allow an individual to pass on genetic material from both parents.
ID061	Sex linked traits are traits that are carried on the sex chromosomes. Many traits are X-linked and are carried on the maternal X chromosome. Sex limited traits are traits that are not on a sex chromosome but are found exclusively in one gender. Sex influenced traits are traits that are not limited to one gender but have a greater effect in one gender. For instance, a man heterozygous for the gene that would make them bald will go bald while a woman with the same genotype would not.
ID061	Conditional mutants are mutations caused by certain environmental conditions. These are commonly influenced by temperature which can lead to variation in an animal's fur coloring depending on the part of the body.Also, nutritional mutations can make it difficult for some to digest certain substances.
ID061	Crossing over and recombination provide variety in gametes. Crossing over allows for new gene combinations on a chromosome rather than producing a chromosome exactly identical to the parent. Recombination also allows this, creating new combinations of alleles.
ID063	This describes how sex linked traits are passed from parents to offspring on the X & Y chromosome.
ID063	This describes offspring with uncharacteristic traits to that of their parents.
ID063	Occurs during prophase 1 of meiosis.  This topic relates to how offspring inherit genes from both parents.
ID129	Sex limited- only one sex expresses a certain phenotype
ID129	A mutant allele will cause a mutant phenotype in a certain environment but in another environment it will cause a wild type phenotype. (heat sensitive lethals in Drosophilia.
ID129	Recombination is the way in which new allele combinations on chromosomes form.
ID075	There are all different. The sex linked is talking about the X chromosomes.
ID075	Conditional mutants help us compare other alleles with.
ID075	This plays a huge role in variability in populations.
ID123	Traits inherited based on the sex of an individual.  Sex-linked traits are carried on the X chromosomes.
ID123	Conditional mutants are mutations due to environment or other stimuli.
ID123	Crossing over and recombination are the cause of genetic diversity.
ID008	Sex linked traits are found on the X chromosome, sex chromosome. These features are associated with the genes. When there are sex influenced traits, these traits are mostly found in one particular sex
ID008	Mutations of the wild type can be a positive outlook in terms of that population to be more fit for the environment and adapted
ID008	Crossing over and recombination gives us the unique offspring that are made from genotypes of both mom and dad.
ID090	An individual's sex influences the expressed phenotype. The genes are usually present on the autosomes. Sex limited traits are traits that are only seen in one sex or the other (ex: beards in males). Both sexes have the gene for growing a beard, but the gene is only "turned on" by the hormones present in males. Sex influenced traits can be in both males and females, but have the expression of the gene depends on the hormones present.
ID090	Conditional mutants. The expression is influenced by an environmental condition such as temperature, nutrition etc.
ID090	Crossing over/recombination occurs in Prophase 1 and increases the genetic variation as genes between the maternal and paternal chromosomes in a homologous pair are exchanged to create a new combination of alleles.
ID000	Genes on the X chromosome have a distinctive mode of inheritance that can be detected in pedigrees but some autosomal genes can mimic inheritance of sex-linked genes
ID000	Conditional mutants are functional at one environmental condition and not another.  This difference allows comparison of function vs non-function in the same organism
ID000	The unit of inheritance is the chromosome but recombination re-assorts linkage groups
ID043	sex linked influence is when the x chromosome influence most of the traits that you may receive such as colorblindness which is recessive trait. sex influenced traits are the traits that are affected due to the presence of male or female hormones such as hen has feather due to testosterone presence but female doesnt have it.  i dont excatly yet understand sex limited traits.
ID043	i am guessing since i dont actually know it for sure. Conditional mutants may be the one which are mutated based on the environment or other factors such as temperature or PH changes.
ID043	This is the most important aspect since it is the cause of diversity in our population and keeping at least some of us safe to various disease. For instance, due to crossing over and independent assortment there are few individuals that have cured HIV without external help whatsoever.
ID131	Sex linked genes are primarily on the X chromosomes of females or on the X chromosome of the male, and they carry certain genes and conditions. Sex influenced and sex limited refers to the role that an organisn's sex plays on certain phenotypes.
ID131	Conditional mutants display certain characteristics or phenotypes under certain conditions and other phenotypes in other conditions.
ID131	Genetic recombination involves the pairing of homologous chromosomes. Recombination ensures that the offspring will be genetically different from either parent.
ID115	Sex linked traits involve genes on the sex chromosomes that are usually passed on from mother to son, since only one x chromosome is inherited. Sex influenced traits are inherited according to the development of hormones in an individual, and sex limited traits are inherited based on sex alone.
ID115	The value of conditional mutants can determine what proteins are produced, such as temperature conditional alleles that affect the phenotypes of certain organisms or nutrient conditional mutants, such as organisms that cannot metabolize lactose.
ID115	Crossing over is the process of exchanging genes among non-sister chromatids, and recombination of gametes leads to variability in offspring
ID052	Patterns of gene expression may be affected by the sex of an individual even when the genes
ID052	Conditional mutants can develop the expression of a phenotype due to its environment. This is important for selective advantages to occur by mutations.
ID052	The occurrence of crossing over and recombination of genes on chromosomes are sources for genetic diversity.
ID023	Aspects of genetics that involve characteristics concerning male or female assignment. Ex: Male pattern baldness and PWS
ID023	Some mutations depend on aspects such as temperature and nutrition.
ID023	Crossing over and recombination allow for genetic variability and variation within a population.
ID049	Sex linked genes are genes that are expressed in relation the individual's sex. Unlike with autosomal chromosomes, an individuals probability of inheriting or expressing a trait relates to their sex.
ID049	Conditional mutants are phenotypes that are only displayed in certain conditions such as a certain temperature.
ID049	Crossing over is the physical exchange of genetic material at the chiasma on a chromosome. Recombination is when an offspring is produced with unique genotype that is unlike their parent. These lead to variety and diversity within a species.
ID078	Expression of genes based on the sex of the offspring
ID078	A change in the expression of an allele based on environmental conditions, such as temperature or pH
ID078	When chromosomes break and recombine with other sections from homologous chromosomes
ID099	Traits that are x-linked have a unique inheritance pattern
ID099	conditional mutants can be difficult to observe due to their changing phenotypes at different conditions
ID099	crossing over increases the the diversity within a set of genes
ID145	Certain traits are found only on the X chromosome, which means this trait could be only found in males since they have one X and a Y.
ID145	A mutant allele causes a different phenotype under certain circumstances.
ID145	This is responsible for variability in populations.
ID031	These are different ways to pass down gene to your offspring.
ID031	This allows for the same allele to be present within different genes.
ID031	The exchanging of genes through swapping parts of chromosomes.
ID091	How we can be hemizygous for certain traits.  That is, men have an X, and Y chromosome, while women have XX. So, genes on each chromosome can be inherited uniquely to men and women. For instance, Men always get their Y from their dad.
ID091	Conditional mutations allow for adaptations to environmental conditions.
ID091	Crossing over and recombination account for genetic diversity. It also allows us to see if genes are linked or how close together they are on the chromosome.
ID060	Sex linked genes are genes found in only sex chromosomes, such as X and/or Y. Sex influenced genes are in autosomal chromosomes. Sex limited traits are also autosomal but only expressed in one gender.
ID060	Conditional mutants are mutations that have wild-type phenotypes in certain permissive situations and mutant phenotypes in restrictive situations.
ID060	Happens in meiosis, and is a transfer and exchange of genetic material between homologous chromosomes during sexual reproduction.
ID151	Certain disorders or diseases are found on the sex chromosomes instead of the sutosomes. Furthermore, these diseases are more prevalent in males since they receive only one copy of the X chromosome and one copy of the Y chromosome; whereas, females receive two copies of the X chromosome thus enabling females to have a better chance of masking the allele if it is X-linked recessive. Sex limited traits are those traits that are specific to a gender and are found on the sex chromosomes as well.
ID151	These mutations occur when they are triggered by some environmental change such as an allele that controls color of a cat and becomes darker as the temperature increases.
ID151	These two events aid in creating variability within a population. Crossing over occurs in Prophase in Meiosis I.
ID040	Sex linked genes are genes that are located on the sex chromosomes, mainly looking at the X chromosomal inheritance patterns. Sex influenced genes are when the sex of the individual influences the phenotype. Sex limited traits is when the expression of a phenotype is limited to one sex.
ID040	Conditional mutants' phentotypic expressions is determined by environmental conditions. They are useful to study mutations that interrupt processes during development that are normally detrimental.
ID040	Crossing over and recombination is when alleles on homologs switch places to increase genetic variability.
ID086	They are all linked to the X chromosome and are completely traceable.
ID086	They can be used to identify a certain environment using conditional mutants that will exhibit their traits.
ID086	This is the most important phenomenon because it causes offspring to have a unique mix of genes.
ID058	Sex linked are genes that are connected with either the X or Y chromosome. When looking at a pedigree, one can see the effects. Sex influenced is autosomal in nature and not sex-linked. However, sex does contribute to how it is expressed. Sex limited is involved in secondary sex characteristics.
ID058	Conditional mutants are the explanation for strange phenotypes that one can not explain from the genotype such as blond labs when they have the genes for black fur.
ID058	The source of genetic variability which is extremely important in order for a species to not end up with a lot of lethal genes. Crossing over is when part of one chromatid exchanges with another chromatid. It gives variability that could not be accomplished with just the gametes.
ID092	These genes rely on whether they're on the X or Y chromosome.
ID092	These only happen in certain conditions
ID092	This allows for increased genetic variety.
ID121	these specific traits only appear in a specific gender of the organism
ID121	the mutations of the species that is only activated when exposed to a certain triggered environment
ID121	this process reshuffling of alleles on different chromosomes and breaks up allele combinations in linkage group
ID074	sex linked: the gene is on the X or Y chromosomes. Sex influenced is on an autosomal chromosome but its expression is influenced by the gender of the organism. sex limited is present in both sexes but expressed in only one
ID074	when the wild type is under permissive environmental conditions and the mutant is under restrictive environmental conditions
ID074	crossing over and recombination happen during mitosis and it is when parts of sister chromatids are switched over on to different sister chromatids
ID034	the phenotype of F1 generation is inheritance from mother and father. the chromosome of offspring is receive one from mother and one from father in X-linkage
ID034	the mutants is describe as a species try to adopt to the environment. some is good, some is bad
ID034	higher mutation rate, haploid DNA mutation can't be marked
ID120	Genes located on the x chromosome
ID120	X
ID120	Allows variability in a population
ID064	Sex linked traits are located on the sex (usually X) chromosomes. Sex influenced and sex limited traits are traits that are located on autosomal chromosomes, but whose expression depends on the sex of the individual.
ID064	I found the example of Siamese cats to be an interesting phenomenon explained by conditional mutations, which are only expressed under a certain set of conditions. This can be determined by temperature, as in Siamese cats, whose different fur colors are explained by varying body heat on different parts of the body, or by other factors such as pH or nutrition.
ID064	Crossing over is a major source of variation between individuals of a population. This variation is caused because crossing over, a physical act of two chromosomes linking at a point called a chiasma, results in the recombination of genes, thus creating genetic variation.
ID088	Things influenced by sex chromosomes.
ID088	Mutations that diversify the population.
ID088	Crossing over the get new genes.
ID137	A sex linked trait is a trait that is expressed based on its position on a sex chromosome. A sex influenced trait a genotypic trait that is differentiated based on whether the chromosome belongs to a male or female. Sex limited traits are differentiated based on one sex being able to express a trait that other cannot.
ID137	Conditional mutants are conditional because their expression is determined by temperature or other environmental conditions.
ID137	Crossing over occurs in prophase 1 of meiosis in which two non-sister chromatids share DNA. Recombination occurs during anaphase 1 in which the homologous chromosomes are separated into the daughter cells. The chromosomes are separated randomly which increases variation.
ID082	Sex linked traits are traits that are found on the sex chromosomes.  X-linked inheritance is a common example of sex linked traits. Sex limited inheritance refers to traits that are only expressed phenotypically in one sex or the other. Sex influenced inheritance occurs when phenotypic expression differs between sexes, but the gene is still expressed in either sex.
ID082	Conditional mutants have varied phenotypic expression of their mutation depending on the environment. These mutants are valuable in genetics research because they provide an easy way to study the effect and influence of environment factors in gene expression.
ID082	Crossing over and recombination describes the process in which genes are translocated to sister chromatids during prophase I of meiosis. Without crossing over, there would be much less genetic variation, as genes would be passed down in blocks, always with one another, giving no opportunity for new combinations.
ID124	sex linked is specific traits to different sex. sex influenced is based on the sex chromosomes some genes are turned on or off based of what the sex turns out to be.Sex limited are traits limited only to certain sexes like breast tissue and varying reproductive organs.
ID124	conditional mutations are conditional upon the environment or the ability of a gene to be on or off at different conditions such as temperature
ID124	cross over dna promotes genetic variability among the gametes produced during meiosis
ID045	Sex linked traits are the differences in the X and Y chromosomes, and why certain traits are expressed differently depending on which chromosome the gene is found on.
ID045	Conditional mutants lets different phenotypes adapt to their surrounding environment.
ID045	Recombination allows for there to be different mixtures of genes expressed differently in future generations.
ID002	Traits linked to the X or Y chromosomes
ID002	Conditional mutation is a mutation that has wild-type phenotype under certain permissive environment. The environment defines what traits are expressed
ID002	results in the shuffling of alleles on different chromosomes
ID005	Expression of alleles are determined on the x and y chromosomes.
ID005	mutant alleles cause mutant phenotypes.
ID005	Crossing over and recombination allows for there to be variation in populations.
ID051	Sex limited: expressed only in one gender usually autosomal
ID051	conditional mutants can be used to mimic chemical inhibition of the corresponding gene products which can be used with developing new anti biotics
ID051	During crossing-over, homologous chromosomes line up along their length, intersect a section of their chromatids, and exchange corresponding parts of that chromatid with each other.
ID144	Sex-linked inheritance: genes located on sex chromosomes
ID144	a mutant allele causes a mutant phenotype in only a certain environment
ID144	ways that we can achieve genetic variation
ID076	sex influenced genes are not on the 23 chromsome; sex linked - on x chromosome, typically will see a mother carrier (or the rare mother with the trait) and sons with the trait; sex limited- another autosomal (non x chromosome) gene that is exclusive to males or females even though not sex linked
ID076	the wild type can be toned down but still expressed to some degree
ID076	variation*** which is key in genetics and for populations survival
ID140	Sex linked are genetically based on x and y chromosomes, sex influence will be influenced by those and limited will change based on gender
ID140	recombination is important because it is responsible for the variation in genes
ID101	The sex linked genes are related to the x chromosome. If it is a recessive gene, then it will be visible when present in all males. But females can be carriers, and only show it when both recessive genes are present.
ID101	The conditional mutants indicate the change in phenotype as the physical attributes or environments change for the organism.
ID101	The Crossing over of genes enables the recombination of chromosomes that creates greater diversity in the population. For the parent genes are no longer identical but have exchanged genome segments.
ID108	Dominant or recessive sex linked genetic disorders occur on the x chromosome.
ID108	Mutations create variability in the gene pool and through natural selection the unfavorable mutants will be eliminated.
ID108	Crossing over and recombination creations variation in the population. This ensures the potential to adapt to various situations
ID011	Sex linked genes are found on the X and Y chromosomes. Sex influenced genes are expressed differently between males and females but are located on autosomal chromosomes. Sex limited genes are found in both males and females but are turned off in one sex while active in the other.
ID011	There are certain mutant alleles that only function when a certain temperature is reached for the cell.
ID011	Crossing over and recombination is when sections of chromosomes crossover to the other chromosome and recombine causing genetic diversity in the new cell.
ID098	Sex linked traits are the those influenced by the genes on the sex chromosomes. Sex influenced traits are the result of varying gene expression based on the sex of the organism (e.g. baldness in humans). Sex limited traits are like sex influenced traits, except they are more of an all-or-nothing type of expression, instead of varying degrees of expression.
ID098	Conditional mutants play a role in gene expression by reacting to the environment. Depending on certain factors such as temperature, nutrition, and the age/developmental stage of the organism, certain genes may be in a permissive or restrictive range, which will impact whether or not they are expressed.
ID098	Crossing over and recombination are two of the big ways in which genetic variation is increased within a population. In crossing over, paired homologous chromosomes exchange genetic material during prophase I in meiosis. In recombination, chromosomes line up arbitrarily during metaphase I, and the manner in which they are pulled apart during anaphase I is therefore random. This means that the two daughter cells can receive any combination of the genetic material.
ID107	Sex-linked traits are from genes found on the X or Y sex chromosome. Sex influenced traits are from genes found on non-sex chromosomes (autosomal) but their expression can differ in each gender. Sex limited genes are found in both males and females but will only be expressed in one gender. So both sexes will have the same genotype but have different phenotypes.
ID107	Conditional mutants have a wild-type phenotype under normal environmental conditions but have a mutant phenotype when the environment is manipulated. They can be very helpful in research to discover to what extent the environment affects an organism and exactly what factors in the environment can affect the organism.
ID107	Crossing over and recombination is a reassortment of alleles on the chromosomes into different allele combinations that differ from the parents.
ID029	Traits that are carried on the sex chromosomes.
ID029	These mutations could help organisms better adapt to environmental changes.
ID029	This provides a source of genetic variation in genomes.
ID128	Sex linked traits are those located on the X chromosome
ID128	Conditional mutants are ones who only display characteristics under certain conditions
ID128	Crossing over and recombination is the exchange of genes that causes genetic variation
ID093	If an allele does not have a copy on a homologous chromosome then the individual may show hemizygosity. Sex linked traits lead to hemizygotes - all the alleles on the X chromosome are different than that of the Y chromosome.
ID093	Some alleles are only expressed under certain conditions. The value in this would be conserved energy.
ID093	Crossing over, which occurs during the diplonema stage of prophase I during meiosis, increases the genetic variation within a population.
ID114	Sex linked traits are those that are only expressed in one gender Sex influenced traits are traits that have different expressions depending on whether they are present in a male or female. Sex limited traits are present in both sexes, but are only expressed in one gender. In the other gender, the trait is "turned off."
ID114	Conditional mutants are those which have different levels of expression depending on what type of environment they are in. For example, temperature can play a role in how the mutant is expressed.
ID114	Crossing over occurs when two chromosomes exchange a part of their gene. This is essential to genetic variation. Recombination is when chromosomes form new strands of DNA that are not found in either parent.
ID119	These are traits that are expressed in an offspring specifically because it is found in only of one of the genes in one of the parents of that offspring.
ID119	Conditional mutations cause mutant alleles to form which give of mutant phenotypes in specific environmental conditions.
ID119	Crossing over is the exchange of chromosomal parts during mitosis. This exchange between the chromatids causes recombination, which is the pairing of these chromosomes.
ID016	Sex linked refers to phenotypes that are inherited based on the traits of the x chromosome. Sex influences means the expressivity of the phenotype changes based on sex.
ID016	I am not well versed in this topic, so I can not give a good explanation.
ID016	Crossing over occurs between two sister chromatids of homologous chromosomes. Recombination is the process of producing recombined gametes.
ID113	sex linked: traits based from the X or Y chromosome
ID113	it important because of the restrictive and permissive temperatures that produce a nonfunctional and functional protein, respectively
ID113	show the different possible gene combination
ID133	Sex linked traits are those that are passed down to offspring by a gene on the X chromosome. Sex influnced traits are those that are not determined by the genes on the X chromosome and are not limited to one sex. Sex limited traits are not determined by the genes on the X chromosome and is limited to one sex.
ID133	Ones phenotype can change based on the environment that they are in. This change could be due to temperature.
ID133	This occurs when the genes on different chromosomes are in proximity to each other that they are able to combine and form new combinations of alleles. This could create new phenotypes in a population.
ID111	These are all different variations of genes that are linked by the sex chromosomes during reproduction.
ID111	This is when the mutation has a wild type under some environmental conditions and a mutant in other environments.
ID111	The exchange of genetic material between chromosomes during sexual reproduction that results in a recombinant gene.
ID044	certain traits are passed on specifically by the sex chromosomes
ID044	some mutations are only expressed under certain conditions
ID044	crossing over and recombination provide variation in genes and phenotypes
ID080	These are just talking about how ones gender can influence multiple different things.
ID080	I'm not really sure.
ID080	Occurs during anaphase I. Its vital for genetic diversity.
ID102	-sex limited is traits that can be seen in one sex.
ID102	I am thinking that the value is dependent upon the mutation especially in this case because a conditional mutation is based on the environmental conditions
ID102	In crossing over, the homologous chromosomes basically take a piece of DNA from one another.
ID149	Most sex linked genes are X-linked. They show up in males mostly because since they only have one X chromosome, so they cannot be carriers.
ID149	Conditional mutants only express a certain characteristic at a certain temperature or pH or something else.
ID149	Crossing over allows for more combinations of alleles in linked genes.
ID127	These are traits that influenced by the parent
ID127	These are mutants that are important for determining the function or location of a gene.
ID127	Crossing over and recombination happens during sexual reproduction to produce genetically different offspring.
ID021	All are different. Linked are on the XY chromosomes. Limited and influenced on are autosomal.
ID021	Valued in experiments to determine effect of mutations.
ID021	Allow for variation.
ID047	Genes on the X and Y chromosome can determine certain traits that are different from autosomal genes.Certain genes are only present on the X chromosome show specific patterns of inheritance.
ID047	The phenotype of certain genes can be altered by the environment under certain conditions.
ID047	crossing over is when portions of chromatids of homologous chromosomes exchange portions. Recombination is where which chromosome each cell gets is completely random
ID100	Some genes are located on sex chromosomes affecting their inheritance patterns.
ID100	Not to versed in this topic. Need to go back and look at it. Basically though, some genes are only displayed in certain environments.
ID100	Crossing over and recombination occur during meiosis and contribute to genetic variation.
ID073	Traits that are inherited by the sex chromosomes, XX for females, XY for males.
ID073	Conditional mutants only occur in restrictive environments. They are only expressed under certain conditions.
ID073	Crossing over is a step in recombination where homologous chromosomes exchange genetic material with each other.
ID096	A sex linked trait is a trait found on a specific chromosome, x or y. A sex influenced trait is expressed differently depending on which chromosome (x or Y) it is on.
ID096	Conditional mutants are only expressed under certain conditions such as temperature.
ID096	Recombination: the chromosome the cells receive is completely random. Crossing over: Chromosomes exchange or switch parts of their chromosomes. Causes genetic diversity.
ID059	Sex linked genes determines the gender of the offspring and depending on the situation, some genes on the male X chromosome are recessive allowing only the mother to pass down her traits to the offspring.
ID059	Conditional Mutants are expressed and nonfunctional depending upon the environmental conditions.
ID059	Crossing over and recombination are the key to variation in a gene population.
ID132	If the gene is only found on the X and Y chromosomes.
ID132	It's like a wild type phenotype under permissive environmental conditions. Under a mutant phenotype its restrictive conditions.
ID132	Exchange of different segments in their genetic material.
ID084	linked- males more likely (XY) to have it than females (XX)
ID084	mutations seen under certain conditions- i.e. temperature or pH sensitive, complementation mutations
ID084	exchange of genetic material between homologous chromosomes. occurs during prophase I of meiosis. results in new combinations
ID054	Sex linked traits are traits found on sex chromosomes that is typically passed from mother to son.
ID054	Conditional mutants are helpful in the case of species that possess temperature sensitive alleles, which allows them to be able to tolerate a large range on climate.
ID054	Crossing over occurs between two chromatids, exchanging genetic material resulting in variability between offspring.
ID020	These traits are traits that can affect your phenotype and genotype depending on what traits you receive from your mother or father. Sex-linked traits are traits that you receive from your mother or father and is passed down from generation to generation
ID020	conditional mutants are mutants that are expressed in certain temperatures/ (hot or cold)
ID020	Crossing over and recombination is when two chromosomes share segments of their DNA during meiosis.
ID146	These traits are based off the different genes in X and Y chromosomes. Due to their individual natures, certain traits are only associated with one of these chromosomes and therefore only one gender. The Y chromosome contains so many fewer genes that there are many instances in which a male will only have one copy of a gene, and that will be located on its X chromosome.
ID146	Conditional mutants is related to the expression of a gene as a wild-type in "permissive or favorable" conditions in an organism's environment, and the expression of a mutant phenotype under "restrictive or non-favorable" conditions
ID146	Crossing over/recombination is the exchange of genetic material between homologous chromosomes that takes place during gametic production and provides vast variability in a population that allows for evolution
ID027	Haplotype networks can show what kinds of mutations had to occur between the sets of DNA being compared before they would be identical. This can give an idea of how long they have been separated and how closely they are related.
ID027	Microsatellites allow geneticists to pick out an individual's DNA purely based on the tandem repeats seen in the DNA. This sequence and number of repeats is unique to the organism in question.
ID027	Cis or Trans configuration affects which gene is labeled as the "Middle" gene, throwing off all following calculations if it is incorrect.
ID094	Haplotypes are sets of genes that tend to move together and can be used to trace inheritence in a population.
ID094	Microsatellites are mutations that are passed on so they can occur at any time and are easlity followed through populations.
ID094	Knowing this makes it possible to follow the behavior of crossovers in populations.
ID039	A haplotype network reveals the relationship among haplotypes. It can help determined origins.
ID039	Microsatellites are areas of sequences in the dna sequence that repeat. In the population exist many variations of these sequences and repeats
ID039	Cis configuration is when both wild typevalleles are on the same chromosome and both of the mutant alleles are on the other chromosome. With trans configuration one wild and one mutant is on the same chromosome. It is important to know and understand this because this info can help determine the configuration of the offspring.
ID089	Haplotype networks reveal information about populations by showing differences in part of their DNA. It seems to me to be similar to a taxonomic tree demonstrating evolution from species to species
ID089	I only know what I learned in class today, that microsatellites serve as a sort of gene marker..? So it makes sense that the gene markers and therefore DNA sequences would be different among populations
ID089	In order to properly analyze crossing over in linkage, we must know the parental genotype. Knowing that the parental genotype is cis or trans is important so we can tell what allele was double crossed over into the next chromatid
ID116	These are a series of unique sequences in a genome. These end showing different information with a community.
ID116	These microsatellites occur in non-coding DNA. These are repeated sequences.
ID116	Cis configuration is on the same side, while trans configurations are on opposite sides.
ID033	It is a representation of the different dna combinations of each individual in the network
ID033	some individuals have the same repeated difference in their dna sequence.
ID033	a dominant gene may be on one side with another dominant making it cis or on the opposite making it trans.
ID098	A haplotype is a sequence of base pairs that is shared by many different people. That is to say, a large number of individuals in a group will all have the exact same sequence. We can map the relative similarity between different haplotypes in a haplotype network, and these networks can provide quick and concise information about how genetically similar one haplotype group is in comparison to another. This is helpful in determining the phylogenetic relationships between populations and between species.
ID098	Microsatellites are sequences of repeating base pairs in a section of DNA. Some individuals will have just a few repeats, while others could have hundreds or thousands. This variation is useful in determining genetic relationships. For example, paternity tests the varying length of DNA strands between the microsatellites to determine whether a potential father's DNA is similar to the child's in question.
ID098	In order to determine the linkage between genes, we must know whether the parental organisms have cis or trans configuration because that greatly impacts how the gametes will be formed. If, for example, a parent organism was cis (AB & ab), then the most common gametes would be AB and ab. If that same organism were trans (Ab & aB), then the most common gametes would be Ab and aB. Without knowing the original configuration of the genes, it would therefore be impossible to determine whether the genes were linked or not in the first filial generation.
ID056	A haplotype is a polymorphism that are closely located on the chromosome and are normally inherited together.
ID056	Microsatellites are locations with short repeated segments. These are also called SSRs which stands for simple sequence repeats. They are polymorphic and can be used as markers to distinguish variation in populations.
ID056	Order is important because it tells us which gamete crosses over in a double cross (the one in the middle). Knowing cis and trans is important as well because it tells us the location of the wildtype gametes.
ID035	Haplotypes are used to compare individuals between populations in order to infer which populations are most closely related to which others, and thereby also to infer the recent evolutionary history of diverse and dispersed populations.
ID035	Tandem repeat (or VNTR) is a location in a genome where a short nucleotide sequence is organized as a tandem repeat.
ID035	The mutations are said to be cis-dominant because the expression of the genes downstream on the same piece of DNA will be turned off regardless of the genotype present in the trans copy.
ID065	This involves nucleotide sequences and how they have the ability to be mutated, studied, and crossed to further reveal information about the gene.
ID065	This is when nucleotide sequences repeat on a gene.
ID065	This is important in figuring out where genes came from or where they are located on the original parental chromosomes.
ID000	A haplotype is the arrangement of bases or genes on a single chromosome.  Haplotypes vary within and between populations.
ID000	VNTRs are short 2-3bp repeats that are scattered throughout the genome.  Individuals vary with respect to these repeats as do populations.
ID000	In any test cross involving linked genes, the investigator may not know the orientation of alleles on the chromosome but different configurations will produce different phenotypic recombinant classes.
ID090	Haplotypes are unique sequence of bases in the genome. When people share a haplotype they have the exact same sequence for that entire region and can be assumed to be closely related to one another. Haplotypes can be used to compare individuals between populations in order determine which populations are most closely related to which others or when a population evolved.
ID090	Microsatellites are molecular markers used for mapping, linkage analysis and to trace inheritance patterns. They consist of repeated sequences of DNA and are extremely important in forensics as well as in determining parentage.
ID090	Parental and cross over gametes will have different combinations of alleles depending on whether the genes are in cis or trans formation.  It is necessary to know the order and arrangement the genes on the parental chromosomes as we use this information to determine recombination frequencies between genes.
ID105	It shows us the relationship between haplotypes.
ID105	Microsatellites are areas of repeated segments of haplotypes.
ID105	If it is cis both wild type alleles are on one chromosome and both mutant alleles are on the other. Trans means that one wild type and one mutant allele are on each homolog.
ID124	The genes for different populations have a correlation between them  created by separation, interbreeding, and gene expression based on the environment
ID124	The sequence of code that each person has can differentiate where their ancestors are originally from based on common mutations in populations
ID124	i have no idea about this topic
ID081	Halotype networks show the alleles of genes in a population. As a result, this can reveal specific information for a population's variablity or diseases found within it.
ID081	Microsatellites are classes of genetic polymorphism, which is used in mapping genomes. These can be used to link inheritance and diseases.
ID081	Knowing the order of genes on chromosomes help one understand whether single or double crossovers occur. By knowing the order and the counts of offspring outcome, the distance in map units can be found.
ID088	They reveal information.
ID088	There is variation.
ID088	This helps to determine genotype.
ID109	This is because a network of traits that a population has can show what is favorable in that population. It also tells us what genes were in the population before, because genes were passed down.
ID109	This is because different locations of populations can cause certain traits to be favorable, and therefore those traits will be expressed.
ID109	It can determine the phenotype of the offspring. For example, if they are linked, then the phenotypic classes would be equal and trans.
ID104	Mitochondrial haplotypes are used in gene sequencing to show the similarities between different individuals of a species. They show the individual base pairs and compare them in order to figure out the outcomes of each gene.
ID104	microsatellites are used as markers in gene sequencing as well as haplotype networks. The variations observed in populations can be used to determine why certain mutations are more common in one population over another. Variations in the base pairs are also used to determine lineage and how genes have traveled around the world.
ID104	If we can determine whether the alleles are in cis or trans configuration then we can determine through test crossing if they are linked or not. We also need to know the order of the alleles for genetic mapping as well as determining the affects of the crossing over of the alleles.
ID014	Haplotype are typically inherited together so by examining the haplotypes we can learn more about populations. They are also found on a single chromosome which is important considering that you receive a chromosome from each parent.
ID014	Microsatellites are known for being their consistent variability across populations and they also have a high mutation rate. This could be one of the main reasons that there is variation in the number and order of the nucleotides known as micro satellites.
ID014	This is important when trying to determine what the phenotype will resemble after crossing over.
ID092	I understand
ID092	I understand
ID092	I understand
ID144	Halpotype networks give us the populations genotypes. This gives us information about the given population.
ID144	These microsatelites are one way we can get the genetic variability we have in populations.
ID144	Yes, because the cis has both wild type alleles. Trans has one wild type and one mutant.
ID129	A haplotype is a sequence of genes than an offspring inherits from a parent. Haplotype networks reveal information about populations because as sequences of genes are transmitted through offspring, the population exhibits more similar phenotypes.
ID129	Tandem repeats are typically associated with noncoding genes. Microsatellites lead to the gain or loss of an entire repeat unit. These have high mutation rates which lead to variation in populations.
ID129	It is necessary to know whether the alleles of a heterozygous parent are in cis or trans configuration because if cis, the arrangement has both wild-type alleles on one homologous chromosome and both mutant alleles on the other. In a trans configuration, both have on mutant allele and one wild type allele on each homologous chromosome. This is important in gene mapping.
ID079	Haplotype is a group of alleles that are often inherited as a unit, due to a small distance between the alleles, which reduces the probability of recombination.  Haplotypes in a population may shed light on breeding patterns within the population and genetic disorders present in the population.
ID079	Microsatellites are polymorphic (I think), so they differ between individuals and populations.  Microsatellites are unique for individuals; however, micro satellite variability will be greater between populations than within a population.
ID079	Cis and trans configurations reveal which alleles are on which homologous chromosome, so when looking at the gametes, we know if crossing over occurred.  The order of alleles on a chromosome allow us to figure out the relative distance between them.  The gene that changes position is the gene in the middle.
ID085	Haplotype networks show relationship between haplotypes, which allows us to see what chromosomes have come from where by comparison with previous generations.
ID085	Microsatellites can vary in how many times they repeat in a DNA sequence, and how many there are in a DNA sequence.
ID085	The cis and trans configurations vary where the dominant alleles are on a pair of chromosomes. The configuration will determine what the phenotype of the children will be.
ID148	This helps map the genes on the chromosome
ID148	This
ID055	They may be able to help identify a series of genes that are inherited as a unity that cause certain diseases.
ID055	Microsatellites are DNA markers that help with chromosome mapping. They can be identifies by the number of replications or sequence that are found within the genome.
ID055	If the linkage analysis is in cis then both wild types will be on the same chromosome, and if a crossover occurs then you will have one wild type allele and one recessive allele on a chromosome. Trans configuration is the opposite. There is a wild type and a recessive allele on the same chromosome to begin with and if a crossover occurs then you will get both wild types on a chromosome and both recessive alleles on a chromosome. There order is important to because it will determine when and if there was a crossover and whether it was a single or double.
ID134	Haplotype is a collection of alleles from linked loci carried by an organism that are inherited together. Haplotype in populations reveals the different genes, genotypes, and phenotypes found in that population. It also reveals patterns of inheritance, effects of mutation, and types of mutations that affect that population.
ID134	Through microsatellites, which mark sequencing for different genetic research methods.This aid in the scientific observance of patterns of inheritance and differences in inheritance in a population.Also can aid in revealing recessivity or dominance for different genes in that population in order to formulate probabily or map the possiblities.
ID134	Cis and trans mean whether the same genes are on the same allele, or on different alleles. This affects on which alleles they are inherited. Also, their order allows us to see if a gene has changes position. The one that changes positions is the one found in the middle. Both cis and trans configurations and order not only help us to predict how they will be inherited, but also how they behave.
ID095	A given haplotype allows the experimenter to know the genes given in the gametes.  Knowing which genes coming from generation to another allows us to have a better understanding of the population.
ID095	Individuals have unique microsatellites, no matter how interbred a population is.  This information is useful for identifying individuals using only DNA.
ID095	Crossing over can change the cis/trans configuration of the genetic code.  If we know the haplotypes  of an individual, we may be able to determine whether recombination took place, and in what order.  The cis/trans configuration also helps us to map the genetic code on chromosomes using map units.
ID103	Haplotype networks are sequences in the genome that are identical between individuals.  They can be used to determine how closely related individuals within a population are.
ID103	These variable repeats can be used to screen for a trait or to locate a gene in proximity to them.
ID103	Knowing whether alleles are cis or trans can be used to predict which combinations of alleles will occur in the progeny in greater frequency as the parental genotypes will be more well represented.
ID064	An individual's haplotype is a set of genes inherited together, from one parent. It can also be thought of as genes on the same chromosome. Haplotypes are inherited together, so they are broken up if crossing over occurs. Haplotype networks are a visual way of representing differences between individuals in a population by drawing circles for each individual and connecting them with lines, where the lines label what gene is different between the two. These networks show the variation in alleles in a population, which can then demonstrate how a species evolved.
ID064	Microsatellites are sequences of DNA where a couple of bases (most examples I've seen only have two or three) repeat for variable lengths of the sequence. These repeats are genetic markers, and different combinations of repeats are different microsatellites. One individual can have many, but the length can be different for different populations, but analysis with a PCR gel demonstrates that microsatellites can also be different lengths betweeen individuals of the same species.
ID064	For two genes on two homologous chromosomes, there can be two possible configurations: cis, where both genes are wild type on one homolog and mutant on the other, or trans, where one gene is wild type and the other is mutant on one homolog, and vice versa on the other homolog. This order determines which gametes are the parental genotype, which is more prevalent than the recombinant genotype.
ID075	A haplotype is a set of DNA variations, or polymorphisms, that tend to be inherited together. Haplotype can refer to a combination of alleles or to a set of single nucleotide polymorphisms found on the same chromosome. Studying them offer scientists a great deal of information.
ID075	Variation between populations
ID075	A test cross between two unlinked genes gives a 1:1:1:1 expected ratio in the progeny so deviations
ID049	Haplotype networks describe the positions on chromosomes where individuals differ. They can provide information about the where different alleles and mutations occur in a population
ID049	Microsatellites are regions of DNA that repeat base pairs. These areas can be cause higher variations in populations because of high mutation rates that change the protein.
ID049	During linkage it is more likely for an offspring to demonstrate parental linkages rather than recombinant ones.
ID126	Describes where the polymorphism has occurred.
ID126	Microsatellites are repeats and that they happen at differing points within the chromosome so it makes it easy within large populations.
ID126	Same or different homolog tells a relationship between parents
ID069	Individuals sharing haplotypes (unique sequences of nucleotide bases) can be inferred to be closely related. This information can be used to speculate on the total relations of populations as a whole.
ID069	Microsatellites consist of nucleotide sequences that are repeated another of times. Because these usually exist at non-coding regions, mutations can accrue at these sites over generations, creating variation within and between populations.
ID069	The alleles of a heterozygous parent can reside on either sister chromatid, and they can be in any order. In order to predict and understand the effects of crossing over and independent assortment, one must know the location and arrangement of the alleles on a heterozygous parent.
ID077	Haplotype networks can show whether individuals are closely related to each other using unique sequences of bases over the genome. Basically if an individual shares the same haplotype as another they are closely related.
ID077	Microsatellites are an array of short repeats. One can use microsatellite diagrams to see whether people are closely related or not. They can be used to locate mutations and genes.
ID077	It is necessary to know whether the alleles of the heterozygous parents in a linkage analysis are cis or trans because it can lead to complementation (restoring wild type in progeny) or no complementation. Cis configuration both wild type alleles on one homologous chromosome and both mutants on another. Trans configuration one wild type and one mutant on each homolog.
ID066	A haplotype is a set of DNA variations, or polymorphisms, that tend to be inherited together. A haplotype can refer to a combination of alleles or to a set of single nucleotide polymorphisms (SNPs) found on the same chromosome
ID066	Microsatellites are segments of repeating nucleotide bases that occur in non-coding DNA sequences, these segments may vary depending between populations.
ID066	In order to tell if genes of an offspring were linked in the parents, the orientation of those genes in the parent chromosome must be known.
ID063	Individuals that share a haplotype have the exact same sequence for this entire region, and are therefore inferred to be closely related to one another. Usually a haplotype sequence is relatively short.
ID063	These are areas of short repeated segments. We are able to run DNA sequences after PCR amplification on a gel and can detect the parentage to a specific progeny.
ID063	This just means that if 2 mutations are on the same allele then it is said to be in cis configuration. If they are on different alleles then they are considered to be in trans configuration.
ID061	Haplotype networks consist of only one set of chromosomes. Thus they can show if genes are linked because if the gametes of a parent always show the same genes from generation to generation then it can be assumed that they are linked. If there is variation in the haplotype networks then there would be reason to believe crossing over took place.
ID061	Microsatellities vary within a population, as they vary in the number of repeats at any given site. These microsatellites may be identified not only by the number of repeats but by the DNA sequences that flank them. Variations may be used to screen for an association with specific diseases or traits. These variations thus allow an opportunity to identify and locate related genes.
ID061	It is necessary to know whether alleles of a heterozygous parent are in cis or trans configuration because this determines what the outcome of noncrossover gametes for the parent would be. It also will determine whether the gametes produced were formed through cross over or are exhibiting linkage. It is important to know the order of the alleles so that the likely genotype of a double crossover between two alleles may be discovered. This also helps to determine the distance between the alleles and the likelihood that they are linked.
ID015	It is thought that identifying these statistical associations and few alleles of a specific haplotype sequence can facilitate identifying all other such polymorphic sites that are nearby on the chromosome. Such information is critical for investigating the genetics of common diseases.
ID015	Micro satellites are a tract of repetitive DNA where base pairs are repeated sometimes from 2-50 times. They occur thousands of times in the human genome and are a sight of high mutation rate and high variability.
ID015	Cis is on the same parental homolog and trans is on different ones.
ID097	Depending on the genetic makeup of a population can help identify variation in alleles.
ID097	Yes, variation can help define the variance with in a population on the mirco level.
ID097	Yes
ID013	Haplotype networks show the relationships between holotypes, which helps to identify the sequence on one chromosome. It shows the SNPs or single nucleotide polymorphisms that are between certain genes. An example of a haplotype is tracing the mitochondria, which are inherited from the mother. This haplotype has been used to trace the origin of people from Africa to America.
ID013	Microsatellites are short segments of repeats in the DNA strand. Because these micro satellites vary in individuals, they can be used in forensics to identify a child and its parent because the child must get one of the parents' micro satellites.
ID013	Cis and trans configurations can help determine the order of genes on a chromosome. Also, cis means the wild-type alleles are all on the same chromosome while trans means there is a mixture of wild-type and mutant alleles on one chromosome. These configurations affect the crossover and parental genotype possibilities which in turn affects the order genes on this chromosome. This will also result in a different in map distance between these genes.
ID067	Beacuse they can be linked back to other genes.
ID067	These clusters have served as useful molecular markers for genome analysis
ID067	cis-acting sequence is one that is located on the same chromosome as the gene that it regulates. This is in contrast to trans. They are required for the accurate and regulated transcription of genes transcribed by RNA polymerase II
ID007	Haplotype networks are used to study the phylogeographic history of a population.  The same sequence from a variety of different individuals is studied, and from which mutations are observed and how many, it is possible to infer which individuals or subpopulations diverged and at approximately what time in their history.
ID007	Vntrs are inherited and vary among individuals.  The amount of repeated segments can be determined and compared to determine whether two individuals are likely related or not.
ID007	Without knowing the configuration of the parent, it is impossible to determine relative distances in a linkage analysis because the basis for these analyses is recombination.  If one does not determine which offspring are recombinants and which are not, the test will show nothing.
ID145	By mapping haplotypes, you can determine the relationship between haplotypes and trace back to common ancestors and use that to map evolution and population movements.
ID145	The repetition of nucleotide sequences can be found and used to show variation between people.  The VNTR must be the same for one person, or will have one from each parent.
ID145	The order determines the frequency of crossing over, and the configuration can affect recombination or what alleles are paired together.
ID091	Genetics is useful for the study of populations as a whole. Haplotype networks aid in the analysis of a gene pool.  Haplotype networks consists of genes and sometimes restriction enzyme sites are close enough together that they are inherited together frequently.
ID091	Micro satellites are generally very susceptible to mutations.  One mutation could be passed down in one population while another is spread in a separate population. Genes can be linked with these micro satellites.
ID091	Being Cis or Trans greatly affects the results of genetic linkage.  If both dominant alleles are on the same chromosome (cis) than the recombination with produce a heterozygote.  While recombination of a trans set up will produce homozygous generations.
ID022	Haplotype networks are a kind of marker that mark or show genes on the chromosomes and are helped in gene mapping.
ID022	Micro-satellites is a repeated sequence of the same genes over and over again.
ID022	The presence of the genes according to cis or trans matters because it changes the way how phenotypes are express.
ID045	These networks reveal information about populations because they are a set of DNA variations typically inherited together.
ID045	Microsatellites are short DNA segments that have repeating units.
ID045	Alleles in a linkage analysis are important to know the order of the genes on the chromosomes.
ID068	Haplotype networks can show the similarities in DNA between individuals and what SNP's cause individuals to vary from each other. This can be used to trace populations in areas.
ID068	Microsatellites are repeats in the DNA gene sequence. The number and length of microsatellites in individuals varies and can passed down to offspring. These repeats can be used to identify an individual.
ID068	If a heterozygote is in cis configuration, one chromosome will have both dominant alleles on one chromosome for 2 genes and the other chromosome will have both recessive alleles. If they are in trans configuration, both chromosomes will have the dominant and recessive allele for both genes on each chromosome.
ID031	Since haplotypes are a region of bases that code for a specific region they definitely help identify certain things. They do this because whatever population is in the region will have the same haplotype network.
ID031	This concept helps provide genetic variation between many different populations and can help distinguish many different populations from each other.
ID031	This helps you determine whether or not the allele is on the front or back chromosome. This will quickly narrow down whether something is linked or not.
ID040	Haplotypes are sets of alleles from closely linked loci carried by an individual that are inherited as a unit. Haplotypes can be used to look at populations to see which are more closely related than others and see where dispersed populations may have diverged.
ID040	Microsatellites are very short DNA sequences found widely throughout the genome that are used as molecular markers. They have high mutation rates and are found to be very diverse within populations.
ID040	Knowing whether the alleles are in cis or trans configuration is important because the configuration affects the resultant progeny. Cis configuration has both wild type alleles on one chromosome and both mutant alleles on the other chromosome while trans configuration has one wild type allele and one mutant allele on each homolog. Knowing this helps determine what phenotypes progeny will have.
ID008	The individual circles indicate a specific haplotype and they are connected based on similarities to one another. The diameter of the circle is equal to the individuals of that haplotype sampled.
ID008	Microsatellites are short repetitive sequences that are found throughout the genome and can be identified by DNA sequences that flank them.
ID008	In order to understand crossing over and producing new recombinants we have to understand where the alleles will be located after this process to determine if specific genes will show up in the phenotype.
ID141	yes, since haplotypes are regions of the genome in which certain individuals of the population share the exact same segment of bases along part of the genome, we can infer how closely related populations are.
ID141	This variation helps geneticist locate specific genes and mutations responsible for certain traits or diseases.
ID141	Knowing whether the alleles are in cis or trans can let the geneticist know before hand what the gametes will be once meiosis has occurred.  In cis the gametes will be 0.5 homozygous dominant, and 0.5 homozygous recessive.  In trans, the gametes will be 0.5 heterozygous for one trait and 0.5 heterozygous dominant for the other trait.
ID111	A haplotype is a set of alleles closely linked carried by a person and they are inherited as a group. This makes them easily detectable among groups of individuals since they are only inherited as the group.
ID111	There is always going to be variation because no two individuals are going to be exactly the same. Microsatellites are are short repetitive sequences that are found throughout a genome and vary in the number of repeats.
ID111	It is necessary to know due to the process of meiosis and the passing of genes from parent to offspring. Cis and trans have different outcomes.
ID136	With haplotype networks we can trace back to generations from a certain population and learn more about that populations growth and adaptation.
ID136	With microsatellites and their variable number tandem repeats it gives variation in a population causing a larger diversity between species.
ID136	It is necessary to know whether alleles of the heterozygous parent in a linkage analysis are in cis or trans configuration and their order because with that information we can know how the crossing over occurred.
ID118	Haplotype is a specific genes that are inherited to offspring from the parent. It can carry SNP: single nucleotide polymorphism also called point mutaiton, VNTRS: variable number tandem repeats, RFLPS: restriction fragment length polymorphism.They are DNA molecular markers allow the chromosome mapping in organism where hundreds of offspring is not viable or ethical like human as location and sequence are known, making them useful landmarks for mapping purposes..
ID118	Microsatellites (variable number tandem repeats) are short repeats on DNA segment and tend to occur in non-coding DNA. Its variation within and between the population allow us to determine population demographic history and ancestral origin.
ID118	It is necessary to know whether alleles of the heterozygous parent in a linkage analysis are in cis or trans configuration and their order as it allow us to predict the outcome of the next generation and to see if crossing over has occurred or not. It allow us to compare parental and crossing over by predicting outcome of the next generation. It shows that parental are more frequent and more common than crossing over.
ID115	Haplotypes are unique sequences of bases over a specific region of the genome, that can reveal information about populations by identifying individuals that have the same sequences and are thus related, or even infer the evolutionary history of a specific population and the genes it holds.
ID115	Because microsatellites are a wide range of repeated sequences along the genome and have a high mutation rate in DNA, these can help forensic analysts decipher the kinship that an individual may have had with another, due to the fact that mutations are inherited during meiosis and the fusion of gametes. However, because mutation rates in the noncoding sequences of DNA in each individual is high, this may lead to variation between individuals and can be less accurate in testing kinship.
ID115	In a linkage analysis, the sequence of genes and their position along a chromosome is determined to know whether or not those genes in question are so close together that they must be inherited in meiosis together or if they are far enough apart that they might be exchanged with other genes from another parent. It is important to determine whether the alleles of a heterozygous parent are in cis or trans configuration to establish whether wild type alleles will be inherited together in offspring or if the offspring will inherit heterozygous traits again.
ID083	haplotypes allow us to know what genes are being passed from generation to genration
ID083	Variation is due to recombination between non sister chromatids
ID083	Cis or trans configurations are important because the gametes that results from them different.
ID070	You can use haplotypes of a sensible size to compare individuals between populations in order to infer which populations are most closely related to which others, and thereby also to infer the recent evolutionary history of diverse and dispersed populations
ID070	A variable number tandem repeat is a location in a genome where a short nucleotide sequence is organized as a tandem repeat. These can be found on many chromosomes, and often show variations in length between individuals.
ID070	Linkage between loci is indicated when th
ID023	Alleles on the same chromosome that are inherited together can cause variability and variety to the population. Knowing that a chromosome contains certain haplotypes, it is possible to predict genetic frequencies.
ID023	Microsatellites are pieces of chromosomes that contain a repeating gene sequence. That repeating sequence is identified and causes genetic variability.
ID023	Cis vs Trans orientation is important especially when it comes to determining gene sequencing. Trans tend to be more difficult than cis orientation because trans orientation has both mutant and wild type alleles on the chromosome.
ID082	By seeing what groups of alleles are inherited together as a unit, geneticists can make predictions about the sequence of genes on the chromosome. Haplotypes are contiguous, and do not undergo recombination during meiosis.
ID082	Because of crossing over and variable gene inheritance by offspring, differences in the DNA of all individuals allows DNA markers, such as microsatellites, to aid geneticist in sequencing DNA. These short, repetitive sequences vary in the number of repeats at any given sites, and are identified by these repeats as well as the DNA on either side of them.
ID082	In linkage analysis, the genotypes of offspring between heterozygous parents are commonly evaluated to see whether or not crossing over occurred during meiosis. Cis configuration and trans configuration refer to the configuration of the linked genes on the chromosomes. If both dominant alleles are on the same chromosome, meaning that both recessive alleles are on the other chromosome, then the genes are in a cis configuration. If each of the homologs have the dominant allele for one trait and the recessive trait for the other, then the genes are in a trans configuration. This is important to know because without this information it is impossible to determine if recombination occurred.
ID107	Haplotypes are a set of polymorphisms that tend to be inherited together. You can compare haplotypes of individuals from different populations and infer which populations are most closely related to each other by how similar their haplotype sequences are. You can create a haplotype network that shows the relation between populations and therefore gain insight on the evolution of diverse populations and make connections to populations that might have migrated from the original population.
ID107	The variation in tandem repeats (meaning the repeats are of varying lengths) can cause different gene expression and lead to differences in phenotypes. Also the variation in microsatellites can be useful genetic markers for sequencing DNA.
ID107	Cis configuration means that both the dominant alleles are paired on one chromosome and the recessive alleles are paired on another (eg. AB on one and ab on the other). Trans means that one dominant is paired  with one recessive on each chromosome (A and b and a and B). If the genes are linked, that means that they do not behave independently from eachother. In a linkage analysis, the genotypes of offspring from a heterozygous parent are evaluated to determine whether or not crossing over occurred during meiosis. It is important to know if the alleles of the heterozygous parents were in the cis or trans configuration to accurately determine if recombination happened.
ID132	Haplotypes are DNA variations that are inherited together. They can be a combination of alleles found on the same chromosome.
ID132	If we didn't have variation within populations then everyone's genes would be similar. Microsatellites are repeated short segments of DNA. It gives our population high diversity.
ID132	Yes it is necessary to know whether the alleles of the heterozygous parent is in cis or trans because these linkages determine the proportion of recombinant and parental types and if a crossover occurs between certain genes.
ID030	A haplotype is a group of genes which is inherited from a single parent. Using Hardy-Weinburg equilibrium, the linkage of genes can be determined.
ID030	The repetition of microsatellites occurs  to different degrees between individuals and populations. By analyzing the similarities and differences in these repeating DNA sequences, geneticists can determine how closely related to individuals are.
ID030	This is an important concept to understand when determining how two genes will be inherited. If two alleles are cis, they are on the same chromosome and are more likely to be inherited together. Trans alleles each reside on a chromosome in a homologous pair and are therefore inherited separately unless crossing over takes place. This affects the probability of an allele being inherited and expressed.
ID093	If individuals are said to share a haplotype, they have the exact same sequence for a region of a given gene. Therefore, they are closely related. Haplotypes can be used to to compare individuals of a population in order to infer which populations are most closely related.
ID093	Short nucleotide sequences can be used to identify persons or parents due to the variations in length between each individual.
ID093	Whether specific alleles are in the cis/trans configuration on homologous chromosomes determines the frequency of recombination. This frequency can be used to determine the location of alleles on a chromosome. If the parent has a trans configuration between two alleles on sister chromatids and the child receives a cis configuration (alleles on same chromosome) then a crossover even has certainly occurred.
ID026	Haplotype networks can show us which genes are located on the same chromosome as well as how statistically associated the two genes are with each other.
ID026	Although organisms within a population can have many similarities in their genetic code, certain parts can still be different. Microsatellites can have variable numbers of tandem repeats which alters the genetic material from organism to organism.
ID026	This allows us to find out which genes are taken from each chromosome during crossing over, which determines the offspring's genotype.
ID028	Since haplotypes are passed down from generation to generation, by observing these variations one can find out a lot about a population such as where it originated.
ID028	Microsatellites are short repeats that can be seen in gel electrophoresis that can be used as genetic markers.
ID028	By determining the configuration of alleles in a heterozygous parent, one can then place the alleles correctly on a chromosome sketch.
ID060	Haplotypes can be used to determine whether two individuals are closely related to one another or which populations are more closely related to one another.
ID060	Microsatellites are variable tandem repeats in a population, but would be another sort of tandem repeat in another population, which then would distinct which population is which.
ID060	Yes because by knowing in which configuration they are at we can determine how genes will be passed on during meiosis.
ID019	Haplotype networks can reveal information about populations through letting us know about the number of alleles, localilities, and number of individual with that haplotype. Clades of DNA can also be given.
ID019	Microsatelites are short tandem repitions of DNA that occurs at thousands of places in the human genome. There can be variation in these repeats in the same population because of mutations and environmental factors.
ID019	Knowing if the alleles are cis or trans can help determine whether there is coupling or repulsion in the genes.
ID147	Haplotype networks depend on similar sequencing, so if two populations contain an identical sequencing it can be said that they are related.
ID147	Similar populations can be detected by similar microsatellites, while populations who do not share similar microsatellites will be easily distinguished and shown to be distinct between the different populations.
ID147	Cis configuration is when both wild types are on the same homologous chromosome and the mutants are on the other. Trans is when there is one wild type and one mutant on each homolgous chromosome.
ID150	By understanding what genes are inherited together, scientists can pinpoint what genes lead to certain traits found in a population.
ID150	The common repeated sections of DNA undergo more mutations than normal DNA, leading to the accumulation of mutations over time. These mutations provide the basis of variation between individuals.
ID150	Knowing the location of alleles with respect to each other can tell whether a recessive trait will be inherited or not due to multiple mutations.
ID002	This allows geneticists to locate genes that associate with genetic disease such as cystic fibrosis. This allows them to identify specific mutations in the genes that cause the disease being examined.
ID002	This is due to mutations throughout the population. Not everyone will have the same number of repeats. Microsatellites can vary at different sites along the genome.
ID002	It is necessary to know this concept in that depending on if the genes are in cis or trans configuration, it can result in complementation, thus providing the wild-type phenotype. Helps to determine double crossovers.
ID131	Haplotype networks show the codons on a gene, and the differences in these sequences or networks show the variations within the populations.
ID131	Microsatellites are regions on a chromosome of tandem repeats that are generally unique to an individual. these microsatellites can be used to identify people in crime-linked or pregnancy situations.
ID131	cis and trans has an effect on whether a gene is expressed. The configurations plays a large role in the phenotype of the progeny,
ID123	A haplotype is a set of alleles from closely linked loci carried by an individual that are inherited as a unit.  So, a halpotype network can reveal information about populations by looking at the set of alleles in an individual that is a part of the population.
ID123	Microsatellites are short repetitive sequences that are found throughout the genome.  Variation can be caused in populations depending upon the location and number of microsatellites.
ID123	The location of the alleles is necessary to know because the pattern of inheritance could be skewed based on where the alleles are located.  The cis and trans configurations of alleles havr different patterns of crossing over, so it would be important to know how they would be inherited onto another gene.
ID078	knowing which genes are present and in which quantities they appear, allows for them to be traced and predictions to occur of their future frequency
ID078	Knowing where microsatellites are allow you to use them as DNA markers.
ID078	Knowing whether a parent was cis or trans allows you to determine whether crossing over occurred
ID117	Haplotypes are a set of DNA variations (also called polymorphisms) in chromosomal regions. They will be passed on in tact unless crossing over occurs. I understand that because they will usually be passed on in tact, they can be studied to observe similarities in populations and origins.
ID117	Microsatellites are repetitive sequences that are found throughout the genome. The amount of repeating sequences can vary dramatically, which can influence phenotypes and the expression and/or severity of a given disorder.
ID117	If alleles are in the cis configuration, they are located on the same gene. Trans configuration is when the alleles are located on different genes on the choromosome. Complementation only occurs in the trans configuration
ID149	Haplotype networks show the genetic diversity of a population. They are really cool to look at and interpret.
ID149	VNTRs are random nucleotide sequences that separate genes. They show variation because they change from individual to individual.
ID149	Honestly, I missed this day in class because I had an Organic Chem exam to study for. I haven't read the material yet, so I don't understand why this is important.
ID120	The haplotypes give specific location of different genes that are inherited together.
ID120	Between each generation, chromosomes crossover which contributes to different variations in a population, so the DNA micro satellites can change depending on the alleles of the parents and if crossing over occurs
ID120	It is important because if they are cis then they will configure together while the trans are on separate chromosomes
ID058	A haplotype is a group of alleles or SNP's that an individual inherits together from a parent. These mutations can reveal information about a population's environment as well as mating habits.
ID058	Microsatellites are short, repetitive segments found throughout the genome that vary in number from location to location. Variation can arise with the number of microsatellites, how many times each sequence repeats, as well as the location of the microsatellites. This allows for great variation.
ID058	One must know if they are in a cis or trans configuration and their order in order to be able to determine the distances between genes properly.
ID051	haplotype networks can indicate varying patterns of genetic structure among ecologically similar sympatric species
ID051	Most of the microsatellites are noncoding, and therefore variations are independent of natural selection, which makes microsatellites ideal genetic markers for conservation genetics
ID051	Cis and trans allows you to see the arrangement of alleles. there are two arrangements in regards to heterozygous parents.
ID142	Haplotypes are unique sequences in DNA. The more similarities in haplotypes the closer two people or populations are related. The less similarities, then the less they are related. Thus, by studying haplotypes we can infer if populations are related or even different species.
ID142	Microsatellites are DNA specific for an organism. Similarly to haplotypes the variation in microsatellites allows us to determine how related species or populations are.
ID142	It is very important to know the order of alleles during linkage analyses because it will better help predict outcomes, especially because of recombination, environmental factors, and influence of other near by genes.
ID076	Tells us much about genes and ratios in the population
ID076	*Need more work on this topic
ID076	location can be determined if enough is given
ID020	Haplotype networks show the genetic diversity within a single group. When looking at a haplotype network we would be able to determine the genetic make-up throughout that group.
ID020	Microsatellites are repetitive sequences that are found throughout the genome. Depending on the site the sequence can vary. Because of the variation and repeats we are able to identify the microsatellites.
ID020	Cis and trans configuration is the way alleles of a gene are organized on chromosomes and it is necessary to know whether the alleles of the heterozygous parent is cis or trans because that can help determine the crossover of the mother/father.
ID034	a haplotype is a set of DNA variations that tend to be inherited together. based on the information of the haplotype network, geneticists understand what kinds of DNA exist in population and can determine population
ID034	A variable number tandem repeat (or VNTR) is a location in a genome where a short nucleotide sequence is organized as a tandem repeat. These can be found on many chromosomes, and often show variations in length between individuals. Each variant acts as an inherited allele, allowing them to be used for personal or parental identification. Their analysis is useful in genetics and biology research, forensics, and DNA fingerprinting.
ID034	to understand genes mapping, cis and trans play an important role to determine the genotype of parents and offspring
ID074	haplotype networks are a type of a tree based representation of the relationship between haplotypes within a population and the single nucleotide polymorphisms and microsatellites. They can reveal the sequence of mutations that were the cause of variation in the population
ID074	microsatellites are differences in the length of repeated sections of bases with in a gene. The differences in length causes variation. This variation can differ between individuals in a population and between to different populations
ID074	it is necessary to know whether alleles of heterozygous parents are cis or trans and the order of the alleles so that you can predict the possible alleles for the next generation and to account for double crossing over. Also it is important for mapping of the genes and to know the distance between.
ID114	A halpotype is a select sequence of bases in section of the genome. Individuals in a population can have identical halpotypes, which means that they have the same halpotype for an entire region. The length of the halpotype depends on how closely related the individuals are. A longer halpotype indicates that they are more closely related.
ID114	Microsatellites are a portion of the DNA in which there can be repetition of bases, such as GAGAGAGA. Usually this occurs in non-coding DNA. The number of repeating bases can vary per organism. In diploid organisms, there will be two copies of any diploid segment.
ID114	It is important to know the conformation of alleles in a heterozygous parent because it tells us a lot about what genes they will have. For example, a trans conformation could mean that there are two dominant alleles each on a chromosome, while cis will reveal that one chromosome has a recessive gene.
ID146	You can use haplotypes of a sensible size to compare individuals between populations in order to infer which populations are most closely related to which others, and thereby also to infer the recent evolutionary history of diverse and dispersed populations.
ID146	Microsatellites occur at thousands of locations in the human genome and they are notable for their high mutation rate and high diversity in the population.
ID146	Knowing the arrangement and order of the parental alleles allows you to more accurately predict tendencies regarding the ratios of crossover phenotypes in progenies as well as identify multiple crossover occurrences.
ID053	Haplotype networks give information about the nucleotide sequences that are the same in various populations, and they oftentimes give a geographical representation of how populations with similar sequences are dispersed.  This is very important when studying the evolution within these populations.
ID053	The variation within microsatellites (VNTRs) can help to show us how closely populations are related because the variation of the VNTRs is characteristic for different populations as a result of interbreeding.
ID053	This allows us to determine which gametes it is possible for the offspring to receive from the heterozygous parent.  If two alleles are in trans configuration, crossing over may be necessary for the offspring to receive both dominant alleles from the heterozygous parent.
ID016	A haplotype is a closely linked set of single nucleotide polymorphisms that serve as markers on chromosomes. It is a site on a chromatid pair that is associated with its homolog.
ID016	VNTR acts as an inherited allele. It is unique and can be useful in DNA fingerprinting.
ID016	It is important to understand the order of genes on a chromosome so that we can make gene maps. In order to make a gene map, one must compare recombinant frequencies between non crossovers and double crossovers.
ID062	haplotypes are unique sequences of bases over the span of the genome. people who share haplotypes also share the same base sequence. geneticists can follow these haplotypes through varying populations.
ID133	A haplotype network shows the relationship among haplotypes, or alleles that appear as a group.  Once reviewing a haplotype network, one can see where there are SNPs, microsatellites, or indels. This can be useful information to distinguish populations from each other with these unique distinctions.
ID133	Microsatellites can be combined and placed in gel electrophoresis to distinguish any similarities between a sample already taken or another person.  Because DNA is polarized, it is pulled through the wells of the gel by the charge it holds. Although this test is useful, it can be inconclusive at times.
ID133	By knowing if the heterzygous parent is cis or trans, it lets you know if all the dominant alleles are one side of the chromosome or if the dominant and recessive alleles are shared between the two chromosomes.  The order of the alleles is determined by the map distance between each other and the switched allele of the three-point linkage cross is always placed in the middle.
ID100	Basically a haplotype is basically a unique sequence of bases over a region of the genome. They can show whether individuals are closely related. If individuals have the same haplotypes they are related.
ID100	Microsatellites are short repetivive sequences of DNA. They are often used to determine a gene or mutation location within the genome. Microsatellites can be used to carry out genetic fingerprinting, which can be used to identify individuals in forensic analysis.
ID100	Knowing whether alleles of a heterozygous parent in a linkage analysis are in the cis or trans configuration is important because if they are trans complementation can occur thereby bringing back the wild-type to the progeny. In the cis configuration, one homologous chromosome has both wild type alleles while the other homologous chromosome has both mutant alleles. In the trans configuration, both homologous chromosomes have one wild-type and one mutant allele.
ID127	This helps geneticists study the haploid genotypes within a certain species. This is mainly to determine if this particular species is related to another species based on their haploid genes(haplotype)/
ID127	There is variation within and between populations because of mutations and different species. Some may repeat two sequences 5 times or some may repeat the same sequences 15 times.
ID127	This is important because it effects the outcome of the cross over. If they are cis, both wild types are on one chromosome and both mutants are on another chromosome. So if a crossover occurs, recombinant gametes will get a mutant and a wild type. If it was trans, than the recombinant gamete will have 2 mutants or 2 wild types after cross over.
ID151	Haplotype networks depict relations among haplotypes which are inherited tandem repeats found in the offspring.These networks portray a list of tandem repeats and can compare these repeats to see the differences between haplotype 1 and haplotype 3 within two different populations which could be SNP 3 ( a base change). A chart was given in lecture on the slides to enable us to see the differences better. Furthermore, since the mutations inherited in these repeats are caused by the mother only, one can trace the "original" mother from numerous generations ago.
ID151	Single Nucleotide Polymorphism or SNPs are a main cause for variation within microsatellites since it is the result of a change in one nucleaotide base. Since these microsatellites or tandem repeats are passed on from one generation to the next, any mutation in the length of the repeats or nucleaotide base change such as a point mutation or a deletion would result in the classificstion of a different tandem repeat. Furthermore, these microsatellites are used for gel electropheresis when examining multiple organisms' DNA. This method is commonly used for paternal testing and crime investigations and is only enabled by the variance in microsatellites within a population.
ID151	The arrangement of the linked alleles is very crucial in determining teh possible genotypes of the offspring. For instance, a parent who is heterozygous for two traits in a cis configuration and is crossed with a homozygous recessive parent for those two traits, then the offspring would have to be either the wild type for those traits or homozygous recessive for the traits if no crossing over occurs. In the course of crossing over occurring, it is easy to detect where the change in alleles occurred if we know the configuration of the parents and the children's genotype. The crossing over in a trans configuration would result in the same as the noncrossover heterozygous parent. In addition, in order to detremine the gene that was swapped, the order of the genes must be correct. In the case of a double crossover the swapped allele must be in the middle because the two crossovers result in keeping the outside genes in their initial locus. For two genes, however, it does not matter.
ID004	they give us information about species interface or below the species level.
ID004	microsatellites are a common area for mutation in populations.
ID004	cis means that one of the homologues would have both recessive alleles and the other would have both dominant alleles. trans would mean that each chromosome would have one recessive and one dominant allele.
ID011	Haplotype networks can reveal similarities and differences between the genomes of two organisms. This can allow insight into how similar the two organisms are and what type of traits they share between the two of them.
ID011	These variations in microsatellites between populations are signs of mutation between the populations.  These microsatellites assist in locating genes on a chromosome that pertaing to a particular trait or allele.
ID011	If the alleles are in cis, the two wild type alleles share the same chromosome and are linked. If they are in trans, they are on oppposite chromosomes. This orientation is important because it can tell us what the genotype of the parent was and what the genotype of a gamete could be.
ID113	haplotype can reveal information about populations because it is a specific group of genes that a progeny inherits from one parent,
ID113	a class of genetic polymorphism commonly used for mapping, linkage analysis
ID113	the configuration is important because it can affect the map distance
ID021	Haplotype are traits or alleles that appear in a group. They are used to describe groups and species. Haplotypes will be passed on to generations unless crossing over occurs. Haplotypes are relatively uniform among members of a population and reveal similarities within the population and differences from other populations.
ID021	Microsatellites are areas in the genome where there are short repeated segments. They help describe a population and show the genetic basis for similarities of a species.
ID021	In a gene with alleles at two locations, it is necessary to know the orientation of the configuration to determine if the progeny will have the wild type alleles or a mutant allele. The cis arrangement, with crossing over, produces a heterozygous wild type. But the trans arrangement, with crossing over, produces one wild type gamete and one mutant gamete.
ID096	A haploytpe is a group of genes that is inherited from one parent. It is a specific DNA sequence that contains information about the individual.
ID096	This is a specific set of repeating DNA. This occurs at multiple locations on the human genome. It causes mutations and variability within a population.
ID096	Cis configuration is when the alleles of two more more pairs of genes on one chromosome are present. Trans configuration contains one pair of genes.
ID044	A haplotype is a set of DNA variations that tend to be inherited together. A haplotype can refer to a combination of alleles or to a set of single nucleotide polymorphisms found on the same chromosome.  These can show relationships.  It shows the sequence on one chromosome.
ID044	Microsatellites are areas where there are short repeated segments of DNA.   These variable microsatellites allow us to differentiate between people and are used in forensics because a person will not have the same microsatellies as another person.  The child gets 1 of the 2 tandem repeats from it's parents.
ID044	Linkage is important in knowing where genes are located on chromosomes.  Crossing over is responsible for genetic variation.  Cis configuration is when the wild type allele is on the same chromosome.  Trans configuration is when the wild type is on opposite chromosomes.  Cis or trans can help determine if the alleles are linked or not.
ID080	A haplotype network are used to compare individuals in a population to see how closely related they are. They use a haplotype which is a unique base sequence found in the genome.
ID080	Microsatelites are used for mapping. The variation between and within population help confirm diversity among population members.
ID080	CIs configuration the alleles are in the same order but in trans they are in reverse. You must know this or your mapping results will be skewed.
ID036	A haplotype network shows the relationship among haplotypes
ID036	Yes, there is variation within and between population because a Microsatellites are areas where
ID036	Yes it is important because cis and trans configuration will tell were genes are located in a chromosome. Cis is where the wild type are both located on the same homologous chromosome, and trans is when there are on different homologous chromosome.
ID102	The use of a haplotype network helps us understand the relationship between haplotypes, and haplotypes are used to trace the relationships between populations of a different environment.
ID102	I read it and still am not too sure about it. However, I think it helps understand what individual has a possibility of belonging to a parent.
ID102	Not clear on the reasonings behind this concept.
ID073	Haplotype networks represent the relationships among the different haploid genotypes observed in the population
ID073	Individual repeats can be removed from or added to  the the number of tandem repeats by recombination or replication errors, leading to alleles with different numbers of repeats hence the variation in and between populations.
ID073	Crossover depends on whether the mother is cis or trans. In linkage studies, you are looking at meiosis on the side of the mother.
ID050	A haplotype is a set of SNPs on the same chromosome that are associated statistically. Maps some how much correlation individuals have on their chromosomes.
ID050	Micro satellites are repeats of noncoding DNA. There is a high mutation rate between different species and provide high diversity in the population.
ID050	You need to know this because it will affect how the cross is performed and thereby how the genotypes will come out.
ID128	The purpose of a haplotype network is to display relationships that occur between haplotypes. The study of the networks not only reveals information about a population but it can also display relation between two populations and help geneticists decide where to draw the line between different species seeing as that can be difficult because the true definition of a species is not always clear.
ID128	Microsatellites are short sequences that are repeated throughout an organism's genome; however, they give variation to a population because different organisms within a species can vary by the number of repeats. Also, these variations can be used to screen for certain diseases and help locate genes for further study.
ID128	Cis and trans configuration is important to be able to distinguish when studying a gene because without using sequencing, it is necessary to know the order of two alleles to be able to determine their location on a chromosome. Also, in order to predict whether or not mutant alleles will be passed on to offspring, the orientation of alleles is important because it indicates whether or not crossing over occurred.
ID059	Haplotype can be used to show what specific genes an offspring inherits from its parents. It can span from generation to generation and basically show the history of genes overtime.Therefore, it reveals information about different populations and their genetic make up.
ID059	Microsatelittes are the main cause in variation of a population. Due to their repetitive patterns of change in DNA sequence, microsatelites are the known to be one of the main causes of high mutation rates in populatins.
ID059	Cis and trans configuration is important because if a parent that has two wild type alleles on the same chromosome and the recessive type alleles of the some gene on the homologous chromosome, then one, mutations can occur which leads to specific diseases,and two, it will determine if the offspring  is dyhybrid for linked or unlinked genes.
ID047	Haplotypes are segments of DNA that are next to each other that do not undergo recombination. They are inherited as units and can be useful as markers.
ID047	Microsatellites serve as good genetic markers because they are very polymorphic, mendelian inherited, can be easily made, and naturally codominant.
ID047	The configuration of the heterozygous patent can help determine recombination. When wild types of both alleles are contributed by one parent they are considered cis. When the alternate is true the trans configuration holds.
ID086	Haplotype Networks can show areas that have more or less of a gene. They allow geneticists to map where different genes migrated to in population areas.
ID086	This is a very helpful aspect in mapping diploid genomes. Every person has different amounts of variations in their genes.
ID086	This is key to mapping successfully and accurately. Without it one can only guess as to where the original genes were located and where they crossed.
ID084	A haplotype is the sequence on one chromosome: it could be a short series or all of the alleles. A haplotype network shows the relationship among the haplotypes in a mapped form. Haplotypes can be used to trace human origins. Haplotypes are usually seen in the same groups.
ID084	Microsatellites are areas where there are short repeated segments. There is a wide range of available numbers of vntrs. The vntrs are molecular markers which can help us to identify the location of a gene on a chromosome.
ID084	This is important because it determines the genotype/ phenotype of the progeny. If you do not know the correct order of parents, you will get the final order of the genes in a DCO, linkage analysis, or test cross,  wrong.
ID054	A haplotype network will show the relationship between multiple haplotypes, which will allow you to trace an individual's genes back multiple generations.
ID054	Microsatellites are short, repeating segments of DNA. Therefore, the more of them there are within a population the less variability there will be. Between populations, you have to compare the amount of microsatellites, the bigger the difference, the more variability there will be between populations.
ID054	If the alleles are in cis configuration, the wild type alleles will be on the same chromosome, and therefore the progeny will be heterozygous. If they are in trans configuration, the progeny will always be homozygous for either the wild-type or the mutant.
ID024	Haplotypes are a set of SNPs that are associated statistically and they reveal if people in a population are related.
ID024	Microsatellites are short pieces of DNA that have repeated sequences. There is a high mutation rate in microsatellites, thus giving rise to variability in a population.
ID024	Cis configuration is when both wild type alleles are on one homologous chromosome, while both mutant alleles are on the other homologous chromosome. Trans is when there is a wild type and mutant allele on each homologous chromosome. It is necessary to know whether the configuration of the parent in order to know how crossing over will look.
ID140	Haplotypes are groups of alleles that are inherited together. They can reveal information about a population by indicating which alleles generally move together and perhaps where that originated.
ID140	This is important because without variation, we could not progress.
ID140	Cis and trans configuration can indicate whether a parent is hetero or homozygous for a trait.
ID130	Haplotypes are blocks of DNA that tend to be inherited together.  This can reveal a lot about a population and the phenotypes present in it.  As more and more haplotypes form, the geneti variation steadily decreases.
ID130	Yes.  This is how you can get things like multiple types of dogs, and dogs with different spotting patterns and sizes.  These tandem repeats have on the individual as a whole and make up most of the genetic diversity within a population.
ID130	The cis or trans configuration determines how the gametes of that parent will separate out.  If you know the cis or trans configuration it becomes easier to predict the gametes and the genes that organism will pass on.
ID072	Haplotypes are sets of variations, that tend to be inherited together as a group.
ID072	Variability associated with microsatellites have to due withe the speed of processing and the potential to isolate a  large number of loci to provide a marker system that has detectable differences among closely related populations.
ID072	based on the the heterozygous parent in a linkage being cis or trans allows us to depict which gene is crossed over.
ID087	Haplotypes are unique sequences of the genome. They can be shared by individuals. When a network is created of the different haplotypes in a populations, then the similarities and differences can be evaluated.
ID087	Kinship can be determined by microsatellites. They can be amplified using PCR techniques. The larger the microsattelite, the less distance it travels in the gel.
ID087	The cis configuration is when two or more dominant alleles are on the same chromosome in a heterozygous individual, while the recessive is on the homologous chromosome. The trans configuration is when the dominant alleles of two or more genes are on separate chromosomes. This is helpful when determining if an individual will get more than one dominant gene in reproduction. The order of the alleles will determine if the genes have the ability to recombine or not.
ID003	I don't know much about this. I have to look over the content.
ID003	There are different variations because the way different people interact with one another.
ID003	Know these things to no how crossing over occurs and recombination.
ID001	Sample
ID001	Sample
ID001	Sample
ID000	Haplotype networks describe relationships between individuals in populations and may be based on haplotype DNA sequence information or other molecular markers such as indels or microsatellites.
ID000	Given HW equilibrium in a population, gene frequencies and genotype frequencies will stay the same from generation to generation and can be predicted from the binomial expansion. Genotype frequencies are p2  2pq  q2  1 for autosomes; allele frequencies are p+q=1
ID000	Natural selection results in a change in population allele frequencies over time. Selection may be stabilizing selection which occurs when homozygotes are at a selective disadvantage compared to heterozygotes (sickle cell anemia), directional in which a specific allele frequency increases as a result of selection or disruptive in which selection is against heterozygotes.
ID039	Haplotype network show the relationship among many haplotype. A haplotype is dna sequences
ID039	The two formulas, p+q=1 and p^2+2pq+q^2=1, are used to make the prediction. One of the values has tonbe known, once it is known the other values can be solved for. This whole situation of using binomial expansion to make a prediction can only be used when Hardy Weinberg is considered
ID039	Hardy Weinberg states that no mutation, migration, or selection is occuring. Therefore the population is staying the sae overtime. When Hardy Weinberg is not occurring, the is migration, mutations and selection acting upon the population and therefore there will be evolution and changes taking placed.
ID105	A haplotype network show the relationship among haplotypes which are a set of DNA variations that tend to be inherited together.
ID105	By plugging in the frequency of two different alleles dominant (p) and recessive (q) into the formula (p+q)^n=1 where n equals the number of alleles you can calculate the frequency of homozygous dominant, homozygous recessive, and the frequency of heterozygous individuals in a population.
ID105	If the criteria for HW equilibrium are not met them populations are evolving through things like natural selection, genetic drift, gene flow, or mutation.
ID014	A haplotype is a set of polymorphisms that are inherited together.
ID014	The equation for binomial expansion may be used to help us determine how frequent alleles, genotypes, and phenotypes are seen in the population.Initally, we could use the binomial expansion to determine the phenotype of individuals but now other factors can be estimated.
ID014	The Hardy Weinberg equilibrium states that allele frequencies and if the Hardy Weinburg is broken then there evolution has occured.
ID125	represent the relationships among the different haploid genotypes
ID125	using the algorith you can see frequency
ID125	there are other factors that effect evolution
ID069	A haplotype is a unique set of genetic code. Individuals possessing the same haplotypes can be inferred to be related. This information can create haplotype networks.
ID069	Binomial expansion can be used to create a prediction of allele, genotype, and phenotype frequency based on the total number of alleles in a population, so long as the population adheres to the Hardy-Weinberg equilibrium conditions.
ID069	Populations that are not in Hardy-Weinberg equilibrium are being effected by other evolutionary tactics that are impacting the allele, genotype, and phenotype frequencies. Such tactics include mutations, natural selection, gene flow, and genetic drift.
ID089	Demonstrates relationships between organisms based on sections of DNA.
ID089	Where p^2 is the frequency of homozygous dominant individuals and q^2 is the frequency of homozygous individuals, 2pq is the frequency of heterozygotes.
ID089	To my understanding, this is not actually necessarily true. Just because a population is not in HW equilibrium does not mean that the assumption(s) of HW equilibrium being violated is the assumption of no evolution. An evolving population would not be in HW equilibrium, but the population could be violating the large population assumption but yet not be evolving.
ID079	Haplotype networks show the relationship of a population in terms of gene sequences.  Haplotype is a group of alleles that are inherited as a unit; looking at a network of haplotypes in a population or in populations over time can show relations within a population and lineages over time.
ID079	Binomial expansion calculates the probability of a specific outcome involving two separate events within the outcome.
ID079	Hardy-Weinburg assumptions are: no mutation, no migration, infinite population size, no natural selection, and random mating. If a population meets all the requirements, it is not evolving. Naturally, a population cannot meet these requirements, and thus, evolution takes place.  Mutation produces new alleles and migration brings new alleles into a population, both increasing genetic diversity.
ID027	It shows how the organisms are related to each other through differences in the genetic code. If I am understanding correctly, it can help determine the "common ancestry" of an organism.
ID027	The frequency of alleles, genotypes, and phenotypes are statistical probabilities and can be analyzed just like any other probability through binomial expansion. This can tell the chances for specific outcomes.
ID027	If a population is not in hardy Weinberg, then it violates one of the underlying assumptions, meaning that evolution is occurring. These assumptions are that there is no: selection, gene flow, genetic drift, or mutations occuring.
ID098	Haplotype networks are a neat and concise way of organizing and representing the genetic differences between different groups of organisms. They display the relative genetic closeness of certain organisms based on how many different haplotypes they have.
ID098	Using different variables to represent different alleles, binomial expansion can be used to estimate allele frequencies, and thus the frequency of certain genotypes and phenotypes. Pascal's triangle can be used to find the appropriate binomial equation for a given gene. A common example of binomial expansion as related to genetics is the Hardy-Weinberg equation.
ID098	The Hardy Weinberg equation makes five assumptions: that the population is infinitely large, that mating is random, that there is no natural selection, that there are no mutations, and that there is no migration. In nature, virtually all of this conditions are continuous broken, resulting in allele frequencies that do not follow the Hardy-Weinberg predictions.
ID133	A haplotype network shows the relationship among haplotypes. These may be used to trace back to ancestors.
ID133	(p+q)^2 can be factored out using the binomial expansion.  If you know the frequency of homozygous recessive alleles in a population, you can calculate the others using this equation.
ID133	If your calculated value of allele frequencies in a population does not correspond to what is found naturally, then at least one of the Hardy Weinberg conditions is being violated.  The population then could be experiencing mutation, drift, gene flow, or natural selection.
ID068	Haplotype networks show how DNA is related among indviduals. It also shows how the DNA differs between individuals and what gene has the variations. These variations are determined by SNPs (single nucleotide polymorphisms) which is a single base change in DNA.
ID068	There are different equations that help to determine the frequency of alleles and genotypes. The letter "p" is used to describe the frequency of the dominant allele and "q" is used to describe the frequency of the recessive allele. p + q will always equal 1. We can use this to determine frequencies of genotypes as well using the equation p^2 + 2pg + q^2. p^2 is the frequency of the homozygous dominant genotype, q^2 is the frequency of the homozygous recessive genotype, and 2pq is the frequency of the heterozygous genotype.
ID068	Hardy Weinburg Equilibrium states that allele/genotypic frequencies stay the same from generation to generation. If the frequencies stay the same, the population will not be evolving. For a population to be evolving, the allele/genotypic frequencies will be moving in various directions for different genes.
ID050	They represent the different relationships between between different haploid types.
ID050	Frequencies can be predicted with binomial expansion by accounting for all the different frequency possibilities and then dividing them by the one that are wanted.
ID050	Populations that are not in Hardy-Weinberg evolve through natural selection, mutations, genetic drift, gene flow, and nonrandom mating.
ID015	a haplotype is a unique sequence of bases over a region of the genome
ID015	the probability of the occurrence of some arrangement of two mutually exclusive trials, where the final order is not specified, is defined by the binomial theorem
ID015	they can evolve in ways such as genetic drift, mutation, gene flow
ID036	A haplotype specific group of genes that a progeny inherits from one parent.
ID036	The hardy Weinberg  can tell you to determine the phenotype, and the phenotype as if no genetic drift, natural selection, and gene pool affects the population.
ID036	Individuals are not in the hardy Weinberg equilibrium evolve differently because there are different
ID070	haplotype networks are of critical importance for studies at the population-species interface or below the species level, in particular those that estimate parameters based on network topologies such as nested clade analyses
ID070	The binomial expansion helps determine the different frequencies in biology
ID070	The Hardy-Weinberg equilibrium is a principle stating that the genetic variation in a population will remain constant from one generation to the next in the absence of disturbing factors. When mating is random in a large population with no disruptive circumstances, the law predicts that both genotype and allele frequencies will remain constant because they are in equilibrium.
ID116	Haploid networks show the relationship between different haploid genes.
ID116	When these are expanded, these are populations within a Hardy-Weinburg population.
ID116	These populations have mutations, genetic drift, natural selection and gene flow.
ID013	Haplotype networks are used to show the relationship among haplotypes, which are short pieces of a chromosome with traits that usually appear as a group or are linked. These create maps that show how individuals differ by SNPs, or single nucleotide polymorphisms.
ID013	The binomial expansion uses Pascal's triangle to predict how a family could have so many children with a certain phenotype or genotype thus allowing us to predict the probability of certain alleles occurring in that generation.
ID013	Populations in Hardy Weinberg equilibrium have no migration, no mutation, no selection, random mating, and a infinitely large population. If a population is not in Hardy Weinberg equilibrium, then they must be evolving as a result of one of these five characteristics being broken. Most populations are not in Hardy Weinberg equilibrium.
ID038	The group of genes present on the same chromosome are called haplotype network.
ID038	Binomial expansion can be used to assess genotype and phenotype frequency directly while working the punnet square.
ID038	Hardy Weinberg equilibrium describes a situation when the population is not evolving due to absence of various evolutionary forces. Thus when the equilibrium is not there, at least one force is working on the population to evolve it.
ID137	Haplotypes are short sequences found in the genome that can be compared to other individuals t determine how closely related they are. The networks show these relationships, for example, in phylogenetic trees.
ID137	You can use binomial expasion in order to find the frequency of alleles for a certain number of 'n' factors. If you are given the frequency of a recessive or dominant allele you can use the quadratic formula with polynomial ^2 in order to isolate the variable. You can then use p+q=1 when finding genotype frequencies.
ID137	Hardy Weinberg equilibrium occurs when there is no mutation, no selection, no migration, a large population, and random mating. This ensures no evolution so that allele frequency doesn't change.
ID094	Haplotype networks can be used to identify individuals by genome compared to a sample haplotype. Haplotypes move together.
ID094	HW Equilibirum predictsthese frequencies vie binomial expansion but is not always accurate because of evolutionary forces.
ID094	HWE assumed ideality but this is not the case; species migrate, mutate, and involve in various ways that cause deviation from ideality.
ID056	Haplotype networks are how different haplotypes are in relation to one another. A haplotype is a group of polymorphisms generally inherited together. They can serve as genetic markers to identify certain chromosome areas.
ID056	Just as we used binomial expansion previously in determining the likelihood of developing a specific trait out of a set number of individuals (n), we can use it to predict the frequency of alleles, genotypes, and phenotypes. It is best explained by Hardy Weinberg where the square of the two traits can give you the frequency of both homozygous traits and the heterozygous.
ID056	Hardy Weinberg equilibrium has many requirements such as no mutation, no migration, random mating, no selection and large population requirements. When a population fails these requirements it can no longer be analyzed using Hardy Weinberg. Those that fail are evolving such as having new mutations or new individuals migrating into the population. These new traits can affect the population by changing the offspring.
ID096	A haplotype network maps evolutionary and geographical history.
ID096	Using the equation of binomial expansion, we are able to predict the frequency of alleles, genotypes, and phenotypes given a situation of offspring.
ID096	Populations not in Hardy Weinberg equilibrium can be a small population size that allows for genetic variability. Mutations can cause variability.
ID028	We can trace our ancestors origins using haplotype networks.
ID028	Using pascals triangle, we can predict phenotype ratios in a population that is currently in Hardy-Weinberg equilibrium.
ID028	A population in Hardy Weinberg equilibrium is not subject to any outside forces that may disrupt the allele frequency.
ID131	Haplotype networks are when the sequence of the gene are laid out in a way they can be compared. We use them to identify people as well as to look at the differences in people. Sometimes, they are used to predict conditions.
ID131	Binomial expansion allows us to easily predict the likelihood of something happening using a formula borrowed from mathematics.
ID131	Hardy Weinberg equilibrium means that a trait within a population doesn't change in frequency over time. There are many rules to be in Hardy Weinberg equilibrium and few genes meet this criteria.
ID007	Haplotype networks are representations of how different individuals (or populations) have genetically diverged based on how a certain DNA sequence has mutated over time.
ID007	Given two alleles with one of their frequencies known, genotypes within a population can be predicted using (p +q)^2 where p+ q = 1. With 3 alleles, the formula would be (p + q + r)^3 and so on.
ID007	The following conditions must be met for Hardy-Weinberg equilibrium: large population size, random mating, no migration, no selection, no mutation.  Any violations of these conditions would result in the frequencies of alleles and/or genotypes changing from generation to generation, which is to say, evolution.
ID026	Haplotype networks are relationships between different haplotypes on a chromosome, or a set of single nucleotide polymorphisms. These haplotypes can be associated statistically based on traits like how far away they are from one another.
ID026	Binomial expansion can be used to predict the likelihood that the offspring will have a certain genotype from the parents. This is calculated from the probabilities of each allele occurring and the number of trials that take place.
ID026	Most populations do not follow the Hardy Weinberg equilibrium method exactly. Natural selection is a factor in this which throws off the value of probability generated from the Hardy Weinberg equation.
ID097	A haplotype network is a way to organize the small DNA sequences comparing and contrasting similarities within a species.
ID097	(a+b)^n=1 allows us to identify frequencies in a population as well as predictions.
ID097	Hardy Weinberg in its equilibrium shows no variability or evolution within a population. Therefore, a postulate has to be broken in order for more genetic diversity leading in to evolution.
ID022	Haplotype networks shows the relationship among the different haploid genotype seen in the database. The haplotype are basically set of genes or alleles that are found closer to each other (linked together) are inherited together.
ID022	if we know the genotype of the parent then we can predict the genotypes, alleles, and phenotypes. For example if a heterozygote male (Hh) mates with a homozygous recessive (hh) female that have cystic fibrosis. Thus based on the the above information it can be said that they would produce half children that are recessive (hh) and half children are (Hh) and half would be normal. Thus based on above information we can say half progeny has H//h alleles and the other half has h//h alleles. Also, the phenotype for half is Hh normal and other hh are recessive so they have cystic fibrosis.
ID022	According, to Hardy Weinberg you can calculate the  genotype of the parents and genotype, phenotype and alleles of the progeny but only if there is no mutation, large population, no mutation, no gene flow etc. However, in normal circumstances there is gene flow which allows the gene from different race or different people exchange between them would increase the chances of them evolving. Besides that mutation- a random gene mutation is a big cause of evolution, genetic drift- a random change in the alleles would also increase the chances of evolving in different population.
ID095	These type of networks represent the genotypes that get passed on to generations, and we can study what we know to predict the ratios a population will be (heterozygous/homozygous).
ID095	The binomial expansion is an equation used to estimate the ratios of genotypes in a population.  We may represent an allele (A) as the variable p, which is the ratio of its frequency (normally in decimals, hundredths out of one), and the other allele (a) as q (its ratio as well).  If p + q = 1, then we may find either variable if we know one.  This can be used for the amount of genes we are observing, where n may represent the total amount of genes, then we can say (p+q)^n = 1.
ID095	Hardy Weinberg Equilibrium uses binomial expansion to predict these ratios of genes, and this can be calculated for a series of generations.  The Hardy Weinberg Equilibrium suggests that there is no mutation, no selection, no genetic drift, nor any gene flow.  If we may assume these rules, then a population's possibly genotype frequencies may be calculated.  If there is an observable change over generations, then the system is not in Hardy Weinberg Equilibrium, therefore some factor is favoring an allele (mutation, selection, gene flow, or genetic drift).
ID061	Haplotype networks can reveal how closely related members of a population are. They are important because a haplotype is a set of genes that are linked in a region and thus are passed on together. So if individuals have the same haplotypes then they share many of the same genes.
ID061	The frequency of alleles, genotypes, and phenotypes can all be predicted through binomial expansion. Specifically, they can be predicted through the Hardy-Weinbeg Law. According to this Law, as long as a population is free from mutations and migrations, has random mating, is infinitely large, and each individual has the same chance of survival, then you can predict all of these frequencies if you know the frequency of just one allele or genotype. They can simply be predicted by plugging them into an equation and solving for the frequencies.
ID061	If a population is not in Hardy Weinberg Equilibrium then it is not staying the same fro generation to generation. This population could be producing new alleles from mutations, experiencing migration from individuals entering or leaving the population, experiencing genetic drift if it is a smaller population, or the individuals could engage in nonrandom mating. Also, natural selection would cause the population not to be in Hardy Weinberg Equlibrium and evolve.
ID063	This is the group of genes a progeny inherits from both parents. It can also refer to a set of SNPs on a chromatid.
ID063	This pretty much says that we can predict the next generations alleles, phenotypes, and genotypes using the (a+b)^n equation of binomial expansion.
ID063	Any population not following Hardy Weinberg's assumptions are said to be evolving. This could mean that there are mutations in the populations or that natural selection may be occurring.
ID101	The importance of haplotype networks, is that they trace the origin of a species and relate it to other species and phyla as reflected in the tree of life.
ID101	The recessive gene is observed to be q squared and is the total recessive gene. From taking the square root of this binomial expansion and finding p by 1-q, then were can find both the dominate and heterogeneous percent of the population. This can only be used if in Hardy Weinberg equilibrium.
ID101	The testing for HW equilibrium is when evolution is not taking place, if it is occurring, as it normally is, then the probabilities of the HW equilibrium will not match the population.
ID049	Haplotype networks are graphic representations of mutations that can happen in an organisms genome.
ID049	Using the binomial formula the probability of frequency for certain alleles can be predicted. This does not always mean that this will be the phenotypes that are observed.
ID049	Populations that are not in hardy Weinberg violate one of the hardy Weinberg principles. Populations could evolve due to genetic drift, migration, mutation, these mutations have to be selected for, for them to stay in the gene pool
ID020	Haplotype networks represent the relationships among the different haploid genotypes that are observed in dataset (a population).
ID020	Binomial expansion helps people know predict the genotypic outcome when breeding. We are able to us the binomial expansion to predict the chances of certain phenotypes or genotypes appearing the next generations.
ID020	The Hardy Weinberg equilibrium describes what happens to alleles in " ideal" population, so populations that aren't evolving, that don't have other genetic factors to interrupt.
ID082	Haplotypes are groups of linked genes that are commonly inherited together because they are close together on the chromosome. These groupings allow geneticists to compile haplotype networks and find out where genes are on the chromosomes.
ID082	Binomial expansion allows us to calculate and predict allele frequencies in populations. The Hardy Weinberg principle is rooted in this expansion, but goes a step further to allow us to identify the proportion of genotypes in the population (p squared is homozygous dominant individuals, 2pq is heterozygous individuals, p squared is homozygous recessive).
ID082	Hardy Weinberg equilibrium requires a large population, no new mutations, no natural selection, no movement in or out of the population, and random mating. If a population is not in this equilibrium, that means that one or more of these conditions are being violated. Hardy Weinberg equilibrium predicts that the allele frequencies will stay constant in future generations, but if a population is evolving (mutations, natural selection, non random mating), the allele frequencies will change from generation to generation. The alleles that are being selected for will increase in frequency, while the alternate alleles for that gene will decrease.
ID081	A haplotype is the specific group of genes that the child inherits from one parent. This means they are very similar to the parent and its genes.
ID081	When using the binomial expansion, p^2=homozygous dominant; q^2=homozygous recessive; 2pq=heterozygous. The frequency of p and q add up to equal 1. The frequency of the alleles does not change, but the frequency of genotypes can change. The equation is (p+q)^2 = 1 - which also equals p^2 + 2pq + q^2 = 1.
ID081	When populaitons are not evolving in Hardy-Weinberg equilibrium, there are a various number of assumptions that could be broken. These include: natural selection, mutation, migration, small populations, and nonrandom mating.
ID115	A haplotype network is a method to connect individuals to one another based on the unique set of gene sequences they have on their DNA, single nucleotide polymorphisms. Since these are inherited from parent to child, an established link between the two can be used to identify the distance of the relationship an individual might have with one another as well as the ancestry they might share.
ID115	Using the Hardy Weinberg equation, which assumes that under random mating, no migration, no genetic drift and no mutation in an infinitely large population, genotype and allele frequencies can be predicted for that population using a binomial expansion. Depending on the number of genes whose frequency you want to identify, more or less variables may be used to identify the homozygous dominant and recessive as well as the heterozygous in the population.
ID115	Because there are a set of assumptions that should be held for Hardy-Weinberg equilibrium to occur, which include no mutation, random mating, no migration,  gene flow or genetic drift in an infinitely large population, if any or more than one of these conditions are not met, one can assume that evolution is occurring within a population as the frequency of alleles in a genotype will change.
ID085	Haplotype networks are a useful way of comparing mutations within haplotypes.
ID085	Binomial expansion and Pascal's triangle use simple statistics to predict the ratio of dominant, recessive, and heterozygous alleles in a population; the frequency of genotypes and phenotypes can be inferred from that.
ID085	For a population to be in Hardy Weinberg equilibrium, it cannot be changing in any way.
ID040	Haplotypes are sets of alleles from closely linked loci carried by an individual that are inherited together. Haplotype networks show the relationship among haplotypes. They can be used to compare individuals between populations to infer which ones are more closely related.
ID040	Binomial expansion can be used to predict different frequencies with the use of the Hardy-Weinberg equation which is p^2+2pq+q^2 = 1. The equation can be expanded to make predictions for multiple alleles. p+q=1 can be to determine the allele frequencies and this is squared to get the Hardy-Weinberg equation for frequencies of genotypes and phenotypes.
ID040	No population is actually in Hardy-Weinberg equilibrium. In populations that would meet Hardy-Weinberg equilibrium, there is no evolution. Without that, populations can experience genetic drift, mutation, gene flow and natural selection which could potentially change allele frequencies in a population.
ID024	A haplotype is a set of DNA variations, or polymorphisms, that are often inherited together. It is a combination of alleles or to a set of SNPs found on the same chromosome. Haplotype networks show how different people are related to each other.
ID024	The process of binomial expansion allows us to predict both the frequencies of the different alleles in the population and how those alleles are distributed across the population. It can tell the number of homozygotes and heterozygotes in the population.
ID024	Not being in Hardy Weinberg equilibrium means that evolution is occurring. When not in Hardy Weinberg equilibrium, populations are affected by different kinds of selection, including directional, stabilizing, and disruptive selection.
ID076	inheritance of an allele from one parent
ID076	binomial expansion is a mathematical tool that has been proven to be useful in calculating genetic variables. in genetics, a key to understanding and learning more about genes is seeing whether the numbers do or don't match up, which then leads researchers to ask more questions on what is going on if the expected numbers are way off from observed
ID076	Hardy weinberg is unrealistic for real life scenarios; however, it is useful since it involves simple math and if a certain situation's values do not match the hardy weinberg values, then one looks at which parameter(s) of HW it is violating - which helps better understand what is genetically occurring.
ID135	The haplotype network is a series of genes on a chromosome. If we know the distance between SNPs, microsatellites,indels or genes that consistanly are on the same position of a chromosome, we can use these known relative positions to figure out the relative distance of other genes.
ID135	Using the Hardy-Weinburg equilibrium equation: p^2+2pq+q^2 and p+q=1
ID135	No population can ACTUALLY be in hardy weinberg equilibrium, because the conditions require totally random mating, an infinitely large population, no mutation, and no immigration/emigration. So any given population experiences a small amount of mutations, but when these mutations are advantageous, the become reproduced and proliferated in the offspring of the original mutant. However if two of the same species had been separated by a river at some point and Population A  acquired mutations independently from Population B, then eventually they would be too genetically different to create fertile offspring, thus resulting in two different species, thus evolution.
ID008	Haplotypes are usually short in length so that the possibility of similarity  will be greater the closer they are together.
ID008	When there are various amounts of alleles on one chromosome that can show many different phenotypes, we can use the binomial expansion in order to predict the frequency of the amount of times that allele will show up in each generation.
ID008	The law says that alleles and genotype frequencies in a population will remain constant from generation to generation in the absence of other evolutionary influences.
ID126	A haplotype is a set of DNA variations, or polymorphisms, that tend to be inherited together. The network connects them through the DNA.
ID126	This is stating  that allele and genotype frequencies in a population will remain constant from generation to generation in the absence of other evolutionary influences.
ID126	Allele and genotype frequencies will remain constant in the absence of evolutionary mechanisms. Ultimately, the Hardy-Weinberg principle models a population without evolution under the following conditions: no mutations.
ID134	A haplotype collection of alleles that are on closely linked loci being caried by an organism that are passed on together. A haplotype network shows the relationship among haplotypes, which means it can be used to find connection between individuals, poplulations, possibly even species. It can also be used to trace histories and the origin of certain alleles.
ID134	The formula p^2+2pq+q^2=1 along with p+q=1 is used to find allele and genotype frequencies for both homozygous recessive inheritance, homozugous dominant, and heterozygous. If phenotype is know then it can be used to figure the projected genotype, and as we know known genotypes can prredict phenotype.
ID134	Hardy Weinberg Equilibrium limits a population to having no migration, no mutation, and no natural selection. Also, it is used for an infinitely large population with random mating. However, because the qualifications are so specific it can be used to determine which characteristics are changing and which cause that population to not be within Hardy Weinberg law.
ID043	unique sequences of bases in a genotype
ID043	(a+b)^n= 1
ID043	Hard to predict their evolutions.
ID052	A haplotype network is a system which consist of paired genes on a chromosomes usually leading to the inheritance of both genes.
ID052	The binomial expansion is useful to predict whether paired alleles are homozygous or heterozygous.
ID052	To be in Hardy Weinberg equilibrium, a population must be very large, have no mutation, natural selection, and/or gene flow, and have random mating.
ID065	Haplotype networks are areas of DNA sequences that can be studied and used to locate areas of genes. They serve as identification tags for genes and through these identifications, more economical methods can be used in order to recognize and locate specific genes and DNA sequences.
ID065	Binomial expansion is used in order to determine the probability of various events happening in a given population. This can be used to determine the amount of genotypes and phenotypes in a population, and this information can ultimately be used in order track history or populations and how genes, alleles, and phenotypes have changed in frequency over time.
ID065	This occurs when it is determined that Hardy Weinberg equilibrium is not the current condition of the population being studied. This is relevant because of the implications and consequences that come with Hardy Weinberg equilibrium. If is it determined that a population is not in this equilibrium, various aspects of the population, including types of selection, can be evaluated and further understood.
ID023	Haplotype networks involve genes that are inherited together.
ID023	binomial expression can help determine allele frequency of either a D allele, a d allele, or heterozygotes. You can take the heterozygote frequency as well and determine the dominant and recessive frequencies.
ID023	Populations that are not in Hardy-Weinberg equilibrium are evolving because of migration, genetic drift, natural selection, or mutation. Populations that are in the equilibrium do not have variations in allele frequency from generation to generation
ID090	A haplotype is a unique sequence of bases over an area of the genome. People who share a haplotype are thought to be closely related since they have the same sequence for that entire region in the genome. We can use haplotypes to compare individuals between populations in order to decide which populations are most closely related to which others/which evolved more recently.
ID090	We can use the equation p2 +2pq+q2 to determine the frequency of alleles and genotypes in a population that is experiencing Hardy Weinberg equilibrium.
ID090	Populations that are not in Hardy Weinberg equilibrium must be violating on of the criteria for a HW population. It could be a very small population that is inbreeding or undergoing nonrandom mating, mutations could have occurred, selection could be acting on a particular trait, and gene flow or genetic drift could also be occurring.
ID118	Haplotype network is series of traits or allele on chromosome or series of short piece of chromosome that is typically linked and generally very stable. Haplotype network is used to find relatedness between population. Haplotype network must be in middle not too long not too short, which shows accurate and informative information regarding the relatedness of individuals to others
ID118	Under "ideal" population of hardy-weindberg, allele frequency in the population is found by p+q=1, where p is dominant alleles in population and 1 is recessive allele in population. (p+q)^n=1 where is n is number of alleles in the population. so for 2 alleles, individual frequency can be found by p2+2pq+q2=1, where p2 is homozygous dominant, 2pq is heterozugous or carrier and homozygous recessive individuals. X-linked individual frequency=allele frequency, can be found by p+q=1 as hemizygosity lacks one of the X chromosome.
ID118	Factors the influence the evolution to occur are natural selection, sexual selection or nonrandom mating, gene flow, genetic drift, and mutation. Natural selection is when one variation of trait is favored more over the other describing their fitness, which is defined by survival and reproduction, they contribute more to the next generation, evolving the population over time. Gene flow is when immigration or emigration influence the gene frequency, leading to evolution. genetic drift when small population looses gene variation for very long period of time, genetic bottleneck. mutation introduces new allele to population. non random mating where selection on the choice of mate. This all evolution leads to speciation which is reproduction isolation.
ID075	Haplotype networks are traits or alleles which appear as a group and are typically linked. They are generally very stable. A haplotype network shows the relationships among haplotypes.
ID075	Binomial expansion lets you find this essential information. The p squared plus 2pq plus q squared is very useful the geneticists when trying to find the frequency of the recessive or dominant allele.
ID075	There are many factors which violate the Hardy Weinberg equilibirum. These factors that cause populations to evolve include: migration, non random mating, natural selection, gene flow, and genetic drift.
ID053	Haplotype networks allow us to see how populations not only evolved, but also how the populations moved geographically based on examining haplotypes, which are characteristic to each population.
ID053	Using the formulas (p+q)^2=1 and p+q=1, where p is the frequency of one allele and q is the frequency of the other, we can predict the frequency of alleles, genotypes, and even phenotypes.  As long as we are given one of the frequencies, we can determine the others through these two formulas. Calculating these frequencies in this manner, however, requires Hardy Weinberg assumptions, so most real-life frequencies cannot be determined through this accurately.
ID053	Because Hardy Weinberg equilibrium postulates that the population in question must be under certain conditions that prevent evolution, we know that if a population is not in Hardy Weinberg equilibrium it must be breaking one of the 5 assumptions, which indicates that it must be evolving
ID149	Haplotype networks is a map that shows the frequency of alleles in a certain population.
ID149	The bionomial expansion is really easy to use to calculate the probability of a certain event.
ID149	Hardy-Weinberg states that a population that has no selection stays constant, so if the population has selection, it is evolving.
ID109	It is a web that shows how species and individuals are related. Also,it shows the relationship between different haploid genotypes in a group of species. It could also be described as the summary of genetic diversity in a group.
ID109	This method is a predictor to show evolution in and changes in the gene pool. It uses the different phenotypes and genes to show what the predicted alleles, genotype, and phenotype of offspring can be.
ID109	It is almost impossible for a population to stick to all of the requirements for the Hardy Weinberg equilibrium. All populations are evolving, always. So if these populations are not in Hardy Weinberg equilibrium, of course they will still evolve, just in different ways.
ID150	A haplotype is a series of bases over a region of a genome. By comparing haplotypes between different populations and species, scientists can infer relatively how close the two are related.
ID150	Populations follow the same probability rules that individuals follow. By comparing expected probabilities to experimental proportions, scientists can identify discrepancies.
ID150	Although populations are never in this equilibrium, it sets a base to help understand and study the amount of variation a population goes through over time.
ID033	A haplotype network shows the closest evolutionary ancestors to a species.
ID033	assuming hardy weinberg equilibrium the binomial expansion formula can show the frequency of alleles.
ID033	When populations evolve in ways that are not in hardy weinberg equilibrium it shows that a variable is causing the change that's not accounted for.
ID092	Haplotype networks are when theres a short group of bases that are identical in the same location and same order in individuals.  This can be used to determine which populations of animals are more closely related to one another.
ID092	Binomial expansion can be used to show the probability of events happening in various types of ways.  This can be used if theres two types of alleles in an organism, can be used to predict genotype probabilitites along with phenotypes.
ID092	Hardy Weinberg says that in order to be in equilibrium, meaning that the projected genotype possibilities are correct, that there must be no migration, no selection along with 3 other variables.  If a population does not abide by all 5 of these then they are evolving differently than populations that are in equilibrium.
ID037	A haplotype is a set of DNA variations, or polymorphisms, that tend to be inherited together
ID037	p=dominant allele. q=recessive allele
ID037	allele frequencies in populations in hardy weinberg equilibrium remain the same from generation to generation
ID123	Haplotype networks are a group of individuals who are closely related and have the same sequence of bases over a region of the gen
ID123	Binomial expansion combines different probability formulas into one universal formula.  It allows the calculation of alleles, genotypes, and phenotypes by calculating the total number of individuals and the number of affected or not affected and placing each in their respective variables in the formula.
ID123	Populations that are not in Hardy Weinberg equilibrium are evolving in many different ways.  They are mutating and migrating.  They are mating non randomly.  Natural selection is occuring.  The population is small.
ID051	A haplotype is a unique sequence of bases over a region of the genome
ID051	the sum of all genotype frequencies is the binomial expansion of the square of the sum of p and q. The sum represents the total of all possibilities, must be equal to 1. (p + q)2 = p2 + 2pq + q2 = 1.
ID051	If no mechanisms of evolution are acting on a population, evolution will not occur highly unlikely that any of these seven conditions, evolution is the inevitable result.
ID103	I understand that haplotype networks are short sequences of repeating DNA that can be used to determine how closely related individuals are.
ID103	I understand that if the frequency of the allele's in a population is known that can be used to predict the genotype frequencies in the next generation.
ID103	If the assumptions of Hardy Weinberg are not met for example if a population experiences nonrandom mating then the nonrandom mating will influences genotype frequencies.
ID031	This is when there are many different haploid genotypes linked together in one way or another. This is what helps certain species be considered family.
ID031	Using binomial expansion which is a mathematical formula you can predict the percentage chance that a certain allele, genotype, or phenotype will appear.
ID031	If populations are not in Hardy Weinberg this means that they are evolving because they could have mutations arise, gene flow, genetic drift, or just natural selection.
ID147	Comparing DNA base sequencing between populations to discern differences and similarities.
ID147	Using binomial expansion we can take a certain set of alleles, genotypes, or phenotypes and following the binomial equation can determine how many will occur based upon differing restriction.
ID147	Popuations in equalibrium will not have any new variation in their population. Populations not in equalibrium will have active components of evolution and newly developing variation.
ID093	A haplotype is a unique sequence of bases over a specific region of the genome.
ID093	Binomial expansion in conjunction with Pascals triangle can be used to predict the frequency of alleles, genotypes, and phenotypes. p+q=1 where q is usually the recessive allele and p is the dominant allele.
ID093	Populations that do not follow natural selection, gene flow, or genetic drift will not follow the Hardy Weinberg principle. Inbreeding and other factors can contribute to an allele becoming fixed within a population.
ID048	haplotype networks shows the relationship among a set of DNA variations that tend to be inherited together. This can be used to determine various things, for example, it can be used to trace different origins of different countries and prevalent Y-chromosome traces.
ID048	binomial expansion that can be used to determine the probability of sets of events. It is used in genetics to determine frequencies. It can determine the probability of any given alleles, genotypes, phenotypes and what are the chances of numerous situations.
ID048	Hardy-Weinberg populations are the "ideal population". This means infinitely large population with no evolution therefore no migration, mutation..etc. With all this being a law, populations that are not in Hardy-Weinberg can constantly be evolving, mutations can occur, there is gene flow, selection, and it can occur in any number of populaitons.
ID100	Haplotype networks are groups of closely linked alleles that are inherited as a unit. Haplotype networks that consist of SNPs can serve as markers for specific chromosome regions. These regions can then be used to identify other regions that are may contain genes linked with specific diseases. This technology is incredibly important because it allows scientists to locate what gene mutations are linked cause illnesses such as cystic fibrosis and schizophrenia.
ID100	The frequency of alleles, genotypes, and phenotypes can all be predicted by using binomial expansion.(p+q)^2=p^2+2pq+q^2=1.  p stands for the allele frequency of the dominant allele and q represents the allele frequency of the recessive allele. These numbers can then be used to calculate the number of homogeneous dominant, heterogeneous, and homogeneous recessive organisms there are in a population. This is important because it allows scientist to determine a lot of information about a given population. Also, when used in successive generations, binomial expansion can be used to determine whether or not the population is evolving in regards to a specific gene.
ID100	Populations that are not in Hardy Weinberg equilibrium are evolving in different ways depending on what phenomenon and is occurring and for what reason. The conditions that must be met in order for a population to be in Hardy Weinberg equilibrium include: large population size, natural selection, no mutation, no migration,and no natural selection. When one of these is occurring/not occurring allele frequencies begin to change and evolution occurs. This is an important topic to understand because it is the basis for why populations evolve and change and sometimes become new species (speciation).
ID129	Haplotype networks show the relationship among haplotypes. Haplotypes are a set of alleles from a closely linked locus that are inherited as a unit. They can be used to trace human ancestries.
ID129	By using the equation p^2+2pq+q^2=1, it is possible to determine the frequency of alleles, genotypes, and phenotypes in a population. p^2 is the frequency of the homozygous dominant individuals. 2pq is the frequency of the heterozygous individuals, and q^2 is the frequency of the homozygous recessive individuals. By using p+q=1 (p=dominant, q=recessive) you can find the frequency of different alleles.
ID129	Hardy Weinberg Equilibrium has 5 stipulations. 1-Non random mating must occur. 2-Infinite population size 3-No migration in or out 4-no mutation 5-no selection So populations not in hardy-weinberg equilibrium are evolving by natural selection and gaining new alleles by mutation.
ID141	A haplotype network is a visual representation of how closely related populations are based on segments of identical base sequences in each individual.
ID141	Binomial expansion is extremely useful in genetics.  With the aid of Pascal's triangle, complex quantitative genetics problems dealing with alleles, genotypes, and phenotypes can be solved.
ID141	If populations are in Hardy Weinberg equilibrium, then no evolution can be occurring.  When populations are in Hardy Weinberg equilibrium, and then shift out of Hardy Weinberg equilibrium, some form of evolution must be occurring.  When in Hardy Weinberg equilibrium, there must be no selection, no new alleles can arise, no migration, very large population, and random mating must be occurring.  All of these are a necessity for the model.
ID002	A haplotype network shows the relationship among haplotypes. Mitochondrial haplotypes can be used to trace human origins to Africa. Haplotypes in a given chromosomal region will be passed on to the next generation intact, as long as crossing over does not occur
ID002	After one generation of random mating, genotype frequencies for alleles can be calculated.
ID002	Hardy Weinburg equilibrium depends on a multitude of factors and if the factors are met, the system is said to be in Hardy Weinberg equilibrium. One such is that if no mutation is occurring, then the population is in equilibrium. However if mutation is occurring, then the population is evolving, in that new alleles are being created
ID143	Haplotype networks express the idea that variations, or mutations, in DNA are typically inherited together, meaning that they are passed down amongst generations within a population.  Haplotype networks convey the significance of variation within a population as well.
ID143	Binomial expansion utilizes probability of frequency of particular genes to determine which genes are more or less likely to be passed down to future generations.  Using Pascal's triangle, one can figure out the frequency of all alleles, genotypes, and phenotypes, as long as they have the specific probabilities of each of the genes being passed down.
ID143	Using the Hardy Weinberg equation, the frequency of genes can be determined amongst populations; however, if populations are not in Hardy Weinberg equilibrium, this signifies that extraneous factors are affecting the expression of genes throughout the population, such as fluctuations in the environment, genetic drift, or gene flow throughout the individuals within the area.
ID120	These are alleles or genes that usually are passed together to the offspring as a combination.
ID120	This is the equation called the chi square where the dominant and recessive allele frequencies can be determined from known genotypes or phenotypes
ID120	These populations gene frequencies are not the same from generation to generation, so the chi square square laws are not being obeyed.
ID083	Haplotype networks show the relationship between haplotypes. Haplotypes are a group of SNP's on and section of DNA
ID083	Using binomial expansion, we are able to predict how often a certain allele, genotype or phenotype will be evident in a group
ID083	When a population is not in the H-W equilibrium, it means that one or more of the 5 H-W assumptions are not met
ID088	Haplotype networks show how different haploid genotypes are related.
ID088	Binomial expansion can be used to predict the frequency of alleles, genotypes, and phenotypes because when you know one frequency, you can use the Hardy Weinberg Law to estimate the other frequencies.
ID088	The Hardy Weinberg equilibrium shows perfect populations that do not have and mutations or migrations. Therefore if a population does not evolve according to the Hardy Weinberg equation, there must be something affecting the population.
ID119	Hapolotype networks are used to show the relationship between haplotypes. Haplotypes are sets of DNA that are usually passed down to the next generation together. The network is how two alleles are affected by one another and is the relationship between these sets of is linked alleles.
ID119	Binomial expansion is a thereom that can be used to determine the genotype frequencies in a population when under Hardy Weinberg principle. This formula helps us predict the frequencies of both homozygous and heterozygous alleles.
ID119	Populations under Hardy-Weinbergy equilibrium are ideal populations. When populations are not under Hardy-Weinburg principles, one cannot predict allele frequencies. The populations under Hardy-Weinberg laws behave much differently and most follow five specific rules. When not under these rules, the population will evolve much differently.
ID108	Haplotypes are unique sets of bases in a genome. Haplotype networks are used to see how closely related individuals are within a population. These are incredibly useful tools for determining diversity within a population.
ID108	Using binomial expansion the different combinations of numbers can be used to find allele frequencies. This is important for calculating a variety of allele outcomes. Hardy Weinberg is an example of a binomial expanded  phrase (p+q)^2= p^2 + 2pq + q^2)
ID108	Hardy Weinberg has many rules in order to maintain its equilibrium. This equilibrium is unrealistic in real and growing populations where mutations and selections happen in genetics.
ID035	Haplotype networks is a two-dimensional summary of genetic diversity within a single group, the size of each node (circle) represents the frequency of a haplotype, and the length of (or number of tick-marks on) the links represents the amount of genetic divergence.
ID035	The genotype proportions p2, 2pq, and q2 are called the Hardy–Weinberg proportions. The sum of all genotype frequencies of this case is the binomial expansion of the square of the sum of p and q, and such a sum, as it represents the total of all possibilities, must be equal to 1.
ID035	The Hardy-Weinberg principle assumes that a given population is large and is not experiencing mutation, migration, natural selection, or sexual selection.
ID006	A haplotype is a set of DNA variation that tend to be inherited together. A haplotype can refer to a combination of alleles or to a set of single nucleotide polymorphisms found on the same chromosome.
ID006	The three terms of this binomial expansion indicate the frequencies of the three genotypes:
ID006	Individual organisms don't evolve. Populations evolve. Because individuals in a population vary, some in the population are better able to survive and reproduce given a particular set of environmental conditions. These individuals generally survive and produce more offspring, thus passing their advantageous traits on to the next generation. Over time, the population changes.
ID060	A group of alleles that are inherited  together, this may be due to their closeness in the chromosome. Experts can determine how related a population is to another population.
ID060	The binomial expansion can be use to predict the outcome of genotypes/phenotypes of the next generation in different quantities.
ID060	Evolution needs genetic drift, mutation, selection, and gene flow in order for species to evolve in different ways. The Hardy Weinberg equilibrium mostly states the opposite of evolution, so most of it's population would stay the same; which is unlikely.
ID151	A haplotype network depicts the relations between haplotypes by using a map like structure to show the differences in microsatellites, SNPs, and an indel. In addition, these differences, if not many, can also portray the closeness between the haplotypes and this network is a great way to trace back events or compare the relationship between two species. This network has been used to determine the possible origination of humans which claims that the "Eve" of humans lived in Africa when Pangea existed.  This network is highly crucial in determining similarities and differences within a species or amongst multiple species and with that information a possible hypothesis could be created explaining the network's results.
ID151	Binomial expansion is a method that utilizes Pascal's Triangle and probability to determine the frequency of alleles, genotypes, and phenotypes including the gender of an offspring or multiple offsprings. This tool is crucial for multiple reasons since it gives an individual to approximate the percentage of outcome X to occur within Y trials. Binomial Expansion takes the general form of (a+b)^n where n is the total number of independent events or outcomes and 'a' and 'b' are the chances of outcome a and outcome b. Although this is basic probability, I sometimes get confused on how to utilize this mathematical model in problems, but I do understand the concept and the importance of this.
ID151	The Hardy Weinberg Equilibrium states that certain conditions must be met be fore ideal results are put into action. The conditions are the following: no mutation, no migration or immigration, no selection,  random mating, and an infinitely large population. These conditions correspond to ideal results in which allele frequencies of both the dominant and recessive allele will remain constant overtime throughout generations. This concept can also be used for sex linked genes but with a slight variation. If the Hardy Weinberg law is discredited through the violation of at least one condition that was earlier stated, then the population(s) is left subject to evolutionary influences (both micro-evolution and macro-evolution) which results in variation from one population to the next. These evolutionary factors include mutation, gene flow, genetic drift and even speciation, thus, creating differentiation a between populations.  This law consists of two equations that can be used to calculate frequencies of an allele or the frequencies of homozygous dominant and recessive and heterozygotes.
ID110	variation among populations can reflect geographical isolation (such as bottlenecking) or local adaptation.
ID110	binomial expansion of (p+q)^n expressed as Hardy Weinberg equilibrium allows one to predict the frequency of alleles, genotypes, and phenotypes, as long as the laws of Hardy Weinberg are met.
ID110	Populations that do not fit the model of Hardy Weinberg equilibrium differ from "natural" evolution. Evolution is showing trends and shifts occur within population. Bell curve changes from standard.
ID091	Haplotype networks allow for the visualization of genetic diversity within a population. The frequency of alleles are shown as well their linkage.
ID091	Binomial expansion is used to predict the frequency of alleles in conjunction with the Hardy Weinberg assumptions. (p+q)^2 where p and q are recessive and dominant alleles is used. p^2 represents the pp genotype, 2pq represents the heterozygous and q^2 represents the qq genotype.  (p+q+r)^2 can also be used where the numbers are similar.
ID091	Hardy Weinberg equilibrium makes assumptions that are usually not completely true. Natural selection occurs over generations favoriting genetically fit organism.  Also, populations migrate, mutate, randomly mate and much more to cause equilibrium to shift.
ID078	It is all the genes on a chromosome that are inherited together
ID078	This tool allows for the prediction of various genetic properties using some simple math
ID078	if a population isnt in hardy weinburg equilibrium, then there is a driving force that will change the frequency of the different alleles
ID055	A relationship between haplotypes. These networks can connect to trace back human origins.
ID055	By using the binomial expansion, we can calculate and predict allele frequencies, genotypes, and phenotypes. Binomial expansion is an algebraic form of calculating these frequencies.
ID055	Hardy Weinberg equilibrium has a list of assumptions that a populations must stay under in order to maintain equilibrium. However, most populations break at least one of these assumptions if not multiple of them. If these assumptions are broken then Hardy Weinberg law is no longer maintain. Meaning that the populations is not an "ideal" population.
ID072	Haplotype networks represent the relationships among the different haploid genotypes observed in a data set.
ID072	by using binomial expansion or by using the pascals triangle, researchers are able to predict the frequency of alleles, genotypes and phenotypes of offspring.
ID072	Popultations that are not in the Hardy Weinberg equilibrium are evolving in different ways due to not being in an environment supported by naturals selection and random mating.
ID059	These are networks similar to a phylogeny where there id a unique sequence of base pairs over a region of the genome that used to identify a specific population and can also be used to show the connections between several populations and generations. With a haplotype one can figure which species are closely related and how close they are.
ID059	Binomial expansion is used to calculate the varying frequencies of organisms and their genetics. It can tell the percentage of alleles, the number of boys vs. girls that a particular couple may produce and it can even tell the probability of how many children a couple may have.
ID059	Those that are not in hardy weinberg equilibrium have violated one or more of the five concepts that leads to evolution. each concept; no mutation, no natural selection, random mating, population is infinitely large, and no migration/immigration.
ID058	Haplotypes are short series of bases that are the exact same between individuals. Haplotype networks are used in order to determine which species are closely related and also what recent evolutionary changes have occurred.
ID058	The equation is p^2 + 2pq + q^2. The equation can be used to figure out both the frequency of the individual alleles p and q as well as homozygous dominant, homozygous recessive, and heterozygous frequencies.
ID058	Populations not in Hardy Weinburg can evolve through mutation, migration, and natural selection. They can also evolve through non-random mating and having only a small population of individuals which results in inbreeding.
ID080	Haplotype networks are sequences in individuals' genomes (usually short) that are used to see how closely related individuals are.
ID080	These frequencies can be used to find the probabilities of the outcome of offspring using factorials or other formulas. I know this is done to determine the outcome of linked genes that don't follow the rules of independent assortment.
ID080	I know the five stipulations that are needed for a population to be considered to be in Hardy Weinberg equilibrium. I know the four ways that a population can evolve and they are gene flow, genetic drift, mutations, and natural selection.
ID042	Haplotype Networks are a way of tracking closely linked genetic markers as they are inherited by progeny.
ID042	Binomial expansion can be used to determine the probability of alleles being passed down in certain situations. For example binomial expansion can be used to solve the probability that out of five children four are female.
ID042	For populations not in Hardy Weinberg equilibrium they could be evolving by several different routes. Some examples include; mutation, genetic drift, gene flow or natural selection. This would mean that at least one of the five assumptions would have to fail.
ID130	Haplotype networks are the mapping of evolutionary lines based on differences in haplotypes.
ID130	Allele frequency is predicted using probability.  Binomial expansion provides a way to calculate that probability.
ID130	Hardy Weinberg equilibrium implies that there is no change occurring in the population.  There cant be anything that causes any sort of variation.  If hardy weinberg is not in effect then there is either mutation, migration, or a selective drive in the population.
ID009	Haplotypes are unique regions in an organisms genome. They can be used for comparison of relativeness between populations. They are a better visual than microsatellites.
ID009	This is important in many ways. Whether to produce more of a certain combination of alleles, or less of one, this can help to do that. The binomial expansion shows probabilities.
ID009	Some of these populations have different requirements than others. In the big scheme of things, it's all about survival of the fittest. Traits that are assets are going to be passed along while the contrary will eventually be weeded out.
ID136	Binomial expansion is a mathematical formula used to make predictions of the frequencies that could come out of a population.
ID136	populations that do not meet the Hardy Weinberg requirements must have an outside or other factor affecting the evolving of the population. Those factors could come from mutations, gene drifting, or from gene flow.
ID145	Haplotype networks can be used to trace ancestry.  Haplotypes are sequences on one chromosome.
ID145	The frequencies predicted can be used to see if a population is Hardy Weinberg equilibrium.  By knowing one or more the frequencies, you can find the frequencies of the other (i.e. knowing genotype and finding phenotype frequency)
ID145	There are very few populations not evolving.  Hardy Weinberg equilibrium is beneficial to determine what type of evolution is taking place.
ID047	A haplotype are closely linked base sequences seen in a genome. You can compare haplotypes among populations to infer evolutionary history
ID047	Binomial expansion is an easy way to expand i binomial expression and get the probabilities and frequencies of alleles, genotypes and phenotypes.
ID047	In Hardy Weinberg equilibrium there is no mutation, no genetic drift, no gene flow, no natural selection. So populations not in Hardy Weinberg are evolving through all these concepts.
ID010	Polymorphisms.
ID010	Binomial expansion used to predict different probabilities depending on how many variances occur i.e. # of births or male/female.
ID010	In the event that no instruments of development are following up on a populace, advancement won't happen - the quality pool frequencies will stay unaltered.
ID011	Haplotype networks are often used to compare how closely related specific populations of a species or multiple species are to each other.  These haplotype networks are often depicted in a diagram, chart, or table showing the genetic similarities between populations.
ID011	Essentially you can use a formula to determine the frequencies of alleles, genotypes, and phenotypes.  This equation can work with multiple alleles and you can often you one allele frequency to find a second one if you are given two alleles.  This formula is related to binomial expansion because the more alleles, the bigger and more complex the formula becomes.
ID011	Populations that are not in Hardy Weinberg equilibrium are experiencing one of four evolutionary methods. These four methods are natural selection, mutation, genetic drift, and gene flow. No random mating also is not in the equilibrium which also causes changes in allele frequencies and evolution.
ID113	a way to track individuals who may share the same haplotype, which have the exact same sequence for this entire region, and are therefore inferred to be closely related to one another
ID113	when knowing the predicted genotypes from binomial expansion, we'll be able to tell what factors are acting on the gene frequency if deviated from
ID113	this shows the different variations of genetic drift when not in an ideal HW
ID077	A haplotype is a set of alleles from closely linked loci carried by an individual that are inherited as a unit. Individuals that share a haplotype are inferred to be closely related to another. Haplotype networks are used to compare individuals haplotypes.
ID077	When 1 of 2 outcomes can occur in a number of trials, the probability of the outcome of several trails can be predicted using binomial expansion. (p + q)^n=1 is the equation. p stands for probability of one event, q stands for probability of another event, and n stands for the number of trials.
ID077	Under Hardy Weinburg equilibrium individuals of all genotypes have equal rates of survival and equal reproductive success (no selection), no new alleles are created (no mutation), no migration, large population, and random mating. If one of these are not met then the population evolves in a different way via individuals from other populations mating or migrating and alleles are created/converted to new alleles via mutation leading to new phenotypes and genotypes.
ID102	Haplotype network is a comparison of population through a haplotype sequence. This network shows which different population could share the same relation. If too short or too long it is impossible to see the relations.
ID102	This is to be true. This is known as the Hardy Weinberg law/theory. It allows the use of a mathematical process to gives us a prediction of the genotypes, etc.
ID102	H-W equilibrium allows us to see the predicted frequencies in genotypes, however, I am not certain on if the equation has the ability to provide whether the populations are evolving in different ways that are outside of the H-W equil.
ID144	The different types of haploid genotypes in populations
ID144	Using the Hardy Weinberg equilibrium we can calculate these things with p and q
ID144	Not all individuals in a population contribute equally to the gene frequency
ID111	a group of genes that someone inherits from one parent.
ID111	Binomial expansion is an equation that helps you to predict the probability of something occurring in various situations.
ID111	In the Hardy Weinberg equilibrium, an allele frequency in a populations stays the same from generation to generation with the absence of evolutionary influences.
ID128	Haplotype networks can be used to compare multiple organisms and show relationships. They also can be used to help define where to draw the line between multiple species which can often be a grey area.
ID128	Binomial expansion is an easy way to how often an allele, genotype, or phenotype will occur in a population. For example, it can be used to determine the likelihood that a couple with 5 kids has two boys and three girls or any combination of the genders.
ID128	For a population to be in Hardy Weinberg equilibrium, the frequency of alleles must stay the same from generation to generation. That means that no evolution can occur because that would mean new or different alleles/frequencies within the population throwing off the equilibrium.
ID034	a haplotype network is a specific group of genes that a progeny inherits from one parent. there are, however, several specific definitions of the term being used in the field of genetics
ID034	if the genes at Hardy- Weinberg equilibrium, the frequency of alleles in the gene pool do not change over time, and the frequency in the gene is calculate by p+q = 1 and p2 + 2pq + q2 = 1 where p= frequency of dominant allele and q= frequency of recessive allele
ID034	mutation, migration, and gene flow can alter allele frequencies
ID146	Haplotype networks can show us relationships between individuals and populations due to distinct combinations of alleles in given sections of genes.
ID146	Under specific circumstances that are inherent in Hardy-Weinberg problems, the p^2 + 2pq + q^2 = 1, can help us identify these values when given one or more of the others.
ID146	Populations outside of these circumstances can be affected by factors such as natural selection, inbreeding, nonrandom mating, mutations, migrations, etc. that change the observed/expected ratios when attempting to calculate these genetic values.
ID138	I honestly do not know much about Haplotype networks. I know that haplotypes are things that tend to be inherited together, but that is about it.
ID138	This is really important, pretty much a fundamental concept of predicting the outcomes of different crosses. I know that there are some set genotype frequencies 1:2:1 and 9:3:3:1 for single and double heterozygote crosses respectively. I know the formula for binomial expansion in both the ways and how to plug in the different variables to solve for the answer.
ID138	Hardy Weinberg equilibrium is based on the idea that, in equilibrium, gene frequencies in a population will stay the same. Pretty much by definition this means that there is no evolution happening. So, if it is not in equilibrium, it would mean that there is some sort of selection/mutation/gene flow/genetic drift happening leading to evolution in some way.
ID029	Haplotype networks are important for understanding how closely related organisms are and how they have evolved.
ID029	The binomial expansion is an equation that can be used to find the probability that certain traits or alleles will be inherited.
ID029	Populations that do not follow hardy Weinberg can evolve due to mutations, genetic drift, gene flow, or natural selection.
ID064	Haplotype networks compare similar regions of different individuals' genomes and illustrate the differences visually with lines and circles representing which genes are different. These networks show with lines how closely related different individuals are.
ID064	We can mathematically predict the frequency of traits in a population if some of the frequencies of alleles or phenotypes are known.
ID064	If a population does not meet the criteria for Hardy-Weinberg equilibrium
ID107	A haplotype is a unique set of gene sequences or set of single nucleotide polymorphisms on the same chromosome. A haplotype network can be used to compare haplotypes in a population and across different populations.
ID107	The Hardy-Weinberg Law can be used to describe allele and gene frequencies in a "ideal" population. If the population is ideal, the population is infinitely large, has random mating, and is not affected by mutation, migration, or natural selection. If the population is ideal, you can use binomial expansion to solve for allele and gene frequencies. To find allele frequencies, you would use the equation p+q=1, where p=the dominant allele and q=the recessive allele. To determine gene frequencies you would square the first equation to get p^2+2pq+q^2=1. P^2 would represent the homozygous dominant gene frequency, 2pq would represent the heterozygous gene frequency, and q^2 would represent the homozygous recessive gene frequency.
ID107	Populations not in Hardy Weinberg equilibrium are not infinitely large and do not always randomly mate. Populations not in HW eq are also susceptible to mutation, migration, and natural selection. (They can be affected by genetic drift, gene mutation, gene flow and natural selection.)
ID132	Haplotype networks are those that contain a set of DNA variations or polymorphisms that are normally inherited together. It can also include a combination of alleles found on the same chromosome.
ID132	Binomial expansion is a way of predicting frequency of alleles, genotypes and phenotypes by using this equation P = n!/x! (n-x)! p^x q^n-x This is an easy way to determine all possibilities for a given set of unordered events.
ID132	The Hardy Weinberg law states what happens to an allele and genotype frequencies in an ideal population that is large and randomly mating. If a population meets the criteria that 1) the frequencies of alleles in the gene pool do not change over time and 2) If two alleles at a locus are considered and after one generation of random dating then they are said to be in Hardy Weinberg.
ID084	Haplotype networks map the relatedness of different haplotypes, or sequences of nucleotide bases (SNPs) common in genomes. These networks can be used to trace populations histories and evolution.
ID084	Binomial expansion is very useful in determining frequencies of alleles in populations. We know that p is related to the dominant allele, and q represents the recessive one. From frequencies of alleles, we can determine genotypes and phenotypes.
ID084	For populations to be in Hardy Weinberg equilibrium, they must meet several assumptions. We know that if populations are not in equilibrium, some outside force must be working on the population. Some of these assumptions are mating is random (and sexual), generations do not overlap, population size is infinitely large, and genetic drift or mutation does not occur.
ID073	A haplotype is a set of alleles inherited by an individual from a single parent. By comparing haplotypes of a mother and father with those of a fetus, scientists can study how new genetic changes arise.  Haplotype networks are the genetics of a population and can show certain genetic components of the population.
ID073	Number of gametes = 2 (alleles # - gene #)
ID073	CHanges in the environment such as  genetic drift, genetic flow, natural selection and mutation cause variation from the Hardy weinberg equilibrium.Physiological response to the environment operates at two levels they are long-term evolutionary changes that characterize all individuals within a population or species. And there is also  short-term  physiological responses that are  called acclimatization.
ID140	The set of several SNPs on a chromatid that are associated together.
ID140	Using binomial expansion, the frequency can be determined by expanding out the equation, choosing the one that fits the predicted situation and calculating the probability of it occurring.
ID140	Hardy-Weinberg has very specific parameters which would have everything evolving the same way over time. Because of variation and limited mating pools, populations evolve very differently.
ID021	Haplotypes show the interconnectedness of haplotype networks. They can be used to establish the extent of relationships. The network looks like a sentence diagram and maps out the likes and differences between genomes.
ID021	Frequency is predicted using the equation (p+q)^2. p is the dominant allele. q is the recessive allele. The equation can be expanded to give the number of homozygous and heterozygous individuals in the population by the equation p^2 + 2pq + q^q = 1
ID021	If a population does not fit the above equations, then it breaks one or more of the assumptions of h-w equilibrium. These are no mutation, no migration, no gene flow, infinite population, and random mating.
ID035	You use the HW to find out the expected and predictions but then you can use the Chi square test to determine if the frequences are consistent or not with the results.
ID035	Determining the frequency of alleles and genotypes with sex linked genes is by using q^2 and p^2
ID035	Generalized transduction is usually mediated by certain lytic viruses. Specialized transduction results in the movement of only specific genes. The viruses that carry out specialized transduction are called lysogenic viruses
ID044	chi square tests are used to determine if there is any difference between the observed experimental values and the expected theoretical values. It can be used to check whether deviation from expected frequencies is larger than expected by chance or by error. If the deviation is larger, then you could try to determine which of the Hardy-Weinberg assumptions is being violated.  This would be important to use to come up with the correct frequencies.
ID044	You can determine the frequency of alleles and genotypes by sex linked genes by using the equation p2+2pq+q2=1 for females and p+q=1 for males. the male equations gives you the phenotype and the allele frequency.  If you know the frequency of recessives (q2), you can determine the frequency of dominant and heterozygous by using the two equations.  therefore, determining allele frequency and genotypes using sex linked genes is simple by applying the hardy-weinburg equation.
ID044	In generalized transduction, a large number of bacterial genes are transferred.  Bacterial DNA is packed into phage particles and transferred to the host cell.  In Special sized transaction, only a few bacterial genes are transferred and strain-specific genes are transducer instead of a wide variety of genes. in both, viruses may be used to transfer the genes because viruses are able to invade the host cell and then force the cell to replicated and produce it's genes.  By doing this, the bacterial genes transferred will then be produced by the cell.
ID150	No population is in Hardy-Weinberg equilibrium, so by using the chi square test, frequencies can be compared. The degree of difference between the expected and actual frequencies can be determined.
ID150	The effect of sex linked genes in populations can be determined using the same methods that frequencies in families uses.
ID150	Bacteria gain genetic diversity through 3 methods, one of which is transduction. Viruses can part of bacterial chromosomes attached to their own. When they infect a host cell, they can pass this bacterial piece onto the host chromosome.
ID013	The chi square test is used to determine if one criterium of the five characteristics of the Hardy Weinberg equilibrium is being violated. Using the degree of freedom, x squared is calculated. If x squared is less than 0.05, then the hypothesis is rejected and the deviation is not due to chance.
ID013	Allele and genotype frequencies can be found for sex linked genes as well. The Hardy Weinberg equilibrium is used, but is different for males. Females use p squared + 2pq + q squared while males use p + q =1.
ID013	Transduction is a form of genetic recombination using viruses. In generalized transduction, only bacterial DNA is added to the phage head and thus bacterial DNA is transferred to a new bacterium. In specialized transduction, only a few bacterial genes are transported along with the viral DNA, so the recipient cell only receives some bacterial genes.
ID000	Chi square tests ask whether an observed outcome fits an expected outcome.  In the context of genotype frequencies in a population, the null hypothesis is that the genotypic frequencies are in Hardy Weinberg equilibrium.
ID000	In men, population allele frequencies are the same as genotypic frequencies because men have a single X chromosome (no homologue).
ID000	Generalized transduction involves infection of a bacterium with a virus, fragmentation of the bacterial chromosome and the reformation of the viral heads but some heads form around a piece of the bacterial DNA which can now be transferred to a new bacterium by infection and incorporated by recombination.  Specialized transduction occurs when a prophage such as lambda excises incorrectly and the excision includes a piece of the bacterial chromosome.
ID037	The Hardy-Weinberg Equilibrium can be used as a null hypothesis, compared to values from a real population, to describe statistically significant deviations from the Equilibrium. If the deivation is significant, then the gene frequencies are changing and thus, evolution is occurring.
ID081	In a chi square test you figure out the (observe-expected)^2 / expected. After this is found, one compares the value with the degrees of freedom in mind to the value on the chi square chart. This is then compared to the value of 0.05. If it is greater than 0.05, we reject the hypothesis. If the it is less than 0.05, we accept the hypothesis. When dealing with Hardy Weinberg, our hypothesis is that the populaiton is at Hardy Weinberg equilibrium. If we reject this hypothesis, we know that one of the Hardy Weinberg assumptions has been broken, so mutation, selection, migration, a small population, or non-random mating has occurred.
ID081	For males, most genes are found on X chromosomes. As a result, their allele frequency (p or q) is equal to their genotype. However, females allele frequency (p or q) is not equal to their genotype. Their genotype is equal to p^2, 2pq, or q^2.
ID081	Generalized transduction occurs when bacterial DNA is packaged into phage particles and transferred to the host. It is then used in chromosomal mapping. This transfers a large number of bacterial genes. Specialized transduction occurs in strain-specific genes. This transfers few bacterial genes.
ID098	In general, chi square analysis is used to compare expected values and observed values. This is done in order to determine the likelihood that the observed values are due to chance. When this likelihood is low, then the values are considered statistically valid. In this particular example, the allele frequency predictions from the Hardy-Weinberg equation are the "expected" values. The observed values are compared to these predictions, and a chi square test can be used to determine how statistically valid the observed values are.
ID098	Because males and females have different sex chromosomes, calculating the frequency of alleles of sex linked genes is different than autosomal genes. When calculating the frequency in female humans, you use the standard p2+2pq+q2=1 formula. This is because females have two X-chromosomes. However, in males, who have only one X chromosome, the formula is simplified to p+q=1. Therefore, it is crucial to consider which half of the population you are measuring.
ID098	When viruses invade a cell, they overtake the DNA replication process and begin copying their own (viral) DNA. This extra viral DNA is then repackaged into new viruses and expelled from the cell. However, sometimes during this repackaging phase of viral infection, the new viruses pick up bacterial DNA. They can then carry this bacterial DNA from one bacterial cell to another, where it can be integrated into a new bacteria's DNA.
ID045	The chi square test is used to determine the consistency of the Hardy-Weinberg predictors. Since it uses a chart and a p-value, it is typically a safe estimation of random allele frequencies, given that all the necessary standards are present.
ID045	The frequency of alleles and genotypes with sex linked genes deals with mainly how the male and female have different sex chromosomes so the recombination and allele frequency is different from normal chromosomes.
ID045	Transduction and specialized transduction to exchange bacterial genes by viruses is how the bacteria are able to pass on certain genes to other bacteria or how allele frequencies are able to be different.
ID058	Chi squared test is used to compare actual results to the expected results so it allow one to compare actual allele frequencies in a population to one's that Hardy Weinberg would predict. This would help establish if a population was in Hardy Weinberg equilibrium or not.
ID058	The Hardy Weinberg equation can also be used to predict allele frequencies for sex linked genes. This is very important  because many sex linked disorders are very destructive. It is important to try to avoid these disorders from occurring in offspring and to be able to study how allele frequencies for certain diseases vary among populations.
ID058	Scientists were confused when placing two auxotrophic bacteria species in a U tube on separate sides of a filter that some bacteria when plated were prototrophic even though conjugation couldn't have occurred because the bacteria only shared media. They discovered when reducing the filter pore size to not allow bacteriophages that none of the bacteria were prototrophic. This is how transduction was discovered.
ID109	Since the Hardy Weinberg equilibrium is 9:3:3:1, we can tell that anything that doesn't match the ratio is inconsistent with the Hardy Weinberg prediction. For example, we can use a chi square test with members in a population to see if the allele frequencies are either consistent or inconsistent with the Hardy Weinberg prediction.
ID109	Sex-linked genes occur at different frequencies in different sexes. For example, X-linked traits are more frequent in males almost never occur in females because it is harder for the female to get two copies of a "defective" gene.
ID109	Generalized transduction is where the bacteria DNA is packaged into phage particles and transferred into the host. Specialized transduction is where strain-specific genes are transduced and a few bacteria genes are transferred.
ID077	Chi square analysis evaluates the influence of chance on genetic data. Outcomes of independent assortment and fertilization are subject to random fluctuations from their predicted occurrences as a result of deviation. Hardy Weinberg is contingent upon 5 factors: no mutation, large population, random mating, no migration, and no genetic drift. Chi square test can be used to determine whether or not a population meets these terms.
ID077	In species where sex is chromosomally determined, females have two X chromosomes while males only have one. In females, gene frequencies are the same as for any other locus. In males, frequencies of the genotypes are the same as frequencies to the alleles in the population. Males = p + q and Females= p^2 +2pq + q^2. One can use the Hardy Weinberg Law to determine the frequencies of the alleles and genotypes with sex linked genes.
ID077	Generalized transduction is when bacterial DNA instead of phage is packed into phage particles and transferred to host. Generalized transduction can transfer a large number of bacterial genes. It is used in linkage and chromosomal mapping. Specialized transduction involves strain-specific genes that are transduced and transfers few bacterial genes.
ID099	Chi square is a tool used to check the accuracy of a common method of checking allele frequency in populations
ID099	Determining the frequency of sex linked genes follows many of the normal steps for determining allele frequency but it with the added element of sex linkage
ID099	General transduction is effective on a large scale though cannot be as specific for unique organisms and genes
ID129	If the deviation from the frequencies are different than what is expected by chance, then researchers study which of the Hardy-Weinberg assumptions (no migration, no mutation, random mating, infinite population, and no selection) are being violated. This is done by calculating the allele frequencies from the genotype frequencies then using the allele frequencies in the parental generation to predict the genotype frequencies of the offspring. If the frequencies are not relatively close to the frequencies observed in the parental generation, then there is evidence that Hardy-Weinberg is being violated, and chi square can be performed.
ID129	Since females have 2 X chromosomes, gene frequencies are the same for any other locus. Since males only have one X chromosome, genotype frequencies are the same as the allele frequencies in the population. Males: p+q=1 Females: p^2+2pq+q^2
ID129	Transduction is when a virus replicates bacterial DNA from one bacterium to another through viral replication cycles. Generalized transduction is when instead of a phage, bacterial DNA is packaged into phage particles and transferred to the host. This transfers a large amount of bacterial genes. In specialized transduction, transfer of bacterial DNA is not random and only strain-specific genes are transduced. This type of transduction only transfers a few bacterial genes.
ID014	The Chi Square value along with the degrees of freedom can help you determine the statistical p-value between the different groups tested. The p-value will help determine if evolution did occur or if the population is still in equilibrium.
ID014	The benefit of determining the frequency of alleles and with sex linked genes is because we know that the probability to females will always be q^2. Also, the frequency of the X-linked allele in the gene pool and the frequency of males expressing the X-linked trait will always be the same. So if the frequency of one is known then the frequency of the other can always be determined.
ID014	Transduction is when bacterial DNA is packaged into a phage particle and then transferred to a host. Specialized transduction is when the bacterial gene is donated to only a few bacterial genes and the genes are strain-specific. In generalized transduction, there is a large number of bacterial genes transferred. The generalized transduction is also used in linkage and chromosomal mapping techniques.
ID027	The chi square test is a statistical tool used to determine the likelihood of something happening due to chance. If the likelihood is less than 5%, then the populations violates H-W.
ID027	The frequency of alleles in males with X- linked chromosomes is equal to the frequency of the corresponding phenotypes since they only have one set of the chromosome. Females are calculated normally.
ID027	Virus DNA can integrate itself into the host bacteria. When it separates, it sometimes pulls some of the bacterial DNA with it and carries it to the next cell the virus infects.
ID094	Chi square tests can explain deviation from ideality in Hardy-Weinberg and explain how this might effect frequencies.
ID094	Some genes are called sex-linked because they are only expressed in one sex or the other; this ties one trait directly to another and therefore provides a tool for following the behavior of some genes.
ID094	As viruses are diverse, their mechanisms of propogation are diverse, so as some evolve their methods differ.
ID085	The chi square test can be used to indicate whether variations in Hardy Weinberg equilibrium are due to chance, or whether they are actually due to the population changing, therefore indicating that it is not in HW equilibrium.
ID085	You can determine the frequency of alleles and genotypes with sex-linked genes because of non-random mating. Using the coefficient of inbreeding, we can find the probability/frequency that the two alleles of a gene in an individual are identical. They would be identical because of a common ancestor.
ID085	Transduction is virus-mediated DNA transfer in bacteria. In generalized transduction, the transduction is random, meaning each part of the bacteria's chromosome has a chance of recombining. In specialized transduction, the transfer of bacterial DNA is not random. Only genes specific to the strains of virus are transduced.
ID046	the Chi Squared test involves calculations with expected and observed values as well as degrees of freedom, which is one minus the number of cases. Hardy Weinberg frequencies involve 5 factors: no mutation, no migration, infinite population, random mating, and no selection.
ID046	For women, the formula is p^2+2pq+q^2 = 1, while for men it is simply p+q=1since men only have one x chromosome.
ID046	The F- is the recipient while the F+ is the donor. Phages carry viral DNA as well as bacterial DNA and transmit the genetic information to other bacterial cells.
ID104	Based on the data and allele frequencies, we can determine whether the two alleles are completely independent from one another. If the two alleles are independent then they will follow the Hardy Weinberg predictions.
ID104	If we can determine the frequency of sex linked genes then we can then use that to help map the sex chromosomes.
ID069	If the chi square test produces results inconsistent with Hardy Weinberg predictions, it signifies that the population did not adhere by one or more of the following Hardy Weinberg stipulations: a large population, random mating, lack of mutation, and equal survival and reproduction rates. This variance occurs because the population may be affected by one or more possible factors, including mutation, genetic drift, gene flow, or natural selection.
ID069	The frequency of alleles and genotypes with sex linked genes can be determined using a variance on the Hardy Weinberg equilibrium equation. For females (XX), the p^2 + 2pq + q^2 = 1 equation can still be used. For males (XY), however, only p + q = 1 can be used.
ID069	Lytic phages mediate generalized transduction, in which the virus' DNA takes over and destroys the cell. Lysogenic phages mediate specialized transduction, in which the virus' DNA merges with the cell's existing DNA.
ID132	This test can be useful when seeing if the allele and genotype frequencies are in Hardy Weinberg equilibrium. Like if we have a series of genotypes that counts from a population, then we can compare these to the ones predicted in the Hardy-Weinberg model.
ID132	This can be a little difficult because males only have one single X-linked alleles but for X linked alleles in females the Hardy Weinberg frequencies are the same as the autosomal loci.
ID132	Generalized transduction is when any bacterial gene can be transferred. Hence the general in the name. It transfers through a bacteriophage and normally only carries bacterial DNA.
ID063	We are able to take the numbers from a population and plug them into the chi square analysis. Then we are able to use the results to see if evolution is occurring. If evolution is occurring then we can say the populations are inconsistent with the HW predictions.
ID063	Through this we must use p + q = 1 for males and p^2 + 2pq + q^2 = 1 for females. Males cannot be heterozygous for a sex linked trait as they only have 1 X chromosome. Females can be a carrier as they have 2 X chromosomes.
ID063	Bacteria undergo 3 types of genetic transfer. Transduction is one of those types. It is an exchange of DNA through bacteriophage.
ID089	By calculating the chi squared value based on observed vs expected frequencies and then using the degrees of freedom to determine the p value, one can detect whether allele frequencies in populations are inconsistent with hardy weinberg predictions.
ID089	Because sex linked genes, specifically those on the x chromosome, will be fully expressed in the male population, it is easy to determine the frequency of alleles and genotypes with x linked genes because all males will have the same phenotype as genotype.
ID089	Viruses can enter into a bacteria and lyse the DNA while using the machinery in the bacteria to produce more parts of viruses. The heads then take up DNA thats in the Bacteria, some of which is bacterial DNA, some of which is viral DNA.
ID131	The Chi square test is used as we did earlier in the semester to identify if genes' frequency remain consistent over generations. If the tests do not fall within a specified range of the chi square test, then an allele frequency is not in Hardy Weinberg equilibrium.
ID131	Using this we can determine how often you will see a certain allele withinin a population. The formula used is p^2 + 2pq + q^2 = 1 because you just need the probability of one thing since they equal 1.
ID131	Transduction is when a virus infects a bacteria and then more viruses are set free, taking some of the bacteria's DNA with it and then giving it to another bacteria. A specialized transduction would be when specific genes are transducted.
ID125	This algorithm allows us to analyze the data and it showcases the inconsistencies with Hardy Weinberg.
ID125	In a population we are able to determine how frequently alleles and genotypes occur due to them being sex linked.
ID125	Generalized transduction is usually mediated by certain lytic viruses. A lytic virus is one that normally infects a host cell and redirects the resources of the host away from its own cellular replication toward viral replication and the eventual lysis, or breaking open, of the bacterial cell.
ID033	The chi square test validates how much a statistic is random or if there is another factor affecting it.
ID033	We use Hardy Weinberg equations to determine the frequency of alleles.
ID033	Bacteria can either pick up plasmids from the environment or they can be shocked into the cell.
ID095	We know that Hardy Weinberg equilibrium represents a population that hasn't undergone any type of selection.  This population must also be large enough to sample its data and other factors can't affect it, i.e., migration, mutations, gene flow, genetic drift.  When we apply the chi square test, we are able to observe the expected population and it's gene frequencies and compare this data against what is observed.  This ratio can help us determine if the factors that deter Hardy Weinberg equilibrium exist in this population.
ID095	Sex linked genes exist on sex determining chromosomes, and, while only comprising 2 chromosomes out of the 23 pairs in humans, these chromosomes also contain genes that don't define the sex of an individual.  We know that color blindness is an x linked recessive trait.  Since women have two x linked genes, the likelihood of her being homozygous for this recessive gene is significantly low.  Men, only having one x chromosome, have a much higher risk of being colorblind.  We can also apply binomial theorem to sex linked traits.  p^2 + 2pq +q^2 = 1; this equation represents the ratio of individuals in a population and the variables are their genes, e.g., p is allele A, and q is allele a.  Since men have only one x chromosome, we can use the square root of this equation, and the q frequency will actually represent their frequency of either the dominant or recessive x chromosome.
ID095	Bacteria have a long history of mutating or developing resistance to antibiotics.  One way of achieving this differentiation is to receive chromosomes from an outside source.  When a bacteria receives chromosomes from a bacteriophage, we call this process transduction.  Generalized transduction results when a bacteriophage directly injects its prophage into the host cell and the cell takes up this material into its genetic code.  Specialized transduction is when a bacteria already has a prophage in its genetic code, and this prophage gets transferred to another bacteria.  We see this during the lysogenic cycle.
ID057	The chi square is used to test the goodness of fit and help you determine the probability of your results or the results less likely. This is helpful with the idea of Hardy Weinberg's predictions allele frequencies remaining constant from generation to generation. The chi square can be used to test the principle of Hardy Weinberg.
ID057	Recessive X-linked traits are more common in males than in females due to the fact that males only have one X chromosome where as females have two X chromosomes and can just be carriers. The frequency of these alleles in the males with depend on the frequency of alleles in their mothers.
ID057	Generalized transduction occurs at random where as specialized transduction is specific DNA segments. In specialized transduction specific DNA segments are integrated into the host chromosome. When the viral DNA integrates with host DNA it's called prophage. This prophage goes into replication along with the bacterial genome. In generalized DNA recombination between the injected DNA and the recipient chromosome occurs. This also may not integrate the DNA segment into the bacterial chromosome.
ID015	Hardy Weinberg predictions are just that, predictions. A population will never be in Hardy Weinberg equilibrium because there will always be outside forces acting on the population.
ID015	In producing gametes, each genotype passes on both alleles that it possesses with equal frequency, therefore the frequencies of A and a are also p and q.
ID015	In generalized transduction, a piece of host DNA gets packaged by mistake, instead of the phage DNA.This rare event results in a phage delivering only bacterial DNA to the next host.  The DNA then recombines homologously, replacing the host allele.
ID103	The Chi squared test can be used to determine the probability that allele frequencies vary from those predicted by Hardy Weinberg are the result of random chance.
ID103	For females the normal Hardy Weinberg rules apply.  For males they only have one copy of the allele so the frequencies of genotypes in males will be the same as the allele frequencies.
ID103	Viral transfer of bacterial genetic material is an important source of genetic diversity in bacteria. The process is called transduction.
ID052	When the results of a chi square test indicates that a population's allele frequency is inconsistent with Hardy Weinberg predictions, then at least one of the following must be true: 1) The population is small, thus leading to genetic drift. 2) The population is not isolated, thus causing gene flow. 3) Mutation occurs, thus introducing new alleles into the population. 4) Natural selection occurs, thus expressing the more "fit" trait.
ID052	The phenotypic expression frequency of certain sex-linked genes can be predicted due to whether the gene is located on the X chromosome or the Y chromosome. Traits can be either X-linked dominant, X-linked recessive, or Y-linked. If a mother with a X-linked dominant trait and a non-affected father mated, then 50% of her offspring would be affected with that trait. However, a father with a X-linked dominant trait passes this trait to all of his daughters. His sons do not receive this trait, because they inherit their father's Y chromosome.
ID052	Transduction is responsible for the diversity seen in bacteria. Generalized transduction is most common. This occurs when a phage inserts its viral DNA into a host cell. The viral DNA uses the host cell's cellular machinery to replicate its viral DNA, which is then packaged into virus capsules. A small percentage of that host cell's bacterial DNA are also packaged into capsules. This packaged bacterial DNA becomes the transducing phage. When a phage injects this DNA into a new host cell, part the DNA may be integrated into the bacterial chromosome via recombination. Specialized transduction occurs when a small part of the bacterial DNA remains attached to the phage DNA when transduction occurs. A piece of the phage DNA is left attached to the bacterial chromosome.
ID079	Chi square analysis mathematically calculates how well a null hypothesis fits the numerical results. The Hardy Weinburg equation provides data for the alleles in a population that is not evolving. Performing a chi square test can tell researchers how far a population deviates from the ideal population.  A large deviation indicates that the population encounters mutation, migration, natural selection, mating is non-random, and the population size is not infinite.
ID079	Because males are heterogametic, the number of males in a population that express an x-linked trait will be equal to the frequency of x-linked alleles in the population. Because females are homogametic, q^2 in the Hardy-Weinburg equation will calculate the number of females homozygous for a specific x-linked allele.
ID079	Transduction occurs when bacterial DNA is transported by a bacteriophage. In generalized transduction, a large number of randomly selected bacterial genes are transported to a different host bacterium. Specialized transduction is the transfer of a small number of genes that are specific for one strain of bacteria. Both types of transduction produce genetic diversity in the population.
ID026	The chi square test can be used to see if there is some kind of mistake in how a population is sampled or if the population is inconsistent with the Hardy Weinberg predictions. A value is given based on the number of degrees of freedom and depending on what number is calculated it can tell if the data is reliable.
ID026	Some genes are transferred on the X or Y chromosome, influencing certain sex offspring to have a particular trait in some cases. If a trait is passed down on the Y chromosome, only males can have it.
ID026	Bacteria can use viruses to take some of their own genetic material and insert it into other bacteria through phages. This can increase diversity among a population and spread genes.
ID021	The chi square test uses statistical analysis to determine if populations are evolving, or do not meet Hardy-Weinberg assumptions. The degrees of freedom is matched with the probability value for that set of conditions. The degrees of freedom is the number of classes minus one. If the p-value is less than 0.05, then the null hypothesis is rejected and H-W conditions are not met.
ID021	Sex linked genes are different from autosomal genes. Sex linked genes are either on the X or Y chromosome. Males get XY and Females get XX. Sex linked genes are usually on the X chromosome because most of the Y chromosome is not expressed. Males are at a higher risk of receiving sex linked genetic diseases because they only receive one X.
ID021	Generalized transduction happens when random fragments of bacterial DNA is packaged along with the bacteriophage head and then recombined. Specialized transduction occurs when the repackaged bacteria DNA is not random, but strain specific. A larger number of bacterial genes are transfered in generalized than specialized transduction.
ID068	Chi square tests can be calculated to determine if the difference in the frequencies of an actual population vs a population in Hardy Weinberg equilibrium are due to chance or not. When the deviation shows significance we can determine what is different than a population in Hardy Weinberg equilibrium.
ID068	We can determine allele/genotype/phenotype frequencies of sex linked genes, and diseases cause by them, using the equation p^2 + 2pq + q^2 according to Hardy Weinberg equilibrium. From this, we can tell that since females are XX and males XY, genes causing a disease on the X chromosome will have a higher frequency in males since they only have one X and nothing can mask one recessive allele. Because of this, if we have the frequency of q in the male, it is also the frequency of q^2 (and the same for p & p^2).
ID068	Transduction is donation of bacterial DNA to another by using a virus to carry the DNA. Generalized is the transduction of any genes of bacteria DNA, specialized is only a certain set of genes from the bacterial DNA.
ID101	The Chi square test is used to determine how well the data fits test a particular theory's fit to that data set.
ID101	The frequency of alleles and sex linked genes is important in understanding the inheritance of traits that are sex linked. By the trace of genes, it can be determined the probability for some, such as males with only one chromosome, the likely hood of having a disease.
ID101	The use of viruses to unintentionally add genes to a host. This occurs for viruses that typically let their genes remain in their host.
ID029	The chi square test uses observed phenotype as compared to expected phenotype. If the p value is greater than .05, then hardy Weinberg does not hold.
ID029	By using the binomial theorem or chi squared test, you can predict outcomes for crosses that involve sex-linked traits.
ID029	Specialized transduction involves a virus injection its DNA into the bacterial cell, where the viral DNA incorporates into the bacterial chromosome. Then new phage's are transcribed and burst out of the cell. Generalized transduction works the same way except the virus+bacterial chromosomes breaks apart into fragments that are replicated. This way each new phage is genetically different.
ID022	In order for allel frequencies to follow Hardy Weinberg predictions there has to be no mutation, no selection, random mating, infinte population size, and no migration or gene flow. Thus, if all of this is followed then the results can be predicted. However, even if a single thing changes there could be different results. Therefore, Chi square test- a statistical method for calculating the goodness of fit between observed value and expected value. The chi square test can be used to calculate data, is the variation in data due to chance or is it due to one of the variables that we are testing. Then based on the critical value and degree of freedom, one can determine weather the data is due to chance or due to variables. As well as they can determine weather the data is consistent with Hardy Weinberg predictions.
ID022	We can calculate the frequency of alleles and genotype for a sex linked gene/traits in Hardy Weingberg population. In humans, the female is XX and male is XY, the frequency of x linked alleles and the frequency of male expressing those x linked traits would be same. This is because each male has only one x chromosome and thus an male receiving x chromosome would be equal to the genotype and allele frequency in population. For example, color blindness is female is 1/1250 =0.0008. q2=0.0008 and q= 1/34. It means 1/34 women have colorblindness.
ID022	In generalized transduction, a small segment of DNA is carried from the bacterium by a bacteriophage or virus and then inserted in another bacterium. After this happen, an enzyme is produced that breaks down the host DNA into smaller fragments. Then phage DNA is replicated and coat and phage proteins are produced. During formation of the phage few of the phage cell have bacterial DNA instead of the phage DNA and then they are carried to other bacterium causing little genetic diversity. Specialized transduction involves the transfer of only specific genes using the phage form one bacteria to another. The lambda phage, that infects E. coli is a good example.  in doing this, some part of the phage DNA is left in bacteria and some of the bacterial DNA is taken away with phage DNA. this is rare though. However, due to this when the phage infects other bacteria the older bacteria's DNA is transfer to it and thus causes genetic variation.
ID082	The chi square test is an important statistical tool that is crucial to determining if results are statistically significant. The prevalence of certain alleles can be calculated for specific populations, according to the Hardy Weinberg principle. A practical use of the Hardy Weinberg principle is that it allows us to make predictions about allele frequencies in populations. The chi square test allows us to compare predicted allele frequencies with observed allele frequencies in a population. This  is important because predicted patterns based probability need a way to be "tested" and verified as meaningful.
ID082	With sex linked genes, such as X-linked genes, we use a slightly modified equation in evaluating the allele frequencies and genotype frequencies via the Hardy Weinberg principle. If an allele is X-linked, then a female will have twice the number of alleles or copies of the gene than males in the population since males only have one copy of the X-chromosome. By doubling the allele frequency for females, we can account for this discrepancy. As far as genotype frequencies, males are hemizygous since they only have one copy of the X chromosome with no corresponding homolog. While females in the population will still be homozygous or heterozygous, the hemizygous males will have only one allele (no combination) of the X-linked gene.
ID082	Transduction describes the transfer of bacterial genetic material from one bacterium to another via viruses. When a virus leaves a cell that it was previously infecting, it can sometimes take some of the genetic material from the bacteria along with it, incorporated into its own viral genome. When the virus goes on to infect another bacterium, it inserts its viral genome as before, but now that viral genome also contains genetic material from the previous bacterium that will also be transferred.  Generalized transduction refers to situations where the bacterial genetic material is floating in the cytoplasm, and all the genes have the same random  probability that they will be incorporated into the phage head.  The chance that certain bacterial DNA will be recombined through transduction via the virus is random. Specialized transduction refers to situations where only strain specific genes are transduced via the virus to the bacterium, meaning this type of transduction of DNA is not random.
ID083	By using the chi squared test, we are able to determine whether or not a population is in HW equilibrium. If it is not, then we know that migration, mutation, or selection must have occured
ID083	HW law can be used to determine the frequency of sex linked genes. In doing this however, we must remember that in some systems females have 2 X chromosomes while males have an X and a Y.
ID083	I understand that transduction is the movement of bacterial DNA from one bacterium to another via a viral vector.I am not quite sure what the difference between generalized and specialized transduction is.
ID115	Chi square test can be used to identify whether or not deviations from the Hardy Weinberg equilibrium can be significant and identify whether or not evolution is occurring within a population
ID115	Using the hardy weinberg equation p^2+2pq+q^2=1, the frequency of alleles and genotypes within a population can be determined for sex linked genes. If we know the frequency of a sex linked disorder for one sex of the population, it can be determined for the frequency in which it will occur for the other.
ID115	Because viruses that are injected into a host cell can carry virulent DNA that has been incorporated into the cell, they can also carry genes of the host cell to other bacteria they infect once lysis occurs. In generalized transduction, bacterial DNA is taken up by bacteriophages and not virulent DNA. In specialized DNA, prohages perform a less efficient transduction that involves lysogeny of the DNA, where the host lives symbiotically with the phage until it enters a lytic phase and the DNA is excised with the help of other bacteriophage factors.
ID007	Because there is always variation in populations due to chance, it is necessary to use the chi square test to determine whether any deviation from equilibrium is in fact due to some evolutionary process or whether it is only due to chance.
ID007	Because sex-linked genes are not in a homologous pair in males due to them having only 1 X chromosome, the normal definitions of p, q, p^2, and q^2 do not apply.  The genotype and the allele frequency are functionally the same since males only receive one allele.
ID007	Transduction occurs when a virus transfers bacterial DNA from one bacterium to another.  In generalized transduction, this occurs because the virus picks up some of the hydrolyzed bacterial DNA instead of or in addition to its own.  In specialized tranduction, when the viral DNA excises itself from the bacterial chromosome, it takes some of the bacterial DNA with it.
ID143	The chi square test is used to detect allele frequencies by comparing the probabilities of each allele occurring within a population.  The chi square test can be inconsistent with Hardy Weinberg principles because Hardy Weinberg equilibrium states that there must not be any extraneous factors that influence the frequency of certain alleles within a population.  As long as there are no other factors affecting the population, allele frequencies can be determined using chi square tests can be utilized by scientists to see how many people within a population will likely inherit particular genes.
ID143	Sex linked genes are those that do not occur within the autosomal region of the chromosome, but they are strictly placed within the sex chromosome; however, the frequencies of these alleles and genotypes can be determined using probability and observing populations.
ID143	Transduction is the transfer of genetic material among cells that occurs through viruses within these cells.  Bacterial exchange is an outcome of this process.  Generalized transduction occurs simply, through a bacteriophage or a phage, through a segment of DNA.  Furthermore, specialized transduction occurs in a more complex way, depending on where the phage is located within the chromosome.
ID149	When using the chi square test, if the value on the chart is a below a certain value, we do not consider the outcome to be related to chance.
ID149	This is different than autosomal genes because the males only have one X chromosome, so they only need one recessive allele to have a certain phenotype.
ID149	I can't explain this topic. I haven't read that far into the book.
ID060	Chi square test is a method used to compare frequencies that have been observed and expected to be of some type. Since we cannot assume a significant difference in frequencies just my looking at them we can use the chi square test to prove it.
ID060	If we have want to know the allele frequency of a recessive gene for color blindness and we get that 8% of males have it then we can use it in the Hardy Weinberg equation to determine the frequency in females.
ID060	When viruses get injected into the bacteria, they can bring about a process known as generalized transduction where the phage breaks down parts of the bacterial DNA chromosome into fragments and parts of the fragments are attached to the phage's head. After that, the phage can then generally transfer the fragment to another bacteria.
ID053	Chi square tests are vital in determining the validity of experiments.  If observed allele frequencies in a population are different than what is expected in a Hardy-Weinberg equilibrium population, one can use a chi square test to determine what the probability is that the population is not in Hardy-Weinberg equilibrium.
ID053	Determining the frequency of alleles and genotypes with sex linked genes is the same as that of autosomal genes except with a slight variation. p^2 + 2pq + q^2 = 1 is used for X linked genes in women to find genotype frequencies, and p + q=1 is used to find allele frequencies. For males, only p+q+1 is used because males can only have a single X, so the allele frequency is the same as the genotype frequency.
ID053	Generalized transduction is associated with the lytic cycle of viral infection, where the viral DNA/RNA is used by the cell ribosomes to create more prophages and eventually destroying the cell.  Specialized transduction is associated with the lysogenic cycle, where the virus injects its DNA into the bacterium, merging it with the bacterial DNA, ultimately not having much effect on the bacterial cell.  Generalized transduction can exchange bacterial genes between bacteria, but usually they are lysed.  Specialized transduction is much more likely to help exchange bacterial genes among other bacteria.
ID105	Using the chi square test you can find out if the allele frequency is not 1 to 1 which would mean they are not in Hardy Weinberg equilibrium.
ID105	By using the p + q for males and the p^2 + 2pq +q^2 you can determine gene frequencies in a population.
ID105	If a virus goes through generalized transduction it will exchange genes for the bacteria. However, in the process of doing this it has to lyse which destroys the host. In specialized transduction the virus doesn't have to lyse so the host survives.
ID061	Using the chi square test allows a person to compare the observed allele frequencies in a population to the allele frequencies that were expected for the population. If the value yielded by the chi square test is greater than the value at the .05 probability level corresponding to the degrees of freedom for the population in consideration, then it can be inferred that the population is not consistent with the predictions. However, if the chi square value were less than the number taken from the table, then there is a good chance that the frequencies are consistent with the predictions.
ID061	The frequency of alleles and genotypes with sex linked genes can be found by analyzing the offspring of a parental cross. If all males have the same phenotype for the gene, then it can be assumed that the mother if homozygous for an allele. If the males have different phenotypes, then the mother is heterozygous. If all daughters produced have the same phenotype then it is likely that the father either has the same allele as a mother who is homozygous for the allele or has a recessive allele while the mother is homozygous dominant for the allele. However, if there are different phenotypes in the daughters then it can be assumed that the mother is heterozygous for the allele.
ID061	This takes place when phage DNA is injected into a cell, then proceeds to destroy the host DNA. Replication synthesis of phage DNA will occur and cause phage proteins to assemble. These phages will mature and then proceed to infect other cells and integrate bacterial DNA into the recipient chromosome. Generalized transduction occurs when the packaging of DNA fragements and the genes that are transduced is random. Most cases of this are abortive. Meanwhile, in specialized transduction the transfer of bacterial DNA is not random, but instead, only strain-specific genes are transduced.
ID090	This is extremely important since it allows us to determine if the observed allele frequencies are in agreement with Hardy Weinberg equilibrium. If there is no difference between the two, we know that the population is is HW equilibrium. If, however, there is a measured difference we know that the population in not in Hardy Weinberg equilibrium and something must be influencing the observed deviation in the population (Ex natural selection, nonrandom mating etc)
ID090	Sex linked genes are unique because we can still use the HW equilibrium equation, but which part of the equation we use varies with males and females. For males (since they only have 1 X chromosome) the genotype frequency is equal to the allele frequency so we can use p+q=1. In females (2 X chromosomes) the genotype frequency is the same as the autosomal genes so we can use P2+3pq+q2=1
ID090	General transduction is a broader category and it is random which bacterial genes are picked up by the virus. Specialized transduction is a more specific category and occurs when specific genes are transduced (NOT random). Both of these methods of transduction provide a way for bacteria to obtain new genes (and increase genetic variation) through virus mediated gene exchange.
ID075	The chi square test lets geneticists detect frequencies in population. After the test they can determine whether the population is evolving. If the test determines inconsistencies with the Hardy Weinberg predictions, it means that factors such as natural selection, mutations, and genetic drift are acting on the population.
ID075	The frequency of alleles and genotypes with sex linked genes can be determined using binomial expansion. p squared plus 2pq plus q squared is the formula for females. For males, the p plus q is used.
ID075	Generalized transduction is when bacterial DNA instead of phage is packaged into phage particles and transferred to host. Specialized transduction is when strain-specific genes are transduced.It involves transfer of few bacterial genes.
ID093	The chi square test shows if a
ID064	The chi square test is a statistical test that determines if there is a significant differences between observed and expected values. This test can be used to compare observed allele frequencies with those frequencies that are predicted by the p^2 + 2pq + q^2 =1 equation, and therefore we can compare to see if the actual frequencies are close enough to the predicted for the population to be considered in Hardy Weinberg equilibrium.
ID064	Since female humans have two copies of the X sex chromosome and males only have one, they have different ratios of allele frequencies for sex-linked genes. Therefore for males, the frequency of occurence of a genotype is the same as the frequency of alleles in the population.
ID064	Transduction is the process of DNA being transmitted from bacterium to bacterium by a viral mechanism. This can be generalized transduction, where the virus packages the bacterial genes in its head instead of the phage, which transmits a lot of information, or by specialized transduction, which is strain-specific and occurs if extra bacterial DNA is also included when the viral DNA is excised.
ID050	The chi square will show whether or not the allelic frequencies occur by chance or not if it is lower than 5 percent. Also the chi square frequency will stay the same if it is Hardy-Weinburg.
ID050	You determine this by using chi-square to determine whether it is not in Hardy-Weinburg and than you determine the frequency by making a pedigree and analyzing that.
ID050	Viruses can carry themselves thorhg the phage and transfer themselves into the bacteria by that means of generalized transduction or insert their DNA into the bacteria DNA so it will be transmitted every time the cell replicates.
ID119	Using chi square tests, one is able to determine if the allele frequencies for genotypes of a given population are in agreement with Hardy Weinberg predictions.
ID119	The ability to determine the probability of alleles reappearing in offspring genotypes of sex linked genes.
ID119	DNA is able to be transferred from one bacterial cell to another through the means of transduction. In specialized transduction, only certain part's of the cell's DNA is able to be transferred.
ID124	If the frequency of phenotypes varies from generation to generation the population is not in hardy weinburg. to prove this you can use a chi square to show the frequency change.
ID124	Sex linked genes can be traced through generations based on their sex which can also be monitored in the frequency to see the variation after generations.
ID124	Bacterial gene exchange and one of the only ways of allele variation is through viruses but it is speculated whether the gene transfer happens generally and only carries bacterial dna or if it is specialized
ID049	The chi square test can be used to see how accurate results are and what the probability is that the results are because of chance. Since chance is involved with most of the evolutionary mechanisms (which violate Hardy Weinberg), chi squares can be used to determine if results are consistent with a population that does not violate hardy Weinberg assumptions.
ID049	Sex linked genes can be determined for males by using the allele frequency in a population (p + q = 1) and for females the genotype frequency can be used (p^2 + 2pq + q^2)
ID049	In generalized transduction any part of the chromosome can be inserted the virus while in specialized, only certain genes may be inserted into the new chromosome
ID137	A chi-squared test can be used to determine the amount of deviation of the observed frequences to the expected frequencies. A violation of the Hardy-Weinberg predictions can be assumed if deviation is larger than expected.
ID137	The Hardy-Weinberg law can be used to determine the frequency of sex-linked genes. If alleles are X-linked, then women can be homozygous or heterozygous, and their Hardy-Weinberg frequencies would be the same as their autosomal gene frequencies. Men only have one X gene, so their genotypical frequencies will be p and q.
ID137	Generalized transduction can be mediated by lytic viruses, which undergo transcription and translation. A lytic virus will infect a host cell so that the host cell replicated the viral DNA. Specialized transduction invloved in specific genes and invloces lysongenic viruses. These viruses will directly insert itself into a host cell and when the host cell replicates, so will the virus's DNA.
ID070	the chi-square test is more commonly used in a very different situation to analyze a contingency table. This is appropriate when you wish to compare two or more groups, and the outcome variable is categorical.
ID070	An important use of the H-W law is that it provides a mechanism for determining the genotypic frequencies from the allelic frequencies when the population is in genetic equilibrium. This in turn provides a way of evaluating which assumptions are being violated when the expected theoretical distribution of genotypes does not match an empirically determined distribution.
ID070	Specialized transduction is the process by which a restricted set of bacterial genes is transferred to another bacterium.Genetic transduction is the transfer, from one cell to another, of nonviral genetic material within a viral
ID056	The chi square test, as we learned previously, is used to tell if a change or error in the data is due to chance or if it is not. The chi square test can be used to see if a population is in Hardy Weinberg equilibrium by measuring the statistical rate in which allele frequencies differ from the predicted values.
ID056	The calculation of allele and genotype frequency of sex linked genes can be formulated by a binomial equation similar to Hardy Weinberg. For the sex that is heterogametic they carry 2 of the same x-linked chromosomes while the homogametic species normally only carries one x-linked chromosome.
ID056	Generalized and specialized transduction both make bacterial gene DNA transfer occur. Generalized occurs more often because it is random while specialized occurs less of the time because it is not random because of it's steps.
ID126	The chi-square test is used to test "goodness-of-fit" of data to a model.
ID126	Use the Hardy Weinburg to find the frequency of  cases of sex-linked genes.
ID126	Generalized transduction is the process by which any bacterial gene may be transferred to another bacterium via a bacteriophage and specialized transduction is the process by which a restricted set of bacterial genes is transferred to another bacterium.
ID151	The Chi Square Analysis is a method used by various scientists to determine if a hypothesis and its results are statistically significant. The Chi Square Value is denoted by the letter 'X' and is equivalent to the square root of observed minus expected squared over the expected. This value and the degrees of freedom, which is normally the number of variables minus one, but in this case there is only one degree of freedom since only one value is unknown and that one value, if known, can be used to calculate the rest of the variables, are used together to determine if the value is below or above 0.05. If the values are above 0.05 then the experiment’s hypothesis is generally refuted due to the fact that the value is not statistically significant; However, it is highly likely to get a value above the 0.05 mark when discussing allele frequencies within a population so that variation can be used to determine which condition is being broken in order to “ensure the effects” of the Hardy- Weinberg Law. Thus, the Chi Square Test can be used to portray discrepancies within the Hardy- Weinberg Law and allows one to determine the cause of the said discrepancy.
ID151	The Hardy- Weinberg Formulas, p + q =1 and p2 + 2pq + q2 =1, can be used to determine the frequency of alleles and genotypes associated with sex-linked genes. The first equation is used to determine the frequency of either the dominant (p) or the recessive allele (q), whereas, the second equation is used to determine the frequency of homozygous dominant individuals, heterozygotes, and homozygous recessive individuals, respectively. However in the case of sex linked genes, the first equation is used specifically for males in the sense that in sex linked genes, males can either have the disorder or not have it due to the fact that males have only one X chromosome. Females on the other hand have two X chromosomes, so the second equation corresponds to the frequency of homozygous dominant individuals, carriers, and homozygous recessive individuals. This application of the Hardy Weinberg formulas right fully depicts the fact that males are more susceptible to sex-linked disorders than females.
ID151	Transduction is a process in which viruses such as a bacteriophage carry bacterial DNA to another bacterium as the bacteriophage inserts its viral DNA components into the bacterium. This insertion of the viral DNA becomes incorporated into the bacteria’s genome and sometimes a piece of the bacterial chromosome is detached along with the viral DNA before the lysis of the bacterial cell to allow the manufacturing of the viral parts. Once that infected cell lyses, the new viral cells carry their DNA and the bacterial DNA to infect other bacterium. Through this process, viruses, in a way, allow the transfer of bacterial genes to the other bacteria. In specialized transduction, only certain specified genes of the bacterial DNA are transduced; whereas, generalized transduction deals with the random bacterial genes that are transduced by the assistance of viruses.
ID040	The chi square test can show whether the changes in allele frequency are by chance (the null hypothesis) or whether the deviations are statistically significant and a result of some force that is selecting for certain genotypes.
ID040	For X-linked alleles in females, Hardy-Weinberg frequencies are still calculated normally. In males, X-linked alleles are calculated by looking at only the p and the q alleles because males only have 1 allele for the gene.
ID040	Generalized transduction is by which any bacterial gene may be transferred to another bacterium via a bacteriophage, and typically carries only bacterial DNA and no viral DNA. Specialized transduction is the process by which genes that are near the bacteriophage genome may be transferred to another bacterium via a bacteriophage. The genes that get transferred always depend on where the phage genome is located on the chromosome.
ID120	The chi square test tells if 2 sets of data have a statistical difference based on the P value. It will tell if actual population frequencies differ from the Hardy Weinerg.
ID120	This is the process of using the Hardy Weinberg, by using given allele frequencies to see find the other allele frequencies
ID120	When viruses attaches to the bacteria, it inserts a copy of its DNA. Thereby, it transforms the bacteria into a bacteria phage.
ID118	Chi square analysis is used to determine the influence of chance on genetic data. If the statical test for chi square (x^2) shows the P value less than 0.05 while figuring out the data on the table provided suggests that chance is not the only reason for genetics variation in the population, there must be some violation of hardy Weindberg equilibrium that is causing the variation in the allele frequencies in the population. A chi-square test can be used to check whether deviation from expected frequencies is larger than expected by chance. If deviation is larger, it suggest that the violation has occurred which allows researcher to investigate on why assumption was violates: natural selection, non-random mating, genetic drift, gene flow, or mutation.
ID118	Frequency of alleles and genotypes for sex linked genes is that in hemizygous individual, only one allele is present for gene on sex chromosome, example XY in male of humans. Y lacks all the genes to homologous to X, which says that only X determines the genotype for the individual. p+q=1 is used for sex linked individual genotype ratio and for allele frequency as its only for one allele under hardy weindberg. compare to homozygous individual for sex linked example female in humans. In order to determine genotype of individual under hardy weindberg p^2+2pq+q^2=1 is used while for allele frequency p+q=1 is used.
ID118	Transduction is a way for bacteria to get genetic variation among there species as bacteria have less chance of variation than eukaryotes as they produce asexually compare to eukaryotes that produce sexually. In the process of Transduction, when virus infect the bacteria they have power to lyse the cell and produce a plaque (which is how the lysis is seen in lab). Virus can cause genetics diversity in many ways one is by generalized transduction, where bacteria DNA is packed into Viral protein coat, which can then infect another host, but as it not a virus it won't lyse the cell instead it will introduce genetics variation when recombination with new bacterial DNA will be occurred. Another way to cause variation is by specialized transduction in which infected virus DNA is integrates into chromosomal DNA of bacteria and while after some stress occur which will cause virus DNA to be separated it will pick up some of the bacterial DNA which can then infect another host and also introduce genetic variation when they recombine with chromosomal DNA.
ID107	If allele frequencies in the population are different from the Hardy Weinberg predictions, you can use a chi square test to determine it that difference is due to chance or not.  To calculate chi squared, you would use this equation: (observed-expected)^2/expected. If the chi squared value is improbably large according to the distribution chart, then you can reject the null hypothesis.
ID107	In females, they have two X chromosomes, so to determine the allele and genotype frequencies, you would use the standard Hardy Weinberg equations.  Males only have one X chromosome, so you would use the equation for allele frequency to find genotype frequency as well (p+q=1).
ID107	In generalized transduction, bacterial DNA is made into phage particles and transferred to the viral host. It results in the transfer of a large amount of bacterial DNA. In specialized transduction, only genes specific to that strain of bacteria are transferred to the host. This results in much fewer bacterial DNA being transferred.
ID097	the result of adding p and q (the dominant and recessive alleles) will add up to 1 in a given population.
ID097	Sex linked genes demonstrate the variation of genes, but result in the same phenotype,or the challenges of the mutations when puberty starts.
ID097	The gene exchange allows bacteria to replicate their strands into  other cells.
ID024	Chi square tests are tests that show us expected outcomes of a population. Through these tests, we can determine the allele frequencies in a population and a rough idea of how many people in the population are homozygous or heterozyhous for that allele or others.
ID024	If a gene is sex linked, it has to do with the x and y chromosomes. Females have two x chromosomes, while males only have one. If a disease is x linked, the men are going to be the only ones to be affected by the disease. The females will only be carriers.
ID024	Transduction occurs when a bacteria gives its DNA to another. It is a common process for them, but when visuses are involved, they can take over this process and use it to spread their DNA all throughout the bacterial cells, killing them through implanting their own DNA.
ID065	The chi square test can be used to detect whether allele frequencies in populations are inconsistent with Hardy Weinberg predictions because this test provides statistical information that determines how much of the results are due to chance. If the results show that more than 5% of the data is due to chance, the null hypothesis is accepted.
ID065	This can be determined with the Hardy Weinberg equilibrium equation the same way as you would with normal alleles except in the scope of sex linked genes. The assumptions / consequences that come with the Hardy Weinberg can also be applied to this concept.
ID065	Viruses can mediate bacterial gene exchange through general transduction, which involves DNA transfer from bacterium to virus to another bacterium, whereas specialized transduction involves the transfer of specific genes from bacterium to virus to bacterium.
ID023	The Chi squared test is a test that can be calculated in order to tell a researcher if the data is accepted within the scientific community. The chi squared test in this case can be used in order to determine if a population is being acted on by some form of selection, mutation, genetic drift, etc.
ID023	Sex linked genes in bacteria are viewed differently from eukaryotes. Bacterial sex linked genes are genes that are close to one another on a chromosome that are transferred together.
ID023	Viruses can carry bacterial DNA from one bacteria to another by infecting the bacterial cell and producing several other viruses that contain bacterial DNA rather than viral; thus causing bacterial gene exchange.
ID113	from the chi-square test it will show how much deviation there is from the ideal population
ID113	the frequency of alleles is greatly affected with sex linked genes
ID113	transduction is the transfer of bacterial DNA via viral vector
ID100	The chi square test is a statistical test that can be used to determine inconsistencies of allele frequencies in populations. It can be used to tell if there is a statistically significant difference between expected frequencies and the actual frequencies of alleles within a population. This information can be used to reject or fail to reject the null hypothesis.
ID100	The process for determining the frequency of alleles for sex linked is different than that for non-sex linked genes. For example with x-chromosome linked genes, one has to take into account that females will have two alleles for the sex-linked genes will males will only have one.
ID100	Bacterial transduction is a source of genetic variation in bacterial populations. It is the process by which genetic information is transferred from one bacterium to another via a virus. General transduction is the process by which any bacterial gene is transferred to another via a bacteriophage, while specialized transduction transfers a restricted set of genes. These two processes can be used to help treat diseases.
ID039	If there is deviation from the expected frequencies,the chi square test lets you know if it is larger than expected by chance or if one of the Hardy-Weinberg assumptions are being broken
ID039	By using the formulas from the Hardy-Weinberg principle, the frequencies of alleles for sex linked genes can be determined.
ID039	Viruses play a part in the diversity of bacteria. Viruses help transmit bacteria genes from one bacteria to the next.
ID144	if the expected frequencies are nearly identical to the observed frequencies in the parental generation we can say the Hardy Weinberg predictions are not being violated
ID144	p+q=1 gives you the frequency of the alleles for the WHOLE population. if it is ONE population we are talking about, we use the same frequency for the males, as well as the females. You will, of course, have two different frequencies of each genotype for male and female since it affects each differently
ID144	generalized transduction is random while specialized transduction occurs when the transfer of bacterial DNA is not random, but is strand specific
ID116	Hardy Weinburg can be used as a null hypothesis and then a chi-square test can be used to determine is there is significance from the equilibrium. Therefore, if there is significance through the chi-square test, there is evidence to show that evolution is occurring.
ID116	When random mating occurs the frequencies in alleles in each generation will be reduced by half. Furthermore, when random mating occurs and the alleles are sex-linked and the sexes differ in frequency, equilibrium is reached over generations.
ID116	General transduction turns the bacteria from its normal functions and starts making it perform duties of a virus. In specialized transduction, the virus incorporates itself into the DNA of the host. Therefore, when the bacteria replicates, it replicates the DNA of the virus as well.
ID117	The Hardy Weinberg allows us to predict allele frequencies in populations that follow numerous limitations. The chi square test allows us to test allele frequencies that are detected and determine whether they deviate from the predictions because of chance or whether there is another factor that is causing the frequencies to occur in the numbers they do.
ID117	The frequency of alleles in male organisms is dependent on the frequency of the alleles in the mother's X chromosome when it comes to sex linked alleles and genotypes.
ID117	Bacterial gene exchange can take place with generalized transduction when the plasmid of the donating cell moves into the host cell via a bacteriophage; where crossing over must occur for the integration of the DNA material from the transducted plasmid. It can also happen when a bacteriophage directly integrates the virus into the baterium's DNA with specifically determined DNA pieces.
ID038	Since the chi square test is a test of of probability which determines how close the values are to our judgement, we can use it to test any set of values. Hardy weinberg is a also a probability and thus can be tested with chi square test.
ID038	Sex linked genes are genes present on the sex chromosome and thus they are often present in one sex but absent in the other. This fact can be used asses the frequency of alleles in a population.
ID038	Transduction is the process by which virus injects its DNA into a bacteria. This process can occur in two ways generalized transduction and specialized transduction where the virus uses a special tube to inject its DNA.
ID017	In the chi square test it is determined how likely it is that an observed distribution is due to chance, so when applied to whether allele and genotype frequencies in a population are constant or not, the chi square test can detect whether allele frequencies in populations are inconsistent with Hardy Weinberg predictions.
ID017	Sex-linked genes in a population determine the frequency of alleles in a population based on the outcome of offspring genders. If the offspring genders are tailored towards a certain gender then the presence of a sex-linked gene, such as male pattern baldness, will either increase or decrease in frequency.
ID017	Generalized transduction is the process by which any bacterial gene may be transferred to another bacterium via a bacteriophage, and typically carries only bacterial DNA and no viral DNA and specialized transduction is the process by which a restricted set of bacterial genes is transferred to another bacterium. Knowing this you can determine which category the bacterial gene falls into and how the transduction will take place.
ID092	Chi square is a good means at determining the frequency of alleles in a population because if a population is in Hardy Weinberg equilibrium then there will be no divergence from the calculated chi square value.  If the calculated value is less than 0.05 then we know that the population is not following all of the tenets to be considered in Hardy Weinberg equilibrium.
ID092	Alleles and genotype frequencies can be determined by using sex linked genes because many genes are tied directly to the X chromosome.  If a population does not exhibit a certain genotype very frequently than one possible explanation is that it could be tied to the Y chromosome.  Also, considering the X chromosome shows up in every male and female organism, determining sex linked genes can also show patterns of inheritance.
ID092	Bacteria don't go through genetic variation as much as eukaryotes so viruses are essential to adding new genes to bacteria.  They inject viral DNA into bacteria allowing that bacteria to obtain the new DNA, which furthers genetic variation.  Transduction is responsible for adding completely new genes to bacteria, so without it the bacteria would have a less extensive genotype.
ID031	This is used to help determine if allele frequencies are even and whether or not they are being transferred at the same rate.
ID031	This helps determine the possibility of the genes being passed on together.
ID031	this is when viruses act as a method of gene transfer and can help exchange dna between cells.
ID146	The chi square test analyzes whether the deviation from expected Hardy Weinberg values is statistically significant or not, and if it is significant one of the Hardy Weinberg principles is likely being violated.
ID146	The  expected frequency of any X-linked trait expressed in males is identical to the expected X-linked allele frequency in the gene pool due to the males only having one X chromosome, while females expressing the trait will be equal to q^2.
ID146	Transduction occurs when a bacterial phage injects its DNA into either a bacteria's chromosomal DNA or its plasmid and then having reproduced, lyses the cell carrying the bacterial DNA onto the next bacteria
ID147	Chi square tests for variation between the expected values and actual values. So if the p value is below the accepted value then we can say that the statistical relevance of the occurrence of inconsistency is due to change or not.
ID147	If a population is not in hardy Weinberg equalibrium you can ignore the 5 priciples it lays down. From that we can then determine various allele frequencies.
ID147	When viruses inject DNA into a bacterial cell the viruses that are then reproduced in the Bacteria have a chance to pickup bacterial DNA and then transfer it to the next bacterial cell the virus infects.
ID127	The Hardy Weinberg predictions assume that no mutations, no gene flow, no selection occurs. The Chi Square analysis decides whether the observed frequency occurred by chance alone or not. Therefore, the Chi square test can likely determine if the frequencies are inconsistent with Hardy Weinburg because it would occur by chance alone.
ID127	The frequency of the alleles mostly depends on the female because she will have XX chromosomes while the male has XY chromosomes. The male will give his son the Y chromosome, so the frequency of the allele that occurs in the offspring depends mostly on the female.
ID127	When a virus infects a bacterial gene, the virus takes a certain donor of DNA(specialized) or it takes a random donor of DNA(generalized). The infection of the virus causes bacterial gene exchange and the bacteria to be partially genetically different from each other by sharing genes through viral infections.
ID055	The chi square test helps to determine if the deviation for an allele frequency in a population deviates from Hardy Weinberg equilibrium. If the chi square test shows that the deviation is too large then further test can be ran to determine which of the Hardy Weinberg assumptions has been violated.
ID055	Frequency of alleles and genotypes with sex linked genes can be determined using the equation p+q=1 for males and P^2 + 2pq + q^2 = 1 for females.
ID055	Both generalized and specialized transduction transfer bacterial DNA from one bacteria to the next by using a virus to transfer. This causes variation in the bacteria. The difference is that specialized transduction using a specific bacteriophage and uses the lysogenic cycle and only transfers a few bacterial genes. Where as, generalized transduction uses bacterial DNA packaged into phage particles to transfer large numbers of bacterial genes.
ID123	The chi square test elicits the importance of the allele frequencies in a population by comparing them to expected values calculated in the Hardy-Weinberg equation.  If the chi square value is not in the range of being scientifically significant depending upon the degree of freedom and the p-value, then some form of evolution is taking place in the population.
ID123	The frequency of alleles and genotypes with sex linked genes can be calculated by using the Hardy-Weinberg equation.  The alleles would be located on the X chromosome because they are sex linked.  The Hardy Weinberg equation can be used because because the alleles can either homozygous or heterozygous.
ID123	General transduction is the process by which DNA is transferred between bacteria by the use of a viral vector.  A virus carries the DNA and infects a bacterium with the DNA and the gene exchange occurs.  Specialized transduction is the same process except a lysogenic phage transfer the DNA and only specific DNA fragments are transferred.
ID002	I would assume this is meaning that after the HW is calculated, you would compare that calculated number to the real number gathered and run a chi square test to see if this is upheld or rejected.
ID002	Using sex-linked genes to determine which are passed on to the next generation. This allows one to see what alleles are prominent in the population, thus we can see the genotype present
ID002	Generalized transduction is the transfer of a large number of genes and used in chromosome mapping. Specialized transduction is more specific, in that fewer genes are transferred; this involves strain-specific genes
ID142	If we find allele frequency data for a population that seems different than what is expected, then a chi squared test will be used to detect if the population is in fact out of Hardy-Weinberg equilibrium due to chance results or actually it is. If the results of the chi square analysis are consist with greater than 5% then we shall assume the population allele frequencies are truthfully what we observed, but if the results are less than 5% then we can contribute the results to chance and assume Hardy-Weinberg equilibrium.
ID142	For males, allele frequency will be determined using p+q=1 since males only have one Y chromosome and one X chromosome.
ID142	In both specialized and generalized transduction, bacterial cells are invaded by new DNA via a virus vector. Instead of virus DNA in the carrier, bacterial DNA is accidentally inserted and is then transferred to a different host (bacterial) cell. This is one way variation is created within bacterial cells.
ID096	Hardy weinberg is near impossible to occur. The Chi square test tests the probability of Hardy Weinberg occurring.
ID096	A sex linked gene shows in the genotype. We are able to observe a sex linked gene by looking at the genotype.
ID096	Generlaized transduction occurs when a bacterial gene is transferred to another bacterium by a bacteriophage. Specialized transduction transfers a set of bacterial genes to another bacterium.
ID111	From Hardy Weinberg and known allele frequencies, we can infer the frequencies of the genotypes. Every individual carries and X and a Y we can predict these genotype frequencies using a chi square.
ID111	If alleles are X linked then females can be either homozygous or heterozygous only have one allele on the X gene. This causes recessive X linked traits to be more common among males than females. For predicting genotypes of sex linked, males and females have to be evaluated separately. For males it is simple because genotype frequency is allele frequency. You can still use Hardy Weinberg to predict these frequencies.
ID111	General transduction is process that DNA is transferred from one bacterium to another by a virus. General transduction is random. Specialized transduction is specific DNA fragments. Specialized is mediated by lysogenic phases.
ID108	The chi square test shows the p values and whether they are significant or not. The Hardy Weinberg predictions can be assured or not through the chi square test.
ID108	Sex linked genes can be used to determine the frequency of alleles and genotypes by looking at the X and Y chromosomes.
ID108	Bacterial gene exchange is overseen by viruses through transduction by using bacterial plasmids.
ID028	By applying a chi square test, you can examine a population to see if it is Hardy Weinberg equilibrium. If the test concludes that there is too much variation in the sample, then that would mean it is inconsistent with the predictions.
ID028	With sex linked genes, the X chromosome is the primary target affected. So, the hardy weinberg equation p + q = 1 can be used for males, whereas p^2 + 2pq + q^2 = 1 must be used for females because males only have one x chromosome and females have two, allowing for carriers.
ID028	A bacterial phage injects its DNA into the host cell and if it does not completely destroy the host cell and make numerous copies of itself, the phage's DNA is incorporated into the host cell's DNA.
ID088	When you have a Hardy Weinberg population, you compare the actual outcome to the predicted out come to find the deviation. This helps with predicting outcomes of populations.
ID088	By studying family trees, it becomes possible to map the frequencies of certain genes. Also, it is possible to discover if a gene is X or Y linked.
ID088	Viruses are brought into bacteria, then when the bacteria replicates, it passes some of that virus to its replicates. This keeps the virus going in the bacteria cells.
ID072	The chi square test allows us to predict what is expected to come out in the allele frequencies of a populations. These may follow the hardy weinberg predictions.
ID072	Sex link genes allow us to determine the frequency due to the x chromosome.
ID072	Bacteria can transfer genetic material through conjugation. this means that the bacteria are generally different from one another and can transfer genes that allow them to be resistant to certain substances.
ID114	The chi square test is a much more realistic view of allele frequencies than the Hardy Weinberg test because it actually factors in gene flow and population dynamics, unlike the Hardy Weinberg test, which only predicts allele frequency under certain static conditions.
ID114	Sex linked genes are genes that are almost always seen in one sex or the other. There are ways to determine the frequency of these alleles since they have a 50% chance of showing up in a genotype. For male sex linked genes, we often must look at the mother to determine the frequency.
ID114	Generalized transduction is when any bacteria can be inserted into another bacteria by bacteriophage method. Specialized transduction is when only a select set of bacteria enter another bacteria. Usually this happens when prophage is excised incorrectly.
ID016	Ideally, if all Hardy Weinberg conditions are met, a population's alleles will exist in equilibrium. A chi square test can determine if these conditions are violated, since a small deviation from allele frequency can be attributed to chance, and "a small" deviation is clearly defined in chi squared analysis.
ID016	Using the Hardy Weinberg equation, one can determine the frequency of an allele if given the frequency of the expressed phenotype. Since the alleles follow a binomial squared, the two-variable equation can be used to find the frequency of the individual alleles.
ID016	Transduction is the process in which DNA is transferred to another bacteria via a virus. Generalized transduction involves any bacterial gene being transferred, where specialized transduction involves a specific set of genes being transferred, rather than any general gene.
ID076	chi sq table with calculations gives mathematical predictions (%) if the result is due to just coincidence or if the result is due to actual reliable events; hardy weinburg predictions can be used as the 'hypothesis' being tested using chi sq
ID076	sex linked genes tend to have a pathway of recessive mothers and sons that project the gene - understanding the frequencies aid geneticist with understanding the inheritance and in the case of harmful genes, helps with people wanting to know the likelihood of having a child with the disease
ID076	microbiologist are still learning so much about the relationships of bacteria/hosts/viruses and discovering new species. these relationships have greatly affected evolution of organisms and we are just starting to understand. gene exchange increases virulence factors and survival of species.
ID145	A certain value from the chi square test denotes if a population is consistent with Hardy Weinberg predictions. If the population is not consistent, then it can be inferred that the population is not meeting all of the requirements of Hardy Weinberg equilibrium.
ID145	Sex linked genes are only passed on or expressed in certain sexes.
ID145	Generalized transduction is the transduction of the whole DNA and can be used for genetic mapping in bacteria, where as specialized transduction is it the transduction of certain genes.
ID074	Chi-squared is a mathematical test to determine if results are significant enough to be a scientific explanation so you could compare  hardy weinberg predicted frequencies and actual frequencies and determine if they were significantly different
ID074	you can use hardy-weinberg to determine the frequency of alleles and genotypes in sex linked genes just like any other autosomal trait
ID074	Generalized transduction is when a phage infects a bacteria breaks down the dna makes copies of itself and then creates new phages some containing bacterial DNA which get passed to another bacterium through transduction. Specialized transduction is a bacterial cell has a prophage integrated into the chromosome and it retains some bacterial DNA when it is spliced out that DNA is then Passed onto another baterial cell throught transduction
ID047	We can use chi square tests to see if the observed and expected frequencies from genotype and allele frequencies  are significantly different than the Hardy Weinburg predictions
ID047	Sex-linked traits of males and females are treated separately.
ID047	Bacteria can exchange genes through generalized or specialized transduction. in generalized any part of the bacterial genome can be transferred, whereas in specialized it's restricted and only transfers only specific portions of the bacterial genome.
ID009	Hardy Weinberg rule says that populations will be in equilibrium, with regards to alleles, after just one generation. The chi squared test can be used to see if this equilibrium is actually obtained or not.
ID009	Some genes can only be carried via the X chromosome and only transported to males. X-linked recessive genes, for instance, are carried by the mothers, skip a generation, then will be shown by all males of the following generation.
ID009	Transduction decides what does and does not infect the different plasmids so that control is important.
ID003	A Chi uses statistical analysis to
ID003	Can determine the frequency of alleles
ID003	Bacterial gene exchange
ID130	The chi square test compares actual results to those that would be expected.  The hardy weinberg predictions are generally used as a predictor for the frequency of alleles in a population.
ID130	Hardy weinberg equilibrium tests can be run in a population with a sex linked gene just the same as a population with a somatic gene.
ID130	A virus can implant new dna into a bacterium and that dna will implant itself into the hosts dna.  This bacterium will then lyse and the duplicated virus will then infect new hosts.
ID004	if the p value in a chi squared test is high, the population doesnt significantly differ statistically from hardy weinberg equilibrium.
ID004	generalized transduction can include the transfer of any bacterial gene to another by bacteriophage which carries only bacterial DNA. specialized transduction will result in the the transfer of genes based on where the phage genome is located.
ID011	If certain allele frequencies are found that do not agree with Hardy Weinberg predictions, it could mean that the population is not in Hardy Weinberg equilibrium which would mean evolution is occurring in the population.
ID011	Specific equations are used for male and females due to their different sex chromosomes. These equations are then used to determine allele and genotype frequency within the sex chromosome. Examples of this is determining the frequency of the hemophilia gene in males or females.
ID011	Bacterial genetic material will combine with viral genetic material and get transferred to other bacterial cells when the virus transfers genetic material to the cells.
ID048	the chi squared tests if a hypothesis is rejected or accepted. The formula for chi squared and Hardy Weinberg is similar therefore can be used to predict both.
ID048	sex linked is specific to male and female. There are certain species where sex is chromosomally determined. humans for example, in females are there are two X's and males have an X and an Y.  In females, gene frequencies are the same as for any other locus. In males, frequencies of the genotypes are the same as frequencies of the alleles in the population.
ID048	transduction is where DNA is transferred from one bacteria to a bacteriaGeneralized transduction is the process by which any bacterial gene may be transferred to another bacterium via a bacteriophage, and typically carries only bacterial DNA and no viral DNA. specialized transduction is where restricted bacteria genes are transferred to another bacteria.
ID008	The Hardy Weinberg predictions can detect gene frequency. when Ne = N, in the absence of selection or mutation, gene frequencies will acheive equilibrium after one generation. The equation can be expanded for three or more alleles.
ID008	not necessarily the same gene frequency in males and females after random mating, since the two sexes get their genes from two different sources. Males get their sex-linked genes from there mom and the mother receive her genes from both.
ID008	any bacterial gene may be transferred to another bacterium via a bacteriophage can be exchanged in generalized transduction and only a restricted amount can be exchanged in specialized.
ID078	this tests how close the actual data is to the expected data, so if they are not similar, then the population isn't at hardy weinburg equilibrium.
ID078	using sex linked genes, you can determine the frequency of certain alleles by whether they are turned on or off depending on the sex of the organis,
ID078	When viruses infect bacteria with their genome, they can take sections of the bacterial genetic material when they leave and transfer it to other bacteria.
ID006	You can use a chis square to predict the observed values and then compare them with the expected values and if they do not line up it's inconsistent.
ID006	For sex linked, both males and females will have two different frequencies, because it affects one more than the other. We would need to calculate it separately for them. Since males can only have one or the other, their frequency equation is: p+q=1 and if it is in the same population, you will have to use this same frequency to calculate for the females.
ID006	In specialized transduction only phage genes near the site of the recipient bacteria integration site can be transduced and only a few specific genes from one bacterial cell are transferred to the second bacterial host by a phage. In generalized transduction, a segment of dna is caried from one bacterial cell to the next by a virus.
ID036	The Chi-Sqaure Test allows you see the difference beteween what is observed and what is expected, while allowing you to see how it is deviated. You can observe what is expected under H.W is due by chance.
ID036	When random mating occurs within a population, the equilibrium genotypic frequencies are reached in one generation. However, if the alleles are X-linked and the sexes differ in allelic frequency, the equilibrium frequencies are approached over several generations. This is because males receive their X chromosomes from their mothers only, while females receive an X chromosome from both the mother and the father. Consequently the frequency of an X-linked allele in males will be the same as the frequency of that allele in their mothers.
ID036	Generalized transduction is the process by which any bacterial gene may be transferred to another bacterium via a bacteriophage, and only carries bacterial DNA, No viral DNA is included.
ID086	Hardy Weinberg predictions are without selection, drift, and random mating so when the frequencies fall out of these ranges it is easy to see using a Chi square.
ID086	Many sex linked genes will be not present in females due to them having two X chromosomes. Men will be stuck with whatever genes are on the X they have.
ID086	Generalized transduction is the process of a phage inserting its genes into a bacteria and it resulting in new phages. Specialized is when there is already phage DNA in the bacteria and it itself makes phages.
ID102	With chi square test, there is an equation that is used to find the frequencies, p2+2pq+q2=1. Then compare the results and determine whether or not it is efficient with the principles of Hardy Weinberg. I think that is how it goes, not too clear on this concept.
ID102	The same rules for determining frequencies for autosomal loci is the same for the frequencies with X-linked genes. However, the male has one X-linked gene as well as the heter female, then the homo female has two X-linked genes. Therefore, the equation would change a bit. It would be twice the number of females added to the number of males.
ID102	Both use the same process to undergo bacterial gene exchange; however, the generalized transduction uses all bacterial genes with similar frequencies or number of cells genetically altered as a function of the total number of potential recipient cells as opposed to specialized which uses a specific gene. The virus lytic mediates generalized and lysogenic mediates specialized.
ID141	The purpose of a chi squared test is to first see if populations are or are not consistent with hardy weinberg predictions.  If the deviation is larger than expected, then geneticists must look into what pre requisites are being violated in the hardy weinberg law.
ID141	Determining the frequency of alleles with sex linked genes is slightly different than determining the frequency of alleles for a non-sex linked gene.  The only difference is since males only have one x chromosome and females have two x chromosomes.  We treat the woman normal and calculate her allele frequency as p squared plus two pq plus q squared.  On the other hand since the male only has one x, his allele frequency is calculated as just p plus q.
ID141	Generalized transduction is not strain specific, which is the major difference between generalized transduction and specialized transduction.
ID080	The chi square test uses probability to predict the percentage of certain alleles in the population. It uses a chart and if a value falls in a certain area then it is whats probable. If it doesn't match up with the Hardy Weinberg p+q=1 then you can predict what kind of evolutionary factor is effecting the population.
ID080	You can predict these by mapping out the chromosome and also by using the probability of each event.
ID080	I know that transduction is when a virus has a plasmid from another bacteria in its DNA and attaches it to another bacteria's DNA whenever it infiltrates the bacteria. I'm not sure what specialized and generalized are.
ID073	The chi square test is used to determine whether there is a significant difference between the number of actual and expected genotypes. Using the chi square test tells you the probability of arriving at the differences found in the experiment by chance alone.  The lower the probability generated by the test, the greater the likelihood that the difference you see between the observed and expected result is actually the result of a violation of one of the five conditions of the Hardy Weinberg rules.
ID073	When determining allele frequencies,m male and females have to be calculated separately. Males have only one copy of sex linked genes so for males, the genotype frequency equals the allele frequency. On the other hand, female sex linked genes can be determined like any other inherited autosomal trait because they have two copies. After the p and q from the hardy weinberg is calculated from the males, this information can be used to determine the frequenies present in females who are carriers and who are non carriers.
ID073	Generalized transduction is when a piece of host DNA gets packaged by mistake instead of the phage DNA.  This results in a phage delivering only bacterial DNA to the next host.  The DNA then recombines homologously, replacing the host allele.  Specialized transduction occurs when a lysogenic prophage recombines itself out of the genome by site specific recombination and mistakenly includes a piece of bacterial DNA.  The resulting phage can infect cells to produce lysogens with a second copy of the allele they had packaged, attached to the phage DNA.
ID091	The Chi Square Test is a statistical tool to reject a null hypothesis.  It can be applied to analyze the allele frequencies within a population. The observed in this case are actual frequencies while expected are the Hardy Weinberg predictions.
ID091	Sex linked genes have a different pattern of inheritance compared to autosomal genes. For males, the Hardy Weinberg equation is simply p + q = 1 while females remain the same.
ID091	Viruses are able to uptake bacterial DNA, reproduce, and infect new cells. It is possible to then inject the new DNA, containing virus and Bacterial DNA, into a different bacterial cell, generating genetic diversity.
ID034	Based on the value of chi square test, if the actual and expected values are not significantly different, the null hypothesis is not rejected, we can conclude that the population is at Hardy-Weinberg equilibrium. on the other hand, if the actual values and expected are significant different, the null hypothesis is reject, therefore, the population is not at Hardy-Weinberg equilibrium
ID034	the same principles can be applied to sex-linked traits if males and females are treated separately. males have only one copy of sex-linked gene so for male: genotype frequencies= allele frequency.
ID034	the fragment of bacterial DNA involved in a transduction event maybe large enough to include several genes.
ID128	The chi square test is used as a statistical analysis for determining if the null hypothesis is a good fit for the data gathered. This test can be used to examine the allele frequencies and ratios within a population and determine if they match up close enough to be considered in Hardy Weinberg equilibrium to measure the inconsistencies.
ID128	Determining the frequency of alleles and genotypes with sex linked genes is important because it is easy to predict very accurately and it allows geneticists and students to easily track genotypes that result in genetic disorders. For example, colorblindness is a sex linked disorder that is often used to teach how alleles and genotypes that use the sex linked mode of inheritance can be tracked.
ID128	Transduction is the process of bacterial recombination through the mediation of bacteriophages. This process involves a virus compromising a bacterium and combining DNA with the host. Then the virus leaves the host either on its own or by causing the host cell to lyse. The virus then moves on does the same to a different cell but takes some of the DNA from the first cell along with it and continues to repeat the process. This gives the bacteria genetic diversity.
ID059	Using the chi square test you can find different frequencies with  a population that could also determine the genotype and phenotype of a specific species. When using this square you can figure out why certain genes are behaving a certain way and exactly where it could have possibly been inherited from.
ID059	Sex linked genes are the chromosomes from the mother and father that determines the sex of the child. There are certain genes fro example the SRY that is the key factor in the sex of the offspring as well as other genes on a chromosome that are linked for the sole purpose of determining the sex.
ID059	Bacterial transduction is where viruses are entered into bacterium by a viral vector. This vector injects the virus and new DNA begins to replicate. In some cases the new DNA strands will consist of old and new sequences where you can see that there was a mix in the both the virus and normal DNA with this exchange variation etc occurs in the bacterium population.
ID138	The chi square test is a method of statistical analysis that is used to determine if something is by chance or not. This test could be used to determine if the change in allele frequency is by chance which could mean that the populations are in equilibrium still, or not by chance indicating the populations are changing.
ID138	Sex linked genes behave differently than autosomal linked genes. This is because one parent has XX (the mother in humans) and the other has XY (the father in humans). There is not very much genetic information on the Y chromosome. When there is a certain allele on the X chromosome, it is much more likely to be inherited by a  male offspring than female.
ID138	Most bacterial reproduction is asexual and results in bacteria with the same genetic makeup as the parents. Viruses can exchange genetic material between bacteria resulting in variation. Transduction involved the exchange of genetic material from one bacteria to another from viruses.
ID042	The chi-square is used to see if an observed trait is similar to what we would expect to see.
ID042	This topic is used to study sex chromosomes, and the traits/genes that are passed on these chromosomes from one generation to the next.
ID042	Using viruses for gene exchange in bacteria can be done one of two ways. Either generalized or specialized.
ID064	Because of X chromosome inactivation, as long as an individual has at least one X chromosome, gain or loss of an X does not have a huge effect on phenotype, and individuals can still be very normal.
ID064	Aneuploidy is abnormal chromosome number, and is only tolerated in humans as gain of chromosomes, loss of chromosomes can usually be lethal, probably because of recessive lethal genes.
ID064	Crossing two plant species can sometimes increase its genome by an entire set of chromosome. This can be useful for breeding larger varieties of plants and for getting other qualities too.
ID094	Everyone has an X so it can be altered safely sometimes.
ID094	This condition lends itself to rarity but plants can handle it well.
ID094	This fact is applied in the breeding of seedless fruits.
ID027	Increasing the number of X chromosomes only affects early development since Barr bodies will disable the extra X's after a certain amount of time. This means they will have little time to alter the phenotype outside of a few changes.
ID027	Only certain anueploidies are non-lethal, so many die in the womb, but those that survive end up with birth defects, like trisomy-21.
ID027	This is the reason almost all of the plants eaten today have grown to the size they are now. They were mutated and altered purposefully by farmers. The increase in the number of chromosomes only changes their phenotype. It is not lethal.
ID046	Humans and mammals do not have as sensitive of an X chromosome as others, such as the insect Drosophilla. One X will be inactive in humans.
ID046	Down syndrome is an example of an autosomal instance, occurring on the 21chromosome.
ID046	This holds true for crops such as maize
ID113	the extra X chromosome can randomly be inactivated
ID113	the gain or loss of one or more chromosomes are not highly tolerated in mammals and humans due to letahlity
ID113	with anueploidy and euploidy in plants, it alters the phenotype, thus creating our fruits, vegetables, etc.
ID069	The amount of X chromosome has little effect on human phenotype. In females, this is because each has two X chromosomes, so the normal one can make up for the affected one. In males, this is because the X chromosome is hyperactivated so that it can transcribe the same amount as two X chromosomes as in a female. Therefore, the effect for each is balanced out and not ridiculously visible in human phenotype.
ID069	Autosomal aneuploidy is commonplace amongst plants, producing variable but equally viable phenotypes. In mammals and humans, however, autosomal aneuploidy is rare because it is usually lethal and destroys individuals before it is detectable.
ID069	Plants can tolerate both single chromosome and chromosome set increases. These increases may cause the plants to be infertile, a trait valued by breeders who do not want their specific crops to be replicated by others. The infertile state also results in a lack of seeds, which may be repulsive to consumers.
ID126	The increase or decreases of the x have slight affects but phenotypically the girls look the same
ID126	Aneuploidy is the presence of an abnormal number of chromosomes in a cell, such as having 45 or 47 chromosomes when 46 is expected.
ID126	For example, bananas and watermelons have extra chromosomes which allow for the benefits of seedless fruits.
ID101	The addition of an X chromosome will cause the suppression of all but but one X chromosome that will cause some variety in the phenotype expressed.
ID101	The Autosomal anueploidy often causes the termination of the fetus, however some exist and can cause diseases. One example is Trisomy 21 that causes down syndrome.
ID101	The plant is normally viable regardless of an extra chromosome, however the offspring will be sterile. This is useful for food, for the sterile plants don't produce seeds in their fruits.
ID000	Humans regulate gene dose for the X-chromosome by inactivating all but one X chromosome.  The inactivation is random making females mosaics for functional X-chromosomes
ID000	Few conditions in mammals show aneuploidy.  Three small autosomes in humans may be trisomic and sufficient children are born with these aneuploid conditions that they can be recognized as syndromes.
ID116	Increases or decreases in the X chromosome do not affect the human phenotype because of X chromosome inactivation. When more than one X chromosome is present, only one is active. Therefore, XX and XY have the same number of gene housekeeping products. Therefore, the an increase or decrease in the X chromosome makes little difference phenotypically.
ID116	Autosomal anueploidy occurs when there is an incorrect number of chromosomes in the cell. This occurs most often when there are 45 or 47 chromosomes, therefore this doesn't occur when there is a missing pair. Autosomal anuploidy can cause birth defects, and even occurs in cancer cells. It originates during inappropriate chromosome division during cell division.
ID116	Some of the food that we eat today is triploid, meaning the cell contains three homologous sets of chromosomes. Seedless bananas and watermelons are triploids. The set of three chromosomes prevent meiosis and therefore the fruit cannot produce gametes, or seeds.
ID109	As long as a person has one x chromosome, the x chromosome number can increase or decrease, and there will be little phenotypic effects. For example, a male can have an extra X chromosome and look normal; however, he will have other side effects that you can't physically see, like sterility, low sex drive, and weaker muscles. The loss or gain of the X chromosome is caused by nondisjunction, and individuals can have more or less chromosomal material than normal.
ID109	Generally, aneuploidy is fatal in mammals because chromosomes are so specific and vital to proper development. An example of a specific scenario is when aneuploidy occurs with a sex chromosome, and in this case there are very few phenotypic effects of aneuploidy. Aneuploidy is rare in mammals because it is generally fatal in autosomes but it is generally not fatal if it happens in sex chromosomes.
ID109	Plants can tolerate aneuploidy of both single and sets of chromosomes. Triploidy plants, most of the fruits we eat and crops we harvest, don't have seeds, which people like. However, these plants and crops are infertile because they don't have seeds. We breed hybrids of triploid crop plants to keep this mutation in the population because they grow well, and are seedless, a very popular quality for consumers.
ID037	There is little effect on the phenotype because there one X is always deactivated.
ID037	Anueploidy happens when  gamete does not receive genetic material.
ID063	This effect is seen as their are normally 2 copies. 1 copy can compensate for the other. So genotypic differences can occur but will not be seen as the 1 copy is present.
ID063	This phenomena occurs with select chromosomes in a karyotype. For humans, each disease is associated with a specific chromosomes and is specific.
ID063	Plants with triploidy are able to mate with diploid or triploid. However, with a diploid sterile offspring will be created with a reproductive barrier. With triploidy, offspring will be created with no reproductive barrier.
ID040	When humans inherit extra X chromosomes it has little effect on the phenotype because extra X chromosomes become highly condensed structures called Barr bodies that are inactive and do not contribute to genetic information given to the cells.
ID040	Autosomal aneuploidy is when an organism loses or gains one or more chromosomes. They are rare in mammals and humans because the extra genes often lead to nonviable gametes.
ID040	Plants are more able to tolerate an increase in the entire set of chromosomes than a single chromosome. Having an odd number of chromosome sets lead to uneven numbers of homologs leading to unbalanced gametes. Having an increase in the entire set of chromosomes often leads to increased plant size which is better for crops.
ID026	The X chromosome can give a human some different characteristics when there is an additional chromosome as seen in conditions like Klinefelter's Syndrome, but overall the differences are minor.
ID026	Autosomal aneuploidy is not seen in most mammals and humans, but can occasionally be present in some organisms. Down Syndrome is an example where a person has 3 of the 21st chromosome, but is still able to function.
ID026	Plants have a higher toleration of increased numbers of chromosomes than other organisms. This is shown in many of the fruits and vegetables that we eat, but in many cases produces a sterile offspring.
ID065	Increase or decrease in the X chromosome have little effect on human phenotype because the autosomal chromosomes house a majority of the phenotypic traits. The X chromosome mostly contains genes that deal with the sex of the organism. Because of this, some, but not all, mutations on the X chromosome usually have little affect on the human phenotype.
ID065	Aneuploidy occurs when an organism gains or loses one of its chromosomes, but it does not lose a complete set of chromosomes. This is rare to occur in mammals and in humans because it is a mistake in meiosis, which is vastly regulated and controlled. Also, mammalian aneuploidy has the potential to be fatal and result in miscarriages, thus lowering the percent of the population that have these disorders.
ID065	It is common for plants to exhibit autopolyploidy and allopolyploidy. Unlike animals that would most likely die in the presence of double or triple the amount of chromosomes, plants that have an even number of pairs of chromosomes (such as 4n) can live and reproduce because their gametes will have a balanced number of chromosomes, assuming meiosis carries out correctly. Because plants are capable of this, more phenotypes are likely to result. Plants that are triploids (3n) can not produce gametes because they would be unbalanced, so they are infertile, similar to many of the crops we eat today, like bananas.
ID056	Normally those who are XX are female and XY are male. Aside from oddities, the presence of a Y chromosome indicates a male phenotype. A person who is XXXXY is still male. And the individual who is XX compared with one who is XXXX are still female.
ID056	Anueploidy is more common in plants. Monosomy in humans is not well tolerated and are normally lethal. The specific scenarios occur from trisomy. An example is Down Syndrome or Trisomy 21.
ID056	As chromosome number increases in value so does size. Not the number of cells but the actual size of the cells that construct the plants. The autopolyploids formed are larger in size than their originating diploid family.
ID079	The number of X chromosomes in an individual has little phenotypic effect because dosage compensation is possible via Barr bodies. Barr bodies are condensed, inactive X chromosomes. The number of Barr bodies in an individual is the number of X chromosomes minus one, resulting in only one active X chromosome. Women do not overly express genes that are on the X chromosome because of Barr bodies. However, certain conditions in which an individual has extra X chromosomes can still occur because some X chromosomes that form Barr bodies are not inactivated.
ID079	Anueploidy is defined as a variation in chromosome number.  In other words, it is a gain or loss of a chromosome. The gain of one chromosome is referred to as trisomy, and the loss of a chromosome is referred to as monosomy. Trisomy is usually lethal in humans, but one relatively common trisomy condition in humans is Down Syndrome, which is caused by an extra 21st chromosome.
ID079	Polyploidy is common in plants and can be found in nature as a result of evolution, such as bananas, or can be manipulated by producers, such as seedless watermelons and strawberries. Polyploidy crops tend to be larger, and they are usually infertile due to their chromosome number. Infertility in crops is a benefit for producers, because consumers cannot plants the seeds from a fruit they buy in the grocery store and grow their own food.
ID089	Because all X chromosomes are inactivated except for one, multiple Xs will have limited effect. The only reason that the multiple Xs has an effect on phenotype is because the X inactivation does not occur until after some fetal development.
ID089	Autosomal anueploidy is rare because it is often lethal. Because the autosomes are not inactivated in the same way the X chromosome is, the gene products are produced in too great of a degree.
ID089	Aneuploidy is tolerated in plants and can lead to profitable economic benefits because of the way that many triploid plants are seedless due to incorrect pairing and disjunction in meiosis.
ID043	Each time you have a child there is a 50% chance that you'll have a boy or a girl
ID043	It is rare because of the sperm and egg cells.
ID043	It's because alot of these plants become seedless.
ID013	In an organism with more than one X chromosome, the other X chromosomes are inactivated and are seen as barr bodies near the outside of the cell. Because these genes on these chromosomes are inactivated, they have little effect on the human phenotype.
ID013	Aneuploidy occurs when nondisjunction in meiosis leads to gametes with or without a chromosome. Instead of 46 chromosomes, they could have 45 or 47. In humans, only one monosomy and three different trisomies have been discovered as syndromes, or are repeated over and over again and is not lethal in utero. The monosomy is Turner's syndrome, and the trisomies are Down syndrome, Patau syndrome, and Edwards syndrome.
ID013	Autopolyploidy and allopolyploidy are types of polyploidy that occur in plants. In autopolyploidy, a single type of species is used to acquire extra chromosomes while in allopolyploidy different species are used to acquire extra chromosomes. Crops that undergo autopolyploidy are increased in size; therefore, this technique is used today to commercially grow strawberries, grapes, corn, watermelon, and other foods to make them seedless and sterile.
ID033	Mostly when you add more X chromosomes the phenotype stays feminine so not much phenotype change occurs.
ID033	Nondisjunction separations occur most often in sex linked genes opposed to autosomal chromosomes.
ID033	Plants are better adapted to survive a nondisjunction chromosome separation from meiosis.
ID014	If there are more x chromosomes then they will become inactivated. This is how the individual can be phenotypically normal due to this inactivation.
ID014	Autosomal anueploidy will cause individuals to be phenotypically different. Some anueploidy such as trisomy 21 can be due specifically to three known causes either disjuntion, mosaicism, or translocation. For some chromosome it can be lethal to have aneuploidy and those individuals are less like to survive. Organisms such as plants are able to tolerate an increase in chromosomes.
ID014	An increase in chromosomes can cause the plants to have different characteristics such as being larger or seedless. In some cases the amount of chromosomes are increases purposely so that the plant is able to reproduce with another plant.
ID150	Because X chromosome genetic expression is monitored by X chromosome inactivation. Females have two x chromosomes normally, one of which is shut off by Barr bodies. The same system works for unusual x chromosome numbers, resulting in a lack of expression.
ID150	A missing chromosome contains a lot of genetic material that is essential for the organism to function properly. An extra chromosome ruins the correct genetic balance, meaning that the organism overproduces certain gene products.
ID150	Polyploid plants rearrange their extra chromosomes to serve new functions in the organism. These new functions can help fight diseases or create bigger fruits for example.
ID068	There only needs to be one dose of an X gene to be viable and there are not very important genes on the X chromosome that can be deleted causing no phenotypic change.
ID068	Autosomal anueploidy occurs with the addition or deletion of a copy of a chromosome. A monosomy is the deletion of a single chromosome, and a trisomy is a gain of a single chromosome.
ID068	The change in chromosomes is viable for a plant to live but it can cause infertility because of abnormal meiosis pairing. This condition is used in commercial plants such as seedless grapes and strawberries. There are 2 types of changes with plants, allopolyploidy which is the doubling of chromosomes in a hybrid between 2 species. Autopolyploidy is a duplication of the genome within the same species.
ID038	All humans have at least copy of X chromosome and thus if if a human cell were to have more than one, they would produce the same gene product and thus more X chromosome does not affect as much.
ID038	Since during meiosis, disjunction occurs very reliably, autosomal aneuploidy is not common. If it does arise, the phenotype is normally infertile and thus it is not carried forward. In humans due to advanced treatments, sometimes the individuals live to reproduce and results in heriditary cases.
ID038	Plants may show a different morphology due to chromosomal imbalance but they do survive. many plants such as wheat, banana and watermelon have been given some characteristic features because of their ability to survive chromosomal aberrations.
ID136	Having an increase or decrease in the X chromosome could have a big effect on human phenotype, or it will have no effect on the phenotype. It depends on what occurs.
ID136	It is a rare condition because the chances of having an alteration in the number of chromosomes are very small.
ID136	In plants those events could actually be beneficial. It is normal to see plants with different phenotype.
ID020	Typically human males of one X and one Y chromosome and human females have two X chromosomes, but if a male or female has an extra X chromosome or is missing one they may not develop like normal and express different phenotypes, but they don't change a human's phenotype drastically. For example if a male has an extra X chromosome they develop Klinefelter syndrome where individuals have big hands and feets and usually have testes that are rudimentary and fail to produce sperm. If a female is missing an X chromosome for example they can develop Turner Syndrome which can affect the individual's external genitalia and internal ducts, but the ovaries are rudimentary.
ID020	Aneuploidy is when you have a lost or gain of one or more chromosomes and autosomal means on your autosomal chromosomes.There are only a certain amount of syndromes in humans where there is an addition of a chromosome on the autosomal chromosomes instead of the sex chromosomes. One example is Trisomy 21.
ID020	When plant have an extra set of chromosome they are better tolerated than in humans. Plants can still produce good seeds and do just fine. They may be only phenotypically different. For example polyploid fruits like strawberries and bananas are bigger great for commercial value, but there isn't any new or unique information compared to their diploid relatives.
ID140	Phenotypically, individuals with increasing numbers of X chromosomes appear to look about the same. However, with more x chromosomes the severity of health/mental retardation is more common.
ID140	Generally, autosomal anueploidy is rare because most of the time extra/not enough autosomal chromosomes is a lethal combination, resulting in death. The few times that it doesn't result in death is for additional chromosomes like in Down Syndrome, Edward Syndrome and another one from chromosome 14.
ID140	Plants with increased chromosomes has resulted in improved crops, such as seedless watermelons.
ID007	Because all X chromosomes but one are switched off in each cell due to dosage compensation, additional X chromosomes generally do not affect phenotype.
ID007	Additional autosomes due to nondisjunction in the gametes, because there is no dosage compensation for them, affect phenotype and very often result in non-viable offspring.  The rare cases where the offspring are viable tend to still produce developmental problems.
ID007	Both autopolyploid and allopolyploid plants are able to survive, and in some cases (when meiosis does not fail due to inability of chromosomes to pair up), they are fertile.  Hybrid strains are often created this way, and triploid plants are often sold as seedless varieties.
ID115	Because of Barr Bodies and dosage compensation, females with extra X chromosomes do not differ much phenotypically, but experience genetic disasters within the body that lead to lower IQ's and rudimentary reproductive systems as the increase in X chromosomes progresses. In such conditions as Klinefelter's where males exhibit an extra X, these males may be perceived to have feminine aspects and have rudimentary male genitalia, but other than this their phenotypes appear relatively normal. However, their reproductive and cellular compositions are altered and as a result, a lower intelligence level is achieved.
ID115	Because mammals cannot live without a chromosome pair or in very rare instances, exhibit aneuploidy, those that are monosomic usually die in vitro. The only scenario that exhibits monosomic sex chromosomes is the Turner syndrome, where only one X is to be found due to nondisjunction. even then, function of the individual is limited. Those that exhibit more than two chromosomes rarely survive due to problems in segregation of meiosis, and result in physical and mental defects, such as Down Syndrome, Klinefelter's and Triplo-X syndrome.
ID115	Because plants can self-fertilize and withstand duplications or mutations of chromosomes, they increase in size and can become sterile due to imbalanced pairing in meiosis. To this end, geneticists use this ability to their advantage to create seedless crops or larger fruits.
ID023	Given certain aspects such as polar bodies, have an increase or decrease of the X chromosome causes little effect on phenotypes given that only a small amount of excess X chromosome dosage escapes.
ID023	Autosomal anueploidy basically means having an abnormal number of chromosomes. Humans have 22 autosomal chromosomes. Examples of autosomal anueploidy would be Down Syndrome and Turner Syndrome. They are rare in humans and mammals but occur more often in plants.
ID023	Plants that have an increase or decrease in chromosome number more than likely will only have sterile consequences, but many plants can become fertile again. An increase in chromosome number can be seen in many produce in grocery markets, including bananas and strawberries.
ID103	I understand that extra x chromosomes will be made in to bar bodies and will not have much affect on the phenotype of the organism.
ID103	Mammals and humans typically do not have more then two chromosomes at a give loci and there are some disorders in humans such as down syndrome that result from having an extra chromosome at a give loci
ID103	When plants that have different numbers of chromosomes cross pollinate the resulting progeny sometimes has an odd number of chromosomes and duplicating the genome is necessary to reach an even number which can be used to form gametes.
ID105	This id because of Barr bodies some X chromosomes are expressed and some are not this prevents over expression of the X chromosome.
ID105	It is very rare in humans because in most cases any chromosomal aberration usually occurs in a miscarriage. The scenarios where the organism survives result in genetic disorders.
ID105	Allopolyploidy is doubling of chromosomes in a hybrid in two plants the hybrid is sterile. However, the polyploidy retires the fertility.
ID129	In 47, XXY syndrome, the only shared characteristic is above average height, and in 47, XXX syndrome, most women are born completely normal and are unaware that they even have this syndrome until a karotype is done.
ID129	In humans, aneuploidy can occur by Patau syndrome (trisomy 13) and Edward's syndrome (trisomy 18). Both cause early death. Patau syndrome symptoms include cleft lip and palate, small eyes, mental and developmental retardation. Edward's syndrome symptoms include clenched fists,elongated skull, low set ears, and mental retardation.
ID129	Allopolypolidy is the doubling of chromosomes in a hybrid between 2 different plants. the hybrid is sterile until polyploidization restores the fertility. This is commonly used in plant breeding. Triploid and tetraploid grapes and strawberries are also very common.
ID146	Examples such as XXY, XXX, or XO (Kleinfeldt, Poly X, and Turner Syndromes) mainly result in only physical characteristic changes, whereas autosomal disorders canresult in permanent disabilities, cancer, and even death.
ID146	Autosomal aneuploidy can have serious effects and is rare in humans butis the cause of Down syndrome, Edwards syndrome, and Patau syndrome.
ID146	Allopolyploid can restore fertility to some plants and has been used in the cultivation of wheat, cotton, grapes, and strawberries.
ID133	When non-disjunction occurs with the sex chromosomes, sometimes this leads to an increase or decrease in the X chromosome.  Because only one of the X's genes are usually expressed in a cell, the phenotype can be undisturbed.
ID133	When chromosomes do not separate properly, this could lead to anueploidy in which there is abnormal number of chromosomes. This usually is lethal but sometimes can produce viable offspring with defect.
ID133	Autopolyploids and allopolyploids are examples of plants with an increase in chromosomes.  These are useful because in the second case, two species can combine new traits. Also, these plants tend to be larger.
ID036	A change in chromosome number interferes with growth and development of an organism and is a common characteristic. A cell with increase gene product can produce a more gene material than needed. An example would be tumors that are cancerous.
ID036	Aneuploidy in human are rare, and can be lethal. Chromosome 18, 21, and 13 are usually the ones that are effect. Anueploidy is not the normal hapliod 2n set. An example of a condition is downs syndome trisomy 21.
ID036	In plants, an increase or decrease in a single chromosome does not affect viability of the organism. For the plant Datura, lines have been identified that are aneuploid for each chromosome in the genome.
ID022	The increase or decrease in the number of x chromosome is due to nondisjunction in meiosis 1 or meiosis 2.  Due to increase and or decrease in the x chromosomes sometimes humans have various syndromes and their phenotype changes while in some instances there is no change.
ID022	Aneuploidy means variation in the number of a particular chromosome within a set
ID022	The reason plants can tolerate both increase in chromosomes and entire sets of chromosomes is because after duplication chromosomal sets number are fixed. And due to having all the chromosomes they can function normally however due to extra chromosomes there is an increase in size.  Crops such as large Bananas and Strawberry, maize, and grapes are some of the example of triploidy and tetraploidy.
ID048	X chromosomes are inactivated at random early in development. An X is randomly chosen in each cell for inactivation early in development. The inactive X chromosome is silenced by its being packaged in such a way that it has a transcriptionally inactive structure called heterochromatin. The inactivation of X results in it effecting little to none in phenotypes.
ID048	This type of chromosomal abnormality is rare in humans and only occurs in certain number of chromosomes. Some examples in humans are Trisomy 21(down syndrome), Trisomy 18( Edward's syndrome), and Trisomy 5(Cri du chat).
ID048	In plants, the increase in ploidy over the years has resulted in the common plants we have today. two types of ploidy: 1. Autopolyploids: duplication of the same genome. Leads to segregation problems until the genomes
ID049	Increases in the number of x chromosomes is not lethal. Oftentimes it just results in different phenotypes. If males have an extra x they tend to have wider hips, higher voice, and may experience breast development.
ID049	It is rare because it is lethal. THis means it is not passed on generation to generation. There are specific cases when it is possible such as down's syndrome.
ID049	This causes variation in plant species. Oftentimes this causes plants to be infertile so you must double the chromosome number so that proper separation can occur in mitosis.
ID124	pheno type is not extremely changed because only one x chromosome is expressed and the rest are made into barr bodies.
ID124	Chromosome number is abnormal which either results in death or severe phenotypic abnormalities such as down syndrome.
ID124	the increase in chromosomes causes the plants to become sterile and produce larger fruit which allows farmers to protect their genetic variations of plants.
ID057	An increase and decrease in the X chromosome has little effect on human phenotype because only one X chromosome is activated at a time. If there is more than one X chromosome then the X will be inactivated otherwise known as a Barr body.
ID057	Anueploidy is when an organism gains or loses one or more chromosomes but not a complete set.  It is very rare in mammals and humans because if it loses a chromosome then we have monosomy which unmasks the lethals that are otherwise tolerated in heterozygotes carrying the wild type alleles. When mammals have an addition of an autosome this has serve effects and is usually lethal during development.
ID057	In many plants increases in chromosome are tolerated but usually result in an alteration in phenotype. This alteration can result in seedless plants like many of the produce we eat today such as seedless bananas and seedless watermelon.
ID039	Increases and decreases in X chromosomes have little effect on phenotype because most of these cases are lethal
ID039	Autosomal anueploity is the loss or addition of chromosomes. It's rare in mammals because most of the time it's lethal. The embryo dies before it is born. In humans, in some cases it is lethal,a spontaneous abortion occurs, but in cases were the embryo survive. In the cases of survival, common conditions such as downs syndrome and patua syndrome are seen.
ID039	Plants can tolerate these changes in chromosomes because they can double their chromosome numbers (polyploidization) and can then form gametes again.
ID024	A male or female might have many more X chromosomes than are needed, but it rarely has a  large effect on their phenotype. One or more of the X chromosomes will go dormant and turn into barr bodies in the cell. Normal XX females have this and have one barr body. Sometimes, the X chromosomes will share dominance, and even that does not drastically change the phenotype. Males and females are still normal.
ID024	Autosomal aneuploidy occurs in all the chromosomes except for the sex chromosomes. It is rare in both humans and other mammals alike because if it does occur, many of the individuals do not survive. It leads to spontaneous abortion. If the individual does live, they have a disease very specific to their particular type of autosomal aneuploidy, such as Down Syndrome and Turner's Syndrome.
ID024	Plants can both undergo an increase or decrease of the number of chromosomes they have. The plants with trisomy are viable, but they themselves cannot have children. They are sterile. Plants that have this odd number of chromosomes can be made fertile again by the use of the drug, colchecine. This drug brings the number of chromosomes back to an even 2n number.
ID031	This is when an individual has a loss or gain of an x chromosome and there is little effect from this on the individuals looks.
ID031	This is the gain of a chromosome on a non-sex chromosome.
ID031	Whole sets of chromosomes can be added or lost in plants and have many different effects that can arise from this.
ID093	Increases or decreases in the number of X chromosomes DOES have an impact on phenotype in humans. Turner's syndrome, for example, results in monosomy of the X chromosome. Since one X chromosome is usually converted into a Barr body the individual will most likely live, but still phenotypically different. They will be  female, but with external genitalia and internal ducts, rudimentary ovaries, underdeveloped breasts, cognitive impairment, and short stature is a far cry from "little effect" on phenotype.
ID093	Anueploidy (alterations in a single chromosome not entire set of homologous chromosomes) is rare in mammals due to the perniciousness of extra chromosomes. Most embryos with trisomy 21 are spontaneously aborted. Those few that survive have many alterations to phenotype. They are fertile and have a 50% chance to conceive a child with trisomy 21 due to the pattern of nondisjunction of the 21st chromosome during gamete formation.
ID093	Plants can tolerate single chromosomes or entire sets. Polyploidy (Extra pairs of homologous chromosomes) in plants has provided humans with seedless watermelons and bananas, which are trisomic. Many other commercially profitable genetically altered food items exist such as coffee and peanuts, which are tetraploids.
ID058	Those with only 1 X chromosome (Turner's syndrome) still appear phenotypically female. They are usually short in stature and have some cognitive impairment, but they have all of the genitalia and internal ducts of females. With Poly X syndrome (XXX,XXXX,or XXXXX), some individuals are completely normal. Others though can have underdeveloped secondary sex characteristics such as sterility and mental retardation.
ID058	There are only a few syndromes caused by aneuploidy that are seen (XO, poly X,XXY, and XYY). They have a range of effects from an individual being completely normal to having mental retardation and/or rudimentary testis or ovaries.
ID058	Aneuploidy is very common in plants and they tolerate it well. Farmers often cross diploid and tetraploid individuals to create triploids in order to produce seedless bananas and watermelon. Also, tetraploid individuals are twice as large as diploid ones so they produce more product for the same effort.
ID082	Increases or decreases in the X chromosome have little effect on human phenotype because only copy of the X chromosome in any given individual. Since only one copy of the X chromosome can be active, any additional copies of the X chromosome will be inactivated in the form of Barr bodies. Because of deactivation into Barr bodies, individuals with multiple copies of the X chromosome do not have dramatic phenotypic effects, because regardless of how many extra copies they are, deactivation of extra copies leads to similar gene expression as an individual with the correct number of copies of the X chromosome.
ID082	Aneuploidy refers to variations in chromosome number when an organism gains or loses one or more chromosomes. Aneuploidy can be tolerated by many plants, but in mammals and humans, monosomy in particular poses a high risk for death because having only one copy of each gene unmasks any recessive lethal genes that would have gone unnoticed. With only one copy of each gene, it only takes one lethal allele for the individual to die. Autosomal monosomies are very rarely found, implying that the gametes that lack a single chromosome are functionally impaired in a way that prevents them from reaching term. In terms of specific scenarios in which anueploidy in humans occurs, trisomy 21, also known as Down syndrome, is a relatively common aneuploidy condition in humans. Other than Down syndrome, two other human Trisomies can survive to term. These two trisomies are Patau syndrome and Edwards syndrome.
ID082	Plants often tolerate an abnormal genetic content. This genetic variation can give unique phenotypes, which is important to the evolution of plants. Trisomy, for example, which involves the addition of a large autosome, is usually lethal in animals and humans.  Trisomic individuals in plants can be viable, however, and as seen with jimson weed, each trisomic possibility produces a distinct phenotype. One of the common phenotypic effects of increased chromosomal material is the production of unviable offspring. This may at first seem counter-intuitive that we would use these plants in crop plants today, but producing sterile offspring also means producing seedless offspring, which is desirable for many consumers. Also, these sterile offspring can be larger in volume, further increasing their market value.
ID061	If a human has more X chromosomes or fewer X chromosomes than they should, this does not cause much of an impact on their phenotype because of Barr bodies. Only one X chromosome may be expressed in each cell, with the other X chromosomes instead being coated with RNA and becoming Barr Bodies that have no impact on the cell's genetic expression. This is why increases or decreases in the X chromosome have little effect on human phenotype.
ID061	Anueplodiy is the gain or loss of one or more chromosomes but not the complete set. These chromosomal variations originate as a random error during the production of gametes, through nondisjunction. For a human to survive despite anueploidy taking place, the deletion of a chromosome must not be lethal and the addition of a chromosome must not cause death in the individual. However, minor alterations in a human's genetic content will cause phenotypic changes. The small rate of survival for anueplodic individuals is because humans need a precise diploid complement of chromosomes to maintain equilibrium in the expression of genetic content during development.
ID061	It is very common for plants sold at grocery stores to be triploid rather than haploid. This is because a triploid plant would be unable to reproduce and not produce seeds. Examples of this include some potato species, Winesap apples, bananas, and seedless watermelons. The fact that these plants have been grown as triploid makes an economical impact and is practical for our diet. Plants that are the result of allopolyploidy can be more successful at living in their environment and thus more productive.
ID030	Female humans have two X chromosomes, each containing genes. However the genes on both chromosomes being transcribed would lead to dosage issues for the organism. Therefore, one X chromosome in inactivated at a very early stage in development. As a result, additional or fewer X chromosomes do not cause large phenotypic changes.
ID030	Autosomal anueploidy most often comes about as a result of non-disjuction, a rare error during cell division. However, it can also occur when a portion of one chromosome is attached to another, resulting in a single large chromosome.
ID030	Plants often survive changes in chromosome number that animals can not. This fact is used to the benefit of farmers who intentionally breed polyploid crops to increase their size.
ID131	A person is not generally severely affected by multiple X's because one is muted and wrapped in RNA so only one generally gives effects. Which one seems to be completely random.
ID131	Autosomal aneuploidy is often lethal in humans and mammals. It occurs when there are extra chromosomes, and three somewhat commonly exist in humans, but they are not without phenotypic effects. They normally cause physical differences and mental retardation.
ID131	Whereas animals often die from any extra chromosomes, plants can withstand and even thrive with new chromosomes or entire new sets. This has given rise to many plants we have today such as cabbage, wheat, and corn as they have been so successful with new chromosomes.
ID053	Because of Barr bodies and the inactivation of X chromosomes when there are 2 or more, changes in the number of X chromosomes don't have much of an effect on human phenotype.
ID053	Autosomal aneuploidy usually results in death of mammals before birth.  In humans there are very few instances of autosomal aneuploidy where the individual survives for more than a few months.  Down syndrome is one of these specific scenarios.
ID053	Polyploidy plants are often used by plant breeders to alter characteristics of plants, such as making bananas and watermelons seedless.
ID035	Changes in chromosome number are of two basic types, changes in whole chromosome sets and changes in parts of chromosome sets.
ID035	BPA has also been shown to induce epigenetic changes in developing mammals that differ between species and vary with dosage
ID035	Plants tolerate an abnormal genetic content but as a result they often manifest unique phenotypes.
ID050	They have little effect due to the fact that they have no partner chromosome.
ID050	It is rare occurrence due to the fact that most chromosome separation are highly controlled throughout meiosis so the occurrence of one is considered a defect..
ID050	The increase of these chromosome increases deal with with the fact that if the chromosome set is simply doubled it can lead to plant fertility and the plant will often be a much stronger and viable plants as well.
ID143	Humans can live with many X chromosomes, but at least one X chromosome is needed for the organism to be viable.  If a human has more than one X chromosome, it is rare that he or she will develop phenotypic changes; however, some changes can occur.
ID143	Autosomal aneuploidy occurs when changes in the autosomal chromosomes of an organisms are altered; that is, these mutations or changes in the chromosomes do not involve the sex chromosomes of the organisms.  This process is rare, but this type of aneuploidy may cause some specific types of phenotypic changes in mammals and humans.
ID143	Plants are an extremely different type of organism in a sense that they can develop increases in an entire set of chromosomes and maintain viability.  In fact, many of the crop plants used today have obtained increases in either single chromosomes or entire sets of chromosomes because they provide larger component parts than regular plants.
ID090	When individuals receive too many autosomal chromosomes, the result is usually death. However, individuals who receive extra X chromosomes are not as severely effected. This is largely due to the “inactivation” or silencing of the extra X chromosomes. The inactivated X chromosomes are called Barr bodies. The number of Barr bodies is always 1 less than the number of chromosomes. Not only do Barr bodies/ inactivated X chromosomes make syndromes with extra X chromosomes less severe, but they also allow for dosage compensation between males and females so that females do not have twice as much gene product as males.
ID090	In Autosomal aneuploidy, an embryo gains or loses one or more of their autosomes, but not the entire set. This is most often due to a separation error in meiosis called nondisjunction. In most cases, Autosomal aneuploidy condition results in miscarriage.  Monosomy (loss of a chromosomes) can occur in the sex chromosomes (XO) but is not tolerated in humans in the autosomes (lethal). Trisomies are, however, observed in humans (three copies of a chromosomes). It has only been observed in chromosomes 21, 18, and 13. It is unknown why only these three trisomies can be tolerated, but is thought that the size of the chromosomes as well as the number and type of genes present has some effect.
ID090	Plants can tolerate both increase of single chromosomes as well as entire SETS of chromosomes (which is always lethal in humans). In polyploidy (multiple chromosome sets) odd numbers of chromosome sets are not usually tolerated since there is an imbalance in the number of homologs which will produce genetically unbalanced gametes. Even numbers of chromosome sets are better tolerated since they will produce balanced gametes. When the number of chromosome sets within a cell are increase, the cells size also increases. Many of our crops are autopolyploids (multiple sets of chromosomes that are identical to the parent species) and are larger than the “normal” diploid crops. This is not due to more cells, but larger cells since each individual cell in an autopolyploid has to contain multiple sets of chromosomes. This increases the size of plants making them ideal for crops since the farmers increase the mass of their crops.
ID052	Nondisjunction may also occurs in the X and/or Y chromosome, leading to abnormal phenotypic expressions. For example, the gain of an extra X chromosome may lead to Klinefelter's syndrome. The loss of a X chromosome may lead to Turner's syndrome.
ID052	Autosomal aneuploidy is the result of nondisjunction. For example, Down's syndrome is a result of having abnormalities with the autosomal chromosome pair 21. A person with this defect will have an extra chromosome in this group. This condition is also known as trisomy 21, which indicates that group 21 has an extra chromosome.
ID052	Many plants are able to contain multiple sets of chromosomes which is termed polyploidy. Polyploidy is common is plants because they can reproduce asexually.The production of viable gametes is due to asexual reproduction.
ID091	I honestly do not know  how to explain this topic. I thought that an increase or decrease in X chromosomes causes a variety of syndromes such as
ID091	Autosomal aneuploidy rarely occurs in mammals because the the zygotes are spontaneously aborted.  Aneuploidy is certain situations, such as trisomy 21. Down syndrome is a result of trisomy 21.
ID091	Plants can support multiple sets of chromosomes in numerous ways. Excess gene products result in phenotypically different plants. Strawberries, for instance, grow bigger when they are triploids. However, they can not produce viable gametes.
ID044	Increasing or decreasing the X chromosome does not have a huge affect on phenotype because when there are more than one X, only one of the X chromosomes are activated.  All the others are deactivate and turned into Barr Bodies.  For a female, if they have more than 1 X, they only need one to be a phenotypic female, so the others do not affect this.  For a male, the Y chromosome determines the male sex so duplicate X chromosomes also do not affect the male phenotype drastically.
ID044	Autosomal aneuploidy is rare because most of these mutations are lethal so the fetus never survives.  In humans, we see a few cases where autosomal anuepolidy occurs and is not entirely life threatening.  We usually see trisomy, where the fetus will have three copies of 1 chromosome.  In trisomy 21, the person has all normal chromosomes except at 21, where they have three instead of the pair of 2.
ID044	Plants are capable of dealing with more and odd numbers of chromosomes. Plants can become autotriploids, which sterile, but they can also be crossed and hybridized to make autotetraploids, which regain their fertility.  These plants are not sterile and can produce gametes.
ID081	X chromosomes are inactivated at random early in development these inactivated X chromosomes are called barr bodies. Therefore and increase or decrease in an X chromosome has little affect since all but one is inactivated at an early age.
ID081	Aneuploidy is the variation in chromosome number. Organisms can gain or lose one or more chromosomes but not the entire set. Monosomy and Trisomy are examples and are often caused by nondisjunction.
ID081	Polyploidy is a change in an an entire set of chromosomes. Usually this change is an increase. For plants, instead of this being lethal, it causes them to become sterile. This means they are seedless. As a result, seedless fruit, like watermelon, bananas, and grapes are used in today's society.
ID003	the increase of X has more of an effect
ID003	an increase in a set o chromosomes makes
ID107	Decreases or increases in the X chromosome have little effect on human phenotype because individuals with more than one X chromosome have only one active X and all the other X chromosomes are inactive in the form of Barr bodies. You only need one X to be active to have gene expression, so therefore people with 1,2,3,4 (etc) X chromosomes would have similar levels of gene expression.
ID107	The expression of some autosomal genes are balanced with the expression of other genes. The ratios of gene products must be maintained for proper cell function. Autosomal anueploidy can mess up the gene product ratios and lead to problems in meiosis to produce viable offspring.
ID107	Plants can tolerate increases in chromosomes because plant polyploids can often still produce viable offspring. Most offspring of polyploids are sterile but they tend to be bigger and seedless so they are good crop plants to sell. Fertility can be restored to sterile plant hybrid by doubling the chromosome set.
ID118	Extra X chromosomes in human is not that evident phenotypically as the extra X- chromosomes form Barr bodies, which most of time are inactivated due to X-inactivation center, but 15% of barr bodies escape inactivation. so Barr bodies is number of chromosomes-1, so only one X- chromosome is active in normal female with XX. decrease in X chromosomes have little effect as when only one X and one Y chromosome is present there is no Barr bodies. Monosomic for X chromosomes doesn't survive except turner syndromes.
ID118	Aneuploidy is variations in chromosomes number. organism gain or loses one or more chromosome and not entire set. Monosomy is lost of single chromosome in diploid genome and Trisomy is gain of single chromosome due to non disjunction. They are also Mosaic which means that they show one phenotype in one tissue and another phenotype in other part of body.
ID118	Plant tolerates increase in both single chromosomes and entire sets of chromosomes as they double their chromosomes when the hybrid is form by fertilization of two different number of chromosomes. By doubling the chromosomes the sterile hybrid forms a fertile hybrid, so no genetic dead end and they continue to form gametes and pass down genetic information to next generation.
ID092	Whenever an individual has more than one X chromosome, the phenotypic differences that results are usually very mild and not too obvious.  This is because when there are multiple X chromosomes present, the inactivation and presence of barr bodies allows only one X to be active.  When there is a decrease in the number of X chromosomes, the same results usually follow.
ID092	Situations where autosomal anueploidy occurs in animals is rare because the effects it has on the organism are very deleterious.  Typically, the larger the chromosome then the worsened phenotypic expression.  Usually monoploid is always lethal while triploid has very sever phenotypic consequences.  One example of it in humans is down syndrome where the most common form is for a person to have 3 copies of chromosome 21 leading to many physical and developmental problems.
ID092	Plants are much better suited to be polyploidy because the results of, say, tetraploidy do no affect the plants physical body as much as it would in humans considering we are so much more complex.  With that being said, many of our current crop plants are polyploidy because they are able to still produce viable crops that are usually enhanced to the normal diploid counterparts.  An example of this is when bananas are triploid they produce no seeds because the gametes of the fruit are not fully formed.  These plants are usually sterile also.
ID074	many people are born with abnormal copies of the x chromosome and it has little affect on the phenotype of the individual. this is because extra x chromosomes and deactivated and stored away in the cell.
ID074	Aneuploidy is the presence of an abnormal number of chromosomes in a cell, such as having 45 or 47 chromosomes when 46 is expected. The most common chromosomes in which autosomal anueploidy occurs on is 18, 13, and 21 in which the addition of and extra 21st chromosomes results in downs syndrome.
ID074	it is very common for plants to be subject to polyploidy whether is occurs in self fertilization or when a hybrid is being made. Wheat is a good example of this.
ID149	Increases or decreases in the X chromosomes do not have much effect of human phenotype because one X chromosome is only active in each cell. The other (or others) is/are inactive.
ID149	First, it's spelled aneuploidy. Examples of this are Trisomy 21 (Down's Syndrome), Turner's Syndrome, and Klinefelter Syndrome. It's rare because most of the time having multiple or not enough chromosomes is lethal.
ID149	A lot of seedless fruits are trisomy. I'm not completely sure how to explain this topic.
ID098	An increase or decrease in the number of X chromosomes usually results in only mild clinical significance. This is because the extra X chromosomes will be converted into Barr bodies, and will contribute very little to gene expression. Therefore, their impact is mild.
ID098	Autosomal aneuploidy occurs when an organism has one copy of chromosomes (monosomy), three copies of chromosomes (trisomy), or multiple copies of their entire diploid set of chromosomes (polyploidy). While these conditions are well-tolerated in plants, they are extremely rare in humans and other animals, and they often end in a failure to develop correctly and spontaneous abortion.
ID098	Plants tolerate increases in the number of chromosomes much better than humans/animals. This tolerance has come to play a large role in agriculture, as many of the characteristics that we now take for granted in fruits and vegetables (such as no seeds, large bodies of fruit, and longer shelf-lives) are the result of increases in the number of chromosomes present in the plant.
ID151	Increases or decreases in the number X chromosomes such as in Klinefelter’s and Turner’s  are known to create an effect on the individual afflicted, but that effect itself is on a smaller scale. For instance, an individual with Klinefelter’s Syndrome usually has a genotype of XXY (47 chromosomes) thus having at least two X chromosomes; whereas, an individual with Turner’s Syndrome simply is presented as XO (45 chromosomes) in which the O represents the lack of a sex chromosome. The reason why the majority of these individuals are relatively normal, physically and mentally, even though there is an abnormal chromosomal number is due to the fact that only one X chromosome remains active and the other X chromosome is generally condensed and packaged away to the side of the nucleus (known as Barr Bodies). Therefore, the lack of one X chromosome is synonymous to males in the sense that there are no Barr Bodies. The amount of Barr Bodies is one less than the actual number of X chromosomes present. A possible hypothesis for the presence of some abnormalities within the individuals afflicted with similar conditions is that the possible expression of the genes on the extra chromosome before the processing and packaging of that chromosome to the side of the nucleus.
ID151	Organisms, in general, with the exception of plants do not necessarily tolerate any variation from the general chromosome number of that particular species, thus, leading to spontaneously aborted fetuses or death within that individual within a short period of time. However, in humans certain scenarios are known to defy the statement above since the individual can tolerate the excess or the lack of an autosomal or a sex chromosome(s). For instance, trisomy 21 which is also known as Down Syndrome in humans serves as an example in which three chromosome 21 can exist and that individual can function normally although there is some slow mental development. Two other trisomies are all tolerated as well within the human species and that is trisomy 13 (Patau Syndrome) and 18 ( Edwards' Syndrome). The two disorders described above along with a disorder that results in XYY or even XXX and so on is also tolerated within the human population.
ID151	Plants are generally known for their ability to tolerate triploidy and other genetically related disorders that humans may not be able to tolerate. For instance, a regular watermelon with the proper number of chromosomes is known to produce seeds within that watermelon; however, increasing the set of chromosomes from two to three produces seedless watermelons. Therefore, such manipulation within the fruit is normal to enable populations to eat the food without the seeds. Also, some fruits such as strawberries and grapes become in icreasingly larger in size with variation in chromosome number those allows for preferred consummation by the human population.
ID145	This is because usually only one X chromosome is expressed in each cell.  Other X chromosomes are turned off.
ID145	Anueploidy is rare because many instances of anueploidy do not survive to term.  A difference in the chromosome number usually results in death of the fetus.
ID145	Increases in chromosomes leads to larger fruits or seedless fruits, both of which are preferred by farmers.  Increases in chromosomes in plants can occur naturally or artificially.
ID085	In the case of triplo-X, there is an increase in X in number of X chromosomes, and the phenotype of the affected individual changes slightly, but for the most part, they lead normal lives. This is female differentiation. In male differentiation, addition of an X leads to Klinefelter's syndrome. In this case, the men are more feminine phenotypically, but are still men.
ID085	Autosomal aneuploidy is when a chromosome is either missing or there is an extra chromosome in a gamete. In humans, the only autosomal monosomy  that exists is Turner syndrome - all others are fatal.
ID085	Many of the plants found in the grocery store are triploid. The reason for this is because autopolyploidy will lead to larger plants, while simultaneously rendering them sterile.
ID123	Increases or decreases in the X chromosome have little effect on the human phenotype because the X chromosome is a copy of itself.  The increases and decreases are caused by chromosomal abberations, and usually have little effect on the human phenotype because the X chromosome is either duplicated or reduced.  The X chromosome is a copy of itself.
ID123	Autosomal anueploidy is the presence of an abnormal number of autosomal chromosomes.  It is very rare because it is caused by an error in cell division.  A specific example is Down syndrome.  It is the presence of an extra chromosome 21.
ID123	Plants tolerate chromosomal increases because the increases in chromosomes are not lethal to them.  An example of this would be the strawberry of the 21st century.  Many of them have increases in chromosome sets which allows them to be twice as big as their counterparts.
ID070	The nuclei of human cells contain 22 autosomes and 2 sex chromosomes. In females, the sex chromosomes are the 2 X chromosomes. Males have one X chromosome and one Y chromosome. The presence of the Y chromosome is decisive for unleashing the developmental program that leads to a baby boy.
ID070	An aneuploid is an individual organism whose chromosome number differs from the wild type by part of a chromosome set. Generally, the aneuploid chromosome set differs from wild type by only one or a small number of chromosomes.
ID070	manipulation of chromosome number is routinely used by breeders to improve agriculturally important species.
ID144	Increases and decreases in the X chromosome show phenotypic effects on humans. Turner's syndrome shows phenotypic effects of underdeveloped breasts and usually being short in stature. In Klinefelters Syndrome men are going to experience some female like phenotypic features.
ID144	Down syndrome is an example of autosomal anueploidy and it occurs on the 21st chromosome.
ID144	Increasing either a full set or a single chromosome comes with benefits to plants. We can use this to our advantage and produce food that is sterile and not produce seeds.
ID055	Bar bodies helped to inactivate the extra X chromosomes within the genes therefore it has little to no effect on the phenotype of a person with extra X chromosomes.
ID055	There is a variation in the chromosomal number either one or more chromosomes have been lost or gained. This usual results in complications for the organism.
ID055	These type of increases in the plants are used to produce different variations amongst the same species of plant. Also, this is used to increase the size of particular plants and select for specific characteristics.
ID097	The amount of X chromosomes has an effect on the phenotype, such as an absence of a chromosome is categorized as a lethal gene.
ID097	This proves that the human genome is fragile, and any alleles that affects the whole chromosome creates a negative impact on the human's phenotype which decreases their life expectancy.
ID097	Plants are able to adapt and produce better than mammals because the way they receive energy and a microscopic level as well.
ID127	The reason this is true is because of "dosage compensation". For example, since a genotypic normal male has one X chromosome and a normal female has two XX chromosomes, the male and female will only get the same expression if one of the females X chromosomes did not count. Otherwise, hers would be expressed twice as much as the male's. Barr bodies help this situation. Barr bodies are when one of the X chromosomes is inactivated. The equation for this is "n-1" so the male fully expresses his X chromosome, while one of the female's is inactivated (2-1=1). It is like they both only have 1 X chromosome.
ID127	This is probably because there is an average of 6-8 lethal genes. If humans only have 22 autosomes, this means that close to half of them could be lethal if we were missing a chromosome in a pair(monosomy) and the other chromosome was haploinsufficient.
ID127	Plants can tolerate aneuploidy because they can change number of chromosomes while staying multicellular. Like changing from a diploid sporophyte to a haploid gametophyte. They are used to changing like this in generations which is why they can tolerate it. The phenotype usually results in larger crops which is why it is used today.
ID078	It has little effect because the extra X chromosomes become inactivated
ID078	It is rare because improper dosages of many genes are lethal
ID078	This is seen in mustard plants where the difference in the number of chromosomes resulted in entirely different plants
ID077	Increases or decreases in the X chromosome are due to nondisjunction. Both increases or decreases in the X chromosome can lead to unusual phenotypes on humans compared to the normal XX or XY phenotypes.
ID077	Aneuploidy is when an organism gains or loses one or more chromosomes but not a complete set. Most members of diploid species have 2 haploid chromosome sets, so when there is a change in the number of chromosomes unique and rare genetic outcomes come about.
ID077	Geneticists have produced various hybrids of plants. Polyploidy is where more than two multiples of haploid chromosome sets are found. There are autopolyploids and allopolyploids which differ in the composition of their chromosomes. For example, polyploid plants are often larger and have larger components than their diploid relatives which is the main reason they are used for crop plants today.
ID017	In terms of the increases or decreases in the X chromosome, they have little effect on human phenotype. If males do not have the correct number of X chromosomes it is not good because they would lack the genes on the X chromosome, which account for a lot of genes attributed in the male. The X-inactivation even in females occurs, but is less dramatic.
ID017	This topic dealing with aneuploidy is rare in mammals, but occurs in very specific scenarios in humans due to chromosomal abnormality. For example, an aneuploidy in humans occurs in humans who have Down Syndrome. This aneuploid is the abnormality of the twenty-first chromosome, an autosomal chromosome, in which during mitosis the chromosomes are split indifferently and the child has three chromosomes on the twenty-first, resulting in a total of forty-seven chromosomes (uncharacteristic in humans) and not forty-six.
ID017	For plants that exhibit polyploidy, they are not affected as much as for example, humans, due to their lack of genetic complexity. The presence of polyploidy in plants, such as seedless watermelons, can increase in fruit size, but the three sets of chromosomes in the triploid plant prevents meiosis from taking place and thus these sorts of plants cannot produce fertile gametes.
ID034	increases or decreases in the X chromosome have a lot of effect on human phenotype. they are created new types of syndromes (Klinefelter syndrome, Turner syndrome), which make human's genetic variety
ID034	most embryos cannot survive with a missing or extra autosome and are spontaneously aborted
ID034	plants with a autopolyploidy is made bigger, and produce more quality as well as quantity
ID095	We know that the cell has a mechanism to stop the amount of gene expression regarding the x chromosome.  This feature allows for normal phenotypes regardless if there are more than one x chromosomes in the cell.
ID095	Most often when aneuploidy occurs in mammalian cells, the cell will not grow.  This is a lethal feature and growth never even begins in the first place.
ID095	Plant cells have the ability to form Barr bodies, and these mechanisms pull the excess DNA to parts of the cell where they are out of the way, so to speak.
ID096	While increases or decreases on the X chromosome will alter the genotype, it will have little effect on the phenotype.
ID096	Anueploidy involves deletion of part of all of a chromosome. Autosomal Anueploidy does not occur in the sex chromosomes, which is why it is very rare in mammals and humans.
ID096	Altering the chromosome or entire sets of chromosomes in plants can change the genotype and phenotype of the plant, yielding new traits that are better suited for the environment, pest resistance, etc.
ID067	Individuals with more than one X chromosome maintain only one active X chromosome and inactivate all others in the form of Barr bodies. Thus, the extra copies of the X chromosome do not cause an imbalance in the dosage of functional genes, and people with 1, 2, 3, or 4 X chromosomes all have similar levels of gene expression for most X-linked genes.
ID067	extra x chromosomes in mammals are inactivated; so the expected aneuploidy in these animals is said to be less detrimental. Aneuploidy in humans produces serious  developmental problems that spontaneous abortion results.
ID067	The loss or gain of one (or few) chromosomes is the result of non-disjunction and if it occurs in Variation in chromosome number is common in both plants and animals and the effects are various: a gamete can lead to individuals with less or more chromosomal material than is normal for the species. An increase or decrease in a chromosome is termed aneuploidy.
ID011	Extra X chromosomes cause little to no change in humans primarily due to the deactivation of X chromosomes via Barr bodies.
ID011	This rare instance can cause drastic changes in a phenotype usually results in crippling diseases in humans.  One example of this is Down syndrome where one of the most notable symptoms is mental retardation.
ID011	Plants can tolerate these changes in chromosomes although certain changes can cause the plant to lose fertility which would leave the plant sterile.  This can usually be fixed by doubling the chromosome count.
ID141	The reason that increases or decreases in the x chromosome have liitle effect on human phenotype is because if there are extra x chromosomes, they are inactivated and moved to the edge of the nucleus.  These inactivated x chromosomes are known as barr bodies.  The reason that extra x chromosomes have any effect at all is because as many as 15 percent of the human x chromosomal genes actually escape inactivation.
ID141	In humans, autosomal anueploidy only occurs for 3 specific syndromes.  Down syndrome, Patau syndrome, and Edwards syndrome.
ID141	In most animals, changes in entire sets of chromosomes is usually lethal, but is fairly common in plants.  The reason that polyploidy is so popular in plant breeding is due to the fact that the extra sets of chromosomes make the plants leaves thicker while also making the plants fruit larger.  For example, strawberries and grapes double in size or grow even larger when comparing a tetraploid to a diploid.
ID142	Increases in the X chromosome have little affect on the human chromosome because they become inactivated at the very early stage.
ID142	Increases in chromosome number can actually result in larger food because more genes are being expressed. This is especially used for crops so that the food produced is a lot larger than average.
ID008	As The number of Barr Bodies that a individual has increases, the phenotype doesn't alter much due to the fact that the extra barr bodies are already encoded for in the genes
ID008	This can occur in cancer cells with humans causing related diseases.
ID008	Farmers use these crop plants in order to have bigger portions of that crop. They alter the gene expression.
ID016	Since humans have the ability to mask one X chromosome, the effects of having multiple are generally negligible. Although there are disorders which involve having multiple X chromosomes, it is not as prevalent as having multiple autosomes.
ID016	The human autosomes exist in a delicate balance. Any disturbance can result in lethality. Generally, there are only a select trisomies, and monosomies are almost always lethal.
ID016	Plant can handle aneuploidy since their offspring isn't as delicate as a zygote in the womb. Also, plants undergo alternation of generations, which allows the precision of chromosome movement during reproduction to have a wider window of freedom.
ID080	The X chromosome doesn't affect the phenotype very much because regardless of how many are found in a specific organism only one is truly expressed. This is due to Barr bodies compressing the DNA of any additional C chromosome after one.
ID080	It is rare because any autosomal defect usually results in death of the fetus. This can only be seen in a small percentage of cases because the chances of survival are dismal in other cases.
ID080	The reason that polyploidy plants are found more often than their diploid counterparts in our crops is because they tend to be larger than plants with fewer chromosomes. This makes them more desirable for farming.
ID108	Increases or decreases in the X chromosome have little effect on human phenotype because X chromosomes can be expressed or not expressed in certain areas.
ID108	Autosomal anueploidy is rare in humans and mammals because most are lethal before age 2 or create a natural abortion.
ID108	Plants tolerate increases in both single chromosomes and entire sets of chromosomes and most commercial fruits are polyploid since it creates a larger or "seedless" fruit.
ID086	These X chromosomes create Barr bodies that are inactivated X chromosomes. For the most part they do not effect human development.
ID086	This is a condition where the chromosomes other than X and Y have extra numbers.
ID086	These plants are normally larger and more fruit bearing than the standard plants.
ID073	Individuals with more than one X chromosome maintain only one active X chromosome and inactivate all others in the form of Barr bodies. Thus, the extra copies of the X chromosome do not cause an imbalance in the dosage of functional genes, and people with 1, 2, 3, or 4 X chromosomes all have similar levels of gene expression for most X-linked genes.
ID073	The expression of some genes is balanced with the expression of other genes; the ratios of their gene products (proteins) must be maintained within a narrow range for proper cell function. Extra copies of one of these genes cause that gene to be expressed at proportionately higher levels, thereby upsetting the balance of gene products.When the anueploidy occurs in autosomal chromosomes, the changes usually are expressed phenotypically and are most instances lethal.
ID073	Most polyploidies have an even number of chromosome duplications.  By providing duplicated genes, polyploidization
ID102	The increase or decrease in an X chromosome can cause the individual have feminine and masculine like features. Or rather the male will have women features if there is an increase in the X chromosome. Not too certain on what would happen without the X chromosome.
ID102	Autosomal anueploidy causes either death or a disorder of the individual especially in humans. The extra chromosome depending on which one causes the disorder leaving the individual not like your "average" human being.
ID102	This is a result of polyploidy. During the meiotic phase non-reduction occurs as well as somatic doubling in the mitotic phase. Not too sure on this either.
ID100	Increases or decreases in the x chromosome have little effect on human phenotype because extra x chromosomes are turned off. The existance of barr bodies makes this phenomenon possible. Barr bodies inactivate X chromosomes.
ID100	Autosomal anueploidy is condition where an individual has an abnormal chromosome number. It is rare in mammals because monosomy allows for lethal alleles to take over and cause death.
ID100	Plants are better able to tolerate an increase or decrease in chromosome number than humans. Plants that show an increase in entire sets are often time advantagous over normal individuals.
ID110	aneuploidy, which is the term used to describe an increase or decrease in chromosome, but not a complete set.
ID110	autosomal aneuploidy occurs in Patau and Edward's syndrome. these syndomes have a very high incidence of lethality therefore its rareness can be attributed to the death of the offspring before it is able to reproduce.
ID110	due to plants shorter life span, higher rates of mutagenic crossing over is able to occur from one generation to the next. this allows humans to induce through commercial hybridization a more economical version of the foods we eat today through genetic engineering.
ID111	This is what happens with X-chromosome inactivation. This is the process of transcriptionally silencing one of the X-chromosomes in a complex and coordinated manner. The multiple x's are still there, however one is in a silent stat and the other one is expressed.
ID111	Anueploidy is having an abnormal amount of chromosomes in a cell. This can be either more or less than the normal 46. This happens during cell division and causes many genetic disorders. These are rare because the offspring typically do not survive to the birth stage.
ID111	Having different numbers of chromosomes is why we have variations of the same type of plant. Sometimes having more chromosomes does not affect the plant, but sometimes it does. Genetically modified foods are really big today because scientists can modify the plants to make them more suitable for what people want. (ex seedless watermelons) These are widely used today because they can be grown in mass quantities and in a more controlled environment.
ID075	The reason increases and decreases in the X chromosome have little effect on human phenotype is because of dosage compensation. The extra Xs are inactivated so they are not proven to be very detrimental.
ID075	Autosomal aneuploidy refers to all chromosome abnormalities that do not involve sex chromosomes.Autosomal aneuploidy is rare in mammals and in humans because they are lethal. Most are spontaneously aborted.
ID075	Increases in entire sets of chromosomes do not detrimentally affect plants the way it does to humans. In fact, in some cases, it helps them. Allopolyploidy plants become fertile after the chromosome number in doubled.
ID128	Changes in the number of X chromosomes in humans are very different from changes in numbers of autosomal chromosomes. In humans, changes in autosomal chromosome numbers are often deadly. With the exception of a couple of trisomies like on the 21st chromosome, polyploidy and all forms of monosomy result in spontaneous abortion. However, increases in numbers of X chromosomes in humans have little phenotypic effect because X chromosomes can be deactivated.
ID128	Autosomal anueploidy is rare in humans and other mammals because it almost always results in death of the individual. Anueploidy is the presence of an abnormal number of chromosomes which in human autosomal chromosomes is fatal except in a few cases such as Down Syndrome which is trisomy 21.
ID128	In plants, extra chromosomes are not only survivable but often beneficial. Extra sets of chromosomes can result in large number of fruits which is goal when growing plants today. Also, variations in the chromosomes can result in seedless fruit like bananas and watermelons.
ID138	Since in females one of the X's is inactivated anyways, XO genotype can lead a relatively normal life if they survive birth. With increases in the X chromosome you would see that more than one X was inactivated. This is what we talked about when considering Barr bodies when you see multiple Barr bodies you know that there were more than the normal number of X's, or if you see 1 Barr Body in male you know that there is an increased number.
ID138	In general, cell reproduction is a very accurate process that results in very few mistakes. The most common mistake that can lead to aneuploidy is nondisjunction in meiosis I or II which causes you to either lose or gain a chromosome.
ID138	Plants are able to tolerate increases in chromosomes which is often lethal to mammals.  This is caused by the fact that the plants reproduce asexually. This allows plants to balance out when a chromosomes are lost or gained. Another gamete of the same type can form and will combine with the other which will allow homologous pairing.
ID018	Because the x chromosome can be silenced as a Barr body, increases or decreases in the chromosome don't have many phenotypic effects.
ID018	Because autosomal anueploidy can be detrimental to mammals, it occurs less often.
ID018	Plants can have polyploidy and not be negatively affected by it. The phenotypic effects can be beneficial to farmers today, so it actually can be favorable when growing crops.
ID047	IT usually effects the genotype, but hardly affects phenotype because females have two x chromosomes and males just need one
ID047	Anueploidy is a variation in the number of chromosomes in a set.  It can be lethal and is rarely seen.
ID047	This is possible in plants because they are reproductively successful in forming balanced gametes
ID119	We do not often see a  drastic change in a person's phenotype as a result of either increasing or decreasing in the X chromosome.
ID119	Anueploidy is when there is an addition or deletion of a chromosome. Autosomal aneuploidy is when the chromosomes involved are NOT sex chromosomes. This occurrence is rare in humans and in mammals in general and can be fatal in many cases.
ID119	When there is an accidental addition or deletion of a chromosome or set of chromosomes it is called an anueploidy. Plants are able to thrive off of increases in chromosomes.
ID059	With an increase in the X chromosome, there should be little effect on the human phenotype because all of the necessary genetic information that is needed is there. However, more x chromosomes in males can result in them having a phenotype similar to that of a woman due to the genes that are activated to prevent certain testes development etc.
ID059	Anueploidy is an example of genetic disorders as well as the  cause of some cancer developments. In the case of Down syndrome there is an extra copy of chromosome 21. Because of this extra copy, there is an excessive amount of genetic information that causes mental, physical, and intellectual disabilities.
ID059	Potatoes have 48 chromosomes, wheres as the human body has 46. This difference in chromosomal number does not effect potatoes ability to function or survive. For the most part its the outside, environmental factors that plays a role in the increase of certain functions of genes within the human and plant species.
ID072	People with 1-4 X chromosomes all have similar levels of gene expression. The number of the X chromosome have little effect on the human phenotype since there isn't much difference between the number of chromosomes and gene expression.
ID072	Autosomal Aneuploidy is rare and it is due to the variation between chromosomes on a certain set. One example is the trisomy 21 which causes Down syndrome
ID072	This is due to genetically modified crops. They are genetically enhanced with different genes that allow them to be unaffected by different types of chemicals or bugs.
ID132	Well, all female will have mild phenotypic effect because in each cell all the extra X chromosomes are inactivated. The number of barr bodies is the number of X chromosomes minus one. In drosophila, the Y chromosome doesn't dictate the phenotype of its maleness but the absence of the second X chromosome.
ID132	Anueploidy is an error in cell division that results in the daughter cells having the wrong number of chromosomes. Like there is a missing chromosome or other have extras. Most is caused by meiosis and result mostly in miscarriage.
ID132	Plants can tolerate polyploidy rather than humans. Plants can reproduce asexually, it also depends on their environmental factors because of gamete formation. It has made breeding our crops that are polyploid easier but I think we are still researching.
ID021	Life cycles depend on sexual determination, and in humans this means X and Y chromosomes. The Y chromosome causes maleness, a drastic change in phenotype, but the X chromosome can be thought of as the default.
ID021	Autosomal aneuploidy is the increase or decrease in a autosomal chromosome. It is rare in humans and mammals because most are spontaneously aborted. Specific scenarios in humans include Trisomy 21, 13 and 18.
ID021	Plants survive aneuploidy better than do humans. Autopolyploids have duplications of the same genome. Allopolyploids have doubling of the chromosomes in a hybrid of two different species. Desirable crops with this genetic change are strawberries because they are larger and cannot be copied, bananas so they do not have seeds, and flowering plants.
ID120	The X chromosome is only a sex chromosome, and depending if there is a Y or not will determine the sex. The overall phenotype is similiar.
ID120	Anueploidy is a genetic mutation as a result of nondisjunction, where chromosomes do not seperate to opposite polls equally.
ID120	The plants can share other qualities from other plants due to more X's making them more viable for crop production.
ID076	Research shows that eb and flows of ratios of X chromosomes is something that happens and doesn't have a significant effect on the phenotype
ID076	Specific scenarios are required for this phenomena to allow these to survive
ID076	increased in gene pool and fertility
ID009	This is because 2 X's are possible for a normal functioning and phenotypical human whereas too many Y's would definitely be a problem.
ID009	The specific scenarios are mutations or just mess ups that happen typically during mitosis and meiosis. Some are hereditary but most are due to nondisjunction.
ID009	They help produce more. More crop production is better.
ID102	This occurs when the non-homologous chromosomes are being rearrange causes the abnormalities in the chromosome. Translocation is basically trying to even out the chromosome by taking the extra from one and attaching it to the chromosome that is missing that part.
ID102	Said that the DNA n the two looked similar to bacteria, only issue was that there was an insufficient amount of oxygen for the eukaryotes who then took in the bacteria just for that purpose. They couldn't respire aerobically, so the took in something that could.
ID102	Mitochondrial mutations affect the respiration system of chloroplast and yeast. With chloroplast it causes slow growth due to the absence of specific proteins that are needed for electron transport. The same with yeast, there is an abnormal way the electron system works, which gives the colonies the inability to grow to it maximum size.
ID135	Translocations may occur between chromosomes, not only homologous chromatids. When this occurs, it usually ends up in miscarriage, deformities, and developmental issues.
ID135	Endosymbiotic Theory says that mitochondria and chloroplasts were once independent microorganisms that were engulfed by larger microorganisms. However, instead of digesting the mitochondria and then chloroplast for plants, the microorganisms developed a symbiotic relationship in which the larger microorganism provided nutrients for the mitochondria and chloroplast housed inside of its self. In return, the mitochondria allowed the larger microorganism to harvest aerobically formed energy for the first time in history. The chloroplast did the same thing except it creates energy using light from the sun.
ID135	The mitochondria is the powerhouse of the cell because it creates a large amount of energy using oxygen. However, if the mitochondria was defective the body would not be able to effectively utilize the oxygen going into it, which could lead to a plethora of problems.
ID039	Translocation is the change in a segment of a chromosome. When it occurs betwen chromosomes, a segment of a chromosome is transferred to another. Segments can also be exhchanged between the two.
ID039	There are theories that state that the chloroplast and mitochondria began as small free living cells. A larger host cell engulfed them, and instead of digesting the chloroplast or mitochondria, the host cell preserved them because they benefited the host.
ID039	The mitochondria houses a lot of the proteins and enzymes needed for respiration, so when there are mutations
ID094	Transolacations, such as Robertstonian, can occur and cause anwhere from sever to no issues whatsoever in an individual but often cause problems in children.
ID094	Both have their own DNA so it appears to geneticists as though organisms were taken up by these cells and eventually cae to function as a chloroplast or mitochondria.
ID094	Mitochondria defects can be detrimental because of its neccesity to mammalian life; as the site of respiration's most vital steps, mutations can be deadly if improper function is encountered.
ID057	A translocation is a chromosome alteration that results in a whole chromosome or a segment becomes attached to or interchanged with another whole chromosome or segment. There is no gain or loss of genetic material in a translocation.
ID057	There are several observations as to why we believe that the mitochondria and chloroplast use to be free living organisms. This relationship was believed to be a mutually beneficial relationship between the cell and these once free living organisms. Some of the evidence that suggests that both the mitochondria and chloroplasts are ancient symbiotic events is that both of them have their own cell membranes, mitochondria has its own circular DNA that is passed on to it's offspring and they reproduce in the same process that bacteria do. These are all the major lines of evidence.
ID057	Many mitochondrial mutations result in some type of human disease. In petite mitochondrial mutations in yeast causes deficiency in cellular respiration and causes the mitochondria performs abnormal electron transport.
ID070	Translocations are chromosomal abnormalities which occur when chromosomes break and the fragments rejoin to other chromosomes.
ID070	The endosymbiotic event that generated mitochondria must have happened early in the history of eukaryotes, because all eukaryotes have them. Then, later, a similar event brought chloroplasts into some eukaryotic cells, creating the lineage that led to plants.
ID070	Genes in the mitochondrial respiratory chain complex gene family provide instructions for proteins involved in oxidative phosphorylation, also called the respiratory chain.
ID037	In genetics, a chromosome translocation is a chromosome abnormality caused by rearrangement of parts between nonhomologous chromosomes. A gene fusion may be created when the translocation joins two otherwise-separated genes, it is detected on cytogenetics or a karyotype of affected cells.
ID037	Defects of the mitochondrial respiratory chain are associated with a diverse spectrum of clinical phenotypes, and may be caused by mutations in either the nuclear or the mitochondrial genome
ID000	Translocations occur when a piece of one chromosome is transferred to another chromosomes.  Since there is no loss or gain of genetic material, this should have no effect unless there is a position effect.
ID000	Chloroplast and mitochondria have prokaryotic ribosomes and prokaryotic gene structure but perhaps not necessarily circular chromosomes.  These lines of evidence suggest that chloroplasts and mitochondria may have, at one time, been free-living and been acquired by a eukaryotic organism through endocytosis.
ID000	:  Mitochondria are responsible for aerobic respiration in plants and animals.  Both nuclear and mitochondrial genes interact in the normal functioning of mitochondria in generating ATP
ID109	Chromosome translocations happen when two non homologous chromosomes rearrange their parts and cause an abnormality. Translocations can be harmless or they can cause genetic disorders. For example, a translocation of the 21st chromosome can cause Downs Syndrome.
ID109	Since mitochondria and chloroplasts both have their own separate membranes, this is evidence that they were both symbiotic events. They also both have their own genome so that is further evidence that they were both symbiotic events.
ID109	Since the mitochondria is the organelle that does respiration, a mutation in the mitochondria would affect respiration. These mutations in the mitochondria could be detrimental to respiration, or they could not affect it so much. It just depends on the type of mutation.
ID083	Translocation is the transfer of part of a chromosome to a different position especially on a nonhomologous chromosome or the exchange of parts between nonhomologous chromosomes.
ID083	This is called the endosymbiotic theory, which is an evolutionary theory that explains the origin of eukaryotic cells from prokaryotes. It states that several key organelles of eukaryotes originated as a symbiosis between separate single-celled organisms, including mitochondria and chloraplast.
ID083	This happens because the mitochondria is the powerhouse of our cells and it is where ATP/energy is formed
ID079	When a segment of a chromosome is relocated, translocation has occurred.A segment can be relocated to another position in the chromosome, or the segment can be relocated to another chromosome. Translocation can lead to abnormalities, such as heritable Down Syndrome and tumors.
ID079	In an symbiotic relationship,two organisms have a mutual dependency on each other. The endosymbiotic theory suggests that eukaryotes acquired the ability to utilize oxidative respiration and photosynthesis by engulfing bacteria, resulting in a symbiotic relationship. Mitochondria and chloroplasts are significant because they are prokaryotic in nature.
ID079	Most of the process of cellular respiration occurs in the mitochondria. Therefore, mutations affecting mitochondria can hinder an organism's ability to complete cellular respiration.
ID064	Sometimes two chromosomes exchange genetic information by physically exchanging parts of their chromosome. This can either be reciprocal, where both chromosomes lose part of themselves, or nonreciprocal, where only one chromosome loses part of itself.
ID064	These organelles both have DNA that resembles prokaryotic DNA and is different from nuclear eukaryotic DNA and so it has been suggested that mitochondria and chloroplasts were once free-living organisms that evolved instead to be a part of a larger eukaryotic organism.
ID064	The mitochondria generates energy for the cell by producing ATP by cellular respiration, and if any of the products needed to complete this process are disturbed or not produced properly, the organism can suffer because of an inability to respirate.
ID105	In chromosomal translocations no DNA is gained or lost. There are two types of simple translocations intrachromosomal which is a change in position  inside an individual chromosome, and interchromosomal which is a transfer of a chromosomal segment to a nonhomologous chromosome.
ID105	The endosymbiotic theory suggest that the chloroplast and mitochondria arose independently but at the same time from a free-living bacteria.
ID105	Petite mutations which are named after the small colonies perform abnormal electron transport and have a deficiency in cellular respiration.
ID026	Chromosomes can be rearranged through processes such as deletion and inversion. This can alter the genetic code on the chromosome and change the genes that are on them. Some of these changes can be unnoticeable and others can lead to major differences and problems.
ID026	Mitochondria and chloroplasts are said to have originally been their own systems. They were then engulfed by cells and the cells began to use them to help generate their own energy. At the same time, the mitochondria and chloroplasts were given protection and other resources to help energy production, so they remained in the cells.
ID026	Most of the respiration process occurs within the mitochondria of a cell. This means that if something is wrong with the mitochondria or their membranes energy may not be generated as efficiently which can harm the cell. It may also lead to cell death if enough energy is not produced to continue its processes.
ID101	The translocation of a gene from one chromosome to a different chromosome can result in incorrect activation of genes. For if the promoter region is not completely transferred, diseases can arise such as lymphoma.
ID101	Both the mitochondria and chloroplasts have their own unique DNA. They seem to work in cooperation of the host. Their ability to produce energy, or ATP, and yet have its own membrane around it in similarity to other microorganisms. The photosynthetic and anabolic process observed in these organelles is very similar to that of other Eukaryotes such as fungi.
ID101	The mutations that can occur in mitochondria which is passed from the mother to her offspring in humans, reduce the amount of energy that can be produced by each cell.
ID056	A translocation is an altering of the gene that involves the change of the location of a segment of chromosome. No DNA is lost or gained in the process like deletion or duplication. Translocations can occur within one chromosome or between 2.
ID056	Mitochondria and chlorophasts are prokaryotic in nature which suggests that they came from eukaryotic bacterial engulfment. They existed together in a symbiotic relationship and grew to depend on each other and no longer could survive independently. This led to oxidative respiration and photosynthesis in eukaryotes.
ID056	The whole point of having mitochondria is to have a "power house" of a cell. Mitochondria is the location of the electron transport chain which is used in oxidative respiration. Mutations in mitochondria will change the efficiency of the process and will therefore affect the amount of energy we get out of respiration.
ID044	A translocations of a chromosome is an abnormality caused by rearrangement of sections of chromosomes between non homologous chromosomes.  It is the movement of a chromosomal segment to a new location in the genome. The most simple form is a reciprocal translocation, which involves only two breaks and a simple exchange is made.  Genetic information can be lost of gained. The translocation does not directly alter the viability of individuals.
ID044	There is evidence that mitochondria and chloroplasts are symbiotic events because they both have their own membranes, just like prokaryotes.  They both have their own circular DNA, which is passed to its offspring separately form the host's DNA.  Both mitochondria and chloroplasts are inherited through the material cytoplasm.  This is proof that these two things are ancient symbiotic events, which is what allowed mitochondria to become the eukaryotic power house and the chloroplast to become the plants power house.
ID044	Mitochondria mutations can affect cellular respiration because a mutation in mitochondria DNA can affect the ability to produce ATP for energy.  It contains 16,569 base pairs include 13 of over 70 proteins required for aerobic cellular respiration.  Mutations to these base pairs can damage lipids and proteins that are essential for respiration.  These affected mitochondria are inherited by offspring if the mother is affected, but not if the father is affected.
ID013	Translocations are a type of mutation that occur when a part of a chromosome is moved to another chromosome. These mutations can be reciprocal in which both chromosomes break off a piece and get a new piece or nonreciprocal in which only one chromosome breaks off a piece and loses it.
ID013	The Endosymbiotic Theory states that chloroplasts and mitochondria were once free-living small aerobes that invaded larger anaerobic bacteria to create the eukaryotic cell. Evidence for this theory includes their 70S ribosomes, the small size of these organelles, and their single circular DNA. As a result of endosymbiosis, eukaryotes gained the ability to respire and/or undergo photosynthesis.
ID013	Mitochondria have their own single circular DNA and replicate independently of the cell. Also, aerobic respiration occurs on the inner membrane of this organelle.These mutations are passed down from the mother and include the LHON mutation, the MERF mutation, mitochondrial DNA mutations, and mutations that result in heteroplasmy.
ID068	Translocation is a segment of a chromosome that is broken off of one chromosome and move to another chromosome. It can be homologous chromosomes or two different types of chromosomes.
ID068	It is believed that both mitochondria and chloroplasts arose from free living bacteria many years ago. Eukaryotic cells engulfed the bacteria and it provided them with benefits such as respiration and photosynthesis.
ID068	Mutations in the mitochondrial DNA can cause deficiencies in the cellular respiration process of cells. This is because of an electron transport that does not properly function due to the mutation.
ID040	A translocation is a a chromosomal mutation associated with the reciprocal or nonreciprocal transfer of a chromosomal segment from one chromosome to another another. When the translocation is reciprocal, genetic material is neither lost nor gained. Other times, genetic material can be lost with abnormal synapsis or Robertsonian translocations, where the arm of one chromosome fuses onto the arm of a nonhomologous chromosome.
ID040	It is believed that mitochondria and chloroplast arose independently but both possessed skills that the other required. Bacteria were engulfed by larger eukaryotic organisms leading to a symbiotic relationship. The eukaryotic cells gained oxidative respiration and photosynthesis. DNA in chloroplasts and mitochondria look very similar to DNA in bacteria. Chloroplast DNA encodes for many different gene products including distinct ribosomes. Mitochondrial DNA lacks introns.
ID040	Mitochondrial DNA is extremely vulnerable to mutations. It does not have structural protection from mutations by histone proteins. DNA repair mechanisms in mtDNA are limited. Mitochondria concentrate highly reactive oxygen species, which are toxic to the organelle. Mitochondria carry out respiration so when there is damage to the mtDNA, the mitochondria is unable to mask these mutations which affects respiration.
ID038	Translocation is when a segment of chromosome detaches from one chromosome and attaches itself to another another chromosome or another part of the same chromosome.
ID038	Both of these organelles have double plasma membrane and have their own dna which suggests that an organism engulfed these organelles which were organisms first. They entered a symbiotic relationship whereby these organelles help in energy metabolism and in return they get the ingredients.
ID038	Since mitochondria is the center of respiration, any mutations affecting mitochondria would affect respiration.
ID061	Translocations occur whenever a chromosomal segment moves to a new location in the genome. These movements can occur between chromosomes. In a Robertsonian translocation, the short arms of two nonhomolgous chromosomes break off. The two short segments of the chromosomes are lost but the two long segments join together to make on long chromosome. In reciprocal translocations, two nonhomologous chromosomes exchange segments. The information coded on these segments will still be present in the genome as they have simply been relocated. However, in a Robertsonian translocation, the short segments that break off are lost from the genome. Translocations between nonhomologous chromosomes can cause problems with synapsis in cellular reproduction, while translocation between homologous chromosomes that result in iversions can also cause problems in synapsis.
ID061	Both mitochondria and chloroplasts are believed to be ancient symbiotic events. The belief is that both of these organelles were engulfed by larger primitive eukaryotic cells that lacked the ability to respire aerobically or to capture energy from sunlight. As the host cells have evolved, so have the mitochondria in the host cells. The mitochondria have transferred many of their genes to the nucleus of the host cell. Now, the products of the genes are encoded and transcribed in the nucleus before being translated in the cytoplasm prior to their transport to the mitochondria. In addition, the amount of DNA present today in a mitochondria genome is miniscule compared to the free-living bacteria from which it was derived. Similar changes characterize the evolution of chloroplasts.
ID061	Mitochondrial mutations affect respiration because the mitochondria are responsible for generating ATP during aerobic cellular respiration. In fact, mitochondrial DNA are particularly vulnerable to mutations because they have no structural protection from mutations provided by histone proteins present in nuclear DNA, repair mechanisms for mitochondrial DNA are limited, and mitochondria concentrate highly mutagenic reactive oxygen species generated by cell respiration. Since the mitochondria is the organelle responsible for respiration and producing ATP, if the mitochondrial DNA responsible for producing mitochondria is damaged then the mitochondria will not perform its functions correctly and cell respiration will be impacted.
ID053	Translocations of chromosome segments occur between non homologous chromosomes.  This usually has no phenotypic effect on the individual, but it can affect the progeny by altering the number of copies of a certain chromosome.
ID053	Mitochondria and chloroplasts are believed to have joined into eukaryotic cells in a symbiotic relationship in ancient times. This is known as the endosymbiont theory. Evidence supports these organelles having their own DNA separate from the rest of the cell.
ID053	When mutations to the mitochondria occur, oftentimes they affect respiration, especially since that is the main function that mitochondria fulfill. Even minute alterations to the mitochondrial DNA can affect cellular respiration in the produced mitochondrion.
ID069	Translocation occurs when parts of two different non-homologous chromosomes are rearranged. The non-homologous chromosomes may swap material evenly (as in a balanced translocation) or unevenly (as in an unbalanced translocation) where material may be gained or lost for either chromosome.
ID069	The Endosymbiotic theory asserts that mitochondria and chloroplasts arose independently about two billion years ago. This event allowed eukaryotic cells to achieve cellular respiration and photosynthesis.
ID069	Mitochondria has its own DNA, which is highly susceptible to mutation. Because mitochondria is responsible for cellular respiration in a cell, any mutation to its DNA will affect its ability to effectively carry out respiration.
ID115	Translocations sometimes occur between chromosomes when they are near each other in the nucleus during interphase and genetic information is exchanged between nonhomologous chromosomes. This may result in fused segments of chromosomes that lead to aneuploidy in progeny or that may result in sterility or lack of function of the organism
ID115	Both mitochondria and chloroplasts arose independently about 2 billion years ago and were engulfed by primitive eukaryotic organisms that did not have the ability of aerobic or oxidative respiration/photosynthesis. Thus, with the aid of the bacterial cells they gained the ability, and in return the bacterial cells could live off of the large host cell and integrate its information in the nucleus.
ID115	These can occur because the line of defense for mtDNA is limited, as well as repair is not always established and because of mitochondria's role with ROS, it can acquire mutations through the toxic elements. As a result, respiration is affected due to abnormal ETC and ATP phosphoroylation. As consequence, this affects function of the organism and may lead to blindness, deafness, dementia or other conditions.
ID116	Translocations that may occur between chromosomes are generally abnormalities between nonhomologous chromosomes. There are two types, which include balanced and unbalanced. In balanced translocations, the chromosomes are functional because there is no important information missing from the chromosome. In unbalanced translocation, there is some information missing.
ID116	This is called the endosymbiotic theory. The mitochondria and chloroplasts started as independent bacteria that then formed a relationship inside another cell as an endosymbiont. Under this theory, mitochondria developed from proteobacteria and chloroplasts developed from cyanobacteria.
ID116	The respiratory chain is controlled by both mitochondrial DNA and nuclear DNA. Mitochondrial mutations occur at a relatively high rate and generally are inherited through the maternal side.
ID093	Translocation is the rearrangement of parts between two nonhomologous chromosomes. Translocations can result no extra or missing genetic information or an unequal translocation resulting in extra or missing genetic information. A Robertsonian translocation is a type of translocation where the chromosome breaks near the centromere. This results in one large chromosome and one very small chromosome which contains very few genes.
ID093	It is thought that organelles with two distinct membranes were once separate living organisms which became endosymbionts. This is due to the presence of the original membrane and the membrane created when the organism was phagocytized. The fact that DNA is found in both the nucleus and mitochondria suggests that an organisms consumed another which survived and the relationship eventually became mutualistic.
ID093	Since cellular respiration occurs within the mitochondria, mitochondrial DNA mutations may affect respiration. All mitochondria are inherited through the mother's egg. A male with a mitochondrial mutation will not pass down the mutation to his offspring.
ID096	Translocations in chromosomes occur when chromosome fragments that broke off from one chromosome join another chromosome.
ID096	The endosymbiotic theory states that both mitochondria and chloroplasts arose independently from free living bacteria.
ID096	Mutations in the Mitochondria affect proper cell function. LHON, Alzheimer’s, and MERRF, are just a few examples of disorders caused by mitochondrial mutations.
ID023	Translocations(movement of a chromosome to another chromosome) can occur between chromosomes as long as the chromosome arms are close enough to allow translocations. Translocations do not involve the deletion or addition of any genetic information (just its relocation); thus individuals with translocations usually are phenotypically normal, but their offspring can have effects.
ID023	The endosymbiotic theory states that the mitochondria and chloroplast come from prokaryotic cells that were engulfed and formed a mutualistic relationship with an eukaryotic organism. There are many studies that prove that this theory is supported.
ID023	Mitochondria can have a variety of mutations that range from base deletions and point mutations. Since the mitochondria is involved in ATP production via cellular respiration, any slight mutation may affect the organelle's ability to complete cellular respiration. Cells that have this mutation must complete fermentation in order to survive.
ID043	Pieces of chromosomes can become inverted or switched within the chromosome or between two chromosomes.
ID043	It is believed that mitochondria come from bacterial cells.
ID043	The mitochondria is the location where respiration occurs.
ID059	Translocations an occur from the exchange between two chromosomes leaving either a total of 45 chromosomes or an extra copy of chromosomes (47). Translocations can occur at or near the centromere or even at the very end of a chromosome. Either translocation, will/ can result in severe birth defects or abnormalities. For example, there are different types of translocations (robertsonian being one of the most mentioned).
ID059	Mitochondria and chloroplasts are part of a coevolution. Due to the discovery of one parasite being enguffed but not digested in the other, nuclear membranes began to fold and develop the first eukaryotic like cellular structures and with the production of energy source feeding off chloroplast which benefited the host organism; the mutual exchange was an example of how symbiotic events have changed history forever.
ID059	Mitochondrial mutations are the main cause in certain respiratory problems such as lung cancer. It is the mitochondrial DNA is a key factor in the production of production and make the cell function. Though mutations are random, some of the leading causes to the cells overall activity is the fact that metabolism and energy sources are not functioning properly. These pathogenic mutations are the explanation to many diseases today that is still difficult to get a complete explanation of.
ID092	This concept is the idea that sections of chromosomes can break off of their respective chromosome and attach to another one.  This yields genetic variation while at the same time can be detrimental to the organism that it happens to.  Hence the name, "translocation", a segment is literally being relocated to another chromosome.
ID092	It is known that mitochondria do similar activities for animals that chloroplasts do for plants.  For this reason, many scientists believe that both of these organelles have evolved from a common prokaryotic organism long ago.  Both of these produce energy for organisms and are the main power houses in many.
ID092	Mitochondria are the main organelles responsible in cellular respiration as they are responsible for oxidative phosphorylation, which is very crucial in cellular respiration.  When mutations occur in the mitochondrial, OP can be inhibited, diminished, or blocked which results in decreased respiration and can affect the cell in a negative way.
ID090	Translocations occur when genetic material is moved to a new location in the genome. Genetic material is not lost or gained; it only changes location in the genome. Translocations can either involve the exchange or transfer of genetic material between two chromosomes. In a nonreciprocal exchange, one chromosome transfers genetic material to another chromosome and does not receive any “new” material in return. In a reciprocal exchange, the genetic material is “swapped” between the two chromosomes.
ID090	The theory of endosymbiosis suggests that both the mitochondria and the chloroplast were ancient cyanobacteria that were engulfed by a eukaryotic cell. Biologists who support the endosymbiotic theory point to several similarities between the mitochondria/chloroplast and prokaryotic bacteria.  They have their own DNA, which is separate from the DNA found in the nucleus of the cell and both organelles use their DNA to produce many proteins and enzymes required for their function. A double membrane surrounds both mitochondria and chloroplasts, suggesting that each was ingested by a primitive host. The two organelles also reproduce like bacteria, replicating their own DNA and directing their own division.
ID090	The mitochondrial membrane contains the electron transport chain that is vital for respiration. When mutations occur in mitochondrial DNA, it can affect the gene products that are produced for respiration, in particular for the ETC. Defective or absent gene products due to mutant mitochondrial DNA affect respiration since the ETC is usually deficient of the required proteins.
ID075	A translocation means that there is an unusual arrangement of the chromosomes. This can happen because of: a change has arisen during the making of the egg or the sperm or around the time of conception; or an altered chromosome arrangement has been inherited from either the mother or the father. There are two main types of translocations: a reciprocal translocation and a robertsonian translocation.
ID075	There have been many evidence that suggests this mitochondria and chloroplasts are symbiotic events. The chloroplast genome is
ID075	Mitochondrial mutations do affect respiration. When a mtDNA mutation occurs, there is no structural protection from histones; DNA repair mechanism is limited;and high concentrations of ROS generated by cell respiration
ID118	Tranlocation is one of the chromosome abnormality that can occur between non homologous pair. Translocation. It alters the karyotype of the individual. The person who has translocation shows no effect as it has not lost any coding region of the gene, so it has all the information needed to make the translocation person normal phenotypically. The person with translocation are called balanced translocation carrier. The gametes produced by balanced translocation carrier are not normal as it has weird segregation of the chromosome. The translocation carrier and normal person will have 100% chance of having syndromes offspring. so person with translocation of two 21 chromosome will have 100% chance of having down syndrome children.
ID118	Mitochondria and chloroplasts shows evidence of prokaryote organism as they have circular DNA, single cell, no organelles, and small ribosomes. They can also divide by their own. This was just ingesting one cell into another and maintained into another. They were passed down to their offsprings and they produced symbiont relationship. Overtime some DNA from mitochondria  were transferred to the host chromosomes, so they can't survive by their own. Mitochondria and chloroplast gained safety under the host cell and sufficient nutrients to survive. Mitochondria endosymbiotic relationship allowed cell to produce ATP in large amount and Chloroplast allowed cell to use photon and wavelength for ATP production and to build biomass.
ID118	Mitochondrion mutation effects its ability to do electron transport chain. Electron transport chain is the main step for respiration which will take electron donor organic or inorganic and transfer the electrons so it leaves cell in form of H2O when electron acceptor is O2. Mitochondrian mutation alters the cells ability to transfer electron through energy carriers and to final electron acceptor.
ID113	translocation is a rearrangement of chromosomal segments on the chromosome and could be balanced or unbalanced
ID113	the endosymbitoic theory suggests that the mitochondria and chloroplast arose independently from free-living probacteria
ID113	mitochondrial mutations are transmitted through the cytoplasm, and with impaired functions, respiration leading to ATP synthesis is reduced
ID140	Chromosomes that are close together can have translocation occur in which genetic information is swapped between them.
ID140	I understand that this is something that many people have pointed out and that these organisms have been around for a very long time. It has been proven in fossil records and whatnot.
ID140	Mitochondria affects the ability for a cell to make energy which could affect respiration.
ID033	Gene sequences on a chromosome may be moved to another chromosome
ID033	The evidence explains the endosymbiotic theory
ID033	Mitochondria are the site for respiration so a mutation in the mDNA can cause a change in the respiration of the mitochondria.
ID077	Translocations is when a chromosomal segment moves from one chromosome to another. It may either involve one chromosome donating it's segment of another or it may involve both of the chromosome's donating to each other.
ID077	The endosymbiotic theory says that mitochondria and chloroplasts arose separately from one another a long time ago. The basis of the theory is that these cells with the chloroplasts and mitochondria where "engulfed" by eukaryotic cells who originally lacked the ability to respirate with oxygen or to use energy from the sun. After they engulfed these bacteria they gained these functions.
ID077	Mitochondrial mutations affect respiration because mitochondrial DNA is highly vulnerable to mutations due to the reactive oxygen species (ROS) generated by cell respiration. Mutations to the ROS can affect cellular respiration.
ID031	Whole chromosomes can be moved from the location they are at and end up in a completely new spot causing problems usually.
ID031	Both the chloroplasts and mitochondria contain their own dna which hints they might have been their own cell at one point and this leads for scientists to believe that they evolved to get to the point they are at now.
ID031	Since mitochondria oversees the process of respiration then mutational affects will impact the whole process limiting the way respiration can occur.
ID003	translocations happen during meiosis where different chromosomes become integrated into one another. For example chromosome 21 can become integrated into chromosome 14 causing familial Down Syndrome.
ID003	There is evidence that they are ancient symbiotic events because the mitochondria and chloroplasts show evidence that they once lived by themselves as bacteria because they have there own DNA materials and ribosomes to making energy within itself.
ID003	They affect respiration because mitochondria is used for energy within species.
ID020	In a chromosome there can be incidences where part of one chromosomes exchanges some of its material for part of another chromosome. During translocation this section could be inverted.
ID020	The theory says that mitochondria and chloroplasts arose independently billions of years ago from free-living proteobacteria.
ID020	Mitochondrial DNA gene products include 13 proteins required for aerobic cellular respiration. Since a cell's energy supply is largely dependent on aerobic cellular respiration to generate ATP, disruption of the mitochondrial gene can have a severe impact on the organism's respiration.
ID028	When a translocation event occurs, one chomosomal segment may be transferred to another chromosome in return for an unequal or equal portion of that chromosome.
ID028	The inheritance of the mitochondria in humans is from the mother's cell. This can be examined from the mitochandrial DNA through generations and this phenomenon can also be seen in chloroplasts.
ID028	Mutations in the mitochondrial DNA can effect respiration which produces energy for the organism to grow and survive. So, mitochondrial mutations have the potential to severely impair an organism
ID081	There are different types of translocations. One occurs when the translocation is within the chromosome. This is called an intrachromosomal translocation. Another kind of translocation occurs between chromosomes. This is called inter chromosomal translocations.
ID081	DNA from both the mitochondria and chloroplasts evolve differently from the nucleus. Their gene structures and chromosomes reseble those of a prokaryotic cell. It's believed that bacteria were engulfed by larger eukaryotic cells, which led to a symbiotic relationship, then the bacteria lost the ability to function on its own, which finally led to eukaryotic cells gaining oxidative respiration and photosynthesis.
ID081	The mitochondria is an organelle in which the biochemical processes for respiration occur. If mutation occur in the mitochondria, this in turn will affected the overall rate and ability of repiration for that cell and/or organism.
ID099	Translocation is movement from one chromosome to another or movement within a chromosome of genetic information
ID099	plasmid DNA replicates within itself independently of cell replication and suggest some level of autonomy
ID099	Mitochondrial reactions are the key behind respiration so any mutation to this organelle with affect respiration
ID129	Interchromosomal translocations occur between nonhomologous chromosomes. If a segment is transferred from one chromosome to another, it is nonreciprocal. If segments are exchanged, it is a reciprocal translocation. Reciprocal translocations typically don't cause many problems, but Robertsonian translocations can cause familial down syndrome or may cause children to have an unbalanced set of chromosomes and be infertile.
ID129	The endosymbiotic theory says that bacteria were engulfed by larger eukaryotic cells. A beneficial relationship developed, and bacteria lost the ability to function autonomously. Eukaryotic cells then gained photosynthesis and oxidative respiration.
ID129	Mitochondrial mutations may be associated with impaired mitochondrial function in bread mold. These mutations can also cause a deficiency in cellular respiration in yeast.
ID030	Translocations occur when a portion of a chromosome moves and attaches to a different chromosome. Sometimes this is is reciprocated to the chromosome that lost material. In such a case, it is still possible for offspring to receive a normal set of chromosomes. Some, however, may be affected by aneuploidy if a gamete with unbalanced genetic information is fertilized.
ID030	Using DNA evidence, biologists have determined that the mitochondria and chloroplasts of eukaryotes likely arose from the incorporation of prokaryotes into the cells. One compelling piece of evidence to support this theory is the fact that mitochondria and chloroplasts have their own prokaryotic resembling DNA, seperate from that of the cell.
ID030	The mitochondria is the workhorse of the cell, producing large amounts of ATP via the process of cellular respiration. Mutations to the mitochondria can affect the organelles function, likely impacting respiration.
ID150	A translocation is a chromosome aberration that involves the switching of segments of DNA between non-homologous chromosomes. They can be balanced, an equal portion is shared, or unbalanced, unequal portions are shared.
ID150	Chloroplasts and mitochondia both have single, circular DNA, which is similar to bacteria. They also have bacterial type proteins and membrane structures. These and other similarities between the organelles and bacteria have led to the theory that larger bacteria swallowed up smaller bacteria and used them to perform respiration and photosynthesis for them.
ID150	Mitochondria can undergo somatic mutations that can overproduce toxic substances. The toxins can cause more mutations and more toxic buildup, causing the mitochondria to lose full functionality, which can lead to cell death.
ID100	Translocations are a type of chromosomal aberration in which a piece of chromosome is moved to a new location in an organism's genome. This type of mutation can occur within or between chromosomes. When translocation occurs between chromosomes, diseases such as Down Syndrome can occur where the translocation results in a trisomic cell.
ID100	There is a great deal of scientific research that supports the claim that mitochondria and chloroplasts developed out of ancient symbiotic events. Basically what the evidence shows is that both mitochondria and chloroplasts arose independetly about 2 billion years ago from primitive bacterial cells. These ancient bacterial cells were engulfed by larger primitive eukaryotic cells lacking the abililites to either respire aerobically or capture energy from sunlight. From this engulfment, a symbiotic relationship formed between the primitive bacterial and primitive eukaryotic cell. The bacterial cell lost its ability to live independently and the eukaryotic cell gained the ability to respire aerobically or capture energy from sunlight.
ID100	Mitochondria are the cell organelle that is primarily responsible for carrying out respiration. When mutations occur to this cell structure, respiration can be significantly impacted. Because the electron transport system is such a specific process, any minor changes made to the ETC or mitochondrial membrane can affect how well the cell carries out oxidation phosphorylation, thereby affecting how much energy the cell is able to produce.
ID133	A translocation occurs when a segment from one chromosome is transported to another chromosome. Translocations can be reciprocal or nonreciprocal between chromosomes.  If the translocation is reciprocal, then the individual is usually phenotypically normal. This is because all of the essential genes are present and none are lost.
ID133	Both mitochondria and chloroplasts have prokaryotic DNA that is different from the DNA of the cell itself.  They also have 2 membranes. This gives evidence to the endosymbiosis theory.
ID133	A study with bread mold revealed that when the mitochondria is impaired, there is an absence of several cytochrome proteins essential for electron transport.  Without these cytochromes, aerobic respiration leading to ATP synthesis is slowed.
ID058	Translocations occur when a piece of chromosome is either transferred to a homologous chromosome or a non-homologous chromosome. No information is lost or gained, but affects can still be seen when gametes are formed. Produced gametes with translocations usually have duplications and/or deletions and result in inviable gametes. Also, if one piece for example of chromosome 21 is translocated to say chromosome 14, that person is capable of having Down's syndrome children.
ID058	Evidence such as having different DNA than their host cells suggests that mitochondria and chloroplasts were once independent entities and then one day were swallowed up by larger eukaryotic cells and now are in a symbiotic relationship with them. They receive protection while providing the cell with energy through cellular respiration or photosynthesis.
ID058	Mutations in mitochondria can lead to the cell having an insufficient electron transport chain which is a necessity for cellular respiration. Without the electron transport chain, H+ ions can't be pumped out of the cell and later used by ATP Synthase to produce ATP.
ID108	Translocations may occur between nonhomologous chromosomes causing chromosome abnormalities. Reciprocal, nonreciprocal, and robertsonian are examples of translocations. Translocations can be balanced with an even exchange of material or unbalanced with an uneven amount of material.
ID108	Mitochondria went through endocytosis for aerobic bacteria, while chloroplasts went through endocytosis for photosynthetic bacteria. Anaerobic bacteria benefited as well as the mitchondria and chloroplasts making it a symbiotic relationship.
ID108	The citric acid cycle and oxidative phosphorylation electron transport chain occur in the mitochondria. If a mitochondrial mutation affected the intake of pyruvate into the citric acid cycle or impacted the ATP production process at all, respiration would be affected.
ID098	Sometimes segments are moved from one chromosome to another non-homologous chromosome. These events are called translocations, and they usually have powerful effects on the organisms that they produce (assuming the translocation occurs in a gamete).
ID098	The prokaryotic nature of mitochondrial and chloroplast DNA indicates that at one point, eukaryotic cells may have absorbed prokaryotes that served similar purposes as the modern day mitochondria and chloroplasts. The eukaryotes and prokaryotes formed a symbiotic relationship, and the prokaryotes gradually became a part of the eukaryotes.
ID098	Because the mitochondria are the site at which cellular respiration occurs, mutations in mitochondrial DNA can have a direct effect on an organism’s respiratory ability.
ID048	A chromosomal translocation is an abnormality caused by rearrangement of parts between nonhomologous chromosomes.
ID048	Both organelles have their own genetic material. They both(chloroplast and mitochondria) divide through binary fission and do not exchange membrane vesicles with other organelles. A closer examination of both showed similarities to bacteria rather than eukaryotes. Therefore, this proves that the evidence that both of these come from ancient bacteria.
ID048	In the absence of cytochromes, aerobic respiration
ID065	Translocations occur when one portion of a chromosome is moved as a chunk in its entirety. It does not lose or gain any genetic information, but the position of the genes are moved. A reciprocal translocation occurs when one part of a chromosome is exchanged with another, and a nonreciprocal translocations occurs when one part of a chromosome is moved from one chromosome to another without any exchange.
ID065	There are differences between the DNA of mitochondria and chloroplasts when compared to the DNA of the cell of interest. This DNA is entirely different, and it is thought that chloroplasts and mitochondria came to be in plant and animal cells respectively through these organelles entering the cell and forming a symbiotic relationship. This endosymbiosis theory explains the differences in the DNA between these organelles and the host cell.
ID065	The mitochondria is home to respiration. Mutations in these organelles can result in the organelles not working at optimum function or not working at all. For example, a mutation that minimizes the mitochondria's amount of ATP synthase would result in not creating enough energy in the form of ATP.  A non-functioning organelle that is pivotal for respiration would ultimately result in cell death.
ID117	A translocation is a change in location of a DNA segment. This is a chromosome abnormality and can occur between two chromosomes. It can be a balanced transfer, unbalanced, reciprocal, or Robertsonian--which results in a karyotype of less than 46 and a metacentric chromosome where the  the location of the chromosome segment has been moved to.
ID117	Mitochondria and chrloroplasts have DNA quite similar to that of bacteria, which has caused scientists to theorize that these organelles began as primitive bacteria a couple million years ago. The Endosymbiotic Theory theorizes that these primitive bacteria were engulfed by eukaryotic cells which gave the host cells the abilities of cellular respiration and photosynthesis.
ID117	Mitochondrial gene products account for the 13 proteins crucial for aerobic cellular respiration and the production of ATP. If a mutation--which is a change in the DNA sequence--occurs in one of the protein sequences, a varying degree of affects can be attributed to the mutation, ranging from slight to severe or even lethal.
ID007	A translocation occurs when a piece of one chromosome breaks off and becomes attached to another nonhomologous one.  These can be reciprocal (two pieces exchange places) or nonreciprocal (one chromosome "donates" material to another).
ID007	Relatively recently scientists put together a variety of evidence for the endosymbiotic theory.  Chloroplasts and mitochondria have prokaryotic DNA, prokaryotic ribosomes, and a striking resemblance to ancient prokaryotes (a cyanobacteria and a proteobacteria).
ID007	Because mitochondria are the site of the TCA cycle and the electron transport chain, it naturally follows that any changes to their structures would affect respiration.
ID060	Chromosomes are rearranged in parts.
ID060	Endosymbiotic theory suggests that the mitochondrion and chloroplast evolved through a symbiosis relationship of a Eukary and Prokarya specie.
ID060	If mutations occur in mitochondrial DNA enough to affect different processes of respiration, respiration will either be less effective or halted depending on the mutation.
ID017	During translocation between chromosomes there is a partial transfer from one chromosome to a different position. This traditionally occurs in non-homologous chromosomes.
ID017	In symbiogenesis, or otherwise known as the endosymbiotic theory deals with the theoretical important mechanism of evolutionary change. This deals with the merging of two or more free-living organisms to create a new organism.
ID017	If a mitochondria is exhibiting mutations, they essentially inhibit all of the cells functionality as a whole. The mitochondria's job is to produce energy such as ATP and if a mutation inhibits that, the cell does not gain the required amount of energy to uphold the cells abilities such as oxidation-phosphorylation which is the metabolic pathway in which it release the oxidation of nutrients in order to reform ATP.
ID143	A translocation is the process in which a segment of a chromosome is switched with another segment of another nonhomologous chromosome.  This type of chromosomal alteration may affect the organism in phenotypically minor or drastic ways
ID143	Many scientists believe that mitochondria and chloroplasts were developed from a separate form of bacteria, and they are symbiotic events, meaning that these organelles support each other in their development.
ID143	Mitochondrial mutations may alter the ability of an organism to perform respiration properly because mitochondria is the organelle that generates energy for the functioning of the organism.  If the mitochondria is mutated in a negative way, the organelle may not be capable of functioning the way that it is supposed to, leading to issues with respiration within the organism.
ID021	A translocation is a type of chromosomal mutation in which pieces of a chromosome rearrange onto another nonhomologous chromosome. A reciprocal translocation is when the chromosomes trade pieces. A nonreciprocal translocation is a piece moves to another chromosome, but no piece is exchanged. Robertsonian translocations are a type of translocation in which a  break occurs at or near the centromeres of two acrocentric chromosomes.
ID021	This is called endosymbiosis. A host prokaryotic cell engulfed a smaller eukaryotic cell and used its metabolic processes for its own use. Evidence includes similarities in the genome found in chloroplasts and mitochondria  to eukaryotic cells.
ID021	The synthesis of ATP, the cell's energy currency, depends on processes that occur in the mitochondrial membrane. When the structure of mitochondria are changed due to mutations, the function also changes, and respiration is affected.
ID144	Chromosomes can often see translocations. The severity of these translocations depends on several factors.
ID144	They both present characteristics needed for aerobic respiration and photosynthesis
ID144	Abnormal electron transport results in abnormalities in respirations
ID132	Translocations can occur between chromosomes and this just means that the arrangement is not normal. It can happen because of a change that has risen during egg and sperm formation or during time of conception. There are two types of translocation, reciprocal and robertsonian.
ID132	There are many types of evidence that suggest mitochondria and chloroplasts are ancient symbiotic events. Many of the evidence tells us that' they both have similar DNA to bacteria cells.
ID132	Mitochondrias are the power house of mostly all cells. For humans, and plants we need mitochondria to survive and breath through respiration. If three were mutations in the mitochondira then there would be problems with our respiration.
ID055	Translocation means that no DNA is gained or lost between chromosomes it is simple moved from one chromosome to another. There are different types of translocation such as reciprocal and nonreciprocal interchromosomal translocation and intrachromosomal translocation.
ID055	Characteristics of mitochondria and chloroplasts suggest that they were once prokaryotes living individually. Now these organelles reside in a eukaryotic cell meaning these organelles have a symbiotic relationship with eukaryotic that has evolved over the years.
ID055	A mutation called petites have a deficiency in cellular respiration. The portion that lacks this can survive via fermentation.
ID114	Translocations are abnormalities that can occur between two nonhomologous chromosomes. This causes an exchange of genetic information and can lead to cancer, infertility, or down syndrome.
ID114	The mitochondria and chloroplast both exhibit features that are consistent with two separate prokaryotic organisms that were engulfed by a eukaryotic cell due to a symbiotic relationship.
ID114	Mutations in the mitochondria effect respiration because the mitochondria is responsible for producing much of the cell's energy. When the mitochondria is damaged or experiences mutations that alters its function, then it cannot provide the cell with as much respiratory abilities.
ID054	A translocation is a rearrangement between two nonhomologous chromosomes. The are two kinds of translocations, non-reciprocal and reciprocal.
ID054	Evidence suggests that mitochondria and chloroplasts both had ancestors that were prokaryotic organisms. Independently, they both evolved through symbiosis into the eukaryotic organisms they are today.
ID054	Mutations to an individuals mitochondria has been known to cause various forms of deformities, including to lung development, more specifically lung cancer.
ID127	This is when part or a good amount of the chromosome is exchanged or transferred to another chromosome. Reciprocal translocation is when two chromosomes transfer part of their chromosome to another. Nonreciprocal is when one chromosome gives away a part of itself and it does not get a part of another chromosome in return. This usually does not effect the individual but may affect the children.
ID127	The evidence that points to ancient symbiotic events is the presence of more than one membrane and the similarities to bacteria. Both of these organelles function like bacteria, but they only live within eukaryotic cells, which could mean that they once lived with eukaryotic cells before they were engulfed.
ID127	Cellular respiration takes place in the mitochondria of cells. Therefore, if there was a mutation that occurs in the mitochondria, it will affect respiration.
ID042	Translocation is the movement of a segment of a chromosome to a new place. Genetic information is not lost during this process just relocated. The consequences of this process are not lethal although it can cause other issues.
ID042	According to the endosymbiotic theory, mitochondria and chloroplasts came about independently of each other billions of years ago, however they are both very similar in the fact that they both carry genetic information. This information is passed down from parent to progeny and effects phenotype.
ID042	In the study of a type of yeast, a mutation that caused a deficiency in cellular respiration involving abnormal electron transport.
ID011	Translocations are when two pieces of DNA on two different chromosomes change position.  This can also lead to a third chromosome being developed via Robertsonian translocation.
ID011	Mitochondria and chloroplasts appear to be ancient cells that had formed a symbiotic relationship with ancient eukaryotic cells which has led to their current state as organelles.  This is evidenced by the fact that they have their own DNA and have two phospholipid bilayer.
ID011	Mitochondrial mutations cause a negative affect in the bioenergetic function of the organelle and is transmitted maternally.
ID134	Transferring of DNA can occur between organisms. Information can be inverted, transferred to different parts on the arms of the chromosome, etc. Information is transferred differently or to a different location on the chromosome
ID134	Chloroplast and mitochondria are theorized to have once been independent free-living prokaryotic organisms that entered a symbiotic relationsips with a eukaryotic cell and over time evolved to be organelles of those cells and contributing important functions to those cells
ID134	Different mitochondrial disorders are only inherited from the mother. There is a wide rane of mitochondrial disorders and they affect one pivotal function of the mitochondria, cellular respiration
ID027	A piece of one chromosome may remove itself from one chromosome and attach on another. This can be reciprocal or non-reciprocal, meaning that they can swap or simply give.
ID027	This means that modern eukaryotic cells are living off of the energy provided by the bacterial cells living within them. This is important for evolutionary biology to understand the origins of life. This is important for geneticists because it means that the DNA in these organisms is prokaryotic in nature and likely effects the nucleus of the eukaryotic cell as well.
ID027	Mitochondrion are responsible for producing the ATP needed for the cell to perform its normal functions. If there is a mutation in the mitochondria, it will alter the amount of ATP produced or even stop its production completely.
ID146	A change in location of a chromosome segment is a translocation, but during the change no DNA is lost or gained. Simple translocations are of two types: intrachromosomal (a change of position within the same chromosome) and interchromosomal (a transfer of the segment to a non homologous chromosome) If a segment is transferred from one chromosome to another, it is nonreciprocal. If segments are exchanged, it is reciprocal.
ID146	New mitochondria and plastids are formed only through a process similar to binary fission. If a cell's mitochondria or chloroplasts are removed, they do not have the means to create new ones. Genome comparisons indicate that cyanobacteria contributed to the genetic origin of plastids. DNA sequence analysis and phylogenetic estimates suggest that nuclear DNA contains genes that probably came from plastids. These organelles' ribosomes are like those found in bacteria. Much of the internal structure and biochemistry of plastids, for instance the presence of thylakoids and particular chlorophylls, is very similar to that of cyanobacteria. Mitochondria have several enzymes and transport systems similar to those of bacteria. Both mitochondria and plastids have small genomes compared to bacteria. This is consistent with an increased dependence on the eukaryotic host after forming an endosymbiosis. Most genes on the organellar genomes have been lost or moved to the nucleus. Most genes needed for mitochondrial and plastid function are located in the nucleus. Many originate from the bacterial endosymbiont.
ID146	Mitochondrial disease is a group of disorders caused by dysfunctional mitochondria, the organelles that generate energy for the cell. Mitochondrial diseases are sometimes (about 15% of the time)[1] caused by mutations in the mitochondrial DNA that affect mitochondrial function. Other causes of mitochondrial disease are mutations in genes of the nuclear DNA, whose gene products are imported into the mitochondria (mitochondrial proteins) as well as acquired mitochondrial conditions.
ID050	Translocations between chromosome can be between two homologous chromosomes or on the same chromosome(Robertsonian). This translocations can lead to no change in the phenotype or it can have drastic effects, it generally depends on whether the translocation was balanced or unbalanced and where in the chromosomes it took place.
ID050	This is related to the endosymbiotic theory of which states that these organelles were once their own organisms and they eventually combined into one cell to form eukaryotes.
ID050	The mutations affect respiration because the mitochondria is responsible for producing the energy for the cell and if the cell is not producing energy it will die. When enough cells die it will cause body systems and breathing to become effected and eventually fail.
ID091	A translocation is possible where part (or all) of one chromosome is translocated to another chromosome. This can occur in a variety of ways but has important implications. For instance, a translocation of the 21st chromosome can be the cause of familial downs.
ID091	Mitochondria and Chloroplasts both have their own DNA. This suggests that it is possible that they were freestanding organisms that developed a mutualistic relationship with other cells. This leads to many important evolutionary ideas such as genetic lineage and history.
ID091	The mitochondria houses the process of cellular respiration. Since mitochondria have their own DNA, mutations in their DNA affect the respiration of individuals.
ID151	Translocations are the breaking off and the transfer of a portion of a chromosome to another chromosome. This transfer can be discussed as a reciprocal translocation between nonhomologous chromosomes or as a non-reciprocal translocation of nonhomologous chromosomes. In a reciprocal translocations, two chromosomes that are not homologous chromosomes to each other both actively switch portions of their individual chromosome, thus, creating a hybrid-like chromosome. In this, both chromosomes act like a receiver and a donor for a piece of that chromosome. However, in a non-reciprocal translocation of nonhomologous chromosomes, only one chromosome acts like the receiver and one acts like the donor. Non-reciprocal translocations consist of the transfer of hereditary material from only one chromosome to be attached to another nonhomologous chromosome. Another way to describe the rearrangement of chromosomes is known as Robertsonian Translocation which is the transfer of a portion of a chromosome amongst these chromosomal numbers only 13, 14, 15, 21, and 22.
ID151	The Endosymbiotic Theory is a theory that explains the origination of mitochondria and chloroplast within the current eukaryotic cell. This theory claims that the mitochondria and the chloroplast were previously prokaryotic organisms, proteobacteria and cyanobacteria, respectively, that coexisted with the now eukaryotic bacteria in a symbiotic relationship. There are numerous evidences that show or support this possible explanation. For instance, both the chloroplast and the mitochondria have their own DNA and have their own replication mechanisms separate from the nucleus. The former prokaryotic organelles have DNA, ribosomes, and transport enzymes like that of bacteria. Furthermore, there are other lines of evidence that suggest this symbiosis between the chloroplast and the mitochondria to the eukaryotic cell and the actual process of the symbioitc event. However, in class I was informed that some facts were overlooked that disagreed with the concept of the endosymbiotic theory due to the fact that individuals wanted every detail to match the idea that was created.
ID151	Mitochondria are responsible for producing ATP within the cell to be utilized by the mitochondrial cell and the eukaryotic cell while some energy is wasted in the form of heat. This production of ATP is crucial to allow certain functions to occur within an organism. Furthermore in order to produce this energy, mitochondria must utilize oxygen along with other molecules to initiate the process of aerobic respiration. Therefore, any mutation that results in the reduction of mitochondrias within a cell or inhibits the production of ATP through interference during the exothermic process can result in respiratory diseases. A lower number of mitochondrias directly respond to a lower and limited oxygen absorption or intake and ATP production, thus affecting respiration.
ID084	Two types of translocation can occur between chromosomes. The first, a non-reciprocal translocation occurs when a segment of DNA is transferred from one chromosome to the other. The second type of translocation is referred to as a reciprocal translocation, and is seen when each chromosome gets a piece of DNA from the other.
ID084	Ancestors of both mitochondria and chloroplasts began as prokaryotes until a symbiotic event occured in which each organism was engulfed by or engulfed another organism, where they eventually evolved into the eukaryotic organisms they are today.
ID084	A mutation in just a small piece of mitochondrial DNA has been known to cause a severe case of lung cancer, therefore affecting respiration.
ID088	A section or group of genes from one chromosome may be translocated from one chromosome to a different chromosome. Translocations can either be balanced or unbalanced. For balanced, there is an even exchange of material. For unbalanced, there is either missing or extra genes.
ID088	It is thought that the chloroplast and mitochondria used to exist outside of cells. Evidence points us to believe that this was true until these two were taken into cells and benefitted the cells.
ID088	When mutations occur on the mitochondria, respiration may be affected. That being so, some forms of cancer may form.
ID107	A translocation occurs when a segment of one chromosome rearranges with a segment of a nonhomologous chromosome. Translocations do not lead to a loss or gain of genetic material. They can be
ID107	The Endosymbiotic Theory states that around 2 million years ago, chloroplasts and mitochondria existed as independent prokaryotic organisms. They were engulfed by larger eukaryotic cells and developed a symbiotic relationship. Over time, they lost the ability to live autonomously and eukaryotic cells gained the ability to respirate and photosynthesize. Mitochondria and chloroplasts, unlike other eukaryotic organelles, have prokaryotic gene structure and ribosomal RNA.
ID107	Mitochondrial mutations can effect electron transport which can lead to deficiencies in cellular respiration.
ID142	Translocations are the movement of DNA pieces to another chromosome. They may be reciprocal or not. Reciprocal translocations happen when two chromosome equally swap DNA.
ID141	Translocations occuring between chromosomes can be of two types.  A reciprocal translocation means that both chromosomes swap segments of genetic material.  A non-reciprocal translocation means that only one chromosome segment is moved to another chromosome.
ID141	The endosymbiotic theory explains how the mitochondria and the chloroplast developed a symbiotic relationship with the cells they were engulfed by.  Both the bacteria and the larger cell benefited from the relationship in that the cell could now carry out oxidative respiration and photosynthesis.
ID141	Since the mitochondria is the powerhouse of the cell and is the main source of ATP for the cell, mutations in the mtDNA would obviously affect cellular respiration.
ID138	Translocations are when segments from chromosomes break off and attach to different chromosomes. They can be reciprocal or non-reciprocal. Reciprocal means that it switches places with a segment from another chromosome.
ID138	Mitochondria and chloroplasts both contain their own DNA. It is believed that they were both once individual cells that were engulfed by Eukaryotic cells.
ID138	Mitochondria are considered the "power house" of the cell. They produce ATP through aerobic respiration. When there is something wrong with the mitochondria it can affect mutation because your body will not be getting the energy it normally uses the oxygen you breathe in to produce.
ID002	As with some genetic diseases, translocations between chromosomes can occur. These accidents can be non-lethal, or lethal. Others tend to lead to genetic diseases.
ID002	It is theorized that both mitochondria and chloroplasts were their own forms of bacteria and through various events, became incorporated in modern cells, which catalyzed the evolution of life on this planet
ID002	Mitochondrial mutations that affect respiration are because the mitochondria utilize oxygen in cellular respiration to make ATP. So it makes sense that mutations in mitochondria affect the ability to utilize oxygen and thus affect respiration.
ID016	Chromosomal abberations can occur in which the entire chromosome suffers a mutation. Translocation involves taking a piece of one chromosome and adding it to another. The same genetic material is still there, but the order is not the same.
ID016	Proteobacteria existed alongside the first cells. A theory suggest that a cell engulfed a photosynthetic cyanobacteria to acquire autotrophic capibilities and a proteobacteria to acquire respirative abilities.
ID016	Mutations from mDNA can result in the respiration pathway not working optimally.
ID006	Translocation is a type of chromosomal abnormality in which a chromosome breaks and a portion of it reattaches to a different chromosome.
ID006	Membranes — Mitochondria &chlorolplast have their own cell membranes, just like a prokaryotic cell does.
ID006	A factor that affects the respiratory rate is:
ID123	Translocations may occur between chromosomes because parts of the each chromosome can break off and rejoin to the adjacent chromosome.  There is no loss of chromosomal material because the segments of DNA are exchanged.
ID123	Mitochondria and chloroplasts are symbiotic events because they were bacterial cells that worked together in order for their host cell to survive.  They were both engulfed by a larger cell, and they allowed the larger cell to produce energy while they were allowed to survive inside the host cell.
ID123	Mitochondrial mutations affect respiration because the mutation may have altered the mitochondrial DNA to a point where it is deemed inoperable.  This level of mutation would hinder the process of cellular respiration because the mitochondria can not function properly.  In turn, these mutations can be devastating to people's lives.
ID036	Trans location can occur between chromosomes. It is one of the chromosomal abnormalities that can occur. It is responsible for genetics defects such as robertsonian syndrome an d certain cancers. It occurs when a set of sister chromosome are switched, and become one.
ID036	Symbiotic event occurred between the chloroplast and the Mitochondria. Examples of this event occurring is indicated by Mitochondria have their own cell membranes, just like a prokaryotic cell does. mitochondria has a small circular genome simlar to a bacteria.
ID036	Mitochondria DNA can affect respiration because mtDNA is vulnerable to mutation because they do not contain histones that can protect them from mutegenic events. Reactive Oxygen species are known to damage mtDNA. MERRF is a mitochondial mutation that is caused by miscoding of the TRNA lys. Mutation causes affected individuals lack muscular coordination.
ID095	Translocations alter the location of chromosomal segments in the genome.  Whenever a segment of a chromosome is moved to another part of the chromosome, this is known as a translocation.  Two types are observed: intrachromosomal is a change within the same chromosome, and interchromosomal is a transfer to a non homologous chromosome.
ID095	It is theorized that the organelles mitochondria and chloroplast are prokaryotic organisms that were taken up by other cells and developed a symbiotic relationship.  This is known as endosymbiotic theory.
ID095	The mitochondria is the organelle responsible for synthesizing energy for the organism.  If the mitochondria dna is mutated, a negative effect can arise resulting in poor respiration.
ID111	A translocation is caused by a rearrangement of parts between nonhomologous chromosomes. There are two different kinds:reciprocal and Robertsonian. Reciprocal translocations are an exchange of material between nonhomologous chromosmes. Robertsonian is caused by breaks at or near the centromeres of two acrocentric chromosomes.
ID111	This is referred to as symbiogeneis. This theory suggests that mitochondria, plasmids such as chloroplasts, and other organelles representing formerly free living bacteria (prokaryotes) were taken inside another cell as an endosymbiont millions of years ago.
ID111	Mitochondrial disorders are typically characterized by lack of cellular energy due to oxidative phosphorylation defects (which greatly involves the mitochondria). Mitochondria are essential in cells and support aerobic respiration produce bulk ATP. When there are mutations in the mitochondria and they are unable to do aerobic respiration correctly or at all, then this greatly affects respiration in a negative way and can cause many disorders.
ID085	A translocation does not results in a loss of genetic information. However, they do affect the pairing of chromosomes, meaning that an adult with a translocation can appear perfectly healthy, but his offspring have a chance of monosomies or trisomies in the affected chromosomes, particularly if he has a Robertsonian translocation.
ID085	Evolutionists are trying to link the idea that mitochondria used to be separate organisms which eventually started living inside eukaryotesto provide energy. Although no explanation is provided for how mitochondria and chloroplasts suddenly all decided to start living inside the larger eukaryotic cells, we assume that because mitochondrial DNA is different from the DNA in the nucleus of a eukaryote that they must have come from separate parts of the world.
ID085	Because the DNA repair mechanism in mitochondrial DNA (mtDNA) is limited, the energy-producing capability of mitochondria is limited when mutations occur in the mtDNA.
ID128	Translocations are the movement of chromosomal segments to a different place in the genome. These changes have similar effects to inversions. Since there is no change in the actual genetic material (none is gained or lost) it doesn't typically make the subject inviable.
ID128	This follows the endosymbiotic theory. This theory suggests that these two organelles arose separately from primitive bacteria. The two organelles were then absorbed by larger eukaryotic cells and developed a symbiotic relationship based on their functions (aerobic respiration and photosynthesis).
ID128	The mitochondrial DNA is responsible for 13 proteins that are all necessary for aerobic respiration. Any of the over 400 mtDNA mutations that lead to at least 150 distinct syndromes. All of these mutations lead to issues with aerobic respiration.
ID072	Translocation is an abnormality that occurs during the rearrangement of the sections of chromosomes. Parts of the two chromosomes may be fused together when translocation fuses two nonhomologous chromosomes
ID072	the mitochondria and chloroplasts have had symbiotic events for a long period of time. Mitochondria was thought to have been absorbed or eaten and lived inside the cell to produce the energy while the mitochondria had somewhere to be safe.
ID072	The amount of oxygen that the body can use effectively is related to how active the proton pumping sites are in the mitochondria.
ID047	Translocations generally happen between chromosomal arms between heterologous chromosomes. Usually they result from double stranded breaks in the chromosome. Translocations lead to deregulation of cellular proteins.
ID047	Mitochondria and chloroplasts resemble tiny bacteria. They have their own membranes, circular DNA, and they pinch off to reproduce. This evidence suggests that mitochondria and chloroplasts were once free living bacteria that made a living in eukaryotic cells leading to endosymbiosis.
ID047	The phenotypes of mitochondrial diseases caused by mutation are hetergenous, and it is very hard to examine mitochondria due to its polyplidy , double membrane, and lack of recombination. These mutations affect respiration in turn affecting ATP production
ID082	Translocations of genetic material between chromosomes can result in a variety of effects. When the translocation occurs in a specific individual, that individual is usually phenotypically normal because they still have the correct amount of genetic material, just in different places. However, if influences such as position effect alter or disrupt the expression of the translocated genes, then the individual will not be phenotypically normal. The progeny of individuals with translocations, on the other hand, will not be normal if they inherit one of the translocated chromosomes.
ID082	In learning about the way that both mitochondria and chloroplasts function in the cell, they stand out as distinct from the other organelles in that they have their own machinery and DNA replication systems. In researching further the mechanisms of these organelles, they appear to have striking similarities to independent, prokaryotic organisms. These lines of research have implicated that the origin and incorporation of these organelles into living systems today are a result of an ancient symbiotic relationship between two separate prokaryotic cells.
ID082	Mutations in mitochondrial DNA have led to a variety of impacts on respiration in cells. While over time through evolution, these mutations have led to the incorporation of a prokaryotic bacterium into the host cell, there are also respiratory syndromes that result from deleterious mutations. One such syndrome is Leigh syndrome, which is a severe neurological disorder resulting from mutations in mitochondrial DNA. Since mitochondria have their own DNA, mutations in these genes affect the gene products and functioning of the mitochondria itself. Because the mitochondria is involved in respiration and many events associated with respiration occur in the mitochondria, these mutated or absent gene products directly affect respiration.
ID131	Translocations are rearrangements between non homologous chromosomes. It can be survivable, but also lethal and cause ill effects such as familial Down's Syndrome.
ID131	Since mitochondria and chloroplasts both act very similarly to and look very much like bacterial cells, it is believed that they entered eukaryotic cells millions of years ago and formed a sybiotic, beneficial relationship where they grew to rely on each other for survival.
ID131	Mitochondrial mutations affect respiration largely because the DNA contained within the mitochondria codes for many features required for a cell to respirate. The mitochondria is where a large portion of cellular respiration occurs so this makes sense.
ID104	Translocations in chromosomes, can result in major defects and sterility of the person. It can also result in the consistent production of a specific type of syndrome, for example, down syndrome with translocation of chromosome 21
ID104	I don't remember discussing this topic in lecture. It poses an interesting topic of discussion that I would like to look more in-depth at in the coming future.
ID104	the ability of mitochondria to produce energy can affect just about any processing the cell, let alone respiration. A defect in the mitochondria would limit the lung cells' ability to take oxygen and produce CO2. The defect might also impact the cells' ability to rid the lungs of the waste CO2.
ID149	Translocations are very abnormal. It's caused by rearrangements between nonhomologous chromosomes. Robertsonian translocations are most talked about. It's when the chromosome breaks at or near the centromere and attach to another. The karyotype looks like it has 45 chromosomes.
ID149	Symbiogenesis is a theory that explains how organelles of eukaryotes are derived from prokaryotic cells. Both of these organelles are similar to prokaryotes, mostly talking about their DNA and structure.
ID149	Mitochondrial DNA codes for 37 genes. One of those genes could code for something important in the respiration cycle.
ID086	This occurs when parts of chromosomes are moved and connected to other chromosomes. This can sometimes result in offspring having trisomy 21.
ID086	This was how our oxygen consuming cells came about. Our mitochondria have different dan than our nucleuses.
ID086	These affect how the mitochondria can take in and use ATP.
ID119	Translocation, or the rearrangment of segments of chromosomes between one another, may occur between chromosomes.
ID119	Mitochondria and chloroplasts were once thought to be bacteria cells themselves that were then engulfed by a host cell and became organelles.
ID119	There is a deficiency in cellular respiration as a result of mitochondrial mutations.
ID052	Chromosomes may undergo mutation which include either the reciprocal or nonreciprocal swap between certain gene segments. Translocation occurring without the gain or loss or extra genetic material may not have any defects.. However, the gametes of these chromosomes may have a higher chance of having abnormalities and abberations.
ID052	This means that there is some mode of inheritance for the mitochondria and chloroplast.
ID052	Most mutations are either deleterious, neutral, or confer a selective advantage. Observing how mutations can affect the mitochondria and its mode of inheritance can indicate how respiration is affected.
ID009	They are abnormalities where chromosomes break and join other chromosomes. This can lead to normal phenotype and abnormal fertility, however it can cause variations in phenotype.
ID009	Research suggests that both choloroplasts and mitochondria were brought about by bacteria being sucked into other bacteria.
ID009	These mutations affect the respiratory chain.
ID120	Translocation are swapping of piece of one chromosome with another piece of another chromosome resulting in varying effects.
ID120	Mitochondria and chloroplast both contain DNA. By determining the origin of these organelles, more information about the genome can be discovered.
ID120	Mitochondria is responsible for conducting cellular respiration by breaking down food and turning it into ATP. If cellular respiration is not correct, there will not be an adequate amount of energy for the organism.
ID073	In genetics, a chromosome translocation is a chromosome abnormality caused by rearrangement of parts between nonhomologous chromosomes. A gene fusion may be created when the translocation joins two otherwise-separated genes, it is detected on cytogenetics or a karyotype of affected cells.
ID073	DNA in mitochondria and chloroplasts accounts for certain phenotypes in offspring. THe transfer of genetic material is through the cytoplasm and not through information held in the nucleus. THey are both inhereited through maternal cytoplasm. They both contain specific DNA patterns that contribute to their functions.
ID073	A cell's energy is dependent on the mitochondria.  In humans, we respire aneorobically. Reactive oxygen species are generated by cell respiration and are toxic to the contents of the organelle. they damage proteins, lipids, and mtDNA.
ID082	In 1944, Avery, McCarty and MacLeod published the results of their studies that concluded, definitively, that DNA was the “fundamental unit of the transforming principle”, meaning that they had shown that it was, in fact, DNA that was the source of the necessary information that cells needed to gain new genes or the ability to produce new gene products. Their work established that the transforming principle that had been hypothesized and much sought to explain was DNA. They showed this using virulent cells that they then killed with heat, and after processing to mix and destroy any remaining living aspects of the virulent cells, introduced the left over material to avirulent cells. This “left over” material after the destruction of the virulent cells themselves was then shown to be able to induce transformation in avirulent cells, making them virulent. They also further controlled for possible contamination or combining factors by adding more digestive enzymes to absolutely destroy any RNA, but the transformation still occurred. The only way to prevent the transformation was found to be after the use of a DNA digesting enzyme, which destroyed the DNA. These results established DNA as the “active factor” that was responsible for the effects of transformation.
ID082	Following the studies performed and published by Avery, McCarty, and MacLeod, the Hershey-Chase experiments introduced yet another convincing piece of evidence that implicated DNA as being the genetic material of cells. Using Escherichia coli and bacteriophage T2, they were able to show that when the virus infected a cell, it was the DNA of the virus that was injected into the bacterial cell, not the associated proteins. They were able to show this because DNA has phosphorous, but not sulfur and proteins have sulfur but not phosphorous. By using radioisotopes of phosphorous and sulfur, Hershey and Chase were able to isolate the location of the viral DNA and proteins after infection of the bacteria. They found that the radiolabeled phosphorous was the only material that was present in an infected bacterial cell, and that the associated viral proteins (termed “ghosts”) were left outside the cell. This implicated that it’s the DNA of the virus that can take over the machinery of the bacterial cell, using the bacteria itself to reproduce and create more viral progeny.
ID082	Erwin Chargaff established that there was a relationship in the proportions of the concentration between A and T and G and C. The three most important aspects of Chargaff’s findings were that the amount of adenine residues is proportional to the amount of thymine residues, and that the amount of guanine residues is proportional to the amount of cytosine residues. They also found that the sum of the adenine and guanine residues equaled the sum of the cytosine and thymine residues. Lastly, they established that the percentage of the guanine/cytosine pairings don’t have to equal the percentage of the adenine/thymine pairings. Using this crucial information, Watson and Crick created a model for the structure of DNA that included seven major important aspects. Watson and Crick hypothesized that DNA is made of two long polynucleotide chains that coil to form a double helix, that these chains are antiparallel, that the chains are “stacked”, each turn of the helix is 3.4 nm long, that major and minor grooves appear because of the base paining, the double helix has 2.0 nm diameter, and, most importantly, that the nitrogenous bases described by Chargaff were paired as a result of hydrogen bonds. This last aspect of their model was crucial to its development, and was made possible using the established relationships Chargaff had already described. The hydrogen bonding is important because the structure of adenine and thymine create two hydrogen bonds, and guanine and cytosine form three hydrogen bonds. Because of these structural differences, the bases pair, creating major and minor grooves in the double helix. While some small changes has been made to their model because of more recent discoveries, the major of the Watson and Crick model remains intact to this day.
ID109	They believed that DNA was the substance that caused bacterial transformation. They used strands of purified DNA that was precipitated from solutions of cell components to perform bacterial transformations. The researchers suggested that DNA, not protein, is the hereditary material of bacteria. The experiment included
ID109	Hershey-Chase conducted an experiment using bacteriophages and two types proteins: sulfur labeled and phosphorous labeled. Phosphorous labeled the DNA, and sulfur labeled the protein. They injected the protein into the bacteria, and centrifuged the cells. Since phosphorous is in DNA, if phosphorous remains then DNA is the genetic material. The result was that there was no sulfur left in the sells, but only phosphorous, and thus DNA, remained.
ID109	Watson and Crick are knowledge thieves because they stole both Chargaff and Franklin's ideas. Combining both Chargaff and Franklin's ideas, they came up with "their" idea. However, Watson and Crick did have the knowledge to put these two pieces of the puzzle together creating one of the biggest scientific breakthroughs.
ID014	During this era it was commonly believed that proteins were responsible for carrying genetic material. The experiments carried out by these three scientists help prove that DNA was actually the genetic material that causes bacterial transformation.
ID014	Hersey and Chase used E.coli and bacteriophages to also demonstrate that DNA was genetic material and not protein. He used radioisotopes specific to both proteins and DNA to see exactly what is transferred to the next generation.
ID014	Watson and Crick took information from both Erwin Chargaff and Rosalind Franklin. Chargaff gave the idea that all of the bases are proportional to each other. Franklin used an X-ray technique to show that DNA is actually in a double helix formation. Watson and Crick put both of these findings together to configure the true structure of DNA.
ID065	I know that the experiment involves the transmission of genetic material via bacteriophages, but I do not entirely understand the steps behind the process that were looked at in particular in this study. I understand that it added to the knowledge behind DNA as genetic material, but I am not exactly sure how.
ID065	This experiment used bacteriophages. The protein layers of the outside of the bacteriophages were stained one way, and the phosphorus in the DNA was stained another way. Once the bacteriophages infected the DNA, the stained phosphorous was found inside the cell and the stained protein layer was not. This led to the conclusion that DNA is the genetic material in bacteriophages.
ID065	Knowing that the ratios of C to G and A to T were almost perfectly 1, it was concluded that C was paired with G and A was paired with T. In combination with the X-ray crystallography that implied a double helix shape, it was concluded the A and T and C and G formed hydrogen bonds between the two respective pairs and were oriented in the shape of a double helix with the nucleotide hydrogen bonds on the inside.
ID027	They were able to use a series of treatments to remove certain materials from the bacteria until there was no longer any transforming activity in the filtrate. This narrowed the genetic material down to DNA since it was no longer able to perform after deoxyribonuclease was added to the filtrate.
ID027	They were able to show that bacteriaphages inject their DNA into the host and not their protein by "marking" the DNA with a radioactive isotope of phosphorus and "marking" the protein with a radioactive sulfur atom. Once an injection occured, the materials were separated. The radioactive sulfur remained outside of the cell in the trial containing sulfur, and the radioactive phosphorus was found inside of the cell in the trial containing its radioactive isotope.
ID027	Watson and Crick discovered the genetic structure by using work from Chargaff, detailing the ratio of the nitrogenous bases and work from Rosalind Franklin's x-ray diffractions. They were able to discover the double helix shape with the bases pointing inward and in the rate pairings based on this data.
ID056	The purpose of Avery, McCarty and MacLeod's experiment was to expand the investigation of Griffith's study to determine if the genetic material in transformation was protein or DNA.
ID056	The purpose of the Hershey-Chase experiment was to provide more evidence that DNA was the genetic material and not protein in bacteriophage T2.
ID056	Watson and Crick further proved that DNA was a double helix, had two anti-parallel strands of pairing bases and stacked nitrogen bases based off the conclusion made by Franklin that DNA had a helical structure.
ID094	These 3 demonstrated that DNA causes bacterial transformation, and showed that protein did not move heriditary info.
ID094	DNA and protein were both radiolabeled and tracked through the process of virus propogation to see what the carrier of genetic info was.
ID094	They used this info to show the structure of DNA based on the existing work of others and theorized for the first time the DNA double helix.
ID093	It was thought that DNA was too simple to be what lead to the genetic diversity of all life on Earth. However, Avery, McCarty, and MacLeod would suggest through their research that all diversity of life could arise from 4 (or 5) simple nucleic acids. They were able to separate the components of a virulent organism. Proteins, DNA, and other cytoplasmic components were separated and injected into mice separately. When injected all together the mice died, but when each component was injected into the mice separately, only the DNA would cause infection, and eventually death. This showed conclusively that DNA was the heritable trait, able to be transmitted to other organisms, that leads to the amazing diversity of life as we know it.
ID093	The capsid of a bacteriophage contains a high amount of sulfur, while the backbone of DNA contains phosphorus. Hershey and Chase were able to radiolabel each of these compounds and observe their interaction with a bacterial host cell. The results showed that the phosphorus containing DNA would be injected into the cell while the sulfur containing protein would remain as a "ghost" on the cell membrane. The DNA would become integrated into the host chromosome, while there was no evidence that the sulfur was transmitted to the host. This suggested that a less complex structure, DNA and not complex protein, was the genetic material utilized by all forms of life.
ID093	Watson and Crick showed that the concentration of adenine will always equal the concentration of thymine, and cytosine would always equal guanine. From data about one nucleic acid, it is possible to derive the concentration of all the others.
ID037	They determined that DNA is genetic material by destroying each other proposed material to see if the mouse died and if the live strain was recovered
ID037	They used radioisotopes to determine that DNA is the genetic material
ID105	Avery/McCary/Macleod used in vitro essay to determine what the transforming agent was in Diplococcus pneumoniae. His experiment showed it was DNA that caused the transformation of avirulent to virulent Diplococcus pneumonia.
ID105	Hershey and Chase placed E. coli in a radioactive medium of either Phosphorus or Sulfur. This helped to show DNA and not protein as the genetic material. this showed that DNA entered bacterial cells during infection and directs viral reproduction.
ID105	Watson and Crick proposed the double helix model of DNA with two antiparallel strands connected by base pairs and stacked nitrogen bases. They used the information of Chargaff's findings on base composition and Franklin's findings on structure.
ID089	These researchers showed that DNA was in fact the genetic material by eliminating different parts of cells, and seeing if transformation still occurred. Because transformation did occur when anything except for DNA was removed, and did not occur when DNA was removed, it could be concluded that DNA was the genetic material.
ID089	Hershey-Chase experiments showed that DNA was the genetic material when the researchers labelled the virus DNA and protein with radioactive isotopes of phosphorus and sulfur, respectively. The radioactive phosphorus was found inside of the bacteria while the radioactive sulfur was found in the supernatant after centrifugation, suggesting that it does not enter the cell.
ID089	Chargaff showed that the amount of A is equal to the amount of T and that C is equal to G. Rosalind franklins x ray diffractions showed that the structure of DNA is helical in nature. Watson and crick used this data to physically construct a model of DNA.
ID057	They extracted from Streptococcus pneumoniae S bacteriaDNA, proteins and other materials and mixed R bacteria with these different materials, and only those mixed with DNA were transformed into S bacteria. For this experiment they used test tubes and not mice. This experiment backed up the thought that DNA was the transforming factor not proteins. This was one of the experiments that laid down the basis that genes are made of DNA.
ID057	The purpose of the Hershey Chase experiments was to determine whether a bacteriophage injected DNA or protein to infect a cell. In the experiment they used T2 bacteriophage. After the experiment bacteriophage DNA was found in the cytoplasm of bacterial cells. Thus, suggesting DNA was the genetic material.
ID057	Watson and Crick discovered the DNA structure was in fact a double helix. They provided proof for this model with the help of Chargaff and Franklin's work. Franklin provided  X-ray diffraction data on DNA structure which was used to then confirm the DNA structure was indeed a double helix.
ID061	Avery, McCarty, and Macleod published a report in 1944 concerning  the chemical nature of a "transforming principle" in bacteria that was the the initial event leading to the acceptance of DNA as genetic material. The group of scientists isolated large quantities of type IIIS virulent cells. These cells were centrifuged, collected, and heat killed. Following homogenization and several extractions with the detergent deoxycholate, the researchers obtained a soluble filtrate that retained the ability to induce transformation of type IIR avirulent cells. Protein was removed from the filtrate and polysaccharides were digested and removed. Precipitation with ethanol yielded a fibrous mass that still retained the ability to induce transformation of type IIR avirulent cells. Testing proved that the transforming principle was DNA. This was proven when the enzyme deoxyribonuclease destroyed the transforming activity of the filtrate, since this enzyme can break down DNA.
ID061	Hershey-Chase experiment used E. coli in their experiment. They used a radioisotope of Phosphorous to label DNA and a radioisotope of Sulfur to label protein in phages. Thus, when the phages infected a cell and transmitted their information into the cell, they could tell what was carrying important information based off which isotope was present in the cell. The results of this experiment showed that the phosphorous labeled DNA was present in the bacterial cell following adsorption while the sulfur labeled protein remained outside the bacterial cell. This suggested that DNA is what carries genetic information since it is what was transferred to the cell.
ID061	Chargaff concluded that the concentration of A was very close to the concentration of T and that the concentration of G was very close to the concentration of C. Thus the sum of the purines is equal to the sum of the pyrimidines. This allowed Watson and Crick to properly match Adenine and Thymine and Guanine and Cytosine in DNA. Then, Watson and Crick were able to use Franklin's X-ray diffraction images to determine that the structure of DNA was a sort of helix. Watson and Crick were able to use Chargaff's data to determine how the base pairs batched up in the helix structure and offered a model for DNA structure.
ID081	This experiment was able to find that trypsin, chymotrypsin and ribonuclease did not affect it, but an enzyme preparation of deoxyribonucleodepolymerase destroyed the extract's transforming power.Therefore, it was able to tell that DNA was in fact the genetic material.
ID081	In this experiment, Hersey and Chase looked at a bacteriophage and the information that is passed to their host cell. A bacteriophage consists of both a protein coat and DNA. They labeled DNA radioactive phosphorous because protein amino acids does not contain it. They also labeled protein radioactive sulfur because DNA did not contain it. As a result, after centrifugation, they found that radioactive phosphorous was found in the cell, but no radioactive sulfur was found. This lead to the belief that DNA was the genetic material.
ID081	Watson and Crick was able to take data from both Chargaff and Franklin to put together the model of DNA. They found that since A=T and C=G, they were able to be paired up as bases in a double helix model, thus the model of DNA was born.
ID096	Experiments by Griffin based on Avery, McCarty, and MacLeods' findings showed that avirulent strands of Diplococcus pneumoniae could be transformed to virulence. Avery determined the transforming agent by using vitro assay.
ID096	Hershey and Chase used Escherichia coli and bacteriophage T2 to demonstrate that DNA is the genetic material. DNA enters the bacterial cell during infection and directs viral reproduction.
ID096	Watson and Crick proposed that the structure of DNA is a double helix. Chargaff determined the proposed base compositions of A,T,C, and G.
ID000	Avery et al constructed an in vitro assay for transformation using IIR bacteria and various fractions from disrupted cells of IIIS.  They demonstrated that RNase and Protease treatment did not prevent IIR bacteria from being transformed with IIIS cell components but that DNAs did.  They concluded that DNA was the transforming material.
ID000	Hershey and Chase used radioactive labels to tag phage proteins (35S) and nucleic acids (32P) and showed that the 32P-labeled phage nucleic acid was found in subsequent generations of phage.  Thus, DNA was the genetic material for these phage as well.
ID000	Data from Chargaff demonstrated that A and T had approximately the same concentration in cells as did G and C.  This suggested that G and C somehow were associated 1:1.  Franklin’s experiments demonstrated that the sugar phosphate backbone was on the outside of a helix
ID013	After Griffith's transformation experiment, these scientists performed a transformation experiment to identify which molecule, DNA or protein, serves as the transforming principle. They treated IIR cells with different enzymes and the enzyme DNase, which digests DNA, resulted in a solution that contained only IIR cells. In other words, no other cells were formed because transformation couldn't occur.
ID013	Hershey and Chase experimented with bacteriophages and further proved that DNA is the genetic material rather than protein. They labeled the protein and DNA with 32S and 32P, respectively, and only 32 P labeled phages were produced. They utilized the idea that the phages only inject their DNA into the bacterial cell to use its machinery while their protein capsid remains outside the cell.
ID013	Watson and Crick discovered that C, U, and T are pyrimidines and A and G are purines. Thus, a purine pairs with a pyrimidine in an antiparallel fashion. A and T have two hydrogen bonds, and G and C have three hydrogen bonds. As a result, a DNA molecule rich in GC bonds will require a higher melting point to denature it.
ID063	Through their experiments, we were able to learn that DNA is what causes transformations in bacteria.
ID063	Their experiments confirmed that DNA is genetic material. It was past thought that proteins carried the information due to the simplicity of DNA.
ID063	In the combination of these experiments, it was found that DNA is a double helix in shape with a phosphate sugar backbone connected by nucleotide bases.
ID102	They worked with the same two strains of bacteria as Griffith but they went through multiple steps and used the extract from the IIIS cells to treat the IIS cells and see if any transformation occurred. Transformation occurred in all except the DNased-treated IIIS filtrate plus the IIS cells.
ID102	Based on their existing knowledge, they knew it was either proteins or DNA that entered the bacterial cell which then caused viral reproduction. DNA contains phosphorous and proteins contains sulfur. So they labeled the phages 32P and 35S. They experimented E. Coli growing them in the presence of 32P and 35S then infected it with T2 viruses. The results of this experiments gave answers.
ID102	The comparison of the right-handed nature of the helix and the left-handed counterpart led to the finding of a chemical stability between pairings of A to T and G to C.
ID026	These experiments helped to prove that DNA was the genetic material inside organisms. Before these experiments, it was believed that proteins contained the genetic information that is passed down to later generations.
ID026	These experiments involved tagging different parts of cells with certain tags to see what part was passed to later generations after centrifugation. This also helped to prove that DNA was the part of the cell that was inherited by offspring and contained the genetic information.
ID026	Watson and Crick were able to analyze all of the information that was available to them to piece together the theory that DNA was a double helix. This information would also lead to finding that A bonds with T and G bonds with C.
ID136	Those experiments brought to light that proteins can not be the genetic material. That there must of been something else that controlled these phenomenon.
ID136	By the Hershey-Chase experiments we now understand that DNA is responsible for the genetic material, not proteins. They ran an experiment that proved that DNA carried the necessary genetic material.
ID136	Watson and Crick took data and material from Chargaff and Franklin and combined it with their own knowledge to come up with the structure of DNA, which is the helices. With that we now understand how DNA is put together and how it functions.
ID033	Avery, McCarty, and Macleod conducted experiments that proved DNA passed on the genetic information and not proteins.
ID033	The Hershey-Chase experiment proved that DNA and not protein get passed on.
ID033	Watson and Crick were able to successfully show the correct DNA structure based on a lot of different evidence from other scientists.
ID068	DNA was determined to be the genetic material by these experiments due to destroying certain parts of the cell and seeing if a live strain was recovered or not in the mouse. They destroyed the polysaccharide coat, lipids, RNA, and proteins in individual experiments and all recovered a live strain which meant that transformation was still taking place. When the DNA was destroyed no transformation occurred in the strain producing no live strain and the mouse lived.
ID068	Phage DNA was labeled with P and S radioisotopes to see what was transferred into the bacteria. The P would attach to the phosphate backbone of DNA and the S would attach to the sulfate groups of amino acids. When the phage injected DNA into the bacteria, the P radioisotope was found and not the S. This meant the phage was injecting DNA into the bacteria and not proteins.
ID068	These experiments showed that concentration of A were proportional to T and the same with G and C. This means that A binds to T and G binds to C in DNA base pairing.
ID101	The experiment that tested a deadly bacteria to mice to determine if the genes could be passed on from the dead bacteria to live bacteria that would then kill the mice.
ID101	They used radio active material to trace the sulfur and phosphor content in viruses to bacteria to determine what trait was transferred.
ID101	The equal amounts of A and T meant the molecules were bonded together, and the same goes for C and G. The X-ray photo indicates the position of the molecules and that it is a double helix.
ID070	The results of the Avery–MacLeod–McCarty experiment, published in 1944, suggested that DNA was the genetic material, but there was still some hesitation within the general scientific community to accept this, which set the stage for the Hershey–Chase experiment.
ID070	Hershey and Chase's experiment concluded that little sulfur containing material entered the bacterial cell. However no specific conclusions can be made regarding whether material that is sulfur-free enters the bacterial cell after phage adsorption. Further research was necessary to conclude that it was solely bacteriophages' DNA that entered the cell and not a combination of protein and DNA where the protein did not contain any sulfur.
ID070	The A:T and C:G pairs are structurally similar. In particular, the length of each base pair is the same and they fit equally between the two sugar-phosphate backbones. The base pairs are held together by hydrogen bonds, a type of chemical attraction that is easy to break and easy to reform. After realizing the structural similarity of the A:T and C:G pairs, Watson and Crick soon produced their double helix model of DNA with the hydrogen bonds at the core of the helix providing a way to unzip the two complementary strands for easy replication: the last key requirement for a likely model of the genetic molecule.
ID022	Avery and his fellow researchers created an in vitro assay to determine the transforming agent. Avery and other researcher made an S-strain from the heat-killed bacteria. They then used that strain   enzymes that destroyed proteins, lipids, carbohydrates, and other molecules, including the nucleic acid RNA and injected it in a mouse. Transformation still occurred. However, the molecules had been damaged, they were not responsible for the transformation. Avery and the other researchers repeated the experiment, this time using enzymes that would break down DNA. When they destroyed the nucleic acid DNA in the extract, transformation did not occur. There was just one possible conclusion. DNA was the transforming factor. Avery and other scientists discovered that the nucleic acid DNA stores and transmits the genetic information from one generation of an organism to the next.
ID022	Hershey and Chase used bacteriophage to show that DNA was the genetic material. They allowed bacteriophage to grow in two seperate medium one that has radioactive phosphorous and other had radioactive sulphur. Then they allowed this virus to infect the ecoli bacteria. once they were infected they took the bacteria and used a blender and after doing this they found phosphorus in them. Therefore, they concluded that DNA was the genetic material and not protein, since phosphorus which is found in DNA was being passed on to next generation.
ID022	Watson and Crick already knew that the phosphate group lied outside the helix. They saw the paper from Linus Pauling, who proposed that DNA had a triplex helix structure, but it was wrong, since phosphorus are very negative so they would ripple each other and due to this they would break apart the DNA strand. However, using the data from Rosalind Franklin they were able to figure out that it was a double helix and not triple helix and they also figured out that the phosphate group lied outside the DNA structure.  However, after looking at Erwin Chargaff data of A and T and C and G always forming a complementary pair, they were able to figure out the exact structure of the DNA double helix.  Since the nitrogenous base pairs and are stable compared to negatively charged phosphorus they were able to eventually figure that DNA was a double helix and not triple helix.
ID062	Experiments by Avery, McCarty and MacLeod consisted of expounding on the findings of griffith. they tried to transform R strain using S strain, and found that heat-killed S strain pretreated with protease and DNAase would most likely have some effect on transformation. with the stages of infection it was determined that S train with DNAase trasformed the living R strain. concluding DNA is the trasforming factor.
ID062	Hershey and Chase used T2 phage, a bacteriophage, for their experiments. They put labels on phage DNA with radioactive Phosphorus-32. They then followed the phages while they infected E. coli. They found that the radioactive element was only in the bacteria, and not in the phage.In a second experiment, they put labels on the phage protein with radioactive Sulfur-35. After the phage was attached to the bacterium, the radioactive element was found in the phage, but not in the bacteria. This showed them that genetic material which infects the bacteria is DNA.
ID062	watson and crick determined the structure of the DA double helix with Chargaff's information concerning the differing concentrations of bases A-T and G-C. from this they could figure out how the paring worked, and with Franklins work, they could figure out the orientation of the bases.
ID103	Before this experiment people were not sure if DNA or protein contained genetic material. This experiment demonstrated that DNA carried the information that conferred resistance.
ID103	This showed that DNA was the genetic material shared by many different living things, which was important because at the time no one new if a virus would contain the same type of genetic material.
ID103	Watson and Crick proposed the first accurate structure of DNA. Their work was based off theories of many other scientists including Chargaff and Roselen Franklin.
ID079	Before experiments performed by Avery, McCarty, and MacLeod, proteins were thought to play a large role in bacterial transformation rather than DNA, as we now know today. This discovery was built on information from previous experiments performed by Griffith.
ID079	Hershey and Chase performed experiments that concluded that proteins were not carriers of genetic material in phages, but rather DNA is the genetic material. This experiment showed that DNA is involved in the process of viral infection of bacteria leading to viral replication.
ID079	Chargaff discovered that the four bases, thymine, adenine, cytosine, and guanine, are proportional; A and T are proportional to each other and C and G are proportional to each other. Rosalind Franklin's work suggested that DNA has a helical structure. Based on previous findings regarding DNA composition and helical characteristics, Watson and Crick created their double helix DNA model.
ID113	Avery, McCarty and MacLeod experiment was based off of Avery's experiment and was to determine whether the transforming factor, was a protein or a nucleic acid. In their experiments, they found that heat-killed pre-treated with protease did not affect the ability of 'dead' S to transform R but DNAase did, therefore they concluded that the genetic material in transformation is most likely DNA.
ID113	In their experiments, they used two radioactive markers to label the proteins and the DNA of the phages. The proteins were labeled with 35S (a radioactive form of sulfur) and the DNA was labeled with 32P, a radioactive form of phosphorus. Then, after a set of series of incubation of phage with non-radioactive bacteria, it was found that the radioactive DNA was always found with the bacteria cells and that the radioactive protein was always still left. Therefore, all of the information needed to produce new viruses was contained in the DNA and not the protein.
ID113	With using the information from Chargaff and Franklin, Watson and Crick were able to describe DNA as a anti-parallel chains that are double helix and have nitrogenous bases of opposite chains that are pairs, with a few more features.
ID021	They were trying to answer the question, what molecule serves as the transforming principle? Through a series of centrifuge, heat kill, homogonize, and treatments, they tested for transformation in different mixtures and found that DNA is the transforming material and not RNA or protein.
ID021	Hershey and Chase asked what molecular component of the phage- DNA or protein- entered the bacterial cell and directed viral reproduction. Radioisotopes 32P (labeled DNA) and 35S (labeled proetins) were grown with the E.coli and phage, and centrifuged. Most of the 32P labeled DNA was with the bacterial cell and the 35S protein was with the empty phage coat. This demonstrated that the genetic material in the phage is DNA, not protein.
ID021	Chargaff showed that most organisms do not contain equal proportions of the 4 nucleotides, but rather more or less AT pairs or GC pairs. Franklin solved  the turn ratio of DNA in 1.7 angstroms .
ID069	These experiments attempted to explain the "transformation" phenomenon discovered in Griffith's experiments. They heat-killed IIIS cells,  added them to IIR cells, then treated with protease. After transformation continued to occur (IIIS cells were produced), protein was disregarded as the active factor. After the cells were treated with ribonuclease and transformation continued to occur, RNA was disregarded as the active factor. Transformation ceased after the cells were treated with deoxyribonuclease, indicating that DNA was the active factor and therefore the genetic material.
ID069	These experiments further supported that DNA was the genetic material. They labeled T2 bacteriophages with either radioactive phosphorus-32 or radioactive sulfur-35. After the separated bacteriophages infected bacterial cells, the cells underwent centrifugation. Afterwards, sulfur (a component of proteins) remained in the bacteriophages but not the cell, while phosphorous (a component of DNA) remained in the cell but not in the bacteriophages, indicating that DNA was the heritable material.
ID069	Chargaff's studies showed that there are equal amounts of A and T and equal amounts of G and C in DNA. Rosalind Franklin's X-ray diffraction data revealed the physical measurements of DNA and its double helical structure. Watson and Crick used the above information to create a DNA model consisting of A-T and G-C pairs surrounded by a phosphate backbone, all twisted in a double helix pattern.
ID024	Avery, McCarty, and MacLeod took the S strain and fragmented it. They wanted to know what made the rough strain turn smooth. They fragmented the smooth strain into different biochemicals, including polysaccharides, lipids, RNA, protein, and DNA. Ultimately, they wanted to see what the hereditary material was. When everything, with the exception of DNA, was removed, the mouse died. When  DNA was isolated and removed, the mouse lived, proving that DNA was indeed the mode of heredity for cells.
ID024	Hershey and Chase wanted to know if DNA was the hereditary material for everything, including bacteriophages. Radioactive tags were used to track DNA and protein in the phages as they transmitted their genes into bacterial cells. The bacterial cell was found to be labeled with DNA, so Hershey and Chase showed that it was DNA that was the mode of genetic transfer for bacteriophages.
ID024	Watson and Crick took what other people knew and made it seem like their own work. They took CHargoff's rules about base pairing and Rosalind Franklin's work with x-ray crytallograhy. Chargoff had discovered that the bases paired with each other to make DNA. A paired with T and C paired with G. Rosalind Franklin determined that DNA was a helical structure and that there was a phosphate backbone on the outside. Watson and Crick took this information and determined the structure of DNA.
ID039	Avery, McCarty and MacLeod based the experiment on earlier work done by Frederick. They wanted to know what molecule was responsible for transformation. They performed experiments using large quantities of IIIS cells in liquid culture. The experiment consisted of them centrifuging and collected cell. Then using heat to kill these, and taking the remain homogenize cells and filtering them. Other things were added and tested. Eventually they discovered that the active factor was DNA.
ID039	Hershey and Chase worked with E. coli and used the bacteriophage T2 that effects E.coli to prove DNA was the genetic material. They did this by inserted the phage into the cell and monitored its reproduction within the cell. They added radioisotopes 32 phosphorus and 35 sulfur to the phage. They knew that 32phosphorus would label the DNA because it contains phosphorus. They ran the experiment, and in the end they noted that progeny of the bacteriophages contained the 32 Phosphorus and not the 35 sulfur. Which indicated that the genetic material of the bacteriophage was DNA not a protein.
ID039	Watson and Crick was trying to determine the structure of DNA. They couldn't figure out the exactly how everything fit. They consulted Chargaff's work and eventually knew what paired with what, but still they didn't know the structure. They ultimately used the work of Franklin without her permission. Rosalind Franklin's worked showed that the structure of DNA was helical and with this bit of information, Watson and Crick finished their model.
ID053	These experiments utilized previous findings from Frederick Griffith's experiments with strains IIIS and IIR bacteria.  They centrifuged heat-killed IIIS bacteria and methodically destroyed one candidate for genetic material in each trial from the bacteria, such as lipids, proteins, and DNA.  Afterwards, they added the IIR strain.  When there was no transformation/recovery of live IIIS bacteria, they would know that they had destroyed the genetic material.
ID053	In these experiments, they tagged the protein coat of some T2 bacteriophages with a radioactive isotope of phosphorous-32, and they tagged the others bactiophages' DNA with sulfur-35.  After allowing each of the groups to infect a bacterium, they examined the bacterium to see what isotope was inside of it.  The ones infected by sulfur-35 bacteriophages contained sulfur-35 in them; those infected by the phosphorous-32 bacteriophages did not have a marking in them.  This showed that DNA was the material inserted into the bacteria.
ID053	Watson and Crick are famous for developing the double-helix model for DNA. They realized that Chargaff's data showed that A must bond to T and that G must bond with C, and this allowed them to create a model with the nucleotide bases in the middle and the backbones on the outside.  Also, Franklin's data showed them that DNA must be in a helical structure, and they were able to develop the structure as we know it.
ID049	Using results of Griffith's experiments that pneumococcus S strain properties could be transferred to R strains, Avery, McCary, and MacLeod set out to discover what exactly was being transferred. They lysed the cells to release cellular components. They destroyed the carbohydrates, but the R strain was still able to pick up the sugar coated property of the S strain. They destroyed the proteins, but the bacteria was still able to transform. Then they destroyed RNA, and the bacteria was able to transform. Finally the DNA was destroyed and the bacteria was unable to transform. They determined that DNA was the genetic material.
ID049	They infected bacteria with T2 phages. These phages were radioactively labeled with 32P or 35S. Found that there was 32P (phosphorus backbone) inside the cell.
ID049	They found that dna wound in a double helix
ID118	Griffth's transformation experiment provided foundation for Avery, McCarty, and MacLeod's research. There research answered the question on what molecules serves as transforming principle. In 1944 after 10 years they published their paper that serve as turning point for molecular genetics as before scientists used to think that protein was genetic material. The were able to reach the conclusion as when protein destroyed from virulent strand of pneumonia, it still killed the mouse. When DNA was destroyed in virulent strand for S strain extract that causes pneumonia, the mouse was alive. This showed that the genetic material was DNA and not protein.
ID118	Hershey and Chase was second major piece of evidence that support that DNA was a genetic material. They used E.coli and T2 bacteriophage. They used radioisotopes 32P labeled DNA and 35S labeled protein. They injected the bacteria with T2 phage. The results of the experiment was that 32P labeled DNA had been transferred into bacteria cell following adsorption, while almost all of the 35S labeled protein remained outside bacterial cell was recovered in phage that are empty phage coats. It showed that the viral that causes the lytic cycle in DNA and not protein.
ID118	Watson and crick proposed the structure of DNA as double helix in 1953.Watson and Crick put all the pieces found by Chargaff's and Rosalind franklin together creating the model for DNA. Chargaff's study concluded that A=T and G=C in the nitrogenous base pairing in all organisms. Rosalind franklin in 1950-1953 showed that DNA had 3,4-angstrom periodicity characteristics of helical structure by X-ray diffraction analysis. By taking Rosalind Franklin's unethically and using chargaff's data they come to model of DNA which was double helix, two anti-parellel strands connected by base pairing, and stacked nitrogenous bases.
ID052	Before this experiment, it was unclear whether proteins or DNA contained the hereditary factor. In their experiment, they destroyed proteins in their bacteria sample and hypothesized that colonies should not form. However, colonies did form so the results disproved the hypothesis that proteins were the hereditary material.
ID052	Hershey and Chase radioactively labeled the DNA and protein materials within their sample of bacteriophages using radioactive phosphorus-32 and sulfur-35 respectively. After the bacteriophage inserted its viral material into the bacterial cell, Hershey and Chase used a high speed blender to separate the bacteriophages from the bacterial cell. Their results showed that the solution contained high amounts of sulfur-labeled protein, indicating that the phosphorus-labeled DNA was transferred into the bacterial cell. This experiment concluded that DNA was the genetic material in bacteriophages.
ID052	Erwin Chargaff's rules and Rosalind Franklin's photo-51 provided crucial information that Watson and Crick needed to properly construct their model of DNA. Chargaff's data indicated that the concentration of adenine was roughly approximate to the concentration of thymine, and similarly, guanine to cytosine. This data inferred that DNA contained base pairs consisting of adenine to thymine or guanine to cytosine.
ID059	By using a test tube,DNA, and streptococcous to test whether DNA or proteins were the transforming factor of genetic material, they were able to conclude that it was in fact DNA. They found that DNA is the transforming factor and that genes are made of DNA that consists of proteins and other useful materials.
ID059	Hershey and Chase used mice and two sets of T2 bacteriaphages to determine if DNA was the transferal of genetic information. For example they used one with radioactive sulfur that isn't found in DNA and the other wih radioactive phosphorus bacteriaphage. As a result they found that the one with sulfur did not transfer which entails that DNA is the transferring information.
ID059	Watson and Crick found the base pairing sequence by understanding that DNA is double helix with phosphate and sugar group alternating to form the overall structure. They found that this structure was based off a x-ray chrystaography taken by Roslind Franklin. Then they noticed that the ATGC base matches were determined by the structure and what was holding it together. They learned that the hydrogen bonds of A matched with T and C matched with G were the elements that held the double helix together.
ID034	it was the culmination of research in the 1930s and early 1940s to purify and characterize the "transforming principle" responsible for the transformation phenomenon first described in Griffith's experiment. Avery and his colleagues suggest that DNA, rather than protein as widely believed at the time, may be the hereditary material of bacterial, and could be analogous to genes and/or viruses in higher organism
ID034	they coat the protein with 32P and 35S and insert the virus into the bacteria. Hershey and Chase were able to demonstrate that mist of the 32P-labeled DNA had been transferred into the bacterial cell while 35S-labeled protein remained outside the bacterial cell
ID034	the data available to Watson and Crick crucial to the development of their proposal,came primarily from two sources: base composition analysis hydrolyzed samples of DNA and X-ray diffraction studies of DNA
ID040	Avery, McCarty and Chase began their experiment with a virulent strain of bacteria that they heat killed and obtained a soluble filtrate that was still able to transform. All of the carbohydrates, proteins, and lipids were then extracted. Separate samples were then treated with a protease, a ribonuclease, and a deoxyribonuclease. Each sample was then tested for transformation. Transformation occurred in all samples except for the one treated with deoxyribonuclease. This lead to the conclusion that DNA is the active factor for genetic material.
ID040	Hershey and Chase knew that phages contained both protein and DNA and wanted to discover what the genetic material was. They began by using radioisotopes of phosphorous and sulfur to tag DNA and protein, respectively. Labeled phages and unmarked E. coli bacteria were mixed and complexes were formed. These complexes were isolated and spun to strip off the phages and analyze the phages and bacteria separately. Hershey and Chase showed that the tagged DNA had been transferred into the bacterial cell while the tagged protein stayed behind in the coat of the phage.
ID040	Erwin Chargaff used methods to separate the bases and determine the amounts of the different bases in each sample. He found that the amount of adenine is proportional to the amount of thymine and the amount of guanine is proportional to the amount of cytosine. Using X-ray diffraction, Rosalind Franklin was able to obtain improved X-ray data and her work confirmed that the shape of DNA was some sort of helix. Using this data, Watson and Crick came to the conclusion that DNA was a double-helix molecule with two phosphate backbones running antiparallel and paired bases of A-T and G-C.
ID058	Avery created an in vitro assay to determine what the transforming agent was. For each part of the experiment they destroyed a different component of the cell and then injected it into  mice. All the mice died except for the mouse where DNA had been destroyed. Therefore the genetic material was DNA.
ID058	They used radioactive P and S to label the DNA in the head of bacteriophages as well as the outer shell of the bacteriophage. They then put the cells in a blender after attachment of the bacteriophages and checked which marker was present in the cell and which one was outside. The P was for DNA and was found in the cell and the S was attached to the shells and was left outside. Therefore it supported that DNA was the genetic material because it got transferred.
ID058	Based on both of these pieces of information, Watson and Crick was able to determine that DNA was a double helix with two antiparallel strands where A pairs with T and G pairs with C.
ID142	Bacteriophages contain sulfur and phosphorous parts. They knew that the phosphorous parts corresponding to DNA and sulfur to proteins. By radio-labeling these and allowing the bacteriophage to replicate they could see of phosphorous material or sulfur material were pasted on. They concluded that the phosphorous material was donated to the cell and thus, DNA was passed on as the hereditary material.
ID142	The x-ray diffraction that were stolen from Franklin proved that the structure of DNA was a helix and Watson and Crick were able to deduce that it would be an antiparrallel helical structure. They knew from these images that the bases must face into the helix as well. Chargaff's data was instrumental in discovering the pattern of how the bases would pair. A would pair to T and C to G.
ID124	They destroyed various parts within the cells injected into mice that caused them to die and eventually discovered that when they destroyed DNA the mouse lived.
ID124	They attached different radioactive particles to the phosphates and sulfur and let the bacteria transfer their material and they found out that the material transferred was dna instead of proteins
ID124	They did experiments that showed the relationship between the different bases in DNA have a ratio of approximately 1:1 for the bases that pair together.
ID015	Their experiments definitively proved that the transforming principle was DNA, and not protein or RNA, the other two constituents in the nucleus
ID015	Using T2 bacteriophage, they showed that DNA and not protein entered the bacterial cell. Since new T2 phage particles were produced inside the cell, it was concluded that DNA was responsible for its development.
ID015	They determined that DNA is:
ID111	The experiments of these scientists followed the studies that Frederick Griffith did. They were all trying to determine if it were proteins or nucleotides that served the function of carrying genetic information. They did an experiment following Griffith's that used bacteria and showed that the DNA rather than protein was the hereditary material of the bacteria. They thought this could be the same for genes.
ID111	This experiment followed those of Griffith as well as Avery, McCarty, and MacLeod. They all had the same goal to show that DNA was the genetic material. However, Hershey-Chase focused on bacteriophages. They showed that when bacteriophages (composed of both proteins and DNA)  infect bacteria, their DNA enters the host bacterial cell while the majority of the proteins do not.
ID111	The key to the Watson and Crick model is the specificity of base pairing. Chargaff showed that A was equal to T and G was equal to C in base pairs. Watson and Crick realized that you could pair A to T and C to G and that it would account for these proportions. They also realized that such pairings could occur as a result of hydrogen bonds between base pairs. They published their analysis of DNA structure and it had seven features.
ID115	the experiment by Avery McCarrt and MacLeod demonstrated that DNA was the genetic material by injecting virulent strains into mice and testing to see whether or not the virulent strain remained in cells based on DNA. Those that had DNA removed let the mice live and those that had DNA infected the mice and died.
ID115	Hershey-Chase demonstrated that DNA is the genetic material through centrifuging cells with phosphorous and sulfur. Those that did not stay in the cell (proteins-sulfur) were not genetic material, and those that did (DNA-phosphorous) did have the genetic material.
ID115	By illustrating that the nucleotide combinations of A-T and G-C with hydrogen bonds allowed DNA to build together, they were able to design the model that would become the basis for the model of DNA.
ID035	Their experiments definitively proved that the transforming principle was DNA, and not protein or RNA, the other two constituents in the nucleus. Since new T2 phage particles were produced inside the cell, it was concluded that DNA was responsible for its development.
ID035	The experiments by Hershey and Chase demonstrated to many that DNA was the hereditary material. Using T2 bacteriophage they showed that DNA and not protein entered the bacterial cell.
ID035	They determined that DNA is a double-stranded, the strands are oriented in an anti-parallel manner to each other, purines nucleotides are opposite pyrimidines nucleotides, guanine hydrogen bonds with cytosine, adenine hydrogen bonds with thymine, and hydrogen bonds and hydrophobic bonding between stacked bases stabilize the structure.
ID140	These guys did experiments in which they injected mice with either live or dead bacteria to see if the mouse lived.
ID140	These guys did experiments to test whether it was DNA or proteins that carried genetic materials. They did an experiment that basically color-coded DNA and protein, mixed up in a tiny centrifuge and then were able to see where the genetic material was found.
ID140	Chargoff was able to find a 1:1 ratio in A:T and G:C and Rosalind was able to use her prowess at x-ray diffraction to capture an image of DNA as a double helix. Watson and Crick basically stole Rosalind's find and used Chargoff's find to put together a model of DNA before anyone else could.
ID064	These experiments were done by injecting virulent pneumonia strains into mice. If these viruses had a rough coat the mouse's immune system could fight the virus, but if it had a smooth coat, then the mouse was unable to fight and became sick and died. When they heat-killed the virus, something interesting happened. The heat-killed smooth strain did not kill the mouse, but when the heat-killed smooth strain mixed with the live rough strain was injected, the mouse died. This suggested that somehow genetic material from the smooth strain was transmitted to the rough strain.
ID064	Hershey and Chase used radioactive Phosphorous and Sulfur to label DNA and the viral protein coats respectively. They did this by injecting bacteria with the viruses, let them inject their material into the bacteria, labeled them, and then let them reproduce. After the bacteria reproduced, they found that phosphorous had been transmitted to the next generation and the sulfur had not. Therefore, DNA was also the genetic material in bacteriophage.
ID064	Chargaff's data showed that in all the different species he examined (including humans) A and T have very similar proportions as do C and G, so there is roughly the same amount of A as T and C as G. Rosalind Franklin had produced a very clear image through X-ray crystallography that, when interpreted by someone who knew how, showed that the DNA molecule is a double helix. Watson and Crick used this information to form the model of DNA that still stands strong today.
ID104	Avery, McCarty, and MacLeod used virulent proteins and injected them into mice. They discovered that just because the virulent proteins were heat killed and mixed with the non-virulent proteins would cause the death of mice even though the proteins that would cause death were dead before they even were injected into the mice. This meant that the proteins had to have some way of transferring the genetic material even after death. The only way that could have happened was if there was a crossing over of the DNA that held the virulent genetic material that caused death.
ID104	Hershey-Chase experiments took radio labels phosphorus and sulfur. The theory was that the radio labeled phosphorus would bond to the phosphodiester back bone of DNA and make it detectable if it was passed on to the next generation. The sulfur was bond with the sulfur in the proteins that wasn't in DNA and if it held the genetic material then it would pass the radio labeled sulfur onto the next generation of cells.
ID104	yes they were successful in producing the correct model of the double helix DNA, but they were only able to do so with the help of Rosalind Franklin and her abilities of perfect x-ray crystallography. As well as the knowledge of how the four bases always paired up with the same ones (G-C & A-T). They got lucky enough to get the right information at the right time and they were smart enough to know how valuable that information was. With this information, they were able to successfully model the proper form of DNA which turned out to be a double helix.
ID147	They took two sets of bacteria, one was vvirulent and the other was not. They then injected a series of different combinations of the two bacteria, along with different treatments before injection. They discovered that non-virulent bacteria that was mixed with heat treated virulent bacteria would result in death. This showed that DNA not protein transferred between the bacterias.
ID147	They labeled bacteriophages with either sulfur or phosporus tags for protein or DNA. After allowing the bacteriophages to infect they discovered that only phosphorous tagged DNA was transfered
ID147	Watson and Crick took data from Chargaff and Rosalind to help develop the proper double-helix structure that contained matching A-T and C-G pairs along with the twisting nature seen in the X-rays
ID048	Showed avirulentstrains of Diplococcus pneumoniae could be transformed
ID048	Used E. coli and bacteriophage T2
ID048	Proposed base composition
ID097	By using lab rats they were able to determine how bacteria effects the body and its survival.
ID097	Their experiment helped show geneticists how genetic diversity occurs on a microscopic level, which produces virus in the human body.
ID097	Watson and Crick was able to gather other scientists (Chargaff and Franklin) findings and put the pieces together to create the correct formation of DNA for the first time.
ID129	Their transformation experiment. They took IIIS virulent cells and put them in a centrifuge. Then they were heat killed. Protein was removed, and it was established that DNA was the "active factor."
ID129	Hershey and Chase used E. Coli and bacteriophage T2. It also demonstrated that DNA instead of protein was the genetic material by using radioisotopes 32P 35S. It also demonstrated DNA enters bacterial cell during infection and viral reproduction.
ID129	Chargaff proposed base composition that the amount of adenine and thymine is equal and the amount guanine and cytosine is equal. Watson and Crick took this information and proposed the DNA structure as a double helix.
ID030	This is experiment attempted to determine whether DNA or protein was the molecule of inheritance. To do this, they noted whether or not mice would die when injected with a phage. One strain has its protein denatured while the other had its DNA denatured. The phage with DNA denatured was unable to pass down its genome to create new viruses and thus the mice survived.
ID030	This experiment was vital in determining that DNA was the molecule of inheritance. The DNA and protein of a phage were both marked with radioactive isotopes. The researchers then determined which radioactive isotope made it into the infected cell. This allowed them to conclude that DNA was most likely the molecule of inheritance.
ID030	This was possibly the most important work in the field of genetics during the 20th century. When the molecule was successfully modeled thanks to the work of many people, it greatly expanded the current understand of genetics, propelling the field forward.
ID036	Avery, MacLeod used Streptococcus pneumoniae bacteria nucleoid and purified the DNA, proteins and other materials. They mixed R bacteria with different materials. They saw that only those mixed with DNA were transformed into S bacteria.
ID036	When radioactive bacteriophages infect nonradioactive bacteria, all the infected cells became radioactive and, most  of the radioactivity was passed on to the next generation of bacteriophages.
ID036	Chargaff and colleagues discovered
ID055	DNA was concluded to be the component that contained genetic material because during the experiment eliminating the DNA from the S strain was the only one that allowed the mouse to live and no live S strain cells to be found.
ID055	In this experiment there was two sets of T2 bacteriophages used, one protein coat labeled 35S (not found in DNA), and another set phosphorus in DNA labeled 32P (not found in protein). At the end of the experiment they found that the E. coli injected with 32P was the only one that was produced in the new phages indicating DNA as the necessary genetic material agent.
ID055	They found that A is proportional to T and G is proportional to C but that each pair could be of different percentages. Also, this helped to discover the structure of DNA as a double helix.
ID090	They created an assay to determine the transforming agent of IIIS/IIR. They took a bacterial strain and destroyed different parts of the strain in different trials (i.e. RNA, lipids, proteins etc). In all trials, IIR was still converted to IIIS except when DNA was destroyed. This showed the DNA was the transforming principle.
ID090	Hershey and Chase tagged phages with either 32P(phosphate groups in DNA) or 35S(sulfur bridges in proteins.). after centrifugation, they were able to determine the bacteria were "hot" with P and not S meaning that the phages had injected DNA into the bacteria and not proteins.
ID090	Watson and Crick used the data obtained from the base pairing data from Chargaff as well as Rosalind Franklin's photo 51. This allowed them the determine the DNA was a double helix and that the bases are paired in a complementary fashion.
ID107	In 1944, Avery, McCarty and McLeod concluded after extensive research that DNA was in fact the source of genetic information for cells. To show this, they used virulent cells which they heated to kill any living aspects of the cells. After processing, they introduced the "left overs" of the virulent cells were introduced to avirulent cells. The "left over" material was still able to transform avirulent cells to become virulent, even though the virulent cells had been destroyed. They controlled for possible contamination of the sample by adding an enzyme that would destroy any RNA, but the transformation still occurred. The only way to stop the transformation was by adding a DNA degrading enzyme. Thus they concluded that DNA was the "active factor" in the transfer of genetic material.
ID107	The Hershey-Chase experiments used an E. coli bacteria cell and a bacteriophage T2 to show that it was the DNA of the virus that was injected into the cell and not the associated proteins. They determined this by using radioisotopes of phosphorus and sulfur to figure out the location of viral DNA and proteins after the infecting the bacteria. DNA contains phosphorus but not sulfur and proteins have sulfur but not phosphorus. They found that the radiolabeled phosphorus remained inside the infected bacteria and the sulfur remained outside the cell. This indicated that the viral DNA was able to take over the bacterial cell machinery by using the bacteria itself to reproduce  more viral progeny.
ID107	Erwin Chargaff found that the amount of adenine residues was proportional to the amount of thymine resides and the amount of guanine residues was proportional to the amount of cytosine residues. He also found that the sum of the A and G residues equaled the sum of the C and T residues, but the percentage of A/T pairings don't necessarily equal the G/C pairings. Watson and Crick used Chargaff's findings to create a model for the structure of DNA. They hypothesized that DNA was made of two long polynucleotide chains that coil to form a double helix. The chains run antiparallel and are "stacked" on top of each other, each turn of the helix is 3.4 nm long, and the helix is 2.0 nm wide. There are major and minor grooves that appear due to base pairing. They also hypothesized that the bases described by Chargaff were paired by hydrogen bonds. Their model still stays in tact today, with a few small changes.
ID127	They continued Griffith's experiment with the R and S bacteria that was injected into the mice. They wanted to know if the transformation was from protein or nucleic acids.
ID127	They labeled the virus with sulfur for proteins and phosphorous for DNA and they allowed the bacteriophage to infect the DNA. What they found was that the bacteria had the phosphorous in its genome which meant that DNA was the genetic material of the bacteriophage.
ID127	Rosalind Franklin had been studying DNA before Watson and Crick and she was about to discover the structure of DNA but Watson and Crick used her work and figured out the double helix structure of DNA. In reality, Rosalind Franklin was more responsible for the study of DNA than Watson and Crick.
ID145	They used IIIS virulent cells and IIR cells.  By adding IIS cells treated with either DNA, RNA, or protease to IIR, the three scientists were able to determine that DNA was the transforming factor.
ID145	Using E. coli, phages, and radioisotopes 32P and 35S, Hershey and Chase demonstrated that DNA is the genetical material in phages.  Hershey and Chase grew E. coli cells in either 32P or 35S and added the phages to the E. coli.  Because phages have an outer protein coat and DNA inside of a head, the two scientists knew that the phages would either be labeled on the outside (35S) or the inside (32P).  By infecting unlabeled bacteria with labeled phages, the two scientists concluded that DNA was the genetic material because 32P was found inserted into the bacteria.
ID145	Chargaff separated the four bases in DNA samples from different organisms and determined the amounts of each base in the organisms.  Franklin's data suggested that a helix was the structure of DNA and confirmed the distance between the bases.  Watson and Crick used the results from these scientists to determine that DNA had a double helix and that pairing A with T and C with G would account for the proportions noted by Chargaff.
ID044	These experiments was the initial event that lead to
ID044	This study was of the Escherichia coli and one of its infecting viruses, bacteriophage
ID044	Erwin Chargaff and his colleagues used chromatographic methods to separate the four bases in DNA from various organisms. They then used quantitative methods to determine the amounts of the
ID078	In these experiments they mixed IIR cells with heat killed IIIS cells that were mixed with various enzymes to break down different aspects of the cell to determine what the transforming factor was
ID078	In this experiment, they treated T2 phage with radioactive phosphorus and sulfur to determine which would remain in the bacteria to produce labeled T2 phages
ID078	To get their model, they used the x-ray crystallography to determine that DNA was a helix, specifically a double helix with a 3.4 angstrom  twist. and each backbone is connected via hydrogen bonds between AT and CG.
ID120	These experiments showed that DNA can change. The DNA in Avery's experiment went though conjugation. Then by killing certain parts of the cell they could pinpoint what was the genetic. aterial
ID120	The used phages that injected their genetic material into the bacteria. Based on what chemicals were present, determined the genetic material.
ID120	This provided them that A and T had to be connected while C and G had to be connected, and they discovered the double helix from Franklin. This gave them the knowledge to create the model.
ID123	Avery, McCarty and MacLeod discovered what the transformation factor was in the bacteria that killed the mice.  They thought that it was either a protein or a nucleic acid.  They used protease to treat heat killed S strain bacteria to see if it inhibited the transformation of the R strain bacteria, but it did not.  So, they treated it with DNAase and the transformation was inhibited.  This concluded that DNA was the genetic material.
ID123	In Hershey and Chase's experiment, they used two radioactive markers to label the protein and DNA of a bacteriaphage.  They allowed the bacteriaphages to infect the bacteria, and then they put the cells in a centrifuge.  Once the cells were separated, the bacteria was found with the DNA marker, so they knew that the DNA was injected by the bacteriaphages.  This allowed them to conclude that DNA was the genetic material.
ID123	Watson and Crick were able to build the model of DNA from the information gathered by Erwin Chargaff and Rosalind Franklins photograph.  The photograph showed the shape of DNA, so they had to figure out how the bases went together the form the double helix.  From Erwin Chargaff's work, they used the base pairing data to put the bases together so that a double helix could be made.  They paired adenine with thymine and guanine with cytosine.  This allowed them to construct the model of DNA as a right handed double helix.
ID144	Their experiments concluded that DNA is the substance that causes bacterial transformation
ID144	Hershey and Chase showed that when bacteriophages, which are composed of DNA and protein, infect bacteria, their DNA enters the host bacterial cell, but most of their protein does not
ID144	Their research found out that DNA's structure is a double helix. This was an incredible find and extremely important
ID023	Avery, McCarty, and MacLeod demonstrated that DNA was the genetic material by injecting mice with various strains of bacteria. Only the bacteria that had the DNA destroyed (compared to the bacteria that had the RNA, lipids, polysaccharides, and protein) lived, because the genetic material was destroyed and could not harm the mouse. This proved that DNA was the genetic material in cells.
ID023	The Hershey-Chase experiment also supported that DNA was the genetic material within the cell by determining if bacteriophages injected protein (sulfur) or DNA (phosphorus) into the cell. Since phosphorus was injected, the conclusion was that DNA was the hereditary material injected into the cell.
ID023	Watson and Crick (along with other scientists) determined that 4 base pairs were responsible for comprising the DNA molecule. X ray diffraction aided their research by making it possible to know that DNA was a double helix. Calculations have been performed and the percentage of AT pairs and GC pairs are not equal, which also contributed to creating the model of DNA.
ID002	This experiment showed that DNA was the genetic material. They exhausted experiments of all forms of the cell to see which part would not lead to the IIR strains becoming IIS strains. The only time the strains were not formed was when the DNA itself was destroyed. If the DNA was not destroyed, the IIR cells could pick up via transformation, DNA involving the polysaccharide gene coating so that the strain would become IIIS
ID002	Hershey-Chase experimented to see if DNA was the genetic material in bacteriophages. They grew T2 phages in two mediums: radioactive phosphate and radioactive sulfur. The two hot phages were then placed in a new medium with E. coli to infect. After infection, the new phages were tested to see which radioactive element was present. Since phosphate is in DNA, that would prove DNA is the genetic material, and sulfur would prove proteins. The phages only showed traces of phosphate, thus proving DNA is the genetic material
ID002	Watson and Crick deduced after Chargaff's experiment lead to the discovery of which bases paired. This helped prove the double helix hypothesis in that no other base pairings would create such a structure.
ID143	Avery, McCarty, and MacLeod illustrated that DNA was the genetic material by showing that DNA is the material that is capable of transforming bacteria.  The idea of DNA transforming bacteria signifies that cells take up extraneous genetic material from their surroundings.  Since DNA is taken up through the cell membranes, it is known as the genetic material.
ID143	Hershey and Chase demonstrated and perpetuated the idea that DNA is the genetic material.  At this point in time, many people believed that proteins were the genetic material due to their complex structure.  However, since these researchers used radioactive phosphorus, the backbone structure of DNA, and radioactive sulfur, a component of proteins, they were able to distinguish which element appeared as the universal genetic material among all organisms.
ID143	Using the information from Chargaff and Franklin, Watson and Crick demonstrated the structure of DNA.  DNA incorporates complementary pairing (A pairs with T and C pairs with G).  Additionally, the genetic material is created with a phosphate backbone in a double-helix fashion.
ID091	At first, proteins were thought to be the carriers of genetic information. Twenty amino acids was seen as a complex system that could encode sufficient information compared to the four bases of DNA. These experiments showed that protein did not carry genetic information using radioactive elements and also qualitative methods.  Knowing that DNA is the genetic material opened up the study of genetics.
ID091	Bacteriophages and mammals are very different. However, we share the same genetic carrying vectors, DNA. So, it was shown that DNA is the universal language for life.  The sequence of bases changes, but the chemical makeup is the same.
ID091	The X-ray diffraction pattern showed a double helix nature. Chargaff showed that A and T showed up in the same concentration and so did G and C. So, the antiparallel, A-T, C-G nature was concluded. Knowing the structure lead to the theory of how DNA is replicated (semi-conservative) and allowed for the explaining of other phenomenons.
ID011	These experiments further cemented the fact that DNA was the genetic material passed on to offspring and new cells.
ID011	This proved that DNA was the genetic material passed on and not proteins.
ID011	The information Watson and Crick used led them to develop the model of DNA which is essential to genetics today.
ID132	They discovered the process of transformation they used different strains of bacteria and mice including heat killed bacteria. They found that the transforming factor was the genetic material.
ID132	They showed that when bacteriophages infect bacteria, their DNA enters the host bacterial cell. The used radioactive elements into the bacteriophages by adding isotopes to separate media within the bacteria. The progeny contained the radioactive isotopes in their structures.
ID132	They learned from Chargaff that the four different base rings in DNA appear to be found in pairs. Like A with T and G with C. They also saw Fanklin's DNA X-ray patterns that confirmed the helix shape adding more information to their model.
ID088	They demonstrate that the transformation of Streptococcus pneumoniae from an un-viral type to a viral type is the result of the transfer of DNA from dead smooth organisms to live rough ones.
ID088	Hershey and Chase demonstrated that while composed of protein as well as DNA, bacteriophages that infect bacteria, their DNA enters the host bacterial cell, but most of their protein does not.
ID088	Studies by Roselind Franklin showed that DNA had a 3.4 angstrom periodicity, characteristic of a helical structure. Watson and Crick stole her data, and were able to conclude that DNA was composed of a double-helix pattern.
ID131	Avery, McCarty and Macleod looked at how the different types of molecules within the cell affected the way traits were passed down, and it was determined that neither lipids, proteins, or anything other than DNA passed down genetic material.
ID131	Through infecting bacteria with viruses, Hershey and Chase were able to conclude that DNA was the genetic material as opposed to protein becasue very little of the protein was passed to the next generation.
ID131	Watson and Crick worked on their own research of bases and observed their combinations. They may have unethically acquired information from other resources that lead to their final hypothesis of the double helix.
ID007	These experiments relied on the process of transformation in bacteria.  The transfer of virulence from one strain of bacteria to another did not occur when this strain's DNA was destroyed, but did occur when other components of it (lipids, protein, RNA, etc.) were destroyed.
ID007	These experiments relied on tagging the phage's protein coat and their DNA in separate trials.  When they were allowed to infect bacteria, only the tagged phosphorus in the DNA was found inside the bacterial cells, which proved that it was the DNA that the bacteriophages were injecting into the bacteria and using to replicate.
ID007	Chargaff's rules showed that A-T and G-C were likely paired in some way, and Franklin's crystallography showed that the structure was a double helix.  Thus Watson and Crick, seeing the possibility for hydrogen bonding between A and T as well as G and C, could propose the correct structure for DNA.
ID095	The Avery, McCarty, and MacLeod experiment utilized a a procedure known as the "transforming principle." They began with heat killing a virulent strain of bacteria.  Next, they isolated the carbohydrates, lipids, and proteins from the dead mixture.  They then proceeded to mix these virulent strain ingredients with an avirulent bacteria and different digesting enzymes.  The enzymes used digested particular part of the mixture: protease (protein), RNase (RNA), and DNase (DNA).  If transformation occurred in the final product, they were able to conclude that the virulent strain of genetic information was taken up by the avirulent bacteria.  Their goal was to isolate the genetic information, and in order to prove this, they needed to find which part of the mixture needed to be digested in order for no transformation to occur.  They found that the protease and RNase added to each test resulted in a transforming bacteria.  When the DNase was added to the test, the resulting bacteria did not transform, concluding that DNA was the source of genetic information.
ID095	The Hershey-Chase experiment involved using phage marked with radioisotopes to determine if DNA or proteins were the genetic information infecting bacteria.  The proteins of the phage were marked with a sulfur radioisotope.  The DNA was marked with a phosphorous isotope.  After procuring infected bacteria, they completely separated the phages from the bacteria.  Their results showed that the bacteria, once infected, contained the phosphorous isotopes, and the sulfur isotopes remained in the phage "ghosts."
ID095	By using x-ray diffraction, Watson and Crick were able to composite a picture of the structure of the DNA double helix.  The molecules are subjected to x-ray, and the scattering of the x-rays is dependent of the molecules being examined.  They used Chargaff's data regarding the concentrations of the purines and pyrimidines to distinguish the A-T/C-G nitrogenous bonds within the helix.  These base pairings they proved are hydrogen bonds.  The proved at the molecular level that the adenine will only bond to the thymine, and cytosine will only bond at the guanine, which produces the final structure.  This pairing is known as complementarity.
ID114	Avery, MacLeod, and McCarty studied the factor that could change a lethal strain into a non-lethal strain into a lethal strain. Their experiments definitively proved that the transforming principle was DNA.
ID114	The experiments by Hershey and Chase
ID114	Watson and Crick did not perform any experiments. Their conclusions were based on research of others. Erwin Chargaff noted that the concentrations of guanine and cytosine were always equal in DNA. Likewise, the concentrations of adenine and thymine were equal. Rosalind Franklin and Maurice Wilkins used X-ray crystallography to study DNA. From these structures, Watson and Crick deduced that DNA had repeating structures (nucleotides), the DNA was of a constant width and double-stranded.
ID006	The experiments conducted by Avery and his colleagues were definitive, but many scientists were very reluctant to accept DNA (rather than proteins) as the genetic material. The clincher was provided in 1952 by Alfred Hershey and Martha Chase with the use of the phage (virus) T2.
ID006	Hershey and Chase showed that when bacteriophages, which are composed of DNA and protein, infect bacteria, their DNA enters the host bacterial cell, but most of their protein does not.
ID006	The second set of clues available to Watson and Crick came from work done several years earlier by Erwin Chargaff. Studying a large selection of DNAs from different organisms, Chargaff established certain empirical rules about the amounts of each component of DNA:
ID141	The big question during this time period for geneticist was whether DNA or Protein was the biomolecule responsible for heredity.  First, Avery, McCarty, and Macleod grew IIIS cells in a liquid culture.  They then centrifuged the cells and heat killed them.  They then extracted the carbohydrates, the proteins, and the lipids.  They then treated the filtered cells with different chemicals to arrive at the conclusion that DNA was the "transforming principle."
ID141	Hershey and Chase used radioisotopes to label both the protein and DNA of the phage head.  Only the radioisotopes found on DNA were recovered in the progeny.  This provided even more convincing evidence that DNA was the heritable material, not protein.
ID141	Based on previous data, Watson and Crick made there model consistent with what was already known about DNA.  From looking at the data, Watson and Crick concluded that A is proportional to t and that G is proportional to C.  Also, the X-Ray diffraction analysis suggested that the DNA molecule was some sort of helix, so based on what was known at the time, Watson and Crick built there model.
ID083	Griffith worked with two strains of Diplococcus pneumonia, strain IIR which had a rough surface and was avirulent and strain IIIS which was virulent and caused pneumonia. When live IIR or dead IIIS bacteria were injected into mice, the mice did not get pneumonia and lived. When live IIIS bacteria were injected into mice, the mice died from pneumonia. Paradoxically, when dead IIIS bacteria and live IIR bacteria were injected together, the mice died and live IIIS bacteria were recovered.
ID083	Griffith worked with two strains of Diplococcus pneumonia, strain IIR which had a rough surface and was avirulent and strain IIIS which was virulent and caused pneumonia. When live IIR or dead IIIS bacteria were injected into mice, the mice did not get pneumonia and lived. When live IIIS bacteria were injected into mice, the mice died from pneumonia. Paradoxically, when dead IIIS bacteria and live IIR bacteria were injected together, the mice died and live IIIS bacteria were recovered.
ID083	Griffith worked with two strains of Diplococcus pneumonia, strain IIR which had a rough surface and was avirulent and strain IIIS which was virulent and caused pneumonia. When live IIR or dead IIIS bacteria were injected into mice, the mice did not get pneumonia and lived. When live IIIS bacteria were injected into mice, the mice died from pneumonia. Paradoxically, when dead IIIS bacteria and live IIR bacteria were injected together, the mice died and live IIIS bacteria were recovered.
ID028	Avery, McCarty and MacLeod all worked to straighten out what kind of cellular material represented the molecule of heredity by conducting multiple experiments.
ID028	Hershey-Chase showed in their experiments that the material that was injected into the bacteria cell was the virus' DNA, not the protein.
ID028	Watson and Crick concluded that DNA was double stranded, in a helix, and had purines and pyrimidines that could bind to each other by using information gathered from multiple sources.
ID133	To find out what the transforming agent was in Griffith's transformation experiment, Avery manipulated parts of the S strain. In different mice he destroyed the polysaccharides, lipids, RNA, protein, and DNA in the S strain to see which mice would live with the effects.  This revealed that DNA was the genetic material and not protein.
ID133	Hershey-Chase labeled Phage T2 with radioactive medium. One was labeled with phosphorous and the other with sulfur. Because DNA does not have sulfur and protein does not have phosphorous, they could easily see what was left inside the bacteria.  After the phages infected the bacteria, only labeled phosphorous was found inside the bacteria cell.  This proved that DNA was the genetic material in bacteriophage.
ID133	The data from Chargaff showed that A and T go together and G and C go together.  The X-ray diffraction revealed that the structure of DNA had to be a helix shape.
ID151	I do not know many details about this experiment that was conducted by Avery, McCarty, and MacLeod, besides the basic set-up of the experiment and that this experiment was the important basis that proved that DNA is the hereditary molecule. The group utilized strains of Pneumoccous Bacteria and allowed the process of transformation to occur and one by one destroyed each macromolecule within groups of flasks. The use of enzymes to breakdown lipids, RNA/DNA, proteins and carbohydrates allowed the researchers to determine if the process of transformation was still occurring. Through various, arduous trials, they were able to conclude that DNA is the genetic material since after the destruction of DNA/RNA transformation ceased unlike in the other scenarios during the destruction of other macromolecules in which transformation continues to occur.
ID151	Hershey and Chase radioactively labeled the bacteriophages with radioactive Phosphorous and radioactive Sulfur. These isotopes were specifically chosen since the researchers knew that the viruses had only a protein compartment and a DNA compartment, but they didn't know which of the two macromolecules allow the production of more bacteriophages. The radioactive sulfur labels the proteins because of the formation of disulfide bonds during the tertiary structure of proteins and the radioactive phosphorous labels the DNA strand due to the phosphate sugar backbones present. Once the bacteriophages were given ample time to infect bacteria, the mixture of the cells and the bacteriophages were then centrifuged. The centrifuged results showed that the pellets contained radioactive phosphorous inside the bacterial cell while the radioactive sulfur was found simply floating closer to the top which became the supernatant. This fascinating experiment was conducted multiple times and supported that DNA was in fact the hereditary material in bacteriophages.
ID151	Watson and Crick are two renowned scientists that pieced the model of the DNA molecule together by gathering information from other researchers. They were able to construct a DNA model that was anti-parallel, double helix with the nitrogenous bases on the interior due to their hydrophobic tendencies and the phosphate sugar backbone facing on the outside due to their hydrophilic nature. Chargaff came about an astounding phenomena that later became to be known as Chargaff's Rules. These rules state that the proportion of adenine to thymine is almost equivalent and the proportion of guanine to cytosine is almost equivalent, but the proportions of adenine and thymine to guanine and cytosine is not necessarily equivalent. Furthermore, he determined that different species had variations in the proportions of these 4 nitrogenous bases. However, Rosalind Franklin's story contrasts highly from Chargaff in the sense that she was not able to publish her findings. Franklin specialized in X-Ray crystallography patterns which helped her determined the properties of the DNA molecule from the width to the length of the DNA in Angstroms. Her famous image (Picture 31) showed that DNA was a double helix with a phosphate sugar backbone on the exterior and the nitrogenous bases connected on the interior. Her work was shared with Watson and Crick without her attention, thus she did not get any accreditation on her work until years later.
ID085	Avery Mccarthy and MacLeod determined DNA was genetic material by doing in vitro experiments to determine the transforming agent in mice.
ID085	By radio coating the protein and DNA of a bacteriophage, Hershey and Chase determined that DNA was the genetic material after only finding radio-labeled DNA in the offspring.
ID085	Watson and Crick were the discovers of DNA's double helix. They did much of their work on their own, but also used Chargaff's rules for AT and CG relations and borrowed Rosalind Franklin's X-ray crystallography of DNA.
ID137	They were able to determine that DNA was the source of genetic material through bacterial transformation. Specifically through Griffith's experiment of analyzing how individual strains of dead and live bacteria affect mice.
ID137	They introduced radioactive sulfur and phosphorous into bacteriophages and found that phosphate was absorbed into the bacteria. This meant that protein was not the genetic material.
ID137	Rosalind Franklin had the clearest picture of DNA and Chargaff understood the relationship of the bases on a DNA strand. Watson and Crick put the information together to determine the structure of a DNA strand.
ID073	This experiment implied that DNA is the transforming factor and not proteins or other materials and by this was demonstrated what is known to us as the transforming principle - that genes are made of DNA. They extracted S bacteria nucleoid purified DNA, proteins and other materials and mixed R bacteria with these different materials, and only those mixed with DNA were transformed into S bacteria.
ID073	After Hershey and Chase used radioactive isotopes to show that it was primarily DNA, rather than protein, that entered bacteria upon infection with bacteriophage, it was soon widely accepted that DNA was the material. Their work was widely more accepted than Avery, McCarty and Macleod's findings.
ID073	Although Watson and Crick were first to put together all thef information that was required to produce a successful molecular model of DNA, their findings had been based on data collected by researchers in several other laboratories. The discovery of the DNA double helix used a considerable amount of material from the unpublished work of Erwin Chargaff and  Rosalind Franklin. Franklin's data suggested that DNA is long and skinny and that it has two similar parts that are parallel to each other and run along the length of the molecule. The X-ray data showed the molecule to be helical). Other regularities were present in the spot patterns, but no one had thought of a three-dimensional structure that could account for just those spot pattern until Watson and Crick.
ID086	They used agents that would break down the different suspects and ended up with the conclusion that when DNA did not get transported no genetic material was exchanged.
ID086	These experiments also were vital in the overall study of DNA.
ID086	Watson and Crick were mainly only used other people's work to show as their own. Much of their findings was actually other scientists work.
ID080	Before this experiment was conducted they were not sure where the genetic material was stored in the body. They had suspicions but no definitive evidence.
ID080	He was able to manipulate bacteriophages to transport DNA to see the genetic effect that occurred in the DNA.
ID080	This was critical for discovering genomes because it showed a definitive way to order the nucleotides and see the difference between species. It also allowed to discover defects in the genome.
ID098	Avery's transformation experiment involved killing certain molecular components of a bacteria and then giving that bacteria to a mouse and observing whether or not it become sick and died. The only mice to survive where the ones who received bacteria with destroyed DNA, providing ample evidence that DNA was the means of genetic transmission.
ID098	In these experiments, Hershey and Chase used E. coli and T2 to prove that DNA, not protein, was the genetic material of bacteriophages. They did this by using two radioactive dyes, one that binds to protein molecules, and one that binds to DNA molecules. They then looked at which of the two dyes ended up inside of the host cell. The result was, of course, that the DNA molecules had made it into the host cell.
ID098	Watson and Crick were essentially the first ones to see the big picture on DNA structure. They were able to look at data from a wide array of sources and bring that information together in order to make conclusions. For example, they used data from Erwin Chargaff and Rosalind Franklin. After pooling all of this information, they were able to conclude that DNA was in the shape of a double helix with base pairs of A/T and G/C.
ID075	Griffith's research provided foundation for Avery, MacLeod, and McCarty’s research. They were able to show that DNA was the genetic material. Avery created an in vitro assay to determine what the transforming agent was. His experiment showed that the no live S strain could be recovered if the DNA was destroyed.
ID075	Hershey-Chase experiments involved Escherichia coli and bacteriophage T2. They demonstrated that DNA, not protein, is the genetic material. These experiments demonstrated DNA enters bacterial cell during infection and directs viral reproduction.
ID075	Watson and Crick's model was developed using Chargaff and Franklin's data. Studies by Rosalind Franklin  showed DNA had a 3.4 angstrom periodicity which is a characteristic of helical structure. Chargaff proposed base composition. He showed how the amount of A is proportional to T; and amount of C is proportional to G. Watson and Crick used these data to create their model.
ID050	They demonstrated that bacteria had the ability to transform because they would inject mice with different heat killed and live bacteria and the mice with the heat killed lethal virus and the living smooth living mice would die, they assumed that it was due to transformation.
ID050	Hershey Chases found that a DNA was the basis for genetic material by labeling the different components of a virus and seeing which parts showed up in the infected generations of the bacteria that the virus attacked.
ID050	They used these other scientists data to determine that DNA was a double helix and had uneven bas pairings which lead to the whole field of genetics.
ID108	This experiment created an in vitro assay to determine the transforming agent. They first injected a mouse with live S strain and found that when no components were destroyed, polysaccharides were destroyed, lipids were destroyed, RNA was destroyed,or when protein was destroyed the mouse died and the S strain was still present. When DNA was destroyed the mouse lived thus meaning that DNA is the genetic material not protein.
ID108	Hershey-Chase used E. coli and bacteriophage T2 and in one E coli sample added radioisotope 32 P (part of the Phosphate backbone on DNA)  and in the other sample used radioisotope of Sulphur 35S (Disulfide bonds on protein). On the 32P side, after replicate 32P was found inside the E. coli bacteria while 35S was not found inside. This showed that DNA is the genetic material for bacteria also.
ID108	Chargaff used chemistry experiments to show that the concentrations to A are similar to concentration of T in DNA, while concentrations of G are similar to concentrations of C in DNA. Watson and Crick used this in the model to show AT and GC base pairing and that it provides 2 complementary strands.Franklin's X-ray diffraction data showed that teh sugar phosphate was on the outside while the inside was stacked base spiral stucture and that it was a double helix and antiparallel. Watson and crick used all of this to construct their model.
ID077	Avery, Mccarty, and Macleod worked to investigate the chemical nature of the transforming factor, whether if was a protein or nucleic acid. In their experiments, they found that genetic material in transformation is most likely DNA.
ID077	The Hershey-Case experiments was the second major piece of evidence supporting that DNA as the genetic material provided during the study of bacterium E.coli and one of it's infecting viruses, bacteriophage T2.
ID077	Watson and Crick published their analysis of DNA structure in 1953. By building models based on certain parameters, they discovered the double-helical form of DNA.
ID042	Using mice and viral strains, Avery, McCarty and MacLeod proved that DNA was the genetic material. They proved this by inject mice with either a rough strain, smooth strain, heat killed smooth strain, or combination of rough strain and heat killed smooth strain. The results were that those with just the rough strain lived. Those with just smooth strain died. Those with heat killed smooth strain lived. Those with the combination of rough strain with heat killed smooth strain died. This showed that even though the lethal strain had been heat killed, when its DNA came in contact with that of the rough strain, it made the rough strain lethal.
ID042	Hershey and Chase used bacteriophages to prove further that Dna was genetic material. They did this by using radioactive sulfur and phosphorus. One bacteriophage had the radioactive sulfur, and "cold" DNA. After centrifugation the product was radioactive sulfur labeled proteins but no radioactive label on the DNA. The second phage contained the radioactive phosphorus and "hot" DNA. After centrifugation this result was no radioactive label in the protein but there was a radioactive phosphorus label in the DNA.
ID042	Watson and Crick combined Franklin's ray diffraction data, which proved that DNA had a double helix structure, and Chargaff's data which showed the approximate number of A and T to G and C. By combining these two sets of data, Watson and Crick were able to build a three dimensional model of DNA matching up the bases to form a double helix structure.
ID146	This group used different strains of the same pneumococcus bacteria to show that DNA was the material used to pass down hereditary material.
ID146	These two utilized unique isotopes of phosphorus and sulfur to label protein and DNA components of bacteriophages that were then allowed to infect bacteria, but only the phosphorus had been transferred, further confirming DNA's role as the hereditary material.
ID146	Watson and Crick utilized Chargaff's data to propose the complementary pairs of the DNA bases, and Rosalind Franklin's diffraction photo to support their double-helix hypothesis.
ID128	The experiments by Avery, McCarty, and Macleod showed that DNA was conclusively the hereditary material by putting cells in a liquid medium culture. Then centrifuging them followed by heat. They then removed all carbohydrates, lipids, and proteins and tested the remaining cells with different enzymes (protease, ribonuclease, and deoxyribonuclease) to see under what conditions would transformation occurs. Transformation happened in three of the four experiment types showing that DNA had to be the source of genetic material instead of RNA or proteins. (The fourth was the control.)
ID128	The Hershey-Chase experiments extended the finding of Avery, McCarty, and Macleod by showing that DNA was the genetic material to bacteriophage. They proved this by using radioactive phosphorus and sulfur. This worked because it was able to label just the DNA because DNA contains phosphorus which protein does not while protein contains sulfur which DNA does not. Therefore, Hershey and Chase were able to observe the fact that bacteriophages were absorbing some of the P-labeled DNA while pretty much all of the S-labeled proteins remained outside the cell.
ID128	Watson and Crick used the data from Erwin Chargaff that showed an approximate relationship between the concentrations of A and T and G and C. The model they proposed suggested that DNA was a double helix consisting of two antiparallel strands. The model also suggested that the two strands were 0.34 nm apart and that the nitrogen bases are paired as a result of the formation of hydrogen bonds (A-T and G-C).
ID117	Avery, McCarty, and MacLeod performed an experiment by isolating strains of virulent IIIS strains. Then they obtained a transformation inducing IIIS filtrate. They treated the IIIS with IIR cells and an IIIS filtrate treated with protease, RNase, or DNase then studied the resulting medium's material to see if transformation had occurred. When the protein and RNA was destroyed, transformation still occurred. When DNA was destroyed, no transformation occurred. Leading the scientists to conclude that DNA was the transforming genetic material.
ID117	Hershey and Chase used radioisotopes for phosphorous and sulfur to label DNA (which included P but not S) and proteins (which included S but not P). Then they tested cells and bacteriophages during infection to see which molecule (DNA, protein, or both) was being injected into the cell to direct reproduction. P-labeled DNA was found to be injected into the cells while the S-labeled proteins remained behind in the empty bacteriophage coats. This led Hershey and Chase to conclude that DNA was the genetic material of bacteriophages.
ID117	Watson and Crick were able to combines the findings of other talented scientists in the race to uncover the structure of genetic material. Based off Chargaff's, they knew how base pairs went together in DNA strands. And after obtaining X-ray diffraction data from the extensive work of Rosalind Franklins and an explanation that the structure was helical, they were able to create and accurate double helix structure of DNA.
ID009	Griffith's experiment set a good basis for their experiment. Avery created his own vitro assay to see the outcomes of the mice's lives to determine what material affected them the most.
ID009	They centrifuged bacteriophages with protein and DNA. After the centrifusion was over and the cell was infected, DNA showed up in it but protein did not. They conducted their experiment with E. coli and bacteriophage T2.
ID009	Rumor has it that Watson and Crick took the credit that Rosalind Franklin deserved. They pieced Chargaff's and Franklin's research together and found that the ratios of A's to T's and C's to G's were very similar.
ID010	Avery and his colleagues examination demonstrated that the extents of carbon, hydrogen, nitrogen, and phosphorus in this dynamic part were predictable with the synthetic creation of DNA. To demonstrate that it was DNA instead of some little measure of RNA, protein, or some other cell part that was in charge of change, Avery and his associates utilized various biochemical tests.
ID010	They labeled phage DNA with a radioactive isotope, phosphorus-32. They were able to track the phages while they infected an E.coli strain. They were then able to conclude that the radioactive isotope was only found in the bacteria and not in the phage. They did it again by labeling the phage protein with an isotope, sulfur-35. After attaching the phage to the bacterium, the sulfur-35 was found in the phage and not in the bacteria. This concluded that DNA is the genetic material that infects bacteria.
ID047	They were able to determine the fundamental unit of transforming principle of Pneumocuccus Type III through thorough experimentation and went through extra measures to eliminate any contamination. They would measure the masses' nitrogen to phosphorus ratio to determine it was DNA
ID047	They conducted a study of the bacterium E.coli and one of its infecting viruses T2. They concluded that the protein of the phage coat stays outside the host but phage DNA enters the cell and helps phage reproduction. They also demonstrated that the genetic material in T2 is DNA not protein
ID047	Chargraff found a pattern of base composition in DNA molecules. The amount of adenine is proportional to thymine while the proportion of guanine resembles the amount of cytosine. The sum of the purines equals the sum of the pyrimidines. WIth these conclusions Crick and Watson were able to come up with double helical model for DNA
ID100	The Avery-McCarty-Macleod experiements proved that DNA was the genetic material. The trio also discovered the pathway of transformation. They used 2 different bacteria strains of the same bacterial species, R and S strains. The S strain was pathogenic and the R strain was not pathogenic. He incubated heat-killed S strain and living R strain and injected this into a mice. The mixture was found to be pathogenic killing the mice. When bacteria was isolated from these mice however, living S strain was found proving that some factor of the S strain was assimilated in the R strain transforming them into S strain bacteria. Because this transformation was inherited, they assumed that this factor was the genetic material.
ID100	Hershey and Chase conducted early experiments which demonstrated that DNA was the genetic material of bacteriophages. They did this by using T2 bacteriophages, which is a strain of viruse that infect bacteria by injecting their attaching to the cell and injecting their DNA into the cell cytoplasm. The pair labeled phage DNA with radioactive phosphorous-32 and labeled phage protein with sulfur-35. They then used E. Coli cells to track the transmission of these radioactive elements. They found that the phosphorous was found within the cells and the sulfur was found outside the cells proving that the DNA was the genetic material.
ID100	The experiments done by Watson and Crick provided a model of DNA structure. The two had been working on their model when they received data collected by Franklin proving that DNA had a helical structure. With this information along with the findings of Chargaff, which showed that the concentrations of A=T and G=C, Watson and Crick were able to determine the shape and structure of DNA. This finding has often been considered the most important biological discovery of all time.
ID020	Avery, McCarty, and MacLeod began their experiment with large quantities of liquid cultures of type IIIS virulent cells. Thier experiment was answering the question of what molecule served as the transforming principle. The virulent cells were centrifuged, collected, and heat killed. They were able to induce type IIR avirulent cells by extractions. The experiment concluded that DNA was the fundamental unit for transformation
ID020	The Hershey-Chase experiment also helped support that DNA was the genetic material. The evidence was from studying E. Coli and one of its infecting viruses bacteriophage T2 (phage). Hershey and Chase ran a bunch of experiments and several of the experiments established the independent function of the phage protein and nucleic acid in the reproduction process associated with bacterial cell.
ID020	The key to the model they proposed is specifically the base pairing. Chargaff's data had led Watson and Crick to believe that A was equal in amount to T and G was equal in amount to C. Watson and Crick realized this and these pairs were a result from hydrogen bonds between base pairs. This also provided the chemical stability necessary to hold the two chains together. Their model had the features of a double helix, two long polynucleotide chains that were coiled, two chains that were antiparallel, C-5' to C-3'. There were also other features that were present in their model.
ID072	There experiments proved that DNA was the genetic material, this disproved the original thought that protein was during the time era around the 1940s.
ID072	Hershey and Chase showed that when bacteriophages, which are composed of DNA and protein, infect bacteria, their DNA enters the host bacterial cell, but most of their protein does not.
ID072	The x-ray diffraction data from Rosalind Franklin showed that the DNA was presented by a double helix, which Watson and Crick used to develop their model.  They also used the pairing sequences from Erwin Chargaff.
ID119	The experiments of these three individuals made it known to the world that DNA is the genetic material.
ID119	Hershey-Chase found that DNA was the genetic material, not protein. They also found that DNA enters bacterial cells during infection an directs viral reproduction.
ID119	Watson and Crick were able to determine the double-helix structure of DNA using both the information and findings of Chargaff and Franklin.
ID074	he Avery–MacLeod–McCarty experiment was an experimental demonstration, reported in 1944 by Oswald Avery, Colin MacLeod, and Maclyn McCarty, that DNA is the substance that causes bacterial transformation, in an era when it had been widely believed that it was proteins that served the function of carrying genetic information. It was the culmination of research in the 1930s and early 1940s at the Rockefeller Institute for Medical Research to purify and characterize the "transforming principle" responsible for the transformation phenomenon first described in Griffith's experiment of 1928: killed Streptococcus pneumoniae of the virulent strain type III-S, when injected along with living but non-virulent type II-R pneumococci, resulted in a deadly infection of type III-S pneumococci.
ID074	The Hershey–Chase experiments were a series of experiments conducted in 1952 by Alfred Hershey and Martha Chase that helped to confirm that DNA is the genetic material. While DNA had been known to biologists since 1869, many scientists still assumed at the time that proteins carried the information for inheritance because DNA appeared simpler than proteins. In their experiments, Hershey and Chase showed that when bacteriophages, which are composed of DNA and protein, infect bacteria, their DNA enters the host bacterial cell, but most of their protein does not.
ID074	"Molecular Structure of Nucleic Acids: A Structure for Deoxyribose Nucleic Acid" was an article published by Francis Crick and James D. Watson in the scientific journal Nature. It was the first publication which described the discovery of the double helix structure of DNA, using X-ray diffraction and the mathematics of a helix transform.
ID017	Avery, MacLeod and McCarty used the phenotype of Streptococcus pneumoniae cells expressed on blood agar, this experiment was conclusively able to determine that DNA is genetic and provided heredity.
ID017	By infecting, blending, and centrifuging a cell with a bacteriophhage in which the bacteriophage inject DNA into a cell and not protein.
ID017	Watson and Crick are notorious for seemingly "stealing" information from different experiments such as from ERwin Chargaff and Rosalind Franklin. This led to being able to synthesize the information into one tool to create the structure.
ID060	Avery, McCarty and MacLeod went through experiments to conclude that the DNA was the genetic material and that it is made of genes in chromosomes.
ID060	Hershey and Chase did a blender experiment where they also found that DNA was the genetic material.
ID060	Erwin Chargaff realized that there were the same amounts of G to C and A to T. Watson and Crick got his data and realized that it was true and that DNA is made up of these 4 nucleotides. When Rosalind Franklin took the X-ray diffraction picture of the DNA she realized it was double helix. Watson and Crick got her data and Chargaff's and made a model of the DNA.
ID130	The experimental demonstrations performed by these individuals disproved the preexisting idea that proteins were the genetic material, and established that genes were stored in DNA.
ID130	Hershey and Chase used bacteriophages to inject new DNA into and then watched the cells exhibit change.  These experiments were not by themselves definitive, but aided greatly in proving that DNA was the true genetic material.
ID130	Watson and crick came up with the double helix model of DNA.  They implemented the idea of the 2 stranded DNA molecule where an A on one strand bonded to a corresponding T on the and the same for G and C.
ID087	Telomeres keeps the ends of chromosomes from degrading making chromosomes shorter and shorter. However, the chromosomes must lengthen in order to add more repetitive units in the 5' to 3' direction.
ID087	Polytenes in Drosophila chromosomes showed where duplications and deletions occurred, then identifying chromosomes.
ID087	These sections can be visible on the polytenes of Drosophila. The areas of active transcription correlate with genes and the areas with no transcription are intermediates between genes.
ID131	DNA and RNA code in a 5' to 3' fashion either way. This is always true and allows us to predict the codons formed and the order they are formed.
ID131	Certain chromosomes in Drosophila are large enough to be easily observed under microscopy.
ID131	Regions that are active in transcription show lightly whereas the areas of no or little transcription show up completely dark allowing us to easily tell active regions.
ID057	The chromosome continuously gets smaller with each replication. This shortening occurs on the 5 prime end of the DNA. Thus creating a hang over of the 3 prime end. Each daughter strand with have this 3 prime hang over. Telomeres prevent the loss of the entire DNA when shortening occurs.
ID057	Observing various specialized drosophila chromosomes and the mutations that occur in them. This enabled us to more easily identify the locations of certain genes. This greatly enhanced our understanding of our own gene locations by developing techniques to identify gene locations.
ID057	Euchromatic is a lightly packed form of chromatin and usually has a high concentration of genes present. However, it is not always active due it constantly undergoing active transcription. Heterochromatic regions are tightly packed forms of DNA. These are important in regards to expression of genes. There are two important varieties of heterochromatin constitutive and facultative. One is involved in silencing genes and the other involved in gene expression or repression signals.
ID085	Telomeres are repetitive DNA sequences on the end of linear chromosomes. Because of DNA polymerase requiring a primer to initiate replication, the telomeres will shorten every time a cell divides. Eventually there will be no room for the primer at the 3' end of the DNA, meaning that part of the telomere is lost every division.
ID085	Polytene chromosomes found in larvae of flies can be seen in the nuclei of interphase cells. When seen under a light microscope, a series of bands were seen. Although it is not known how many genes are contained in each band, gene activity creates a "puff" on the chromosome, allowing its location on a band to be found.
ID085	The chromosome is not structurally uniform - some parts of the chromosome remain condensed during interphase. These parts stain and are called heterochromatin. Euchromatin, on the other hand uncoil during interphase. This region is only lightly stained. Heterochromatic regions are inavtive because they lack genes or contain repressed genes. Examples of heterochromatic regions are the telomeres and centromeres.
ID027	The telomeres at the end of chromosomes cannot pair all of the way to the end without the help of other molecules. This strand of DNA gets added to the end, giving it extra material to leave off instead of running through the actual useful genes.
ID027	The chromosome in larval Drosophila allowed geneticists to view all of the genes on a single chromosome. I believe this was found in some gland in the cheeks. This chromosome allowed scientist to see the entire genome in one place.
ID027	This allows geneticists to see which areas of the gene are active and which are not. This is important because if a scientist wants to compare two different phenotypes, they can compare the active sites on the two sets of chromosomes to find if there is a difference in activation. This also tells them how many genes are active in the genome compared to the non-active genes.
ID121	When RNA primers are removed from the lagging strand, it leaves a gap at the 3' end that can't be filled immediately. This problem is solved by the enzyme telomerase. This enzyme has a RNA template that recognizes the G-T rich telomere region at the end of the DNA and starts to elongates the parental strand of the DNA. Then RNA primers come in to fill the original gap in the lagging strand.
ID121	The light staining regions are exon and the dark staining regions are introns.
ID105	After every DNA replication, some telomere sequences are lost at the 5' end of the newly synthesized strand of DNA, but because these are noncoding sequences, their loss doesn't effect the cell. However, these sequences are not unlimited; eventually after several replications, all the telomere repeats are lost, and the DNA risks losing coding sequences with further replications.
ID105	Polytene chromosomes are rarely found in eukaryotes. However, these chromosomes are found in various tissues in Drosophila and most flies. They can be seen under light microscopy and can be viewed in nuclei interphase cells.
ID105	Euchromatin in chromosomes is uncoiled and active. Where as heterochromatin is coiled tightly and mostly inactive.
ID070	DNA polymerase can only synthesize a new strand of DNA as it moves along the template strand in the 3' –> 5' direction. This works fine for the 3' –> 5' strand of a chromosome as the DNA polymerase can move uninterruptedly from an origin of replication until it meets another bubble of replication or the end of the chromosome. However, synthesis using the 5' –> 3' strand as the template has to be discontinuous. When the replication fork opens sufficiently, DNA polymerase can begin to synthesize a section of complementary strand — called an Okazaki fragment — working in the opposite direction. Later, a DNA ligase stitches the Okazaki fragments together.
ID070	In Drosophila, the role of telomerase is carried out by three specialized retrotransposable elements, HeT-A, TART and Tahre. Telomeres contain long tandem head-to-tail arrays of these elements. Within each array, the three elements occur in random, but polarized, order. Some are truncated at the 5′ end, giving the telomere an enriched content of the large 3′ untranslated regions which distinguish these telomeric elements from other retrotransposons.
ID070	There are two varieties of chromatin. They are euchromatin and heterochromatin. These two forms are distinguished in a cytological manner dealing with how intensely each form is stained. The euchromatin is less intense than heterochromatin. This only indicates that heterochromatin has tighter DNA packaging. To find out more about the difference between euchromatin and heterochromatin, this article will provide you a quick look regarding these two chromatin forms.
ID037	Dna has to be replicated in sections when moving in the 3' to 5' direction
ID098	After new DNA has been synthesized, the RNA primers are removed, which leaves gaps in the DNA. In the middle of the DNA strand, the gaps are easily filled because the DNA polymerase can use the newly synthesized DNA upstream from the gap as a landing pad, so to speak. At the end of a linear strand of DNA, however, there is no such framework upstream. Therefore, the structure of the telomere must support an alternative method for synthesizing DNA at this point.
ID098	Drosophila have polytene chromosomes, which exhibit unique banding patterns when viewed under a light microscope. The polytene chromosome itself is actually several copies of the same DNA strand, which reside side-by-side, thus producing the banding effect. Each band represents a specific gene located at that point on all of the individual strands of DNA.
ID098	All chromosomes have euchromatic regions and heterochromatic regions when they are in chromatin form. Euchromatic regions stain very lightly and carry the DNA that is used for active transcription. Essentially, this is where all of our active DNA is located. Heterochromatic regions, on the other hand, stain very darkly and carry inert, non-coding DNA.
ID040	When linear chromosomes replicate, the RNA primer at the 5' end of the new strand is removed but the bases are not replaced. This leads to an incomplete 5' strand and a 3' overhang. Telomeres are at the ends of chromosomes and consist of a series of noncoding bases repeated hundreds of times. Each time the chromosome replicates, some of the bases are lost because of the overhang. This helps to protect the cell from losing any important information.
ID040	The specialized Drosophila chromosomes are called polytene chromosomes. They represent paired homologs even though they are in somatic cells. The DNA undergoes many rounds of replication but without strand separation of cytoplasmic division. Polytenes have bands on them and this is a visible manifestation of genes.
ID040	Chromosomes contain areas that are noncoding and coding. The euchromatic regions are uncoiled and active, and do not stain. Heterochromatic regions are genetically inactive and will stain.
ID094	Telomeres are the reason behind shortening of chrosomes and therefore of aging and in 5' to 3' replication genetic information is lost if the process is allowed to occur unhindered.
ID094	Puffs in these organism's DNA make it possible to see individual genes.
ID094	Heterochromatin is inactive and tightly packed genomic DNA, where as euchromatin is lightly packed and is usually under transcriptions.
ID000	Removal of the primer leaves a gap that cannot be filled by DNA polymerase.  The problem is resolved in Eukaryotes with an enzyme telomerase which pairs with the end and extends the end
ID000	Polytene chromosomes are found in specialized tissues of Drosophila and other flies.  These chromosomes are uncondensed and replicated approximately 1500 times.  Homologs are paired.  This allowed the identification of duplications, deletions and other chromosome abnormalities.
ID022	Linear chromosomes have an end problem.  it is a problem because for each newly synthesized DNA after cell division the DNA is shorter at 5 prime end. This produces a 3 prime over hang. And this small section can not be synthesized in most cell unless telomerase enzyme is present. Therefore, as the cell divide more often the length of DNA keeps getting smaller and smaller. In short, It's a problem because there is no place for the replication machinery to attach at the ends of the chromosomes thus making the chromosomes smaller and smaller after each division.
ID022	Polytene chromosomes are giant or over sized chromosomes that have developed from standard chromosomes. Some specialized cells undergo repeated DNA replication without cell division giving rise to this polytene chromosome.  They can be visualized using light microscope and they are found in tissue of saliva, rectal and midgut. Due to it being over sized than any normal chromosomes it would have been easier to see them using a microscope and understand from that that they are different from any normal chromosome in shape and size and also in function and thus based on this the specialized chromosomes allowed the identification of gene location.
ID022	Euchromatin and Heterochromatin are the two types of chromatin. And based on how they are stained we can distinguish between them.  The heterochromatin region has a very high DNA packaging while the euchromatin has lower or less intense DNA packaging. So when observed undermicroscope euchromatin are light band region and heterochromatin are dark band region. Moreover, Euchromatin is uncoiled and active region whereas heterochromatin is genetically inactive. It lack genes or has repressed genes.
ID089	Because DNA polymerases require a primer, and can only add bases in the 5->3 direction, end of chromosomes where there is no room to place a primer will not be replicated without an additional solution. That solution in eukaryotes is telomerase. It extends the DNA sequence so that more bases may be added to the end to prevent shortening and ultimately destroying of chromosomes.
ID089	Without looking at my notes, i am not 100% sure what this is talking about. however, my educated guess is that it is referring to polytene chromosomes. These chromosomes are HUGE. They show 'puffs' at areas of high activity. Most often, the areas of high activity will be where genes are located.
ID089	Euchromatic regions of DNA are regions that can uncoil to allow for gene activity like that of transcription. Heterochromatic DNA is often found near the centromere, and does not uncoil to allow for gene activity. The euchromatic regions are lighter staining because the DNA is less condenses, while the heterochromatic DNA is darker because it is more condensed.
ID030	When replicated a telomere, the lagging strand is replicated via Okazaki fragments. An RNA primer is, however, required for this to happen. When the primer is removed, DNA polymerase has no way of pairing the end of the strand. Thus, there would be an overhanging single stranded portion of DNA and telomeres would, as a result, become shortened with each replication.
ID030	Chromosomes known as polytene chromosomes are replicated numerous times without the cell division occurring. Thus the genes on polytene chromosomes become greatly expressed. When observing these chromosomes, geneticists can located genes by watching for the banding pattern created by transcriptionally active regions.
ID030	Euchromatic regions of DNA are loosely packed areas containing a high concentration of genes being actively transcribed. Heterochromatic regions, on the other hand, have there histones tightly packed, making the DNA silent. Altering DNA between being tightly and loosely packed is one way cells regulate gene expression.
ID046	5' to 3' directionality is essential to the replication of DNA. DNA is copied in this direction.
ID046	The Drisophola study is crucial to help us understand our own genes and the way genetics work. With this knowledge of Drosophila, we can understand the locus of specific genes.
ID046	These are known as exons and introns.
ID079	Chromosome length is shortened with each replication because the RNA primers removed, leaving the chromosome unfinished and shorter. However, telomerase enzyme is able to recognize the repeating sequence and bind, leaving overhang to which it can loop back on itself to protect the chromosome from being chewed up.
ID079	Polytene chromosomes are large, banded chromosomes that result from repeated DNA replication without cell division. These large banded chromosomes have been used to map genes, providing the location of genes on the chromosome.
ID079	Euchromatic areas of the chromosome have spaced out chromatin so transcription can occur. Heterochromatic regions of the chromosome have tightly packed/condensed chromatin, which is why it stains dark. Because of the condensed chromatin, no transcription can occur.
ID063	The telomere is a region at the end of a chromatid of repeating nucleotide sequences. During replication, enzymes are unable to fully recreate the entire telomere. After each new replication, the telomere becomes shorter and shorter. Thus the genes loose their protection over time.
ID063	These chromosomes have distinct banding patterns. Through these, we can identify deletion or rearrangements. Dark is inactive and light is active.
ID063	Heterochromatin has found to play a role in gene regulation.The is tightly packed DNA with dark staining. Euchromatin is a more lightly packed DNA with light staining regions. This is what is used in the active transcription of DNA to mRNA.
ID109	DNA replication in eukaryotes requires DNA polymerase that unzips the genes. This allows the gene to be replicated. DNA is replicated in the 5' to 3' direction because DNA polymerase adds the bases to the end of the new strand of DNA. Telomeres cause problems because the telomeres shorten after each replication.
ID109	Specialized Drosophila chromosomes allow us to know the identification of gene locations because of  the puffs on the chromosome. The puffs represent areas of gene activity. Bulges where DNA has uncoiled are visible and indicate high level gene activity or transcription that produces RNA. From this, we can determine whee chromosomes separate and therefore where they are located.
ID109	Euchromatic regions are less tense than heterochromatic regions. Euchromatin has great gene concentration, and it is usually undergoing transcription. Heterochromatic regions are tightly packaged regions of DNA. They are usually found around the nucleus. Heterochromatin is also responsible for gene regulation. Both euchromatin and heterochromatic are versions of chromatin.
ID026	Chromosomes that have telomeres have problems with replication in eukaryotes. Enzymes have trouble replicating the ends of the chromosomes which leads to shortening of the chromosome over time.
ID026	Drosophila fruit flies have special parts of certain chromosomes that allow gene locations to be identified. This helps enzymes to both replicate and express these genes easier and provide the genes with a tag-like indicator.
ID026	Within a chromosome there are different sections that are transcribed or are not. You can tell the difference between these two distinct parts by whether they are light or dark staining regions of the chromosome. This difference between sections can help to determine which and how genes are expressed.
ID024	Chromosomes have ends called telomeres that are very important to the functioning of the chromosome. Special enzymes called telomerases are involved in lengthening the telomeres so that the chromosome does not get any shorter. Telomeres are difficult to copy, but they are so important because the telomeres keep the chromosome from getting any shorter.
ID024	Polytene chromosomes of Drosophila have allowed us to identify gene locations because the chromosomes are very easy to see and read. They have chromosome puffs, which are areas of RNA transcription, which allow us to see the different genes. When the DNA is unwound at this time, it is easy to see the genes, thus we can see their locations on the chromosome.
ID024	The euchromatic areas, where much transcription occurs, is important to the chromosome because it is the part of the chromosome that is important to copy for the next cells. Heterochromatic areas, where essentially no transcription occurs is also important because these areas start and end transcription. Without these areas, transcription would not occur. The TATA and CAT box are in this area and they help transcription to begin.
ID013	Telomeres create a problem in eukaryotic DNA replication because these telomeres leave gaps at the end of the chromosomes that can't be fixed by DNA Polymerase. As a result, these linear chromosomes can get shorter and shorter over time. Telomerase can bind to this terminal fragment and leave an overhang to slowly prime the end of the chromosome. Also, telomeres can be capped with proteins to protect it from degradation.
ID013	Drosophila chromosomes have polytene chromosomes. These chromosomes show where DNA replication has become chaotic because no part of chromosome is missing. Also, puffs can be identified that show where mRNA is being copied off at high volumes so the DNA must be genetically active. These genetically active sites allow for gene locations to be identified with banding techniques like C and G Banding.
ID013	Chromosomes have euchromatic regions where genes are on because they are not packed with histones. Heterochromatic regions have genes that are silenced because the are packaged with histones, and therefore can't be copied. The heterochromatic region is usually found near the centromere because that is where the chromosome is tightly coiled.
ID143	Telomeres are regions of nucleotide sequences that protect the ends of chromosomes from deteriorating.  Telomeres are critical to an organism's development because the deterioration of telomeres can cause an increase in the aging process.  When linear chromosomes have telomeres in Eukaryotes, problems can arise because DNA replication may not be completed all the way to the ends of the chromosomes.  In this case, the telomeres may not be truncated as they are supposed to, leading to problems with 5' to 3' replication in the organisms.
ID143	Specialized Drosophila chromosomes, otherwise known as polytene chromosomes, are found in the salivary glands of fruit flies.  These chromosomes are over-sized due to the repeated rounds of DNA replication that the organism undergoes without cellular division.  These chromosomes also possess particular light and dark patterns that allow researchers to determine the identification of gene locations.
ID143	Since chromosomes are composed of euchromatic and heterochromatic regions, researchers are better capable of identifying deletions, insertions, and rearrangements within chromosomes.  Due to this pattern, researchers are able to visualize the chromosomes and determine particular trends among DNA.  Because of this mapping process, research has been made much easier, and predictions and the possibility of finding solutions to these chromosomal issues is more feasible.
ID082	In DNA replication, DNA polymerase has to have something to “hold on” to. This need is resolved with the use of RNA primers that will go a few base pairs away from the area of DNA that will be synthesized, allowing DNA polymerase to hook itself onto the DNA strand and perform replication. This process works well in organisms that have circular chromosomes, such as in prokaryotes, and in non chromosomal genetic information, like mitochondrial DNA. However, unlike these examples, eukaryotic organisms have a linear DNA strand. This creates a problem because the end of the strand, the space occupied by the primer (which is necessary to synthesize the new strand) is left not replicated. This results in successive loss and shortening at the end of the nascent DNA strand, which is lethal because cells with incorrect genomes are ultimately degraded. To solve this issue of “chewing up” the end of the chromatin, eukaryotic cells have long tandem repeated sequences at the end of the linear strands. This region of repeated sequences is called the telomere, and it is crucial, because replication will “eat away” at the ends of the strand. The excess of repeated sequences ensures that no genes are consumed at the end of the strand as the strand shortens. An additional enzyme, telomerase, lengthens the ends of the chromatin, maintaining the repeated sequences that comprise the telomere. Usually, cells are eventually marked for apoptosis, and the telomerase will no longer function in maintaining the repetitive sequences at the ends of the chromosome, and the “chewed up” DNA  ultimately be degraded at the appropriate time.
ID082	Because of the relatively simple nature of the Drosophila chromosome, research has been able to observe and manipulate different aspects of the chromosome and identify gene locations. Polytene chromosomes are one specialized form of genetic information in eukaryotic organisms that have allowed researchers to identify gene locations. Polytene chromosomes are found in various cells in various organisms, but are not the normal form of genetic information. Polytene chromosomes are huge chromosomes that exist as paired homologs present in somatic cells. These chromosomes are the result of extensive DNA replication. Because they contain so many copies of identical genetic information, these excessive duplications appear as bands on the chromosomes. Each distinct band represents more than just one functional gene, and as such, it is still uncertain just how many genes are contained in each particular band. However, polytene chromosomes were nonetheless instrumental in the identification of many gene locations along the chromosome.
ID082	Unlike prokaryotic organisms that have simple circular chromosomes that are made entirely of essential genes, eukaryotic chromosomes have an extra layer of gene regulation accomplished by structural modifications to the linear DNA strands. Heterochromatic regions of the chromosome are areas in which structural modifications have effectively silenced, or disabled, the genetic information in this region from being accessible for transcription. These heterochromatic regions are dense, packed structures that consist of nucleosomes. The reason these regions stain dark is because they represent densely packed regions of genetic information, and they collectively absorb more stain. Nucleosomes are structures that arise along the strand of DNA that are comprised of eight histone proteins, each of which are wrapped 1.7 times with the strand of DNA. By wrapping DNA strands around these histone proteins, and then clumping theses wrapped bunches into even higher level structures of condensation, eukaryotic organisms can fit their massive linear DNA strand in the nucleus of every cell. However, there is a trade off in the condensed packing because the genes contained in the packed regions of the strand are silenced when they are in this form. On the other hand, euchromatic regions of the strand are the regions where the genes are accessible and therefore are active. These portions are linear, with plenty of space for the necessary enzymes and proteins involved in transcription to access the DNA sequence. These regions stain lighter because they much less dense regions of DNA, and the stain is less absorbed.
ID064	When DNA is replicated, the lagging strand with the Okazaki fragments ends up having a gap at the 5' end that can't be filled and causes chromosome shortening. Telomeres add extra bases onto the end so the gap is farther out and there is no chromosome shortening.
ID064	Polytene and lampbrush chromosomes allowed the identification by their "puffs" and lengthening, respectively, where RNA transcription was occurring, showing clearly where activity was happening.
ID064	The way DNA wraps around histones, which are protein molecules that help with the shaping and compaction of DNA, affects its ability to be translated. If the DNA is tightly wound around the histones, those sections are unaccessable to be transcripted, and these regions are called heterochromatic regions. The euchromatic are regions where the DNA is not wound as tight, and is accessible to be transcribed.
ID014	DNA polymerase is unable to replicate DNA at the ends of chromosomes so telomeres help solve this problem by ensuring that the DNA at the ends are not deteriorated. As an individual ages and is overcome by stressful factors there telomere length begins to decrease. Every time that the linear DNA is replicated there are gaps left between the RNA primers which the DNA polymerase is unable to fill.
ID014	There were different chromosomes found throughout the Drosophila genome that helped geneticists find the actual location of genes. There are numerous origins of replication found throughout the chromosome, they occur at the same locations helping scientists find gene locations.
ID014	The euchromatic region is lighter than the heterochromatic region but is loaded with genes and transcription occurs most likely in this region. Heterochromatic region is everything that is left within the chromosome and typically there is no transcription that takes place in this portion of the chromosome.
ID065	This occurs because the enzyme involved in replication is bulky and has trouble near the ends of the chromosome because the base is not substantial enough to hold it place. Telomeres help with this problem because they cap the ends of the chromosomes to ensure the entirety of the chromosome is not lost and the genome is conserved. Telomeres create a problem with 5 to 3 prime replication because they are on the ends of the chromosome and do not contain the regions that need to be coded for because their function is to conserve the chromosome.
ID065	Drosophila chromosomes helped in the identification of gene locations because concepts like recombination frequencies can be used and this statistical data can be interpreted and applied to determine the relative distances from one gene with respect to another. Together all this data interpretation can be combined to make a generalized map of the chromosome.
ID065	Euchromatic regions of the chromosome where the chromosome has lots of transcription going on and where gene content is concentrated, whereas heterochromatin is not as concentrated with genes and is not coded as readily.
ID111	DNA polymerase can only synthesize a new strand of DNA as it moves along the template strand in the 3' to 5' direction. This works fine for the 3' to 5' strand of the chromosome because the DNA polymerase can move uninterruptedly from the origin of replication until it meets another point of replication or the end of the chromosome. However this is only the case for 3' to 5'. Synthesis involving the 5' to 3' strand is discontinuous. Discontinuous replication produces a series of short DNA fragments called Okazaki fragments that are complementary to the template strand. These fragments are then covalently bound together by DNA ligase which forms a continuous chain of nucleotides which completes the replication of this strand.
ID111	Drosphilia showed us that there are polytene and lampbrush chromosomes. Drosphilia is an example of polytene. Polytene chromosomes are many-stranded chromosomes that have undergone numerous rounds of DNA replication without separation of the replicated strands which remain in exact parallel register. The result is a giant chromosome with aligned chromosomes displaying a characteristic banding pattern. Polytene are found in various tissues in the larvae of some flies as well as in several species of protozoans and plants. Polytene chromosomes can be seen in the nuclei of interphase cells.
ID111	While we know that eukaryotic DNA consists of continual double-helical fiber along its entire length, we also know that the whole chromosomes is not structurally uniform from end to end. This was discovered in 1928. Euchromatin is chromatin or chromosomal regions that are lightly staining and relatively uncoiled during the interphase portion of the cell cycle. This region contains most of the structural genes. Heterochromatin is the heavily staining, late-replicating regions of chromosomes that are prematurely condensed in interphase. Heterochromatin areas are genetically inactive because they either lack genes or contain genes that are repressed.
ID101	The final primer on the 3' end cant be primed, therefore it must be elongated by telomerase using a RNA sequence to recognize the sequence.
ID101	The identification of chromosomes can be easily done using polytene chromosomes that are special in the sense that they do not divide. Therefore the genes are expressed much more then the average gene and are all more visible due to the large number of chromosomes in a cell.
ID101	The hetrochromatin are tightly wound and therefore can not be transcribed, however the lightly stained regions are loosely wound and can be transcribed.
ID056	The leading strand is replicated continuously and the lagging strand is back stitched together. The lagging strand, which is synthesized from the original 5' to 3' stand, stops short and cannot finish replication alone. It is telomerase that binds to telomeres to fix the shortened strand.
ID056	The specialized Drosophila chromosomes are also known as polytene chromosomes. The are found in salivary, midgut, reactal and malpighian excretory tubules. They can be identified as gene locations because they paired homologs that are very large in size that never separate or divide.
ID056	Euchromatin and heterochromatin are both contained in chromosomes and are important for proper function. Euchromatin is more active than heterochromatin.
ID048	DNA replication alone cannot happen with DNA polymerase, telomerase helps and prevents chromosome degradation. telomeres protect the important genes from being deleted as cells divide and as DNA strands shorten during replication.
ID048	Drosophila chromosomes are different from human and they are specialized in a certain way. The discovery of telomore is derived from studies done on drosophila. They have over-sized chromosomes, which go through repeated round DNA replication without cell division.
ID048	euchromatic is lightly packed material.Despite the fact, that it is lightly packed in the form of DNA, RNA, and protein, it is high in gene concentration and is usually always going through active transcription. euchromatin is found in both eukaryotes and prokaryotes. Hetrochromatin is tightly packed material which is found in the peripheral areas of the cell. It aids in gene regulation. It is only found in eukaryotes and is not usually going through transcription.
ID049	5' to 3' replication problems do not occur on the leading strand. However, on the lagging strand, RNA polymerase primes in pieces instead of a continuous strand. For this RNA to be converted to DNA there must be another strand of DNA in front of the RNA primer, this is not present on the end of the lagging strand. The RNA is destroyed by enzymes and pieces of the telomere are lost after each replication
ID049	Polytene chromosomes are chromosomes that have undergone many replications and have banding patterns that can be studied in Drosophila
ID049	Heterochromatin is tightly packed DNA. These areas cause inactivation of genes either in that area or in euchromatic areas around the heterochromatic areas. Heterochromatic areas are densely packed making them darkly stain
ID020	Telomeres help preserve the integrity and stability of the chromosomes. They create inert chromosomes ends, protecting intact eukaryotic chromosomes. The problem that arises is that at the gaps left at the 5' ends of the newly synthesized DNA. These gaps cannot be filled by DNA polymerase because no free 3'-OH groups are available for the synthesis. With each round of replication, the shortening becomes more severe in each daughter cell which eventually extends beyond the telomere and potentially deleting gene-coding regions.
ID020	These specialized chromosomes are called polytene chromosomes and they are found in various tissues in the larvae of some flies like Drosophila and plants. The bands of the polytene chromosomes was initially interpreted as the visible manifestation of individual genes. The study of these bands during development in Drosophila revealed differential gene activity.
ID020	The terms euchromatin and heterochromatin are used to describe the parts of chromosomes that are uncoiled and those that remained condensed. Heterochromatic area are genetically inactive because they either lack genes or contain genes that are repressed.
ID091	Chromosomes are thought to be linear in that the sequence of DNA codes for certain proteins and functions. However, DNA is replicated with a leading and lagging strand, so DNA cannot be replicated completely as RNA primers are needed. So, DNA with telomeres are shortened every time DNA is replicated.
ID091	Polytene chromosomes are found in various parts of the Drosophila. These chromosomes are very large and not usual. They can be seen by light microscopy during interphase, which allows scientists to see gene activity.
ID091	Euchromatin and Heterochromatin stain different because heterochromatin is packed tightly. Because of this, heterochromatin does not correlate with a high rate of gene activity.
ID058	Telomeres are repeated stretches of DNA located at the ends of chromosomes that help protect DNA from degradation. The problem is DNA polymerase cannot replicate there. First telomerase has to attach a RNA template and through reverse transcription create DNA.
ID058	Drosophila chromosomes have "puffs" which are easily visible portions of their chromosomes where one can see areas of high gene activity. It allowed scientists to identify gene locations.
ID058	Euchromatin are uncoiled and contain many, active genes. These appear lightly stained during interphase whereas heterochromatin are condensed areas of chromatin with little gene activity that appear darkly stained during interphase. Little transcription takes place for the condensed chromatin because they are wrapped around the histones.
ID103	DNA is replicated in the 5' to 3' direction which means the lagging strand must be copied discontinously. A few base pairs get left off of each round of replication which is why we have the junk DNA sections known as telomeres located at each end.
ID103	polytene chromosomes undergo repeated replication without cell division. This causes these cells to increase in volume and have increased expression due to multiple copies of a gene.
ID103	DNA must be in a loose conformation in order for genes to be accessed. some functional dna such as the centromere are always in the heterochromatic phase always.
ID059	Linear chromosomes with telomerase creates a problem with the 5'-3' replication due to the fact that the information that is replicated from the excessive RNA strand will never make a chromosome that is copied completely. The information from the DNA is always shortened and in order to prevent the loss of more genetic information, telomerase is created to help cap the DNA and copy it from the RNA strand leaving the 3' end short and 5' end complete of genetic information.
ID059	Specialized Drosophila chromosomes are important to our history because they allowed the identification of gene locations within our genes. For example,each band that's on the chromosome is color coded for specific functions and at a specific location. Most drosophila chromosomes are connected at a central location and spread throughout, so if one spot is darkened in a certain area and on the next chromosome the other spot is darkened at the same location, it shows a relationship between the function and location of each gene.
ID059	Chromosomes that are composed of euchromatic information is basically that of which doesn't have tightly packed information therefor doesn't have strong impact on staining regions. The heterochromatin on the other hand has tightly packed information, localized in the nucleus, and sometimes have many genes that are compressed.
ID124	dna is transcribed from dna starting at 5' end because rna polymerase moves from 3' to 5' because its anti-parallel
ID124	Drosophila have specialized chromosomes that allows genes to be tagged to show where the gene lies on the chromosomes
ID124	euchromatic regions are the ones that are transcribed and therefore the essential parts of the chromosome because they code but the heterochromatic regions are not active and do not code
ID093	After each duplication, DNA is shorter at it's 5' end than the parental DNA strand. This causes an overhang at the 3' end of each daughter double helix. Telomeres are repetitive sequences that protect the important genes from being deleted. Once these telomeric regions are lost, the DNA risks losing sequences with further replication. This is being studied to somewhat inhibit the aging process.
ID093	Polytene chromosomes are oversized chromosomes which have developed from standard chromosomes. These chromosomes have characteristic dark and light bands that can be used to locate and identify rearrangements and deletions. The light bands correspond to locations of high transcriptional activity.
ID093	Introns are noncoding regions of the gene while exons are the parts of the gene that code for a final part of mature RNA. The introns are spliced out during transcription. Prokaryotes have relatively fewer introns than eukaryotes. These noncoding regions contain ancient human DNA as well as viral.
ID108	DNA polymerase can only synthesize in the 3' to 5' direction uninterrupted from an OIR. In the 5' to 3' direction replication is discontinuous and the strand is synthesized in Okazaki fragments where DNA ligase has to stitch together the fragments.
ID108	Polytene chromosomes have light and dark banding patterns. Dark patterns mean inactive chromatin and light patterns mean transcription is occurring.
ID108	Euchromatin contains active genes and is loosely packed in loops and has decreased methylation of of cytosines and increased acetylation of histones. Heterochromatin has few or no genes and has decreased ethylation and increased methylation of lysine.
ID150	DNA polymerase moves along the template strand in the 3' to 5' direction so that the new strand is created in the 5' to 3' direction. However, the non template strand cannot be synthesized completely in this direction, so it creates Okazaki fragments, which are connected via DNA ligase.
ID150	Drosophila have cells that replicate DNA for multiple rounds without cell cleavage. The product is oversized chromosomes called polytene chromosomes. These chromosomes have banding patterns when viewed. Dark bands are associated with inactive transcription areas, while light are associated with active transcription areas. Studies of this showed the structure of chromatin.
ID150	Most of the human genome is euchromatic, meaning that it is rich in gene expression. However, some portions are not active; these are called heterochromatic regions. When viewed light and dark banding indicates the two types of regions.
ID118	Prokaryotes are circular chromosomes while eukaryotes is linear chromosomes. Because eukaryotic chromosome is linear, problem arises when DNA replication occurs which always occur from 5'to 3'.Eukaryotic chromosomes end in distinctive sequences called telomeres that help preserve the integrity and stability in the chromosomes. In such cases, the double-stranded loose ends can fuse to other such ends; if they don't fuse, they are vulnerable to degradation. The newly created gap in lagging strand is filled in with addition of nucleotide to the adjacent 3'-OH group, so there is no Okazaki fragments. The gap remains in the lagging trans resulting in shortening the double-stranded with the length of RNA primer. As more replication occurs the length shortens, deleting the potential gene-coding regions.
ID118	Specialized Drosophila chromosomes of polytene chromosomes and lamp brush chromosomes shows the arrangement and function of the genetic information noticed by the puff. Polytene chromosomes are found in various tissues in the larvae of some flies such as drosophila. Polytene chromosomes represents paired homologs its high unusual, since they are present in somatic cells, where in most organisms, chromosomal material is normally dispersed as chromatin and homologs are not paired and large size.Polytene chromosomes the active gene site on the chromosomes by the puff, which can be visual. Lamp brush chromosomes are differentiated by the extended lengths, and uncoiled versions of the normal meiotic chromosomes.
ID118	Heterochromatin are the genetically inactive read because the genes are tightly compacted together that they can't be read example centromere and mammalian Y- chromosomes genetically inert and the x-chromosomes of the barr bodies are genetically inert. Euchromatin are uncoiled area of chromosomes whereas heterochromatin is condensed. Heterochromatin replicates later during the S phase of the cell cycle than does euchromatin.
ID129	Telomeres are located on the ends of chromosomes, and they shorten with every cell division. An RNA primer is synthesized at the 5' end of the chromosome. Removal of the primer (when telomeres are shortened) leaves a gap that is unable to be filled by DNA polymerase, and this causes problems in eukaryotes.
ID129	Polytene chromosomes in Drosophilia have light and dark banding patterns. The light areas usually had higher transcriptional activity, and gene locations were able to be identified from this.
ID129	Euchromatin is uncoiled and active, and it appears unstained in interphase. Heterochromatin is where condensed areas are mostly inactive, and it appears stained during interphase. Heterochromatin also contains fewer genes than Euchromatin. It replicates later in the 5 phase than euchromatin, and the centromere is involved in chromosome movement.
ID007	DNA polymerase can only copy DNA in a 5' to 3' direction, so while the leading strand is synthesized in a linear manner, the lagging strand must be synthesized in pieces, and the very end of this strand is vulnerable to shortening.  Telomeres are repetitive sequences that exist at the ends of the strand and are not essential to the chromosome and so protect against degradation of necessary genetic material.
ID007	Polytene chromosomes exist in certain tissues of Drosophila, for example their salivary glands, and they are the result of DNA replication without mitosis, such that very large chromosomes form that are easily visible and manipulated under a microscope.
ID007	The difference between euchromatin and heterochromatin is essentially that euchromatin contains the actual gene sequences while heterochromatin does not.  However, sometimes the boundary is not as clear and euchromatic regions can be silenced such that they act like heterochromatin.
ID107	Telomeres are regions of repetitive nucleotide sequences at the end of a chromosome. They protect the chromosome from deterioration. During gene replication, the whole chromosome is not replicated and it becomes truncated. The telomere protects the gene and acts as a buffer that gets cut off a little during replication. Telomeres deteriorate over time and telomerase reverse transcriptase is an enzyme that replenishes them.
ID107	Polytene chromosomes were first observed in the salivary glands of drosophila. Polytene chromosomes are specialized cells that undergo several rounds of DNA replication without cell division to form one large chromosome. Their purpose is to expand the cell and increase cell volume.
ID107	During interphase, you can see euchromatin and heterochromatin. Euchromatin are genes that are being actively transcribed and are less compacted (uncoiled) than the heterochromatin which remain coiled and are not undergoing transcription. Euchromatin contain active genes and heterochromatin contain few genes or repressed genes.
ID061	Linear eukaryotic chromosomes end in distinctive sequences called telomeres, which help preserve the stability and integrity of the chromosomes. They create inert chromosome ends that protect the chromosomes from improper fusion or degradation. When eukaryotic cells undergo DNA replication, RNA primers provide the 3'-OH necessary to initiate synthesis. However, at the 5' ends of newly synthesized DNA, there are gaps that cannot be filled by DNA polymerase because there are no free 3'-OH groups for the initiation of synthesis. Thus, the replicated strands are shorter than the template strands since there are gaps at the end. Over time. this could be a problem as strands become shorter and shorter after each replication. However, telomerase serves to fix this problem by pairing part of an RNA strand with the end of the sequence of the single-stranded overhanging DNA while the rest extends beyond this overhang. Then reverse transcription produces an extension of the DNA strand. Then, once the telomere has been lengthened by this activity,conventional DNA synthesis ensues.
ID061	Both polytene chromosomes and lampbrush chromosomes are types of chromosomes discovered in Drosophila that have allowed identification of gene locations. Polytene chromosomes exhibit a linear series of alternating bands and interbands, producing a pattern that is distinctive for each chromosome in any given species. The presence of bands on the chromosome is actually the visible manifestation of individual genes. Bands revel differential gene activity and can help distinguish locations. Lampbrush chromosomes do not condense as normal meiotic chromosomes do but instead extend to great lengths. Lampbrush chromosomes have a pair of lateral loops emanating from each of them, and much more DNA is present in these loops than in the rest of the chromosomes. These loops represent DNA that has been reeled out from the central chromosome axis during transcription.
ID061	Euchromatin describe the parts of the chromosome that are uncoiled and thus able to be transcribed into mRNA to begin the process of producing proteins. Meanwhile, heterochromatin is condensed regions of the chromosomes that cannot be transcribed to produce mRNA and later proteins. This shows that not all of the DNA always encodes proteins.
ID096	In order for 5' to 3' replication to occur, the template has to be discontinuous.Gaps will remain on newly synthesized DNA strands, which shortens the double-stranded ends of the chromosome by the length of the RNA primer. Each round of replication, the shortening becomes more severe in each daughter cell.
ID096	As development in the chromosomes continue, band formation associated with gene activation is observed. There are approximately 5000 bands out of 15,000 genes in Drosophila.
ID096	Heterochromatin are only found in eukaryotes. Euchromatic are lighter than heterochromatic. Heterochromatic's DNA are packed much tighter than euchromatic.
ID134	Genes are arranged linearly on prokaryotes, eukaryotes, and viruses. A strand read 3'to 5' can be read continuously, however a strand read 5' to 3' must be read discontinuously. Because telomeres protect the chromosome this reversed reading of the strand can cause problems.
ID134	Drosophila are known to be an ideal specimen for much DNA experimentation and discovery.Genes were more simple to map, and locate on the drosophila chromosomes.
ID134	The light staining could be due to the action within the euchromatic, while the lack of activity attributes to the dark of the heterochromatic. These correlate with the introns and exons
ID077	After DNA replication, each newly synthesized DNA strand is shorter at it's 5' end than at the parental DNA strand's 5' end. This produces a 3' overhang at one end of each daughter DNA strand. This causes some problems at the end of linear DNA replication.
ID077	Polytene chromosomes are oversized chromosomes commonly found in the salivary glands of Drosophila. Polytene chromosomes have characteristic light and dark banding patterns that can be used to identify chromosomal rearrangements and deletions.
ID077	Chromatin is found in two varieties. Heterochromatin is dark staining with essentially no transcription because it is more tightly packed and unable to be read. Euchromatin is light staining with active transcription because is not tightly bound and can be read for transcription.
ID100	Linear chromosomes are capped with sequences of DNA called telomeres that help prevent degradation of important genetic material. Because DNA polymerase cannot add nucleotides to the ends of linear chromosomes, the 5' end is left shorter than the 3' causing problems in eukaryotic DNA replication. An enzyme telomerase helps alleviate this problem by adding DNA sequences to the end of eukaryotic chromosomes.
ID100	Polytene chromosomes found in the Drosophila are specialized, over-sized chromososomes that are formed from multiple rounds of DNA replication. The resulting sister chromatids remain synapsed. These chromosomes allow an increased level of gene expression. These chromosomes consist of light and dark banding patterns that can be used to identify chromosomal deletions and rearrangements. The dark banding generally correlates with silent non-transcripitionally active DNA whereas the light banding DNA generally correlates with transcriptionally active genetic material.
ID100	Chromosomes are composed of two different types chromatin, euchromatin and heterochromatin. Euchromatin stains light and typically corresponds to areas of high transcriptional activity. Heterochromatin stains dark and typically corresponds to areas of low transcriptional activity. Depending on where a gene lies (in euchromatin or heterochromatin) its level of gene expression varies.
ID140	Telomeres is needed to replicate DNA, but with that, it enables the chromosomes and cells to start degrading. Which is believed to be linked to aging.
ID140	Scientists were able to use drosophila to color code and manipulate the genes to find where they are located.
ID140	This is important because it allows us to know which areas do and do not perform transcription.
ID115	Because DNA polymerase cannot synthesize continuously on the 5' to 3' strand, it poses a problem in finishing copies of the lagging strand. As a result, RNA polymers must create a longer strand to finish copying an cap it with a telomere. However, once the extra strand is delete, it takes a piece of the DNA strand and as consequence continually shortens the segments of DNA.
ID115	Through polytene chromosomes in the salivary glands of Drosophila, the high reproductive rate allows several genes to be identified through control of gene activation.
ID115	The regions that are euchromatic contain high areas of transcription, while hte areas that are heavy contain no transcription. This is because the euchromatin has less compact structure due to high activity for transcription. Those areas that are darker are due to less active transcription.
ID116	After DNA replication, linear chromosomes have a problem. This is because after replication, the newly synthesized DNA strand is shorter at the five prime end than the parent strand. Therefore, there is a three prime overhang at one end of the daughter strand. Therefore, opposite ends have this three prime overhang. DNA polymerase cannot replace the RNA with DNA, therefore, eventually, the entirety of the DNA would be lost. Therefore, linear chromosomes have telomeres.
ID116	Polytene chromosomes are in many tissues of the Drosophila fly, including salivary, rectal, midgut, excretory tubules of the fly. The puffs in these chromosomes represent gene activity. These bulges or puffs show were there is a lot of gene activity or where the transcription produces RNA.
ID116	Heterochromatin can be either constitutive or facultative. Constitutive heterochromatin affects the genes around it and can form telomeres or centromeres. It also can be used for gene expression or repression signals. Facultative heterochromatin is generally the result of silenced genes, however it can become active by losing its condensed structure. Euchromatin generally contains the most active portion of the genome. It is usually under active transcription.
ID031	The telomeres cause a problem when they are read in th 5 to 3 direction causing a problem in replication.
ID031	In drosophila there are special genes on chromosomes that allowed for scientists to use them as a map or guide in comparing them with other gene locations.
ID031	This helps distinguish which areas of the chromosome certain reactions are occurring and it also helps distinguish what reactants are needed.
ID092	When chromosomes are replicated, the telomere is the end of linear chromosomes that keep genes from getting deleted as replication occurs.  This is a problem because it prevents DNA polymerase from being able to repair and the ends of these chromosomes.  This problem is solved from a telomerase enzyme that elongates the lagging strand and allows DNA polymerase to perform its intended function.
ID092	Polytene chromosomes are specialized, over sized chromosomes that result from repeated DNA replication without cell division resulting a giant clump of DNA.  What makes these useful is that the have light and dark banded areas with light areas corresponding to high replication areas while the dark spots indicate inactive chromatin.  Using these light and dark bands show areas that transcription occurs at most often and allowed scientists to follow all genes that are replicated to show that each cell contains the same genes.
ID092	These chromosomes were discussed in the above question.  Like mentioned before, the dark areas represent inactive chromatin while the light areas reveal areas that transcription occurs abundantly.  This is useful in showing scientists how and which parts of DNA are transcribed.
ID127	Telomeres cause a problem because they are the ends of the chromosomes that takes part in the aging of the chromosome. If the chromosome does not have telomerases, it will not add on to the sequences and it will get shorter and shorter.
ID127	The Drosophila chromosomes that allow for identification of gene location was the polytene chromosome which are large chromsomes that have well-defined bands that resemble and map and make it easier for mapping of the chromosomes and finding the locations.
ID127	Euchromatin is heavy in gene concentration and is undergoing transcription, which is why the DNA is not as tightly packaged as in heterochromatin and shows light bands under the microscope. Heterochromatin has dark bands because the DNA is tightly packaged and is not active(not transcribing). It is important for protection of the chromosome.
ID137	DNA polymerase cannot add to the linear chromosome so the 3' end is shorter. Telomerase enzymes come in and use the RNA template the add to the 3' end in order to protect the gene.
ID137	Polytene chromosomes can increase the size of a cell's nuclei and help increase gene expression. Polytene chromosomes have light and dark bands that show where deletions have occurred - this helps with identification.
ID137	Euchromatins are light staining regions because it is coiled and active in gene expression. Heterochromatin are dark staining regions because the few genes are densely compacted and not active in gene expression.
ID081	In a linear DNA chromosome, after DNA replication, each newly synthesized DNA strand is shorter at its 5' end than its parental strand. This produces a 3' overhang at one end of each daughter cell.
ID081	Specialized chromosomes of the Drosophila that are located in the salivary glands allow identification of gene locations due to its large size. These specialized cells undergo repeated rounds of DNA replication without cell division which increases cell volume, forming a giant polytene chromosome. Polytene chromosomes form when multiple rounds of replication produce many sister chromatids that remain synapsed together.
ID081	Euchromatic regions of chromosomes are the light staining regions of active transport. This region correlates to the region of the chromosomes that are expressed. Heterochromatic regions of the chromosomes are dark staining regions that creates essentially no transcription. This region of the chromosome is not expressed.
ID090	Telomeres, the G rich series at the end of chromosomes, present a problem on the lagging strand. When the RNA primer is removed there is not a free 3’ OH group for DNA polymerase I to fill in the gaps. Thus the lagging strand is shorter than the leading strand. If this segment is not lengthened, it will shorten with each replication and can eventually affect the regions of the chromosomes that encode genes. This problem is resolved via Telomerase and enzyme with an RNA segment that pairs with the overhang on the leading strand and extends the lagging strand so that the chromosome is not shorter.
ID090	Polytene and lampbrush chromosomes are unusual and not typically found in most cells but the study of their structure has allowed for identifying gene locations. The study of the many bands in polytene chromosomes shows gene activity. While we do not know how many genes are encoded in each band, it is known that each band contains much more DNA than is needed active genes. Polytene chromosomes are paired homologs which is highly unusual. They have extensive banding and form “puffs” which represent areas of gene activity.
ID090	Chromosomes are composed of two regions: euchromatin and heterochromatin. The areas of light staining (euchromatin) contain active genes that are being expressed, while in heterochromatin there are no active genes present. The differences in staining allow researchers to identify areas where genes are active.
ID033	I haven't read the book about this yet so I do not know. Hasn't been covered well at all in lecture.
ID033	The location of genes can be determined in drosophila chromosomes.
ID033	It makes bands on the chromosomes so you can indicate which chromosome it is.
ID060	Too much telomeres can be as bad as too little telomeres in the human body.
ID060	Drosophila has been a model organism for the use of genetic research. Some species are used to study the locations of many genes.
ID060	In the euchromatic region, there is a compacted gene concentration where it is known as active transcription. Whereas, heterochromatic is the dark region and is essentially where there is packed DNA and is not really transcribing anything.
ID141	Linear chromosomes cause a problem when both the leading strand and the lagging strand are being replicated at the very end of the chromosome.  This is because the RNA primer leaves a gap at the end of the replicating strands that DNA polymerase cannot fill.  The deletion eventually extends beyond the telomere and deletes gene coding regions.
ID141	There are two specialized drosophila chromosomes, the polytene chromosome, and the lampbrush chromosome.  The polytene chromosome stands out in that puffs are created from a high level of gene activity at certain areas along the chromosome.  The lampbrush chromosome is similar to the polytene chromosome, in that they are both sites of  increased gene activity.
ID141	Chromosomes have areas where transcription is actively taking place during interphase, theses areas stain lightly because the chromosome is uncoiled at this location and are known as euchromatin.  On the other hand, areas that stain darkly are known as heterochramatin, and are condensed areas where transcription is not taking place.
ID133	After DNA replication, each newly synthesized DNA strand is shorter at its 5' end than at the parental DNA strand's 5' end.  This produces a 3' overhang at one end of each daughter DNA strand.
ID133	The polytene chromosomes can be seen under a light microscope, which consist of a series of linear, alternating bands.  These are so large and easily seen that the polytene chromosomes were observed before we understood how mitotic chromosomes form from interphase chromatin.
ID133	Heterochromatic areas are genetically inactive because they either lack genes or contain genes that are repressed.  The presence of of heterochromatin is unique to and characteristic of the genetic material of eukaryotes.
ID008	synthesis using the 5' to 3' strand as the template has to be discontinuous.
ID008	genes in the smaller interbands are expressed more than those in the more compact bands.
ID008	These are the two types of chromatins. The heterochromatic is the highly condensed form, while, euchromatic is the less condensed.
ID053	Telomeres are repeating sequences at the end of chromosomes that protect the chromosome from being shortened.  Telomeres prevent this by acting as a buffer to shortening of the chromosome, so the non-coding telomeres are degraded instead.  Telomeres are lengthened by telomerase, and telomeres are always shortened on the lagging strand.  The lengthening of telomeres by drugs and gene therapy has been postulated as a possible solution for aging; however, telomere lengthening has also shown a high correlation with cancer cells.
ID053	These specialized Drosophila chromosomes are called polytene chromosomes.  Polytene chromosomes start out as normal chromosomes but are replicated many times with no cell division.  This causes the chromosomes to have many copies that are paired together, allowing scientists to see intricate banding patterns across the chromosomes.  This allows researchers to see the specific locations of genes on the chromosome.
ID053	Euchromatic regions of chromosomes are regions that are read by RNA polymerase, so these regions code for genes.  Heterochromatin (usually close to the centromere in a chromosome but also elsewhere) is usually non-coding because it is so tightly wound up.  This is also why the heterochromatin stains darker compared to euchromatin—it is denser.  Because it is more tightly coiled, it is not read by RNA polymerase.  Many times the tight coiling is caused by methylation.
ID069	While prokaryotes have circular DNA that does not reach a fatal end point, eukaryotes have linear DNA that must end. These ends risk degradation and therefore ruining of crucial genes. This problem can be assuaged with telomeres. Telomeres add additional non-coding genes with the help of the protein telomerase. With time, telomeres will shorten and induce aging. However, telomeres do not shorten in cancer cells. This produces a problem in that telomeres aid in propagating the life span of harmful cancer cells.
ID069	Chromosomes can and must be packaged in a variety of intricate, specialized ways in order to fit into nuclei that are much smaller than their fully extended length. Drosophila chromosomes are packaged in a special way in which strands of chromosomes are lined up next to their same genes, creating lines indicating areas of certain same genes. This arrangement is referred to as a polytene chromosome and helps identify gene locations since the genes are much more visible in larger amounts.
ID069	Post-transcriptional regulation affects gene expression by manipulating nucleosome packaging. Acetylation causes the nucleosome to "unwind," allowing the DNA and associated histone proteins to relax and stretch out, enabling the DNA to be read by DNA polymerase; methylation induces the opposite effect, causing the nucleosome to reassemble. When a nucleosome is unwound and the DNA can be read, active transcription is occurring and the chromosome stains lightly, indicating it is euchromatic. When a nucleosome is tightly packed and the DNA cannot be read, the histone-DNA clumps stain darkly, indicating it is heterochromatic.
ID043	Telomeres are at the end of linear chromosomes because DNA polymerase cannot alone replicate and repair DNA molecules at the ends of chromosomes.
ID043	Certain drosophila had key genes and from this scientists were able to pin point where these genes were, which was crucial because we then could keep track of reoccurring mutations.
ID043	The heterochromatic region is a tightly compacted region and contains few genes. Euchromatic regions contain active genes that are less compacted.
ID039	Telomeres are present to protect genes, but that can break off and be lost during replications. Overtime, this causes a problem because there is nothing remaining to protect the DNA from being lost
ID039	Drosophila have an very unusual type of chromosome called  polytene chromosomes. These are found in various tissue of drosophila.
ID039	There are two different types of regions on chromosomes, euchromatic and heterochromatic. The euchromatic region is the uncoiled active region were transcription is occurring, and the heterochromatic is the inactive region
ID151	I do not know much information about telomere replication, but I do know that the amount of telomeres have been associated with cancer and youth, but on the opposite ends of the spectrum. Furthermore, shortened telomeres have been noticed and associated with cancer cells but longer telomere sequences have been associated with aging, thus making it a prime candidate for research especially during DNA replication. Telomeres are tandem repeats found at the end of chromosomes that are produced by the enzyme called telomerase so the actual genes on a chromosome do not get affected by cell replication and cell division during mitosis in somatic cells. The enzyme telomerase controls the amount of sequenced telomeres are placed and paired since DNA polymerase cannot add bases to the ends of linear chromosomes due to a problem from RNA primers. I unfortunately do not know the problem that was caused, but would like to know more information about this issue.
ID151	Two specialized chromosomes that brought more knowledge about the basic structure of a chromosome -specifically chromatin- were Polytene and Lampbrush chromosomes. Lampbrush chromosomes were commonly found in amphibians; whereas, Polytene chromosomes were known to be specialized chromosomes  found in Drosophila. Polytene chromosomes were found to be replicated innumerable times, but fail to undergo cell division which results in the large size of the Polytene chromosomes. Furthermore, since the staining patterns indicated that these specialized forms of chromosomes are highly composed of euchromatin, the multiple copies allow an increase in protein production through gene expression of the multiple genes. These Polytene chromosomes enabled researchers to determine certain banding patterns within the Polytene chromosomes as well allowed a structure of chromatin in Eukayotes to be noticed and constructed. These banding patterns provided further confirmation to two notions. The first being that all the cells within an organism contain the same genome or carry the entire set of chromosomes, but gene expression and regulation allows for cell differentiation. The second important thing enabled by specific banding patterns was the ability to notice certain gene mutations with ease such as translocations and frame-shift mutations.
ID151	When staining chromosomes, two methods-- C banding and G banding, can be used. These banding techniques produce banding patterns that are unique for each chromosome. G-banding, in my opinion, is more useful than C-banding in the sense that more information can be told from the band patterns in G-banding especially in the scenario of a genetic disease. These banding patterns are dependent upon if the gene is highly transcribed or rarely transcribed; the genes that are highly transcribed are known as euchromatin since the gene is able to produce an actual element (amino acid), whereas, those genes that are never transcribed are heterochromatin and cannot produce an amino acid in regular circumstances. The centromeres of chromosomes are shown to be highly composed of heterochromatin since the centromere region is stained darkly with the Giemsa stain which is indicated in C-banding.
ID117	Telomeres a long chains at the end of DNA sequences of repetitive base pairs that vary in length. They are believed to be a protector of DNA degradation during DNA replication in cells. In most eukaryotes, the enzyme telomerase is not active, and the length of telomeres reduces with every replication.
ID117	Specialized Drosophila chromosomes such as polytene represent homologous chromosomes.These chromosomes have a lot of DNA replication without separation, so it's a giant chromosome very easily studied under light. Big 'puffs' were seen and this is where there was a lot of DNA replication action and identified to be coding, gene-rich areas. The spaces between these puffs were found to be non-coding strands of DNA.
ID117	A majority of chromosomes are not actively involved in DNA transcription and the copying of genetic material. Genes are scarce in comparison to the vast length of a chromosome. Active transcription occurs in euchromatic regions, where they stain lightly, and heterochromatic regions where very little to no transcription occurs and staining is dark
ID095	Because the DNA is replicated from 5' to 3', there is an overhang of DNA at the end.  DNA cannot replace this overhang, therefore this part of the chromosome is lost.
ID095	The Drosophila produce polytene chromosomes in their salivary glands.  These large chromosomes have distinct large bands representing heterochromatin and euchromatin.
ID095	Euchromatin are the coding regions of the chromosomes and heterochromatin are the noncoding regions of the chromosomes.  These largely packed regions appearing on the chromosomes may be stained and identified.
ID035	DNA strand is shorter at its 5' end than at the parental DNA strand's 5' end. This makes a 3' overhang at one end of each daughter DNA strand. The two daughter DNAs have their 3' overhangs at opposite ends.
ID035	Drosophila has been used as a model organism for genetics and evolutionary studies. A linkage map with 48 microsatellite loci recently published for this species showed five syntenic groups.
ID035	Euchromatins are prevalent in cells that are active in the transcription of many of their genes while heterochromatin is most abundant in cells that are less active or not active.
ID078	This process add to the ends of the chromosome so genetic material isnt lost as the DNA primers cant extend past the end of the chromosome.
ID078	I have no clue what this is, i didnt find it in the book and i dont recall being taught this in lecture
ID078	Heterochromatin replicate later in interphase and dont undergo transcription, theyre stained darker because they are more tightly coiled than euchromatin
ID068	Telomeres are replicated sequences on the ends of chromosomes to protect against pieces being lost from the end during replication. Protein caps are added to the telomeres for further protection. If there is a mutation in telomerase to add the protein caps, pieces of the chromosome can break off at the end and causes quicker aging of cells.
ID068	Specialized chromosomes of Drosophilia are called polytene chromosomes. With staining, different genes of the chromosome can be seen. Puff areas in the chromosome are where there are regions of gene activity such as high levels of transcription.
ID068	Euchromatin is uncoiled and active causing it to appear unstained. Heterochromatin are condensed areas of few genes that there is no activity and shows up in staining. There are two types of staining, C-banding and G-banding. In C-banding, mitotic chromosomes only the centromeres which are heterochromatic show up in staining. In G-banding, the chromosome is digested by the enzyme trypsin and heterochromatic AT-rich regions stain darkly.
ID075	Telomeres of chromosomes shorten with each cell division. This creates a problem because over a period of cell divisions, there will be no chromosome left. However, telomerases help the chromosome maintain their length.
ID075	The identification of gene locations was made possible through specialized drosophila chromosomes (polytene chromosomes). These can be visualized by light microscopy and can be seen in nuclei of interphase cells.
ID075	Chromosomes are composed of two regions: euchromatic and heterochromatic. The euchromatic is the region where there is active transcription. The heterocromatic region is where no transcription occurs. This is the dark staining region while the euchromatic is the light staining region.
ID055	Eukaryotic cells have a shortening problem with the telomeres. Telomere shortening is associated with aging. When RNA replication occurs they are incomplete do to the shortening of the DNA.
ID055	Drosophila polytene chromosomes  showed under a microscope pairs of genes (homologs) and puffs which represent areas of gene activity.
ID055	Euchromatic contain active genes and are less compact. Where as heterochromatin, contain repressed genes, replicate later in S phase than euchromatin, telomere maintains chromosome integrity, and chromosome movement involves the centromere.
ID088	DNA polymerase cannot replicate and repair DNA molecules at the ends of linear chromosomes.The telomerase enzyme attaches to the end of the chromosome. So, complementary bases to the RNA template are added on the 3' end of the DNA strand.
ID088	By studying the telomerase in Drosophila, and how the reproduced, it became possible to map their genes.
ID088	Euchromatin comprises the most active portion of the genome within the cell nucleus. And heterochromatin affects gene expression.
ID028	The DNA polymerase synthesizing the lagging strand cannot bind to the RNA primer synthesized at the 5' end of the cdhromosome, so there is a gap here. Telomerase extends the other strand to allow for the whole chromosome to be replicated.
ID028	Geneticists were able to find these gene locations through staining the DNA and then viewing the different locations of the genes.
ID028	The euchromatin is lightly stained because the DNA is not tightly wound on the histone proteins. Because it is loose, DNA polymerase is able to come in and replicate. However, the heterochromatin is not available for transcription because it is too tightly wound.
ID086	DNA only replicates only in one direction, but the stands are anti parallel so there has to be a way to do it the same way.
ID086	The drosophila DNA can be modified in different ways by different environmental factors causing phenotypes to occur.
ID086	This is the most important part of the idea of DNA. All cells have the same DNA, but different areas are turned on and off.
ID021	Eukaryotes have linear chromosomes, unlike prokaryotes which have circular chromosomes. Telomeres are found as repeated DNA segments at the end of chromosomes. They serve protective functions. But alone, they would shorten the length of the chromosomes. The problem is solved by the enzyme telomerase which adds back bases to the ends of chromosomes and avoids shortening.
ID021	These chromosomes are huge and easy to read because they are repeatedly replicated many times. This makes it easy to identify gene locations.
ID021	Variation in what makes up chromosomes helps maintain diversity and quality. Heterochromatic regions appear dark when stained and are not transcribed. They are condensed and genetically inactive. Euchromatic regions are the opposity. They appear light when stained. They are highly transcribed and genetically active. They are uncoiled and available for transcription.
ID038	The replication of chromosome proceeds in 5' to 3' direction and thus the initial part of the 5' DNA where RNA polymerase cannot reach is not replicated. This leads to shortening of DNA which is identified as the cause of ageing.
ID038	Specialized Drosophila chromosome like polytene chromosome and lampbrush chomosome are larger and can be easily stained and seen under microscope. Thus they allow identification of gene locus with ease due to their larger size.
ID038	Chromosome are composed of the active euchromatin and the inactive heterochromatin. This allows effective regulation of which genes can be transcribed and which cannot and thus prevents abnormal transcription without the need of blockers.
ID123	A problem is created in linear chromosomes because after replication the 5' end is shorter than the parental 5' end.  This leads to the 3' end to overhang.  This leads to the RNA primer of the 5' end not being replaced with DNA, so the daughter cells have incomplete strands of DNA.  These overhanging strands are digested by nucleases, which over time leads to the strand becoming shorter and shorter.  This is why telomeres exist on DNA strands, in order to protect the important genes from being degraded over time.
ID123	Drosophila have specialized chromosomes called polytene chromosomes which allow the identification of gene locations.  These chromosomes are paired homologs and are unusual.  They are found in the salivary, rectal, midgut, and excretory tubules of the Drosophila.  Puffs in the the chromosome represent areas of gene activity and location of that gene.  Bulges show DNA which has uncoiled and are signs of high gene activity.
ID123	Euchromatic chromosomes are euchromatin, while heterochromatic chromosomes are heterochromatin.  Euchromatin is uncoiled, active, and is unstained during interphase.  Heterochromatin is stained and appears condensed in inactive regions.  Heterochromatin usually contain far fewer genes that euchromatin and are also much more compact than euchromatin.
ID052	Due to this replication problem, eukaryotes have special enzymes called telomerase that adds specific sequences of DNA to the ends of linear chromosomes. Once telomerase elongates the lagging strand, DNA polymerase can continue to add the complementary nucleotide bases.
ID052	Because of the large polytene chromosomes found in the cells of Drosophila, scientists are able to study the high frequency of gene expression to determine the location of genes. This was useful in developing a better understanding of the underlying mechanism of how and where genes are expressed and located.
ID052	Euchromatin is found in both eukaryotes and prokaryotes. Euchromatin consists of lightly-packaged DNA, RNA, and proteins. Euchromatin is usually under active transcription. Heterochromatin has more tightly-packaged genetic material, but does not undergo active transcription. Heterochromatin is also only found in eukaryotes.
ID010	After DNA replication has occurred, each new DNA strand is shorter at its 5' end than at the original parent DNA 5' end. This causes each new DNA strand to have a 3' end overlay on one end.
ID146	After DNA replication, each new DNA strand is shorter at its 5' end than the parental DNA strand's 5' end. This produces a 3' overhang at one end of each daughter DNA strand, such that the two daughter DNAs have their 3' overhangs at opposite ends
ID146	The chromosomes located in various Drosophila tissues (namely salivary glands) swell and can be seen using light microscopy, especially during interphase
ID146	Euchromatin are uncoiled and active sections of DNA that appear unstained during interphase, while, heterochromatin are condensed areas that are
ID130	At the end of replication the 5' end of the dna is shorter than on the parental strand.  This produces a 3' overhang.
ID130	The tandem repeated bands on the polytene chromosomes allow the tracking of chromosomal rearrangements and deletions.
ID130	Euchromatin (or "good" chromatin) is the lightly staining region of of the chromosome is loosely wound and is able to be read for gene transcription.  Heterochromatin is the darkly staining region which is wound too tight for dna enzymes to be able to attach and transcribe.
ID044	DNA polymerase cannot replicate and repair DNA molecules at the ends of linear chromosomes. The ends of linear chromosome have telomeres, which protect genes from getting deleted as cells divide.
ID044	The banding patterns of Drosophila chromosomes were sketched in 1935 by Calvin B. Bridges.  This was done in such detail that these maps are still used. The banding patterns of the chromosomes are helpful in research because they provide a visualization of transcriptionally active chromatin as well as the general chromatin structure. The Drosophila chromosomes have been used to support the theory of genomic equivalence, which states that all of the cells in the body maintain the same genome.
ID044	Chromosomes have characteristic light banding and dark banding patterns.  The dark banding usually corresponds to inactive chromatin, and the light banding is usually found at areas with higher transcriptional activity, meaning these segments are active. These banding patterns can be used to identify chromosomal rearrangements and deletions.
ID036	Telomerase adds complementary RNA bases to the 3' end of the DNA strand. Once the 3' end of the lagging strand template is elongated, DNA polymerase adds the complementary nucleotides to the ends of the chromosomes.
ID036	Polytene chromosomes are  found in the drosophila larvae. They are highly replicated DNA sequence. They allow for  faster larval growthPolytene chromosomes are usually found in the larvae, where it is believed these many-replicated chromosomes allow for much faster larval growth than if the cells remained diploid. Simply because each cell now has many copies of each gene, it can transcribe at a much higher rate than with only two copies in diploid cell because each cell replicated copies of each gene. This allows transcription rate to increase.
ID036	DNA contains twoo variety if chromatin. There is a heterochromtin and a euchromotin. They are differed how they are stained . The euchromatin is less intense than heterochromatin which indicates that heterochromatin has tighter DNA packaging.
ID062	Linear chromosomes have an end problem. After DNA replication, each newly synthesized DNA strand is shorter at its 5' end than at the parental DNA strand's 5' end. This produces a 3' overhang at one end (and one end only) of each daughter DNA strand, such that the two daughter DNAs have their 3' overhangs at opposite ends
ID062	Thus, Drosophila telomeres resemble other telomeres because they are long arrays of repeated sequences, albeit more irregular arrays than those produced by telomerase. Drosophila telomeres have strong functional similarities to other telomeres and, as variants, provide an important model for understanding general principles of telomere function and evolution.
ID062	The lightly packed material is called euchromatin. Though it is lightly packed in the form of DNA, RNA, and protein, it is definitely rich in gene concentration and is usually under active transcription. heterochromatin is a tightly packed form of DNA. It is commonly found on the peripheral areas of the nucleus.
ID102	Telomeres aid in protecting the chromosome during replication process. However, some strands become shorter than the parent at strand 5'. This results in DNA being lost.
ID102	These specialized chromosomes that allow the rearrangements and deletions to be identified through the characteristics such as the light and dark band patterns shown on the chromosome.
ID102	Euchromatin is found in both eukaryotes and  prokaryotes, and has gene concentration. This chromosome aids in cell survival.
ID023	At the end of replication, one DNA strand is longer than the other. In order to fix this problem, the DNA strands have to be elongated. Telomeres contain repetitive sequences of DNA that do not code for anything in particular. By removing pieces of the telomere in order to elongate the DNA strand, important genes can be open to deletion.
ID023	Studies of Drosophila has allowed scientists to identify specific gene locations, their functions, specific alleles that correlate to those genes, and has also allowed scientists to understand transcription with a species that takes relatively short amounts of time to alter. These findings eventually led to determining gene locations in humans.
ID023	Euchromatin is true chromatin that codes for genes and contains genes that are turned on and read by mRNA. Heterochromatin refers to the parts of DNA that is highly condensed and contains genes that are turned off. Since these areas are so condensed, they do not proceed through transcription like euchromatin.
ID114	After DNA replication, each newly synthesized DNA strand is shorter at its 5' end than at the parental DNA strand's 5' end. This produces a 3' overhang at one end of each daughter DNA strand, such that the two daughter DNAs have their 3' overhangs at opposite ends.
ID114	Polytene chromosomes have light and dark banding patterns that can be used to identify chromosomal rearrangements and deletions. Dark banding corresponds to inactive chromatin, whereas light banding is found at areas with higher transcriptional activity.
ID114	The lightly packed material is called euchromatin. It is lightly packed in the form of DNA, RNA, and protein and is usually under active transcription.  Heterochromatin is only found in eukaryotes. When stained euchromatin resembles light-colored bands while heterochromatin is dark colored.
ID073	After replication of DNA, RNA primers are removed, creating gaps.  The gaps have free OH groups that initiate the synthesis of new DNA by polymerase and then they are sealed by ligaase. The gaps at the left end of the 5' do not have free OH groups. With each replication, the daughter DNA strand become shorter.  This is fixed by the enzyme telemerase.  Replication of DNA using RNA occurs and is called reverse transcriptase.
ID073	Specialized chromosomes in the drosophila that helped identify specific gene locations are called polytene chromosomes. Polytene chromosomes are specific types of interphase chromosomes consisting of thousands of DNA strands.  The chromosomes have a distinctive band pattern along the axis of the chromosome.
ID073	The DNA in the nucleus exists in two forms that reflect the level of activity of the cell. Heterochromatin appears as small, darkly staining parts scattered throughout the nucleus or accumulated adjacent to the nuclear envelope. Euchromatin is dispersed and not readily stainable. Euchromatin is prevalent in cells that are active in the transcription of many of their genes while heterochromatin is most abundant in cells that are less active or not active. There are more euchromatic parts of the chromosome than heterochromatic.  The presence of heterochromatin is characteristic of eukaryotes.
ID128	The problem with linear chromosomes is that the ends of the chromosomes resembles that double stranded breaks that can occur in DNA which can lead to fusion and translocation. The solution to this is telomeres which are long sequences at the end of the DNA strand. A problem with telomeres is that they become shorter after replication and the shortening becomes more severe with each generation. The solution to this is the enzyme telomerase. Telomerase is able to lengthen the telomeres which essentially eliminates the shortening of the telomeres.
ID128	The two types of specialized chromosomes discussed in the text are polytene chromosomes and lampbrush chromosomes. I didn't remember this from lecture so upon reading the text I have learned that the polytene chromosomes are giant chromosomes that appear in a pattern of bands and interbands. Study of these bands in Drosophila showed a characteristic pattern that is linked to gene activation. Lampbrush chromosomes are chromosomes that actually lengthen instead of condensing before returning to their normal length during meiosis. The chromosomes exhibit loops whose shape represents DNA that has been reeled out from a central chromomere axis during transcription.
ID128	While euchromatin uncoil during interphase to become active and therefore stain light, heterochromatin stay condensed which causes them to stain deeply. The inactivity in heterochromatic areas is caused by genes being repressed or even not present. Telomeres are a type of heterochromatic region of the chromosome. They aren't active but they are important to the maintenance of the structural integrity of the chromosome.
ID080	The linear chromosome causes problems because they can not be properly read due the transcription being used in double helix spiral chromosomes.
ID080	I don't know this topic well at all. I couldn't give a reason.
ID080	The euchromatic and heterochromatic can be used to identify the gene expression of an organism and how certain genes effect organisms in particular ways.
ID132	DNA polymerase can only synthesize a new strand of DNA as it moves along the template strand of 3' to 5' direction. This works well, but when using the 5' to 3' strand as the template has to be discontinuous.
ID132	Chromosomes called polytene chromosomes in Drosophila have been used to support the theory of genomic equivalence which tells us that all cells in the body maintain the same genome.
ID132	Euchromatin is composed of all the genome in the interphse nucleus except for heterochromatin. It is less tightly coiled and contains the active or potentially active genes. Heterchromatin are highly condensed and are late-replicating.
ID047	DNA polymerase can only add nucleotides to the 3' end. Because of this each round of replication the DNA molecules that are produced become shorter and shorter
ID047	Drosophila contain over-sized polytene chromosomes. These increase the volume of the cells' nuclei. Multiple copies of genes allows high levels of gene expression. This makes it easier to study drosophilas to identify gene locations.
ID047	Euchromatic regions usually contains high concentrations of transcribed genes while regions that lack high numbers of genes are typically compacted in the heterochromatin
ID072	Telomoeres,the ends of linear chromosomes, protect genes from getting deleted as cells continue to divide. This helps keep unwanted and problematic substances and genes from attaching to the chromosome where they shouldn't.
ID072	Due to the fact that Drosophila and human Chromosomes are so similar to each other; the specialized Drosophila chromosomes allowed for gene locations on the chromosomes. They were able to be then examined to help find the other genes on the human chromosomes.
ID072	Euchromatin is a lightly packed form of chromatin that is very high in the amount of gene concentration. The euchromatic is often under active transcription in the process. Heterochromatic which is more tightly condensed and transcriptionally inactive. Heterochromatin comes later in S phase.
ID149	I haven't gone to class this week due to four exams in all of my other classes, so I don't know what this is talking about.
ID149	I haven't gone to class this week due to four exams in all of my other classes, so I don't know what this is talking about.
ID149	I haven't gone to class this week due to four exams in all of my other classes, so I don't know what this is talking about.
ID119	When linear chromosomes contain telomeres at the end of the DNA strand, it creates a problem with 5' to 3' replication in eukaryotic cells.
ID119	Fruit fly chromosomes were first used to identify the location of genes.
ID119	Chromatin is found as either euchromatin or heterochromatin. The two types are distinguishable by their light or dark staining regions and also by transcription.
ID074	Every RNA primer synthesized during replication can be removed and replaced with DNA strands except the RNA primer at the 5' end of the newly synthesized strand. This small section of RNA can only be removed, not replaced with DNA. Enzymes RNase H and FEN1 remove RNA primers, but DNA Polymerase will add new DNA only if the DNA Polymerase has an existing strand 5' to it ("behind" it) to extend. However, there is no more DNA in the 5' direction after the final RNA primer, so DNA polymerase cannot replace the RNA with DNA. Therefore, both daughter DNA strands have an incomplete 5' strand with 3' overhang
ID074	Polytene chromosomes are over-sized chromosomes which have developed from standard chromosomes and are commonly found in the salivary glands of Drosophila melanogaster. Specialized cells undergo repeated rounds of DNA replication withoutcell division (endomitosis), to increase cell volume, forming a giant polytene chromosome. Polytene chromosomes form when multiple rounds of replication produce many sister chromatids that remain synapsed together.
ID074	Heterochromatin is a tightly packed form of DNA, which comes in multiple varieties. These varieties lie on a continuum between the two extremes of constitutive and facultative heterochromatin. Both play a role in the expression of genes. Constitutive heterochromatin can affect the genes near them (position-effect variegation). It is usually repetitive and forms structural functions such as centromeres or telomeres, in addition to acting as an attractor for other gene-expression or repression signals. Facultative heterochromatin is the result of genes that are silenced through a mechanism such as histone deacetylation or piRNA through RNAi.
ID138	When a chromosome is replicating and gets to the end. There is a space that it cannot be replicated the normal way in the lagging strand. It leaves an overhang. Telomerase assists in the formation of a new primer which then allows DNA polymerase to fill in the rest of the chromosome.
ID138	In Drosohpila they have polytene chromosomes. There are unusually binded chromosomes that can be viewed using light microscopy.These chromosomes show areas of gene activity in the puffs that can be seen in them.
ID138	Chromosomes that are stained show up having light and dark stripes essentially. The light areas of the chromosome are the active part that contain more chromosomes and the dark areas are the less active areas with relatively few chromosomes.
ID104	Once the replication site reaches the end of the chromosomes, there is no primer to facilitate the replication at the telomere. The telomerase forms around the end and finishes out the replication and adds a cap to the end of the chromosome
ID104	Since Drosophila have such a similar chromosome structure that we are able to relate certain genes to our own chromosomes.
ID104	Euchromatic regions on the chromosomes are considered the exons which code for the genes that we see. Heterochromatic regions are the introns that are just filler codes that don't code for any phenotypic genes. The heterochromatic regions are eventually left out when transcription occurs, taking the DNA sequence to the mRNA sequence.
ID126	After DNA replication from 5' to 3' in eukaryotes, each newly synthesized DNA strand is shorter at its 5' end than at the parental DNA strand's 5' end. This produces a 3' overhang at one end (and one end only) of each daughter DNA strand.
ID126	The drosophila gene of the poly chromosome allowed the identification of where genes are on the chromosome.
ID126	Euchromatin it is lightly packed in the form of DNA, RNA, and protein, it is definitely rich in gene concentration and is usually under active transcript. Heterochromatin are tightly packed DNA.
ID017	The problem with going 5' to 3' is that during DNA synthesis, the strand template has to be discontinuous.
ID017	On a chromosome map of drosophila flies, one can determine such as dumpy wings, curly wings, star eyes etc.
ID017	Both euchromatin, very gene rich, and heterochromatin play a huge role in the expression of genes.
ID144	Linear chromosomes have an end problem. After DNA replication, each newly synthesized DNA strand is shorter at its 5' end than at the parental DNA strand's 5' end. This produces a 3' overhang at one end (and one end only) of each daughter DNA strand, such that the two daughter DNAs have their 3' overhangs at opposite ends
ID144	They are able to find where the gene activity is taking place on the chromosome for specific genes using polytene chromosomes
ID144	The lightly packed material is called euchromatin. Though it is lightly packed in the form of DNA, RNA, and protein, it is definitely rich in gene concentration and is usually under active transcription.On the other hand, heterochromatin is a tightly packed form of DNA. It is commonly found on the peripheral areas of the nucleus.
ID139	Sometimes over time the telomeres have been shortened sdo many times that it can create a problem in DNA replication.
ID139	They are easily identified because these specialized chromosomes are oversized.
ID139	Euchromatin is very lightly packed and heterochromatin is very tightly packed. The combination of these two is what allows so much variation.
ID110	synthesis using the 5' –> 3' strand as the template has to be discontinuous. When the replication fork opens sufficiently, DNA polymerase can begin to synthesize a section of complementary strand working in the opposite direction. Later, a DNA ligase DNA ligase  stitches the fragments together.
ID110	our ability to test cross drosophila to tag and identify gene location is largely due to such advances in technology, but allows us to use a model organism to understand gene function more and more
ID110	the tighter the dna is packaged or wound up in heterochromatic chromosome regions is indicated by a darker staining region than that of its lightly stained unwound counterpart euchromatic region
ID120	When replicating the ends of the chromosome. The primer is needed, but when it comes off that spot is left unreplicated.
ID120	The drosophilias genes have been mapped and based on certain sequences the genes can be located.
ID120	The euchromatic regions are not tightly wrapped and they are able to be transcribed to form RNA which can result in proteins, while the heterochromatic regins cannot.
ID015	As DNA polymerase alone cannot replicate the ends of chromosomes, telomerase aids in their replication and prevents chromosome degradation.
ID015	many of the cells of certain fly larvae grow to an enormous size through multiple cycles of DNA synthesis without cell division. The resulting giant cells contain as much as several thousand times the normal DNA complement.
ID015	Facultative heterochromatin is the result of genes that are silenced through a mechanism such as histone deacetylation or piRNA through RNAi. It is not repetitive and shares the compact structure of constitutive heterochromatin. However, under specific developmental or environmental signaling cues, it can lose its condensed structure and become transcriptionally active
ID050	Promotors regulate transcription by being located upstream of the DNA strand, and provide an initial binding site for the RNA polymerase to bind and then go out to transcribe the DNA until it hits the stop codon.
ID050	Transcription differs in eukaryotes from prokaryotes because there are several more factors that eukaryotes have to meet in order to the RNA polymerase to even bind to the promotor region. In prokaryotes however the polymerase needs to be identified by the sigma factor and then it binds,
ID050	The genetic code has a 3 base codons that have been transcribed from DNA into MRNA. When this happens the MRNA which consists of these bases it is then translated into proteins because each amino acid is coded to a three base codon.
ID079	A promoter initiates the process of transcription for genes. Prokaryotes and Eukaryotes have different promoters involved in transcription. The promoter in Eukaryotes is the TATA box.
ID079	Transcription in prokaryotes takes place in the cytoplasm, while transcription in Eukaryotes takes place in the nucleus. Once again, the promoters are also different in prokaryotes and eukaryotes. The number of RNA polymerases also differs; eukaryotes have three and prokaryotes have only one.
ID079	The genetic code consists of linear triplet codes that convert nucleates to amino acids so that RNA can be translated to proteins. The genetic code is unambiguous, degenerate, and non overlapping.
ID065	The promoter is the sequence that is at the beginning of the coding sequence. If the promoted is activated, then the following sequence can be transcribed and ultimately translated, but if the promoter is not activated, then the sequence will not be coded and the gene will not be expressed.
ID065	Transcription in eukaryotes occurs with multiple promoters and genes being expressed at once, but in prokaryotes, there is only one origin of replication per DNA molecule. Also, only one replication bubble forms in prokaryotic replication.
ID065	mRNA is broken into codons, and then these codons are "read" by tRNAs that assign a specific amino acid to the specific codon. The Rosetta stone analogy means that this codon sequence can be used to entirely translate and form a functional protein from the mRNA sequence without any ambiguity or inconsistencies.
ID035	promoter binds to the promoter(Start of gene, tells which 2 be read +direction located 5’ region, upstream from initial trans) in the DNA
ID035	Due to the differences of bacteria and eukaryotes, different steps must be taken when dna is made into rna
ID035	The change between mRNA to proteins (through translation) requires important code in order to make a complementary strand
ID015	a promoter is a region of DNA that initiates transcription of a particular gene. Promoters are located near the transcription start sites of genes, on the same strand and upstream on the DNA
ID015	Transcription is achieved by three different types of RNA polymerase (RNA pol I-III). These polymerases differ in the number and type of subunits they contain, as well as the class of RNAs they transcribeIn bacteria, all transcription is performed by a single type of RNA polymerase. This polymerase contains four catalytic subunits and a single regulatory subunit.
ID015	The genetic code is the set of rules by which information encoded within genetic material (DNA or mRNA sequences) is translated into proteins by living cells
ID027	The promotor must bind to the DNA before transcription can occur. It must open the strand and allow other proteins to attach. This makes it the simplest way for evolution to limit the amount of transcription occurring in the cell or to increase the amount as needed.
ID027	Transcription in prokaryotes is different from eukaryotes because prokaryotes have a simple ring of DNA that is not intertwined with proteins. The polymerases and a few other proteins must simply attach to the chain and reproduce it. There is also significantly less DNA in a prokaryote than a eukaryote. The opposite is true for eukaryotes. They have histone proteins and a large genome.
ID027	The genetic code is the same for almost all organisms. The mRNA is translated to proteins through the three bases read at the ribosome. This process requires tRNA to carry the amino acids and the ribosome to attach them together.
ID024	The promoter region is very important in the process of transcription. The TATA and CAT boxes are what DNA polymerase bind to. For both prokaryotes and eukaaryotes, the promotor region is very important. Without it, transcription would never be able to happen.
ID024	Prokaryotes do not have introns, while eukaryotes have introns. This means that there is more gene regulation in eukaryotes than prokaryotes. Both prokaryotes and eukaryotes have promoter regions, consisting of the TATA and CAT box, but in eukaryotes, DNA polymerase must have extra help to bind to the promoter region. Prokaryotic DNA polymerase can simply attach and begin the process of transcription. Eukaryotic DNA must undergo chromatin remodeling, while prokaryotes do not have to.
ID024	This is a major part of transcription and translation. The genetic code is what tells the ribosome which amino acid to put in the protein chain. Every three bases translates into an amino acid. For example, AUG is the three base sequence that codes for Methionine, which is the very important start codon. Without this codon, the process of translation does not ever occur, and if it does not happen, proteins are never made.
ID094	Promoters have varying strengths and control expression of any genes 'downstream' of them. They recurit transcription factors and are needed to begin transcription.
ID094	Transcription in eukaryotes is much more complex than in prokaryotes, and includes elements such as methionine caps and the kozak sequence that are not seen in prokaryotes.
ID094	Groups of 3 bases are called codons. Each codon recruits a tRNA to the ribosome and builds the protien of interest via a polypeptide chain. The mRNA gets this info from the genome after transcription.
ID068	The promoter region signals the RNA polymerase for a transcription to take place. This area contains two areas which help the RNA polymerase to bind, the CAT box and TATA box. These sequences occur -35 bases and -10 bases from the beginning base of transcription.
ID068	There are multiple ways prokaryotes and eukaryotes differ in their transcription processes. In Eukaryotes, the process takes place in the nucleus(prokaryotes do not have a nucleus) and then leaves the nucleus to be translated. Eukaryotes also have nucleosomes that prokaryotes do not have that must be removed before transcription takes place.
ID068	The reading frame of the genetic code 3 bases at a time is called a codon. Each codon codes for a specific amino acid. Multiple different codons can code for the same amino acid. However some codons, one base change can convert it to a completely different amino acid. There are 20 different amino acids coded for by the 64 different codon combinations of RNA using the bases A, U, G, and C.
ID131	Transcription is the coding of mRNA from the DNA and this only occurs on exon portions to give codons that code for the formation of amino acids. The promoter is the portion on the gene that begins transcription.
ID131	Transcription differs in various ways in both prokaryotes and eukaryotes. There are differences in the structure of the chromosome since prokaryotes is a single stranded structure and there are no intron regions.
ID131	Genetic code determines which amino acids are formed from the bases that are coded for on mRNA by DNA that is passed from one generation to the next.
ID064	The promoter region is an area of the DNA where the polymerase will bind to begin transcription, therefore if something is wrong with this area, transcription will be affected.
ID064	Transcription in eukaryotes is a simpler process that is generally the same as in eukaryotes, except eukaryotes have extra steps afterwards since the mRNA must be moved out of the nucleus to the ribosomes for translation to start. Prokaryotes do not have this problem, and transcription and translation can occur simultaneously. Prokaryotes have a TATA box, CAT box, and Shine-Delgarno sequence, and transcription starts directly after the promoter region, while eukaryotes have dozens of proteins involved as enhancers and silencers, and require transcription factors as well as RNA Pol1,2, and 3. Eukaryotes have a CAAT box and the Goldberg-Hogness box instead of a TATA box. Also there are many post-transcriptional events that happen in eukaryotic cells.
ID064	The genetic code is the relation between three base pairs (a codon) and a corresponding amino acid. The code allows tRNA to place the correct amino acid in the growing polypeptide chain in the right place and therefore allow mRNA to transfer its information into the form of a protein.
ID109	The promoter is a region of DNA that initiates gene transcription. Promoters are located at the transcription start sites of genes and on the same strand and also on the 5' end of DNA. The promoter
ID109	Transcription in eukaryotes is different from transcription in prokaryotes because transcription in eukaryotes is much more complicated. RNA processing is also extremely different. In prokaryotes, RNA is synthesized by one polymerase. Eukaryotes contain three types of RNA polymerase. Also,
ID109	The genetic code is essentially a set of rules that encodes for proteins. This code defines how sequences of these three nucleotides encodes for what amino acid will be added next during protein synthesis. A codon (three bases) in a nucleic acid sequence specifies a single amino acid.
ID103	There is a sequence of base pairs before a gene that determines if RNA polymerase will bind and copy that gene into a segment of mRNA.
ID103	The promoter sequence is in a slightly different location in eukaryotes vs prokaryotes and the mechanism of initiation is different. Eukaryote mRNA also undergoes additional modification before it leaves the nucleus such as the addition of a 5' cap and a 3' poly A tail.
ID103	The genetic code gives the Amino acid associated with each 3 base pair codon. It translates from 3 base pairs segments of nucleic acid to amino acids in a protein.
ID020	Transcription results in the synthesis of a single-stranded RNA molecule complementary to a region along only one of the two strands of the DNA double helix. The site of the initial binding is established when the RNA polymerase alpha subunit recognizes specific DNA sequences called promoters. The promoters are located in the 5' region, upstream from the point of initial transcription of a gene. The promoter initiates transcription.
ID020	In summary, transcription is different in Eukaryotes and Prokaryotes and one of the big difference is that in eukaryotes transcription occurs in the nucleus. Mostly Transcription has been derived from studies of prokaryotes, but there are some main differences between the two.
ID020	The genetic code was assumed to be universal which applied equally to viruses, bacteria, archaea, and eukaryotes. The genetic code is triplet in nature. mRNA codes translates to amino acids in pairs of threes which get encoded into protein from the genetic code.
ID014	The promoter determines where transcription will begin by RNA polymerase. Without the presence of a promoter, RNA polymerase will not know where to attach on the DNA strands to being replication.
ID014	In prokaryotes, the DNA does not need a promoter region to begin, and transcription and transcription essentially happen at the same time in some incidences. In eukaryotes, transcription takes place in the nucleus with the help of RNA polymerase intiated by a promoter. Transcription also takes place during a certain time(G1 and G2) phase. Also, transcription and translation do not occur at the same time.
ID014	Three base pair encodes for different codes which translate into different proteins. Typically the codons that are translated into amino can be changed simply by a base change. The central base is crucial to determining the amino acid produced.The amino acids can also be changed depending on the reading frame.
ID039	In order for transcription to occur a promoter has to be present. Thus being present or absent allows the promoter to regulate the process
ID039	Transcription is different in prokaryotes and eukaryotes. It occurs in different locations and have different factor.
ID039	Genetic code is the rosetta stone because it serves as the template of understanding.  It allows mRna and proteins to be able to read and translated between each other because the genetic code tell each this means this or this sequence represents that.
ID082	A promoter  is a sequence of nucleotide bases in the genome that serve as a signal and binding site for RNA polymerase. This is important because without these promoter regions, RNA polymerase will not be able to recognize the gene it is supposed to transcribe, and the gene will not be transcribed. The promoter sequence is commonly called the TATA box, and this region is upstream of the gene to be transcribed itself. Mutations in the promoter sequences result in the inability of RNA polymerase to function, therefore the gene is non-functional because it cannot enter the “central dogma” flow to become a polypeptide.
ID082	There are 5 major ways that transcription differs in eukaryotes and prokaryotes. The first difference is that, unlike prokaryotes that have a circular genome, eukaryotes have linear DNA. This creates an issue at the end of the strand that is solved via the mechanisms of telomerases. The second difference is that eukaryotic transcription is much slower (25 x), but this slow speed is made up for by multiple origins of replication. The third difference is that eukaryotes have nucleosomes, which are structures of histone proteins that clump together and compact the DNA in the strand (also turn off genes in clumped region). The fourth difference is that eukaryotes use two different polymerases to copy the leading and lagging strands (versus only one polymerase in prokaryotes). The fifth difference is found during initiation of transcription in that in eukaryotes, there is primer RNA that is then converted in DNA and in prokaryotes there is just an RNA primer.
ID082	The nucleotides bases that make up the mRNA are read via open reading frames by the ribosome in the cytoplasm. Thanks to tRNA, amino acid residues are added to a growing polypeptide chain one at a time as the tRNA delivers the specific amino acid according to the anticodon sequence on the opposite end of the tRNA. The three base groupings are called codons, and every combination of the four nucleotide bases codes for an amino acid, a start codon (AUG), or one of the three stop codons (UGA, UAA, UGG).
ID013	Transcription is the process of converting the DNA code to mRNA, tRNA, or rRNA. mRNA is used to then translate the code into an amino acid sequence, or polypeptide. To initiate transcription in prokaryotes, there is a TATA box upstream along with a CAT box. RNA Polymerase and the sigma factor bind to this TATA box to initiate transcription. In eukaryotes, transcription begins when the TBP, or TATA Binding Protein, and the Transcription factor II-D bind to the TATA box and form a complex for RNA Polymerase to come in. These promoters are upstream from the start codon, so they copy more information than is needed, which helps maintain the entire gene.
ID013	In eukaryotes, transcription begins by TBP and TF II-D binding to the TATA box while prokaryotes bind RNA Polymerase and sigma factor to the TATA box for initiation. Eukaryotic transcription occurs in the nucleus instead of the cytoplasm, so it is matured by capping the 5 prime end with 7-methylguanosine, splicing the noncoding introns out, and adding a poly-A tail to the other end of the mRNA. Prokaryotic transcription occurs in the cytoplasm, so it immediately binds to a ribosome to start translation as it is being transcribed.
ID013	The genetic code includes 64 codons that code for the 20 different amino acids. The genetic code is linear, commaless, ordered, nearly universal, and degenerate in that several codons can code for one amino acid. The codons of the genetic code match with an anticodon on tRNA that carries a particular amino acid during translation, which is how the bases attract a specific amino acid.
ID070	Promoters are regions of DNA that promote transcription, transcription begins with the binding of RNA polymerase to the promoter in DNA.
ID070	After the first bond is synthesized, the RNA polymerase must clear the promoter. During this time there is a tendency to release the RNA transcript and produce truncated transcripts. This is called abortive initiation and is common for both eukaryotes and prokaryotes.
ID070	A codon is defined by the initial nucleotide from which translation starts. For example, the string GGGAAACCC, if read from the first position, contains the codons GGG, AAA, and CCC; and, if read from the second position, it contains the codons GGA and AAC; if read starting from the third position, GAA and ACC. Every sequence can, thus, be read in three reading frames, each of which will produce a different amino acid sequence . With double-stranded DNA, there are six possible reading frames, three in the forward orientation on one strand and three reverse on the opposite strand. The actual frame in which a protein sequence is translated is defined by a start codon, usually the first AUG codon in the mRNA sequence.
ID151	A promoter sequence allows RNA polymerase to bind to it and bring along other factors. In prokaryotes, this action is a little different since it requires a holoenzyme (composed of 2 alpha, a beta, beta prime, omega, and a specific sigma subunit). The sigma subunit is a specific subunit that varies from sequence to sequence, but all the other subunits listed compose the core enzyme which can continue adding RNA nucleotides or ribonucleotides once the sigma subunit has facilitated binding to the promoter region and has added a few bases before it leaves. I'm eukaryotes, different RNA Polymerases have different jobs or functions associated with them. RNA Polymerase 2 which is used to create mRNA requires Transcription Factors such as TFIID and TBD (TATA binding proteins) to first attach to the promoter region. Once the RNA Polymerase II is attached to the pre-initiation complex, transcription can begin once the tail of Polymerase has been phosphorylated by the TFIID protein. In addition, there are suppressors and enhancers that can also regulate the transcription process by determining if there is a change in environment via signals that may require an increase of a decrease in certain proteins ( therefore, more or less mRNA transcripts to allow for the increase or decrease in protein requirement).
ID151	Eukaryotic transcription varies highly from prokaryotic transcription in specific and fundamental ways. For instance, transcription occurs in the nucleus in a eukaryotic cell, unlike prokaryotes, before it can be transmitted out the nuclear pore into the cytoplasm for translation. Therefore, transcription and translation is not simultaneous within an Eykaryotic cell. Furthermore since eukaryotic DNA has his tone proteins associated with it to create nucleosomes ("beads on a string complex"), the histone proteins must be removed temporarily which also requires other specific mechanisms. In addition, the initiation of transcription varies between eukaryotes and prokaryotes, wherein, eukaryotes require TFIID, TBP, and RNA Polymerase II whereas, prokaryotes require a holoenzyme formation at the promoter.
ID151	This statement is a synopsis of the central dogma which claims that a DNA strand is used as a template to create RNA (mRNA, tRNA, and rRNA) in which each play a role in the process of protein formation. These two steps discusses are transcription and translation, respectively. The DNA which is known as the genetic code, is known to be read in codons or 3 base pairs at a time and is non-overlapping, degenerate, and unambiguous. The fact that the genetic code is  non-overlapping implies that during transcription, separate 3 bases or codons are continuous read without "commas". The fact that the genetic code is degenerate also signifies that multiple codons can code for the same amino acid, but more than one amino acid cannot code for one codon; furthermore, since it only codes for a specific amino acid, there is an indication that the genetic code is not ambiguous and is highly specific.
ID098	The promoter is a region upstream from the actual gene that signals to the polymerases and transcription factors where to start transcription.
ID098	Because of evolutionary divergences that resulted in structural differences in the DNA itself, some elements of transcription are quite different in eukaryotes in comparison to prokaryotes. Transcription occurs in the nucleus, away from all of the ribosomes in eukaryotes. DNA must be uncoiled from nucleosomes in order for transcription to occur in eukaryotes. Gene regulation is more complex in eukaryotes, with promoters, enhancers, silencers, transcription factors, and other proteins play regulatory roles. Because the mRNA must travel outside of the nucleus for translation to occur in eukaryotes, it must undergo some processing in order to safely leave the nucleus without degrading.
ID098	The genetic code is the great translator that converts nucleotide-sequenced information into amnio acid-sequenced information. More succinctly, this means that it translates mRNA structures into protein structures. The genetic code is written linearly, with “words” composed of mRNA triplets (or codons) that match up with corresponding amino acids.
ID030	The promoter region is a portion of DNA that lies upstream of genes. Promoter's are essential for gene activation and regulation. They bind, either directly or with the aid of additional proteins RNA polymermase to the DNA strand. Cell can block promoters to regular the transcription of a particular gene.
ID030	Transcription, while serving similar roles in both types of cell, undergoes a vastly different process in each. In general, transcription in eukaryotes is much more complicated than it is in prokaryotes. The first major difference is that, in prokaryotes, RNA is transcribed and translated simultaneously. Additionally, RNA is transcribed during certain phase of the cell cycle in eukaryotes. Transcription can occur at any time during prokayotes.
ID030	This is one of the most important concepts when trying to understand how proteins are created. The sequence of 3 bases, known as a codon, bond with a corresponding anticoding. This anticodon delivers the correct amino acid. This process repeats and an amino acid chain is formed.
ID105	The promoter area is recognized by the sigma factor. The TATA-like region (Pribnow box) is also the first place where base pairs separate during prokaryotic transcription to allow access to the template strand.
ID105	In eukaryotes transcription occurs within the nucleus unlike in prokaryotes. It must leave the nucleus in order to undergo translation. Eukaryotes also do not need a promoter in order to initiate transcription. Where as prokaryotes do.
ID105	RNA is translated to protein. Some mechanism is necessary to change language from nucleotides to amino acids. That is why the genetic code is referred to as the Rosetta stone it is the 3 bases that translate to the "words" for the amino acids.
ID077	A promoter is a region of DNA that imitates transcription of a particular gene. This region is where the RNA polymerase binds proper to the invitation of trancription.
ID077	Transcription differs in eukaryotes because it occurs in the nucleus and in prokaryotes it occurs in the cytoplasm. A single RNA polymerase synthesizes the 3 types of RNA in prokaryotes and RNA polymerase I, II, III synthesizes rRNA, mRNA, and tRNA respectively. In prokaryotes imitation of transcription doesn't need any proteins or initiation factors but in eukaryotes the invitation of transcription requires proteins called transcription factors. In prokaryotes, transcription and translation happen nearly simultaneously with little process of mRNA. In eukaryotes, processing of hnRNA includes addition of 5' cap and addition of a 3' poly A tail.
ID077	The genetic code is the set of rules by which information encoded within genetic material is translated into proteins by living cells. Each codon consists of 3 nucleotides which corresponds to a single amino acid to make proteins.
ID046	The promoter region is upstream of the gene to be transcribed and initiates this process of transcription.
ID046	These differences, such as the location of the process, the speed of the process, the factors needed, and and lifetime of the mRNA are all crucial differences. these differences are important in understanding the processes.
ID046	The genetic code is what allows us to be us. This central dogma of DNA to RNA to proteins is crucial, but without the understanding that a simple three nitrogenous bases can code for an amino acid to make a protein is important.
ID037	a promoter is a region of DNA that initiates transcription of a particular gene. Promoters are located near the transcription start sites of genes, on the same strand and upstream on the DNA (towards the 5' region of the sense strand).
ID037	In bacteria, all transcription is performed by a single type of RNA polymerase. This polymerase contains four catalytic subunits and a single regulatory subunit known as sigma. Eukaryotic cells are more complex than bacteria in many ways, including in terms of transcription. Specifically, in eukaryotes, transcription is achieved by three different types of RNA polymerase
ID037	During translation, RNA is matched 3 bases at a time to protein.
ID026	The promoter is the site where enzymes are told to begin the transcription process. If there was not a promoter, the enzymes would not know where to begin transcription and further processes would not be able to happen as well.
ID026	There are many differences between how transcription happens in eukaryotes and prokaryotes. In eukaryotes, it happens in the nucleus, requires transcription factors, and uses RNA polymerases I, II, and III. In prokaryotes, it happens in the cytoplasm, does not require transcription factors, and only uses one type of RNA polymerase.
ID026	An organism's genetic code is the template for proteins to be made. Every 3 bases is called a codon and is used to make 1 amino acid. These amino acids are then linked together to create proteins. This process is called translation.
ID141	Promoters are sequences of DNA upstream from the point of initial transcription that the RNA polymerase sigma subunit can recognize.  This promoter area is recognized due to the presence of a sequence known as the TATA box.
ID141	Transcription of prokaryotes occurs in the cytoplasm whereas transcription in eukaryotes occurs in the nucleus.  Since transcription occurs in the nucleus in eukaryotes, the mRNA must be transported out of the nucleus so it can find ribosomes in order to be translated.  In Eukaryotes, the chromatin must uncoil so that RNA polymerase can access the DNA.
ID141	The start codon of this rosetta stone is AUG, and is nearly universal for all of life.  The stop codons are UGA, UAG, and UAA, and are nearly universal for all of life.  There are 64 possible arrangements that the 4 different nucleotides can arrange in in groups of 3.  However, there are only 20 amino acids that these 64 arrangements can code for, therefore more that one arrangement can code for the same amino acid.
ID028	Transcription cannot begin unless the RNA polymerase has attached to the promoter region and all other initiation factors have been met.
ID028	Transcription in Eukaryotes and
ID028	The central dogma of heredity goes from
ID089	Transcription is regulated by the promoter in that transcription factors, in the case of eukaryotes, and the sigma factor in the case of prokaryotes must be present to assist in the binding of RNA polymerase to the DNA strand. The promoter is a cis acting factor, as it is part of the DNA itself.
ID089	Transcription in eukaryotes is located in the nucleus, consists of transcription factors, chromatin remodeling, enhancers and silencers influence the rate at which transcription occurs, and RNA processing after transcription is unique to eukaryotes.
ID089	DNA is transcribed into RNA which is then read three bases at a time, known as a codon, by a ribosome, and each codon calls for a single amino acid addition to the growing peptide chain. Experiments confirmed that the codons were three base pairs in length by showing that three deletions or three additions were required to restore initial reading frame.
ID101	The promoter region controls how the RNA polymerase attaches to DNA and if it can access the DNA to begin transcription for the production of mRNA.
ID101	The transcription in Eukaryotes occurs in the nucleus, it requires initiation factors to recognize the tata box. Process of mRNA has polyA tail and cap. In contrast, the Prokaryotes do not require initiation factors, the transcription occurs in the cytoplasm and is fed directly to the ribosomes to be made into proteins before finishing.
ID101	The mRNA transcribed from DNA is paired up with an amino acid by three base pairs. Although multiple base pairs can produce the same amino acid, the base pairs are still specific. This same sequence pairing is found in most organisms.
ID083	The central dogma states that genetic information is stored in DNA, copied to messenger RNA during 'transcription' and converted to a protein during 'translation'. Transcriptions starts in the promoter region and proceeds past the gene start and stop codons until RNA polymerase is removed from the DNA strand.
ID083	The central dogma states that genetic information is stored in DNA, copied to messenger RNA during 'transcription' and converted to a protein during 'translation'. Transcriptions starts in the promoter region and proceeds past the gene start and stop codons until RNA polymerase is removed from the DNA strand.
ID083	The central dogma states that genetic information is stored in DNA, copied to messenger RNA during 'transcription' and converted to a protein during 'translation'. Transcriptions starts in the promoter region and proceeds past the gene start and stop codons until RNA polymerase is removed from the DNA strand.
ID069	Promoters are regions of DNA that initiate the transcription of a particular gene. They are located upstream of the transcription start sites and can be a few hundred to a few thousand base pairs long. RNA polymerase must bind to the promoter before it can begin transcribing the DNA into mRNA.
ID069	Unlike prokaryotes, transcription occurs within the nucleus, which means mRNA must leave the nucleus in order for it to be translated. Additionally, eukaryotic DNA is arranged into chromatin, which must uncoil and move nucleosomes in order for DNA to be accessible to RNA polymerase. Also, eukaryotic RNA polymerases utilize transcription factors to scan and bind DNA.
ID069	A mechanism is necessary for translation that changes the language from nucleotides to amino acids; the genetic code functions as this "Rosetta stone." Three ribonucleotide letters (a triplet codon) of mRNA codes for an amino acid, or start or stop signal, which initiate or terminate translation. The genetic code is nearly universal, in that its coding dictionary is used by both eukaryotes, prokaryotes, archaea, and viruses.
ID053	Transcription of DNA by RNA polymerase is accomplished with the help of a promoter region in the DNA.  This promoter region, which differs between prokaryotes and eukaryotes, usually consists of a TATA box in addition to other promoters.  These promoters allow RNA polymerase to find the correct spot on DNA to start reading.
ID053	Transcription in eukaryotes, as is the trend, is more complicated than transcription in prokaryotes. Chiefly, it is the differences in DNA between the domains that causes these fundamental differences in transcription.  For instance, eukaryotic RNA is transcribed within the nucleus and is then sent through the nuclear membrane, while prokaryotes have no nucleus.  Also, eukaryotic DNA is organized into chromatin which must be uncoiled for RNA polymerase to read it—prokaryotes don't have this problem.  Eukaryotic transcription requires certain transcription factors, enhancers, and silencers, while prokaryotes have none of these.
ID053	The genetic code is read in sets of three bases at a time.  These sets are termed codons, and these codons are used during translation to signal the addition of amino acids.  Each codon codes for one amino acid, but the code is degenerate because a single amino acid may be coded for by different codons.  The anticodons of tRNA "read" the codons of mRNA in the ribosome by bonding to the complementary codons.  This produces a collinear chain of amino acids.
ID061	A promoter is essential for transcription to be able to take place. In prokaryotes, there are specific DNA sequences called prommoters which the RNA polymerase sigma subunit recognizes and binds to in order to begin transcription. These are known as consensus sequences and are repeating. In eukaryotes, there are multiple promoters needed to begin transcription. The first, the core promoter, determines where RNAP II binds to the DNA and begins transcription. There are also regions called the proximal-promoter elements, enhancers, and silencers that influence the efficiency or the rate of transcription initiation.
ID061	In eukaryotes, transcription is far more complex than in prokaryotes. Eukaryotes have a lot more genetic information that must be transcribed into mRNA in the transcription process. In eukaryotes it occurs in the nucleus, while in prokaryotes it is in the cytoplasm. In addition, transcription in eukaryotes cannot take place until the densely coiled chromatin undergoes chromatin remodeling and is accessible by RNA polymerase. There is also more regulation in eukaryotes, as there is more interaction between cis-acting DNA sequences and trans-acting protein factors; and there are silencers and enhancers that also regulate transcription speeds. Finally, RNA processing takes place in eukaryotes where the RNA transcript undergoes changes before it is ready to be translated.
ID061	The mRNA sequence consists of triplet sequences called codons that each code for a specific amino acid. The genetic code is degenerate, commaless, nonoverlapping, and unambiguous. Each codon translates to one amino acid which join to form proteins. The genetic code is universal, with few exceptions, among species. In addition, the code contains one start codon and three stop codons, with the stop codons not actually coding for a specific amino acid but instead the end of a sequence.
ID056	The promoter is the binding site for RNA polymerase that signals the starting area of transcription directly after. The promoter consists of a "TATA boz" which is made of several T and A bases.
ID056	There are several different ways that transcription varies from eukaryotes to prokaryotes. These differences arise from the different shapes of chromosomes (prokaryotes are spherical and eukaryotes are linear) and the different locations of processes.
ID056	3 bases are read off of the mRNA and are called codons. They have a set reading frame that when altered can change the amino acid in which they code for. This can therefore change the structure and function of the resultant protein.
ID075	The promoter is an extremely important factor in transcription of prokaryotes and eukaryotes.It is needed in order to initiate transcription. In prokaryotes, The promoter area is recognized by the sigma factor and then the RNA Polymerase binds to it. In eukaryotes, the TATA box acts as the promoter and regulates transcription.
ID075	Transcription in eukaryotes is much more complex than transcription in prokaryotes. It occurs within the nucleus. The mRNA must leave the nucleus for translation, and Transcription Factors are needed by the RNA Polymerase.
ID075	The genetic code translates three bases at a time also referred to as codons. It is written in linear form using ribonucleotide bases
ID058	The promoter region where transcription begins when it takes place. The various proteins located in the promoter region help in regulation by determining if translation will or will not take place.
ID058	Transcription in prokaryotes is much simpler than in eukaryotes. It requires fewer types of proteins and fewer of each type. Furthermore, most prokaryote DNA is circular so transcription just has to take place in two directions. In eukaryotes, transcription takes place at many places along the DNA because it is linear and extremely long.
ID058	The genetic code is the template that allows the Central Dogma to take place. DNA can be transcribed into mRNA and then translated into proteins with the help of ribosomes and tRNA. Every 3 bases of RNA codes for a specific amino acid that then forms a chain to create a protein.
ID087	Transcription is regulated by the promoter. Without the promoter, genes will not be transcribed when necessary, or they would not be transcribed at all.
ID087	Transcription in eukaryotes is much slower, they contain multiple origin of replication sites, and they contain telomeres.
ID087	The genetic code is read 3 bases at a time. Each codon correlates to a certain amino acid. While these are read by the ribosome it ensures that the right amino acid is added at the right time.
ID140	Transcription cannot begin without the promoter.
ID140	Because eukaryotes and prokaryotes are different on a very fundamental level, transcription varies significantly between the two.
ID140	Genetic code is compared to the Rosetta stone because it was a crucial piece of evidence found for language. Likewise, genetic code is very important for reading DNA.
ID150	The promoter is a sequence of DNA upstream of the transcription start site that tells RNA polymerase to transcribe the gene as well as provide a place for the enzyme to bind.
ID150	Prokaryotes often undergo transcription and translation at the same time whereas this does not happen in eukaryotes. The location of transcription and the enzymes involved also differ. Eukaryotes require transcription factors and multiple RNA polymerases and also process mRNA after transcription.
ID150	The genetic code is the set of rules that associates 3 base pair sets called codons into amino acids to be added to a growing polypeptide chain. This connection is made by the tRNA molecule in the ribosome.
ID055	In bacteria the promoter is recognized by a sigma factor and RNA polymerase. In eukaryotes the promoter being recognized is more complex in that there are seven different factors necessary for binding to the RNA polymerase II.
ID055	Transcription is more complex in eukaryotes than in prokaryotes. In eukaryotes transcription takes place in the membrane-bound nucleus and then translation takes place in the cytoplasm. For prokaryotes, transcription and translation take place in the same compartment. For eukaryotes they have three different types of RNA polymerase that aid in transcription where as prokaryotes have one.
ID055	3 bases code for a one amino acid and then the amino acids code for a protein. Each amino acid is coded from a particular 3 base codon.
ID118	In bacteria, initial step for transcription is Template binding and the site of the initial binding is established when RNA polymerase sigma subynit recognizes specific DNA sequences called PROMOTER.This sequence is located in 5’region, upstream from the point of initial transcription of gene. The point where transcription starts is called transcription start site. Two consensus sequences of DNA in bacterial promoter is Pribnow box (TATAAT) and TTGACA. In bacteria, its sigma subunit that recognizes promoter on the RNA polymerase.in eukaryotes it is recognized by general transcription factors are TBP and TFIID, they bring the RNA polymerase II and transcription activators and repressors
ID118	In prokaryotes the promoter area is recognized by sigma factors they bring RNA polymerase, in eukaryotes it is recognized by general transcription factors, they bring the RNA polymerase II, it is much larger than the prokaryotes. RNA Polymerase and transcription activators and repressors.
ID118	The genetic code is non overlapping, collinear, universal, degenerative, unambiguous, and its coded by codon on the mRNA. Codon is made up three code in mRNA. The genetic code and triplet-binding assay, they used mixed copolymers, repeating copolymers, and triplet binding. The genetic code was discovered by crick that it was made up of 3 bases as it restore the reading frame when deletion or duplication of 3 bases.
ID008	transcription takes place when the enzyme that synthesizes RNA, RNA polymerase, attaches to the DNA near a gene. Promoters have specific DNA sequences that provide a initial binding site for RNA polymerase and for proteins called transcription factors that recruit RNA polymerase.
ID008	In eukaryotes, transcription is done by three different types of RNA polymerase, which, is RNA polymerase I-III. In turn, bacteria is performed by a single type of RNA polymerase that has four catalytic subunits and a sigma factor.
ID008	The codons are read from a 5' to 3' direction but mRNA is synthesized using the 3'-5' direction strand as a template.
ID104	for transcription to occur the dna needs a promoter in the form of a TATA box where the amino acids TATA provide the necessary location for the RNA polymerase to come in and start coding. Without the promoter then transcription will not occur.
ID104	In Eukaryotes, transcription doesn't need a promoter to begin the process. Also, since prokaryotes don't have a nucleus then the transcription takes place in the cytoplasm, whereas with eukaryotes, the transcription takes place in the nucleolus and then the RNA leaves into the cytoplasm where it can be translated.
ID104	Without the genetic code we would have never been able to figure out why certain mutations were possible and happening around us.
ID092	In regards to transcription, a promoter is a region of DNA that initiates transcription to begin.  Promoters have a specific sequence that allows the proper RNA polymerase to attach to DNA and begin transcription.  This way, transcription regulated by the promoter because it tells when and where the process of transcription can begin.
ID092	Transcription varies many ways in eukaryotes than prokaryotes.  First in eukaryotes, transcription takes place in the nucleus of the cell.  Also, eukaryotes process RNA much more extensively.
ID092	The code that is translated from mRNA and proteins is part of the key to life.  mRNA reads the base long codes that code for different amino acids.  These amino acids together then code for proteins which help keep life running.
ID081	Transcription is regulated by a promoter or start codon AUG. At this point the RNA polymerase binds to the DNA to produce mRNA.
ID081	In eukaryotes, transcription takes place in the nucleus and then is processed and moved to the cytoplasm where translation occurs. While in prokaryotes both transcription and translation occurs in the cytoplasm.
ID081	The genetic code comes from information stored in the DNA called genes. This DNA is then transcribed into mRNA. This occurs at three bases at a time. The mRNA then goes through the ribosomes and binds with tRNA to form amino acids, thus forming proteins.
ID095	The promoter region consists of several parts on the DNA molecule.  The RNA polymerase attaches to the DNA, and these regions vary between eukaryotes and prokaryotes.
ID095	Prokaryotes have sigma factors in the RNA polymerase, and these sigma factors recognize the promoter region to begin transcription. Eukaryotic transcription is more complex. Eukaryotes have three different polymerases: 1,2,3. Polymerase one transcribes ribosomes, polymerase two transcribes messenger RNAs, and polymerase three transcribes transfer RNAs.
ID095	The 3 base coding represents which amino acids become linked in the peptide chain to form proteins.  As the ribosome reads the mRNA, amino acids are brought in with tRNAs and these are amino acids become matched to specific series of base pairs, or a "code", to form particular proteins.
ID063	RNA is transcribed onto a DNA. This is directed by RNA polymerase. For prokaryotes, transcription is regulated by a promoter that binds and is recognized by the sigma factor.
ID063	In eukaryotic transcription, it occurs in the nucleus. it also takes place in the G1 and G2 phases, RNAs are released in the nucleus, and is inhibited by RNA polymerase 2. In prokaryotic transcription, it occurs in the cytoplasm. There is no definite phase, RNAs are released in the cytoplasm, and it is inhibited by rifampin.
ID063	The genetic code is the basis for what we use to determine gene sequences. Each set of 3 bases translate into a codon that combine into a protein sequence.
ID052	Transcription is the first step in the central dogma of biology. This process is regulated by many factors, one of which include the promoter region.The promoter region consists of a sequence of specific nucleotides that help certain transcription factors bind RNA polymerase to the strand to be copied into mRNA. Eukaryotic and prokaryotic organisms have differences in how transcription works, which includes different promoters.
ID052	Unlike prokaryotic transcription and translation both occurring in the cytosol, eukaryotic transcription occurs in the nucleus then travels to the cytosol for translation. In prokaryotes, the promoter region allows the attachment of the holoenzyme. The holoenzyme is made up of a core enzyme and a gene-specific sigma subunit. This RNA polymerase is able to run until either a hairpin is formed or a rho protein causes the termination of transcription. In eukaryotes, the promoter region is also called the TATA box. The TATA-binding protein aids transcription factors bind RNA polymerase to the TATA box, thus initiating transcription. However, before the eukaryotic DNA can start transcription, it undergoes chromatin remodeling. Unlike prokaryotes, eukaryotes also undergo heavy post-transcriptional modifications before translation can occur, which include: the addition of a 5' 7-Methylguanosine cap and a 3' Poly-A tail, intron excision and exon splicing.
ID052	The genetic code is conserved across the majority of organisms. Because of its degenerate qualities and unambiguousness, the genetic code provides the mechanism by which part of the central dogma is achieved. Many proteins are gene products, located in the nuclear genome. In order to create these products, the DNA is transcribed into an intermediary-messenger RNA.
ID031	To start transcription a promoter must be added to the beginning of the strand that is needing to be replicated and this signals for the strand to be copied.
ID031	There are many differences between the ways eukaryotes and prokaryotes go about transcription. Some of these ways include the way transcription starts and just how the dna is shaped.
ID031	This is just saying that three bases are read at a time by the genetic code and these three ribonucleotides are turned into proteons.
ID115	The promoter generated by DNA tells RNA polymerase where to latch on for transcription about 20 codons upstream from start. Once it knows where to start, then transcription can begin.
ID115	Both differ in fundamental ways, some due to the size difference of prokarya and eukarya, the fact that eukarya use more DNA for transcription and have more introns than exons than bacteria, requiring processing of mRNA before it can continue. In eukarya, RNA is created in the nucleus before it can be translated in the cytoplasm. In prokaryotes, transcription and translation is continuous.
ID115	This was discovered with the help of Crick, understanding the fundamental information that one amino acid can be specified by several codons through a three letter triplet code. Using the codons then, a protein can be built from a chain of amino acids and help the cell develop and fulfill its functions.
ID044	Promoters are important during transcription. A promoter is a region of DNA that initiates transcription of a certain gene.  They are located near the transcription start sites, upstream on the same strand of the DNA. Promoters work with other regulatory regions to direct the level of transcription of a gene.
ID044	Eukaryotic and prokaryotic transcription differ in many different ways for many different reasons.  eukaryotic occurs in the nucleus and prokaryotic occurs in the cytoplasm because prokaryotes do not have a nucleus.  Eukaryotes have cell cycle phases and prokaryotes do not have a definitive phase for transcription.  A single RNA polymerase synthesizes all three RNAs in prokaryotes and three different polymerases synthesize in eukaryotes.  prokaryotes do not need a transcription factor to initiate transcription but eukaryotes do.  This is important to show the differences between the two.
ID044	The genetic code is a universal set of rules by which information encoded within genetic material is translated into proteins.  These codes, called codons, allow mRNA to know which amino acids to form.  The sequence of amino acids then code for specific proteins that have a multitude of functions. This code is universal for all life.
ID121	they are different in the initiation process
ID090	In order for transcription to start, either the sigma factor in prokaryotes or the general transcription factors in eukaryotes must bind to a sequence called a promoter. Typically the sequence consists of the TATA box and is located upstream of the gene that will be transcribed. Transcription cannot occur without a promoter sequence as there would be no place to RNA polymerase to bind.
ID090	Transcription in eukaryotes is different than in prokaryotes.
ID090	The genetic code consists of ribonucleotide bases. It is converted to amino acids via translation and tRNAs which bring in amino acids and are linked together in peptide bonds. The amino acids then form polypeptides and many polypeptides linked together form proteins. In this way the genetic code found in three bases in mRNA triggers the tRNA to insert a particular amino acid into the growing nucleotide chain, thus forming a protein.
ID143	The promoter is the section of DNA that initiates the transcription process, which involves the DNA strand being duplicated and transcribed into mRNA, which is then translated into proteins.  Furthermore, the promoter region of DNA is located near the start sites of genes, and they are hundreds of base pairs long.  Promoters also work with other transcription factors, such as silencers and enhancers, to control the rate of transcription that occurs within a gene.
ID143	In eukaryotes, transcription occurs in the nucleus of the cell; in prokaryotes, the transcription process occurs within the cytoplasm of the cell.  In prokaryotes, only a single RNA polymerase is involved in the transcription process, but three RNA polymerases are involved in eukaryotic transcription.  In prokaryotes, initiation factors are not required for transcription, but initiation factors are necessary for the start of transcription in eukaryotes.
ID143	The genetic code is the set of regulations, involving three base pairs at a time, that encodes for genetic information being translated into amino acid sequences.  This process is accomplished by the ribosome, which links amino acids in a particular order.  This process is highly similar among all organisms.
ID129	a promoter is a region of DNA that initiates transcription of a particular gene. Promoters are located near the transcription start sites of genes, on the same strand and upstream on the DNA (towards the 5' region of the sense strand). In bacteria, the promoter contains two short sequence elements approximately -10 and -35 nucleotides upstream from the transcription start site.
ID129	in eukaryotes transcription takes place in the nucleus and translation takes place in the cytoplasm. in prokaryotes transcription takes place in the a more spatially compacted place.
ID129	The genetic code is the set of rules by which information encoded within genetic material (DNA or mRNA sequences) is translated into proteins by living cells. Biological decoding is accomplished by the ribosome, which links amino acids in an order specified by mRNA, using transfer RNA (tRNA) molecules to carry amino acids and to read the mRNA three nucleotides at a time.
ID091	The promoter is a region of DNA that acts as the binding site for RNA polymerase. Proteins can bind to these sections to inhibit transcription. When no proteins are present the section of DNA is transcribed to mRNA. For instance, the presence of lactose releases the protein from the lac promoter and allows for transcription.
ID091	Transcription has a few key differences in eukaryotes. There are multiple origin of initiations while there is only 1 in prokaryotes.  This is due to the much bigger genome of eukaryotes. Transcription also proceeds much slower. Also, euakryotic DNA has large sections of introns that require post-translational modifications, while in prokaryotes the primary transcript is the mature mRNA.
ID091	There 4 nitrogenous bases that are the letters in the genetic code. So their are 64 possible 3 letter sequences of DNA. Each tRNA molecule fits these unique sequences.  Each tRNA molecule also carries an amino acid with it. So each 3 letter codon specifies for one out of 20 amino acids, leading to a full length polypeptide.
ID133	The promoter determines where RNAP II binds to the DNA and where it begins copying the DNA into RNA.  In eukaryotes this core-promoter is the TATA box.
ID133	Transcription in eukaryotes occurs witin the nucleus whereas in prokaryotes it occurs in the cytoplasm.  Pre-mRNAs in eukaryotes are spliced before exiting the nucleus.
ID133	Once mRNA binds to the ribosome and tRNAs match their anticodons with the codons, protein can be translated.
ID088	A promoter is a region in DNA that regulates the start of transcription of a certain gene. RNA polymerase reads the promoter and begins the transcription process.
ID088	Transcription in prokaryotes occurs in the cytoplasm, while transcription in eukaryotes occurs in the nucleus. Also, in prokaryotes, a single RNA polymerase synthesizes all types of RNA. But, in eukaryotes, RNA polmerases I, II, and III are needed.
ID088	The three bases, called a triplet, in mRNA are read by a ribosome and code for an amino acid. For every triplet, there is one amino acid. These amino acids are made into a protein.
ID093	Promoters are located at the 3' end of the template strand's structural genes. In prokaryotes it is the site where RNA polymerase and specific sigma factor bind to begin transcription of the operon (the process is more complicated in eukaryotes). The operon can be regulated in many ways. For example, the lacZYA promoter is followed by one site where the tetrameric repressor LacI binds and forms a complex preventing RNA pol from transcribing the gene.
ID093	Transcription in eukaryotes is more complicated than in prokaryotes. Transcription can be immediately followed by translation in prokaryotes due to the lack of a membrane bound nucleus. In eukaryotes, transcription occurs within the nucleus and the mRNA must be delivered to the cytoplasm in order to the translated by the larger eukaryotic ribosomes.
ID093	Every three nucleotides of mRNA is a codon. Each codon codes for a single amino acid which is assembled into a polypeptide chain by the ribosome. Each individual amino acid is carried throughout the cell by transfer RNA (tRNA).
ID033	the promoter region regulates how much dna is transcribed.
ID033	there are many promoter regions and other factors in eukaryotic cells that are more complex than prokaryotic cells.
ID033	the genetic code can be used to determine the amino acid sequence from the template of dna given.
ID060	Transcription can be regulated by promoters and or enhancers.
ID060	Eukaryotes have different components in their cells and especially in their structure. The Central Dogma differs slightly among them
ID060	The mRNA translates the template to make proteins.
ID002	Transcription is regulated by the promotor site in that transcription begins immediately after the promotor site. The promotor site is known as the TATA box and is around -10
ID002	Transcription in eukaryotes differs from transcription in prokaryotes in that transcription occurs within the nucleus. Chromatin must uncoil to be made accessible to RNA polymerase. RNA polymerases rely on transcription factors that help scan and bind DNA at the promotor region. There are also enhancers and silencers that control gene expression.
ID002	The genetic code reads the order of the bases transcribed into specific amino acids. Those amino acids are read in a sequence that eventually creates a protein
ID016	A promoter is a section on a gene that allows RNA polymerase to bond to a section that starts transcription. After the promoter region, the DNA unwinds and starts transcribing the mRNA strand.
ID016	Eukaryotic transcription occurs in the nucleus. Then, the final mRNA product must find its way into the cytoplasm where translation occurs. Chromatin must also uncoil for transcription, while prokaryotic DNA exists in a circle and does not encounter the same need to uncoil as do eukaryotes. Eukaryotic mRNA also contains introns, or intervening sequences.
ID016	The genetic code involves the sequence of 3 RNA bases called codons. These codons are specific to their anticodons which are needed in translation. The genetic code allows a sequence of amino acids to eventually be formed from bases which existed in DNA.
ID023	The location as to where transcription begins depends on the location of the promoter. Transcription begins after the promoter. If there is a mutation or error concerning the promoter region, there will be extreme results affiliated with the transcription process.
ID023	Considering that Eukaryotic DNA has nucleosomes associated with it, the DNA will need to be disassembled in order to undergo transcription. In addition, unlike prokaryotic DNA, eukaryotic DNA is transcribed inside of the nucleus then translated later outside of the cell. Knowing the major differences between these two is fundamental to understanding transcription; especially in eukaryotes.
ID023	In order to predict which amino acid will be assigned to which codon, the genetic code must be used. Without this code, we would have an extremely limited knowledge of amino acids and the codons that signal them as well as the translation process.
ID117	Trascription is the process in the central dogma when DNA double helixes are unwound and broken into a fork for the strands to be coded into RNA. The promoter is the region that initiates transcription, and therefore is the regulator of transcription.
ID117	Trascription in eukaryotes is regulated by promoters and includes the presence of the TATA box within the core promoter region. In prokaryotes, transcription and translation are coupled, where as that is not possible in eukaryotes. While RNA polymerase is all that's needed in prokaryotes, RNA polymerase I, II, and III are needed in eukaryotes. Eukaryotic transcription occurs in the nucleus, not the cytoplasm.
ID117	Crick found evidence that supported coding bases occurred in groups of three. Based Crick's research and others', they were able to prove that multiple codons could code for a single amino acid, but one codon could not code for multiple amino acids. Therefore, the genetic code is what translates sequences of base pairs into amino acid polypeptide chains.
ID124	The promoter region comes before the area of the gene needing to be transcribed and initiates transcription.
ID124	Transcription in eukaryotes occurs in the nucleus. Prokaryotes transcribe in the cytoplasm.
ID124	Three bases together form a codon and creates the reading frame for the rest of the gene
ID127	The promoter is what initiates transcription. This is where the RNA polymerase binds to and this is where the TATA box is located in eukaryotes. Without the promoter, transcription would not happen because it would not have the proper initiation site.
ID127	Transcription in eukaryotes takes a longer time because of the size of the cell and the number of genes in the cell. It requires more RNA polymerases for the different RNA(tRNA, mRNA, rRNA). For prokaryotes, it does not have a TATA box in the promoter and transcription and translation happen simulteously.
ID127	There is more than one codon that codes for one amino acid but two amino acids could not come from one codon. Each codon has 3 bases, which is called a triplet code, and this is what is translated to proteins.
ID057	The promoter initiates transcription. Transcription is the mechanism that the cell regulates the conversion from DNA to RNA. These promoters are located near the transcription start sites. These promoter sequences help direct the level of transcription of a given gene.
ID057	Transcription in eukaryotes occurs in the nucleus and the mRNA has to leave the nucleus for translation. Chromatin remodeling is also different from prokaryotes due to the fact that the chromatin must uncoil to make DNA accessible to RNA polymerase and must move the nucleosomes. Eukaryotes also has three different RNA polymerases. Prokaryotes does not have a TATA sequence, which is a core promoter sequence in eukaryotes. In prokaryotes translation and transcription are coupled, happening simultaneously, however these processes happen in differing times in eukaryotes.
ID057	Crick helped to determine how many bases coded for an amino acid and whether or not the genetic code overlapped. In addition, frame shift mutations were observed this helped Crick in determining that three duplications or deletions would restore the reading frame, which provided evidence that 3 bases code for an amino acid and therefore the order of the polypeptide chain.
ID114	A promoter is a region of DNA that initiates transcription of a particular gene. Promoters are located near the transcription start sites of genes, on the same strand and upstream on the DNA.
ID114	Eukaryotes have many more genes and these are spread across multiple chromosomes. Prokaryotes have fewer genes and these genes are all located on one chromosome. Groups of genes producing proteins with related functions are often organized into operons in prokaryotes but not in eukaryotes. Eukaryotes also have mRNA that must have its introns excised and the mRNA transported out of the nucleus to the ribosomes. The greater complexity of the eukaryote genome means that a greater variety and complexity of control mechanisms is necessary.
ID114	The genetic code defines how sequences of these nucleotide triplets, called codons, specify which amino acid will be added next during protein synthesis. A three-nucleotide codon in a nucleic acid sequence specifies a single amino acid.
ID147	The promoter sequence in both prokaryotes and eukaryotes is essential to allow for the bind of RNA polymerase, and subsequently is then essential to transcrition.
ID147	Transcription in Eukaryotes contains several more factors than in prokaryotes, these includ transcription factors like silencers and enhancers.
ID147	The genetic code allows for there to be degenerate coding of the 20 amino acids. Since there is multiple ways to get the same amino acids it reduces the effects of several mutations. The code also allows for a specific and basic form of transtlation from DNA to mRNA and then again into proteins from the open reading frames.
ID040	Promoter sequences are located in a 5' region, upstream from the point of initial transcription of a gene.
ID040	transcription in eukaryotes takes place in the nucleus so it is not free to associate with ribosomes directly after transcription. Eukaryotic DNA must be unraveled so that the DNA is made accessible to RNA Polymerase. Initiation and regulation of transcription is more extensive. The primary RNA transcript is often altered.
ID040	The central dogma of biology is DNA codes for RNA which makes the protein. The RNA sequence that becomes a protein is found to be read in triplets and each triplet, known as a codon, codes for an amino acid. This code is nearly universal, degenerate, commaless, and colinear.
ID128	Transcription is the first part of the central dogma that describes how DNA leads to the production of proteins. DNA is transcribed into RNA which is the translated to amino acids that make up proteins. Transcription begins when RNA polymerase binds to the promoter region. In bacteria, there are two promoter regions. Therefore, the importance of the promoter region is that it shows where transcription begins.
ID128	Transcription in eukaryotes is different from prokaryotes in several ways. First, eukaryotes have a nuclear membrane that the mRNA must leave the nucleus to be translated in the ribosome. Also, eukaryotes are much more complex upstream of the coding sequence. Eukaryotes have a TATA box as well as a CAAT box. The absence of a sigma factor is another difference from prokaryotes. Finally, in eukaryotes a single gene is copied and in prokaryotes multiple genes can be a part of the same transcript.
ID128	The genetic code is the Rosetta Stone between mRNA and proteins because the Rosetta Stone was the key to the deciphering of languages in the same way that the genetic code allows mRNA bases to be translated into proteins. The study of frameshift mutations through insertion and deletions allowed for the discovery of the bases being read 3 at a time.
ID078	Transcription can only begin after all the initiation factors have bound to the promoter
ID078	in prokaryotes there is only a singular ORI, but in DNA there can be multiple. Eukaryotes have more more DNA polymerase as well as having more DNA Polymerase molecules
ID123	The promoter regulates transcription because it is a region on DNA that initiates the transcription of that portion of DNA.  It is region of bases that draws in transcription factors and allows them to bind to the DNA to allow transcription.
ID123	Transcription differs in many ways in between eukaryotes and prokaryotes.  Eukaryotes transcribe in the nucleus and prokaryotes transcribe in the cytoplasm.  Eukaryotes require transcription factors.  Eukaryotes have three different RNA polymerases.
ID123	The genetic code is a universal code that translates into different amino acids.  The codon is a sequence of three bases.  Each of the three bases links to a anti codon on a charged tRNA molecule in a ribosome to form a polypeptide.
ID011	Transcription requires a TATA box to act as a promoter for the RNA polymerase II to bind onto.
ID011	Transcription in eukaryotes occurs in the nucleus, requires mRNA to leave the nucleus to be translated, and requires chromatin to uncondense and recondense.
ID011	Three bases code for a single amino acid, but multiple three base codes can code for the same amino acid.
ID134	Promotors are upstream from  point of initial transcription. they are searched for and then when found are binded to.
ID134	In eukaryotes the transcription is done within the nucleus, while the prokaryotic is done in the cytoplasm. In eukaryotes the RNA transcript is not allowed to freely associate with ribosomes prior to completion. Eukaryotes must go through chromatin remodeling. Eukaryotic RNA polymerases rely on transcription factors
ID134	3 nucleotide letter stand for 3 nucleotide bases which replicate and then are transcirbed into mRNA complementarily. The mRNA then code for amino acids by analyzing 3 letter sequences with the corresponding amino acid for which it codes. Then this amino acid through translationg because a polypeptide chain and then a protein
ID111	The promoter is the region of DNA that initiates transcription of a particular gene. Promoters are located near the transcription start sites on the same strand and upstream on DNA. Transcriptional regulation is the means by which a cell regulates the conversion of DNA to RNA.
ID111	Transcription in eukaryotes occurs within the nucleus under the direction of three separate forms of RNA polymerase. Unlike the prokaryote process, in eukaryotes the RNA transcript is not free to associate with ribosomes prior to the completion of transcription. For the mRNA to be translated, it must move out of the nucleus into the cytoplasm. Initiation of transcription of eukaryotic genes requires the compact chromatin fiber, characterized by nucleosome coiling, to be uncoiled and the DNA to be made accessible to RNA polymerase and other regulatory proteins.
ID111	The genetic code is the set of rules by which the information encoded within genetic material is translated into proteins by living cells. Amino acids are in an order specified by mRNA using tRNA to carry amino acids and read the mRNA 3 at a time. Each set of three nucleotides has a corresponding protein depending on the combination of letters.
ID102	The promoter initiates transcription as well as regulates it. For the process to begin, RNA polymerase must attach itself to DNA. The process is different in eukaryotes and prokaryotes.
ID102	The difference between the two is that in bacteria transcription occurs through a single RNA polymerase which contains five sub-units, four catalytic and one regulatory. As opposed to bacteria, eukaryotes are more complex because transcription is regulated by three different types of RNA polymerase (I-III). Each class has a different function in the transcription process.
ID102	The 3 bases at a time are called codons and they specify which amino acid will be added next during protein synthesis. mRNA comes in the matter when it the codon are getting decoded, it links the amino acids which then are used by tRNA in order to read the nucleotides.
ID144	Promoters represent critical elements that can work in concert with other regulatory regions (enhancers, silencers, boundary elements/insulators) to direct the level of transcription of a given gene.
ID144	RNA polymerase doesnt need a promoter in prokaryotes and it only copies one strand. Also, in prokaryotes transcription occurs right after the promoter. One thing unique for eukaryotes is that they have self splicing RNA's and splicesomes.
ID144	The genetic code uses ribonucleotide bases as "letters". The genetic code is unambiguous, degenerate, commaless,nonoverlapping, and collinear. Also, the genetic code is nearly universal. There are 64 codons. AUG is the only start codon. UAG, UAA, and UGA are the three known stop condons.
ID006	Some of these are expressed in all cells all the time. These so-called housekeeping genes are responsible for the routine metabolic functions (e.g. respiration) common to all cells.
ID006	In bacteria, all transcription is performed by a single type of RNA polymerase. This polymerase contains four catalytic subunits and a single regulatory subunit known as sigma (s). Interestingly, several distinct sigma factors have been identified, and each of these oversees transcription of a unique set of genes. Sigma factors are thus discriminatory, as each binds a distinct set of promoter sequences.
ID006	The genetic code consists of 64 triplets of nucleotides. These triplets are called codons.With three exceptions, each codon encodes for one of the 20 amino acids used in the synthesis of proteins. That produces some redundancy in the code: most of the amino acids being encoded by more than one codon.
ID145	The promoter signals where transcription needs to start.
ID145	Different proteins are used and needed in transcription in these two types of cells.
ID145	3 bases are important because if a mutation occurs and interferes with the 3 bases then the correct proteins will not be coded for.
ID085	Transcription is regulated by the promoter because it is not initiated until something binds to it. In prokaryotes, transcription is initiated by binding of RNA Polymerase's sigma factor to the promoter. In eukaryotes, general transcription factors such as TFIID are necessary for RNAPII to bind to the promoter.
ID085	Several differences between transcription in eukaryotes and prokaryotes exist. Eukaryotes have three RNA polymerases, each coding different RNAs. Also, transcription factors are required for mRNA transcription in eukaryotes. Enhancers and/or silencers can affect transcription for a good distance away as well. The biggest difference is that transcription in eukaryotes produces pre-mRNA, which requires post-transcritional processing.
ID085	The genetic code is a simple set of 64 combinations of nucleotides into codons. Using these codons, the information in DNA can be transcribed onto mRNA, which is then translated to proteins.
ID100	Transcription is regulated by the presence of a promoter region. Transcription promoter regions in prokaryotes are RNA-coding sequences that are found upstream of transcription start site in the 5' region. There are strong and weak promoters. Strong promoters are present on very transcriptionally active DNA molecules. These promoter regions are recognized by RNA polymerase theta molecules. RNA polymerase scans the DNA until it comes into contact with one of these promoter regions where it binds to the DNA. Once RNA polymerase binds, the DNA helix is locally unwound so that it can be transcribed by the enzyme. Transcription actually begins at the transcription start site. Without these promoter regions, RNA polymerase would not know where to bind the DNA and begin transcription.
ID100	Transcription between eukaryotes and prokaryotes differs in very fundamental ways. First, transcription in eukaryotes occurs within the nucleus. Three different RNA polymerase forms are involved with this transcription. Unlike in prokaryotes, eukaryotic RNA transcripts cannot associate with ribosomes before transcription is completed. Also, eukaryotic chromatin fibers have to uncoil in order for the DNA to be accessible to RNA polymerase for transcription. Prokaryotic DNA doesn’t not form chromatin. Also, eukaryotic transcription produces RNA that has to be processed before it can leave the nucleus and begin translation. This includes, addition of a 5’cap and poly a tail and splicing.
ID100	The genetic code is analogous to the Rosetta stone for gene expression. The genetic code is made up of the four nucleotide bases, adenine, thymine, cytosine, and guanine, found in DNA molecules. These DNA nucleotides are transcribed into RNA nucleotides, which are the same minus the exception of uracil for thymine. RNA nucleotides are arranged into groups of three called codons. During translation, each codon can code for a single amino acid, which can be subsequently linked together via phosphodiester bonds. Therefore the genetic code  found in DNA that is transcribed into mRNA molecules which are then translated into proteins concluding the central dogma.
ID146	For transcription to take place RNA polymerase must attach to the DNA near a gene. Promoters contain specific DNA sequences that provide a secure initial binding site for RNA polymerase and for transcription factors that recruit RNA polymerase. The transcription factors have specific activator or repressor sequences of corresponding nucleotides that attach to specific promoters and regulate gene expression.
ID146	Transcription takes place in the nucleus of eukaryotes, but in the cytoplasm of prokaryotes. Eukaryotes must allow their chromatin to uncoil and allow RNA polymerase to bind as well as shift nucleosomes, while prokaryotes do not. Prokaryotes do not need proteins or transcription factors for initiation, while eukaryotes utilize many types of these. Also, prokaryotes only use one RNA polymerase of 5 nucleotides to produce the 3 types of RNA, while eukaryotes use various types of RNA polymerases of 10-15 nucleotides.
ID146	The genetic code is the sequence of nucleotides transcribed from DNA to mRNA separated into codons which are translated by tRNA which transports different amino acids that form a polypeptide which can be linked to form proteins.
ID036	Synthesis of DNA begins with a type of RNA. A short segment of RNA is Synthesized by the DNA template through primase, which recruited to the replication fork by DNA helicase. DNA poll added deoxy ribonucleotides, initiating DNA synthesis.
ID036	One of the major differences between prokaryotic/eukaryotic DNA; DNA in eukarya with DNA-binding proteins, chromatin consist of nucleosomes wrapped around histones. before, nucleosomes,es ,must be modified replication of proteins
ID036	the discovery of the retrovirus enzyme reverse transcriptase which can copy RNA back to DNA. This does not go  with the Central Dogma rule that DNA to RNA TO protein. This is backwards. This fiding provides strong evidence that RNA is the genetic code
ID059	Transcription is regulated by a promoter which is the region of DNA that starts the transcription of a particular gene. It is located near the start sites of most genes by the 5' end and is upstream on the DNA.
ID059	In eukaryotes, transcription takes place in the nucleus and translation is carried outside the nucleus in the cytoplasm. Prokaryotes on the other hand, have transcription and translation to occur almost in one giant phase. For example, mRNA begins while the transcript is still being synthesized and then immediately translated and made into proteins.
ID059	The genetic code that translate mRNA into proteins are combinations of codons that are coded for specific functions. These combinations are then combined with other codon pairs to make one giant amino acid. These amino acids are the things that aids to the functioning of the body.
ID073	Transcription is initiated by a compact chromatin fiber characterized by nucleosome coiling to be uncoiled and DNA to be made accessible to RNA polymerase and other regulatory proteins.  This is called chromatin remodeling. The first of cis acting core promoter determines where RNA II binds to DNA and where copying begins.
ID073	Eukaryotes have distinct forms of RNA polymerase, each consisting of a greater number of polypeptide subunits than in bacteria. Eukaryotic transcription occurs in nucleus while unlike prokaryotes.  In eukaryotes, mRNA must leave nucleus for translation. Chromatin must uncoil in eukaryotes to make DNA accessible to RNA polymerase. In eukaryotes, RNA polymerase rely on transcription facts to scan and bind to DNA.  Eukaryotes have general transcription factors, activators and repressors, and enhancers and silencers.
ID073	Transcription is the process from DNA to RNA molecules.  It results in mRNA molecule that is complimentary to gene sequence of on of two strands of the double helix.  Each codon is then complimentary to the anticodon of tRNA, which inserts the correct amino acid into the polypeptide chain during translation. Transcription is the initial step in the process of information flow in the cell.
ID048	a promoter is a region of DNA that initiates transcription of a particular gene. Promoters are located near the transcription start sites of genes, on the same strand and upstream on the DNA.
ID048	There is a huge difference between these two. Prokaryotic occurs in the cytoplasm and eukaryotic transcription occurs in the nucleus. Initiation of transcription does not need any proteins in prokaryotes and in eukaryotes there are several proteins required. It differs vastly between prokaryotes to eukaryotes.
ID048	This is transcription.
ID017	The promoter in initiates and regulates transcription and are located near the transcription start sites in the genes and can be 100-1000 base pairs long.
ID017	For starters,transcription is in the cytoplasm for prokaryotes and the nucleus for eukaryotes. Prokaryotes also only have three promoters as opposed to the plethora of promoters for eukaryotes.
ID017	The genetic code is the set of rules thats purpose is to translate encoded genetic material of the DNA or RNA and make proteins.
ID107	A promoter is a sequence of nucleotide bases on the gene that serve as a signal and binding site for RNA polymerase. Without a promoter region, RNA polymerase would not be able to recognize the gene to be transcribed. The promoter sequence, or TATA box, is upstream of the gene to be transcribed. Mutations in the promoter sequence causes an inability of RNA polymerase to function and a disruption of the central dogma.
ID107	Eukaryotes have a linear DNA and prokaryotes have a circular one. This can cause a problem at the end of the eukaryotic strand during replication, which is solved by the addition of telomerases. Eukaryotic replication is 25 times slower than prokaryotic replication, but there are multiple replication sites in eukaryotes and only one in prokaryotes. Eukaryotes have histone proteins that clump together and compact sections of DNA to turn them off. Eukaryotes have two different polymerases that copy the leading and lagging strand while prokaryotes only have one. In eukaryotic transcription is initiated by a RNA primer which is converted into DNA. There is only a RNA primer in prokaryotes.
ID107	The nucleotide bases that make up mRNA are using open reading frames by ribosomes in the cytoplasm. Amino acid residues are added one at at time to the growing polypeptide chain. The tRNA delivers delivers the specific amino acid according to the anticodon sequence at the opposite end of the tRNA. Three base groupings are called codons, and every combination of the 4 nucleotide bases codes for a specific amino acid or the start (AUG) or stop (UGA, UAA, UGG).
ID038	The promoter sequence helps control transcription. RNA primer recognizes this sequence and initiates transcription. This allows control over how much DNA is transcripted into mRNA since the promoter sequence is blocked by the protein product if available. Thus transcription occurs only when the protein is low.
ID038	Transcription in eukaryotes has multiple sites of origin of replication and thus the transcription is going on at several places at once. In prokaryotes, there is only one origin of replication site where the transcription starts. This is important since eukaryotes have larger genomes and to save time, more than sites of origin speeds up the process.
ID038	The four bases of DNA codes for every protein found in an organism and carries all the information. The centra dogma explains how DNA is converted to RNA and then to protein. Ribosomes read 3 bases at a time, a codon, and translate it into an Amino acid. Thus DNA is a Rosetta Stone that can be translated into rna and protein.
ID132	Promotor sequences are DNA sequences that define where transcription of a gene by RNA polymerase begins, they are usually located upstream or at the end of 5' transcription initiation site.
ID132	Prokaryotes only contain 3 different promoter elements and Eukaryotes use promoter elements like TATA box initiator elements. Eukaryotes have 3 different types of RNA polymerases I, II, and III, prokaryotes only have one. It's safe to say that eukaryotic transcription is way more complex than prokaryotic.
ID132	Yes the genetic code is the Rosetta stone of life that translates mRNA and 3 proteins, 3 bases at a time.It can be expressed as either RNA codons or DNA codons. Each mRNA molecule has it's sequence of nucleotides by transcription.
ID080	Transcription is controlled by the promoter. It determines when the cell undergoes transcription and what is transcribed.
ID080	There are fundamental differences between eukaryotes and prokaryotes as the DNA is stored differently for both types.
ID080	The genetic code is read in 3 base pairs called codons. These are what are later read and translated into amino acids then proteins.
ID072	In bacteria, the site of this initial binding is established when the RNA polymerase subunit recognizes specific DNA sequences called promoters.
ID072	Transcription in eukaryotes occurs within the nucleus under the direction of three separate froms of RNA polymerase. Intiation of transcription of eukayotic genes requires the compact chromatin fiber, characterized by nucleosome coiling, to be uncoiled and the DNA to be made accessible to RNA polymerase and other regulatory proteins. Initiation and regulaton of transcription entail a more extensive interaction between cis-acting DNA sequences and trans-acting protein factors involved in stimulating and initiating transcription. Alteration of the primary RNA transcript to produce mature eukaryotic mRNA involves many complex stages referred to generally as "processing".
ID072	The genetic code can be expressed as either RNA codons or DNA codons. RNA codons occur in messenger RNA (mRNA) and are the codons that are actually "read" during the synthesis of polypeptides (the process called translation).
ID047	RNa polymerase is the main enzyme that deals with RNA sythesis. No primer is needed. Instead promoters are sequences of bases om the DNA that signal RNA polymerase to start trascribing and is dtected by the sigma factor of RNA polymerase. It is not included in the  mRNA because transcription occurs right after it.
ID047	There are several key differences between prokaryotic and eukaryotic transcription. First off in prokaryotes do not have a nucleus to transcription and translation can almost happen simultaneously. Chromatin also have to uncoil and require trans transcription factors to assist RNA polymerase to bind to DNA.
ID047	The genetic code uses ribonucleotide bases as letters.The genetic code is written in linear form, and each word consists of three ribonucleotide letters or  triplet code. The code is unambiguous, degenerate, and commaless. It contains start and stop codons and does not overlap. THe genetic code is also conveniently universal in that it can be used for many different organisms.
ID139	Promoters are located upstream on the DNA (towards the 5' end). Promoters contain unique DNA sequences that allow RNA polymerase to begin transcription. They also have transcription factors that help them find rna polymerase.
ID139	In prokaryotes transcription and translation can be coupled together, and it takes place in the cytoplasm. In prokaryotes transcription is followed by translation, but they cannot be coupled together. Transcription for eukaryotes takes place in the nucleus. Prokaryotes do not require any initiation factors, while eukaryotes do.
ID139	This is important because it explains how amino acid triplet groups influence DNA structure and how these groups come to formation.
ID034	transcription start when the RNA polymerase II binding to TATA box, the transcription start immediate after the TATA box
ID034	transcription in eukaryotes is more complicated then transcription in prokaryotes due to the complexity of cells
ID034	that provides thee evident of the triplet nature of the code
ID003	Promoter regulates transcription to begin the initiation.
ID003	Transcription in eukaryotes occurs in the nucleus while prokaryotes occur in the cuytoplasm.
ID003	The code is a triplet that translates amino acids to proteins.
ID009	Promoters are in the 5' region by the transcription site. RNA polymerase and transcription factors bind to the promoter to begin trascription. Promoters can both initiate and stop transcription.
ID009	Transcription is more complex in eukaryotes. Prokaryotes use one RNA polymerase while eukaryotes use 3 to get RNA strand from DNA template. Termination is also dirrerent. There are several signals for termination in eukaryotes and in prokaryotes, there is either intrinsic termination or factor dependent termination.
ID009	RNA is changed to proteins via translation. Rosetta stone is the term that refers to the actual translating from nucleotides to amino acids.
ID113	the RNA pol binds to the promoter to initiate transcription
ID113	transcription on eukarytoes is more complex than prokaryotes
ID113	the genetic code is written in linear form, nonovelapping, commaless, and nearly universal
ID021	The promoter is located upstream from the transcribed region. It is recognized and the proper enzymes are put into place so that transcription can begin at exactly the correct base.
ID021	In prokaryotes, transcription occurs in the cytoplasm. The promoter region recognized by σ70 factor RNA polymerase holoenzyme. Then RNA polymerase initiates RNA synthesis, σ leaves, chain elongation proceeds, proofreading, termination sequence intrinsic (GC) or ρ(C rut) forms hairpin 2° structure. In eukaryotes, RNA polymerase I transcribes 28S,18S,5.8S rRNAs. RNA polymeraase II transcribes mRNAs & snRNAs. RNA polymerase III transcribes tRNAs, 5S rRNA & snRNAs. Transcription depends on general transcription factors. Transcription occurs in the nucleus and the mRNA must be edited before leaving the nucleus to go to the cytoplasm for translation with ribosomes.
ID021	Genetic code is linear, codons, unambiguous, degenerate (wobble hypothesis 3rd position), start (AUG=met), stop (UAA, UAG, UGA), commaless, nonoverlapping (Crick 3 +/- restore frame), collinear, universal (exceptions mt wobble), ordered (chemically similar amino acids share middle bases in codons).
ID110	Promoters contain DNA sequences that provide an initial binding site for RNA polymerase and for transcription factor proteins that recruit RNA polymerase regulating the transcription fctor
ID110	Eukaryotes have positive or negative control close to and also far away from the cell prokaryotes can only demonstrate +/- control in nearby sites. eukaryotes require a mediator complex and prokaryotes utilize the sigma factor as the promotor
ID110	Genetic code is transcribed from DNA into mRNA which is then translated into proteins based on 3 subunit codons that code for a specific amino acid, the 20 amino acid sequence is then transformed into protein
ID120	The promoted controls whether the process proceeds.
ID120	Transcription in prolaryotes does not reuqire transcription factors nor upstream modulators.
ID120	It is the language stands for all of the amino acids, and different combinations of amino acids creates different proteins
ID060	Gerard stated used urine samples from his patients and researched their relativity with each other.
ID060	Each gene is responsible for the production of an enzyme.
ID060	The sickle cell anemia is a single genetic mutation and so it does not relate to the one gene-one hypothesis.
ID065	This experiment determined that there are various factors involved in genetics and development that have the ability to affect the complex metabolic mechanisms in organisms. These organic reactions often entail multiple enzymes and ligands, and altering one enzyme or ligand in one step of these processes has the ability to alter all following mechanisms and reactions. This showed not only the complexity of metabolic reactions but also their delicacy and how easily a mutation in enzymes can alter the function and net outcome of the reaction.
ID065	This explained that one gene is responsible for the production of one specific enzyme depending on the sequence of codons that the transcribed mRNA codes for during translation and the resulting peptide. This hypothesis explained how a gene codes for one enzyme and one enzyme only, but it did not take into consideration overlapping genes.
ID065	Sickle cell anemia is the result of a mutation in a single nucleotide of the gene that codes for hemoglobin. Altering this one nucleotide in the form of a mutation demonstrated that the one-gene-one-enzyme hypothesis is not quite correct because the hemoglobin is composed of multiple proteins, but a mutation in only one results in sickle cell anemia. This showed an exception to the one gene one enzyme hypothesis because a mutation in one gene affected the function of a tertiary, multiprotein complex.
ID109	Archibald Gerard discovered inborn errors of metabolism. Gerard’s research proposed an explanation for epistasis. For example,he found that alkaptonuria, a disease that makes urine black because of the presence of gentesate. His results showed that the disease was prevalent in those whose parents were closely related.
ID109	The one gene-one enzyme hypothesis proposes that genes are regulated by the production of enzymes where every gene is responsible for making one enzyme that affects one step during the metabolic pathway. This founded the science of biochemical genetics.
ID109	The genetic basis of sickle cell anemia demonstrates that the one-gene-one enzyme hypothesis is not quite correct because sickle cell anemia is caused my many mutations that are affected by many enzymes. Many enzymes affect can influence one gene or one enzyme can affect many genes.
ID143	Archibald Gerard suggested the idea that inborn errors of metabolism can occur due to the control of genes; since particular genes code for certain types of enzymes, problems can arise when toxic or abnormal substances are accumulated within an organism that inhibit his or her ability to function.  The specific type of problem that Gerard suggested derives from a group of genetic diseases that cause metabolism disorders, leading to an organism's inability to synthesize compounds, develop normally, or carry a normal life.
ID143	Based on the idea that genes code for enzymes and errors can occur within metabolic pathways, George Beadle was able to discover the one gene-one enzyme hypothesis, stating that each gene produces strictly one single enzyme within these metabolic pathways.  Beadle utilized Neurospora and other model organisms to illustrate this hypothesis, saying that one gene is capable of ultimately controlling one metabolic pathway.
ID143	There are certain processes that have demonstrated the invalidity of the one gene-one enzyme hypothesis, including the genetic basis of sickle cell anemia.  Sickle cell anemia occurs when a genetic mutation affects the hemoglobin of an organism, leading to distorted red blood cells.  This process demonstrates the invalidity of the hypothesis because multiple genes can control the same metabolic pathway within the organism.
ID030	This first realization is what would eventually lead to the "one gene one polypeptide" theory. By studying inheritance of alkaptonuria, Gerard realized that the improper metabolism associated with the disease was the result of genes.
ID030	This discovery was certainly one of the more important in genetics. It was a huge leap forward, increasing our understanding of how DNA information translates to a protein product that the cell can utilize. Later experimentation revealed that Beadle's conclusion was not entirely accurate. It was, however, a step in the right direction. We now know that one protein codes for one polypeptide.
ID030	This evolved the conclusion that Beadle had come to. By altering single proteins in a metabolic pathways, researchers concluded that instead one gene seemed to code for one polypeptide.
ID014	Garrod studied AKU and PKU which are both diseases that are somewhat benign but he noted that they were both diseases related to metabolism. He also saw that most of the carriers were born to parents who happened to be first cousins. He concluded that the errors of metabolism was found to be genetic and each individual had inherited their condition.
ID014	His initial proposal suggested there each gene encoded for one enzyme. This was disapproved when it was discovered that although most enzymes are proteins not all proteins are enzymes. There was then a study conducted studying sickle cell anemia.
ID014	Sickle cell showed that  the one-gene-one enzyme theory was not correct. This was primarily due to the fact that by using the technique of gel electrophoresis and fingerprinting that the HbS and HbA were only different by a single amino acid. This discovery led to the conclusion that it was more like a one-amino acid-one enzyme.
ID070	Archibald Garrod discovered inborn errors of metabolism. An explanation for epistasis is an important manifestation of Garrod’s research, albeit indirectly. When Garrod studied alkaptonuria, a disorder that makes urine quickly turn black due to the presence of gentesate, he noticed that it was prevalent among populations whose parents were closely related
ID070	The one gene-one enzyme hypothesis is the idea that genes act through the production of enzymes, with each gene responsible for producing a single enzyme that in turn affects a single step in a metabolic pathway. The concept was proposed by George Beadle
ID070	The one gene-one enzyme hypothesis is the idea that genes act through the production of enzymes, with each gene responsible for producing a single enzyme that in turn affects a single step in a metabolic pathway.
ID000	Gerard studied alkaptonuria, a disease caused by the buildup of homogentesic acid in the blood which causes black urine, black cartilage and arthritus. He speculated that this was caused by a genetic disorder because it ran in families and coined the term “inborn (i.e., genetic) errors of metabolism
ID000	Beadle and Tatum worked with X-ray induced haploid mutants of Neurospora and identified auxotrophic mutants that could not synthesize specific amino acids or vitamins.  They cleverly used these auxotrophic mutants to determine the steps in critical metabolic pathways.
ID000	The hemoglobin molecule is composed of 2 alpha and two beta subunits which are the products of two genes, a gene for the alpha subunit and a gene for the beta subunit.  Sickle cell anemia was shown to affect only the beta chain and therefore demonstrated that some proteins were the product of more than one gene.  We now know that one-gene one-enzyme should be modified to one-gene, one polypeptide.
ID037	Inborn errors of metabolism form a large class of genetic diseases involving congenital disorders of metabolism. The majority are due to defects of single genes that code for enzymes that facilitate conversion of various substances (substrates) into others (products). In most of the disorders, problems arise due to accumulation of substances which are toxic or interfere with normal function, or to the effects of reduced ability to synthesize essential compounds. Inborn errors of metabolism are now often referred to as congenital metabolic diseases or inherited metabolic diseases.
ID037	The one gene–one enzyme hypothesis, proposed by George Wells Beadle in the US in 1941, is the theory that each gene directly produces a single enzyme, which consequently affects an individual step in a metabolic pathway.
ID061	Garrod studied human disorders that could be inherited. He studied albinism, cystinuria, and alkaptonuria. He looked for patterns of inheritance of alkaptonuria and eventually concluded that it was genetic in nature. Garrod concluded the inherited condition was the result of an alternative mode of metabolism, thus implying that hereditary information was controlling chemical reactions in the body. Thus, this suggests that there are errors in the metabolism caused by changes in the genes.
ID061	Beadle investigated nutritional mutations in the pink bread mold Neurospora crassa. They studied how different mutant strains of the mold would be affected when grown in a medium that was incomplete. If no growth of a culture in a minimal medium occurred then it was inferred that the strain was mutant and not able to produce the nutrient that is missing in the medium. It seemed likely to Beadle that each nutritional mutation caused the loss of the enzymatic activity that facilitates an essential reaction in wild-type organisms. It also appeared that a mutation could be found for nearly any enzymatically controlled reaction, thus suggesting that one gene specifies one enzyme.
ID061	The first direct evidence that genes specify proteins other than enzymes came from the work on mutant hemoglobin molecules derived from humans with sickle-cell anemia. These individuals have erythrocytes that are not shaped like a normal erythrocyte. These erythrocytes are proteins whose genotype and phenotype can be traced back to a single pair of alleles. Thus, if these alleles control the production and viability of the erythrocytes then it can be deducted that genes code for proteins other than just enzymes.
ID024	Archibald Gerard discovered that there are many errors in the human genome that often cause disease. He called them inborn errors of metabolism because the error occurs in a single gene, that codes for particular enzymes, which often play a key role in a cell's metabolic pathways.
ID024	Beadle proposed that one gene codes for one single, particular enzyme, which then has important and very particular function in metabolic pathways. He proposed this because he  demonstrated that one gene in a fruit fly controlled a single, specific chemical reaction in the fruit fly, which one enzyme controlled
ID024	Instead of the one-gene-one enzyme hypothesis, it seems more likely that is a one-gene-one polypeptide hypothesis. This is because some enzymes are composed of different polypeptides, which come from different genes. One gene is very unlikely to code for one single enzyme by itself. It often takes multiple genes to make an enzyme.
ID136	Genes are responsible for mostly everything.
ID136	It is an idea that every gene acts through production of enzymes and that each gene is responsible for creating one enzyme hence the "one gene one enzyme."
ID136	Sickle cell anemia proves the one gene one enzyme hypothesis incorrect because in that case the infected genes are not able to create enzymes.
ID013	Inborn errors of metabolism involves the use of enzymes in metabolism. When these enzymes are not functional, then it blocks the pathway, which prevents metabolism of some important substances that can cause disease. For example, the phenylketonuria block prevents the production of tyrosine, which causes this process to undergo a different pathway that can lead to nerve damage and mental retardation.
ID013	The one gene-one enzyme hypothesis was important because it as the first theory that suggested one non-functional gene could result in an enzymatic block in metabolism. This hypothesis was proposed by studying different mutations in Neurospora, which is a good organism to use for studying since it is haploid.
ID013	The one-gene-one-enzyme hypothesis became the one-gene-one-polypeptide hypothesis by studying sickle cell anemia in hemoglobin. For example, the sickle cell trait occurs by a mutation on the beta chain of a protein in the sixth amino acid preventing the protein from cross-linking. This study shows that only one polypeptide in this quaternary structure protein is mutated and causes this missfolding.
ID021	Metabolism is the anabolic and catabolic reaction that cells use to create energy. An error in metabolism in this sense causes the reaction to not be able to proceed, and the products to not be available. Inborn errors of metabolism are mutations in the genes that encode enzymes that lower the activation energy of the metabolic reactions so they may proceed. The phenyalanine metabolism depends on such enzymes, and a phenylketonuria block cause PKU.
ID021	George Beadle showed that genes are directly responsible for enzymes. His studies with Neurospora lead to the one gene one enzyme hypothesis that one gene specifies one enzyme.
ID021	Sickle cell anemia is a human disease in which under low oxygenic conditions, erythrocytes become distorted due to a mutation in one of the beta protein chains of hemoglobin. This protein is not an enzyme, yet one gene encodes it. Therefore, the George Beadle was expanded to one gene- one polypeptide chain.
ID033	He did many experiments to determine if DNA has errors for metabolism.
ID033	Many experiments were done to determine that a single gene on DNA codes for a single polypeptide
ID033	Many different genes code for enzymes that combine and help form the blood cells for sick cell
ID131	Genes can have effect on entire body processes. This can come from only one enzyme not successfully being produced by an error in folding, translation, or transcription.
ID131	One gene for one enzyme was an astoundingly accurate hypothesis made at the time, though found later not always true. One gene codes for mRNA to be produced in transcription and then a polypeptide to be formed in translation.
ID131	The genetic basis for the disease is evidence because the disease can come as a result from a variety of different things. Also the quartenary structure has an influence as well.
ID082	Archibald Garrod studied alkaptonuria by looking at patterns of inheritance. He found that consanguineous parents, that were unaffected themselves, had a much higher chance of producing offspring with this rare but serious disease. Because of the differences in incidence rates between related and not related parents, Garrod concluded that because this condition was inherited, it must be a result of an alternative mode of metabolism controlled by hereditary information. He labeled what we now know to be genes are “unit factors” and what we now know as enzymes he called “ferments.” Garrod’s work, like many other brilliant scientists before him, was far ahead of his time, and he was only recognized for the importance of his work almost 30 years later.
ID082	Through his research done in 1933, George Beadle was the first to support the idea that genes are responsible for making enzymes in the body through evidence in his research. After beginning his experiments with mutant genes in fruit flies, he collaborated with Edward Tatum in investigation Neurospora crassa. After using X-rays to increase the mutation rate in Neurospora crassa, they used a series of minimal plating techniques supplemented with various molecules and nutrients in the biochemical pathway of tyrosine. They concluded that each of the mutants lacked an enzyme in the pathway, and therefore, when the induced mutations affected one gene that was responsible for one enzyme, the culture could not grow unless supplemented by the end product of the missing enzyme in the pathway.
ID082	Experiments with human hemoglobin and sickle cell anemia provided that the one-gene-one enzyme hypothesis was incorrect because it was too limiting. Different individuals express different specific phenotypic traits even when they have mutations on the same gene. The two factors that modified the one-gene: one-enzyme hypothesis were (1) while nearly all enzymes are proteins, not all proteins are enzymes, and (2) proteins often have a subunit structure consisting of two or more polypeptides. Through these experiments, the one gene: one-polypeptide chain hypothesis was established, as mutants seemed to affect only one polypeptide, which could then go on to form multi-subunit proteins.
ID026	Archibald Gerard did a lot of research over the one gene, one enzyme hypothesis. His research supported the idea that enzyme defects were mainly inherited by recessive genes from the parents and the effects that it may cause.
ID026	George Beadle studied the relationship between genes and enzymatic reactions. From his observations he made the theory that each gene codes for a specific enzyme. He was on the right track, however, in later years his theory was adjusted to mean one gene codes for one polypeptide.
ID026	Sickle cell anemia is an example of how the one gene, one enzyme theory is not correct. The 6th amino acid is different in a sickled cell as opposed to normal functioning one which causes the change. This proves that one gene does not necessarily code for one enzyme.
ID029	Inborn errors of metabolism are rare genetic disorders in which the body cannot properly turn food into energy. The disorders are usually caused by defects in specific proteins (enzymes) that help break down (metabolize) parts of food.
ID029	Each gene in a genome codes for a single enzyme that performs a specific function in a biological pathway.
ID029	The mutation that causes sickle cell anemia is due to the changing of one base pair. If the gene for hemoglobin was the only thing mutated, it should have affected only one enzyme. However, the whole hemoglobin molecule is defective.
ID116	Archibald Gerard studied alkaptonuria, which turns urine black due to the presence of gentesate in the body. During his research he realized that this disorder was more prevalent in persons who had parents that were closed related.
ID116	This hypothesis was suggested when studying Neurospora Crassa. It was the idea that a gene act through enzyme production. Therefore, each, single gene produces a single enzyme that affects single steps in the metabolic pathway.
ID116	With the invention of protein fingerprinting, it was shown that  genetic variations in proteins, like sickle cell hemoglobin, could only be in the polypeptide chain. This led to the "one gene-one polypeptide" hypothesis.
ID079	Gerard made this suggestion that genes control inborn errors of metabolism after studying numerous genetic disorders. His work showed that every step is controlled by proteins/enzymes, and therefore, is controlled by genes.
ID079	Beadle developed his hypothesis of one gene: one enzyme by discovering that genes play a direct role in enzyme synthesis. However, his one gene: one enzyme hypothesis is not always true; we now know that the ratio is one gene: one polypeptide.
ID079	Beadle's hypothesis was tested through studying hemoglobin alleles of sickle-cell anemia. The normal hemoglobin has an alpha polypeptide and beta polypeptide, and the sickle cell hemoglobin has the same alpha polypeptide but an altered beta polypeptide; one peptide of the sickle-cell hemoglobin is altered. The beta chains of the normal and sickle-cell hemoglobin are the same until the 6th amino acid.
ID022	Gerard first gave the idea of Inborn errors of metabolism. He showed that they are rare genetic disorders in which the body cannot properly turn food into energy. The disorders are usually caused by defects in specific proteins or enzymes that help metabolize parts of food. if the food is not metabolized and if energy is not produced can lead to a wide array of symptoms. This was the main idea of Gerard's inborn errors of metabolism.
ID022	George Beadle used Neurospora crass to prove one gene one enzyme hypothesis. He chose Neurospora crass because it spent most of its life in haploid state so they dont have to worry about second set of genes or dominant or recessive alleles.  In the lab they allowed this Neurospora to grow in a minimal culture. In order for it to grow normally it should have all the genes so it has to be protrotrophic. They reasoned that if they mutate any one of the genes that makes an enzyme, for example gene for arganin, then they should get a Neurospora strain that cannot grow on minimal medium. The mutant would be able to grow if only they added the enzyme product as a supplement. There was  7 mutation they found which did not allow the conversion of citrulline to argenine. Based on this and x-ray method they concluded that one gene transcribes one enzyme.
ID022	The reason one gene one enzyme hypothesis was later changed to one gene one polypetide hypothesis is because of more complicated proteins like Haemoglobin. Haemoglobin have 4 polypeptides chain that come together and combine and make a functional protein that functions normally. Therefore, one gene one enzyme hypothesis was later modified when it was realised that genes also encoded nonenzyme protiens and polypeptide
ID040	One of the disorders that Gerard studied was alkaptonuria. He looked at the patterns of inheritance for this disease and found that it was genetic in nature. Of the many families he studied, the majority of the cases of alkaptonuria were confined to families. Families that had cases were sometimes found to have first cousins that were married, which is more likely to bring back recessive genes to offspring. This information led Gerard to conclude that the inherited condition was the result of a different mode of metabolism, which implied that hereditary information controls chemical reactions in the body.
ID040	Beadle and Tatum worked with Neurospora and induced different genetic mutations to produce strains that had genetic blocks of reactions essential to the growth of the organism. They isolated and studied hundreds of mutants and it seemed that each nutritional mutation caused the loss of the enzymatic activity that facilitates an essential reaction in the wild-type organism. This provided solid evidence that one gene specifies one enzymes.
ID040	Work began to arise that showed that not all genes coded for enzymes. Work with sickle-cell anemia showed that genes may specify proteins other than enzymes. People with sickle-cell anemia had mutant hemoglobin molecules. Pauling concluded that a chemical difference exists between normal and sickle-cell hemoglobin. Further research showed that a gene codes for a single polypeptide chain, showing that Beadle and Tatum were on the right path but not specific enough in their original hypothesis.
ID044	Archibald Gerard referred to inherited diseases as inborn errors of metabolism.  These are a large class of congenital disorders of metabolism. Most are caused by defects of single genes that code for enzymes that catalyze reactions. Most of the disorders are due to  toxic substances that interfere with normal function or reduce the ability to synthesize essential molecules.
ID044	This is the idea that genes produce enzymes, where each gene is responsible for producing a single enzyme. The concept was proposed by George Beadle and Edward Tatum in a 1941 paper on genetic mutations in mold, Neurospora crass. This finding is often considered the first significant result in molecular biology.
ID044	Red blood cells normally have protein (not an enzyme) hemoglobin A, which consists of two alpha and two beta chains. People with sickle cell anemia have a mutation in the beta chain, but there can be many different variations. This shows that the genes for hemoglobin do not code for one enzyme only.  It was proven through protein fingerprinting that genetic variations in proteins like sickle cell hemoglobin could be limited to differences in a single polypeptide chain, which lead to a one gene-one polypeptide hypothesis instead
ID103	Only recently have we understood that some diseases are related to genetics. Gerald is the person credited with coming up with the notion.
ID103	Beadle had the idea that each enzyme is produced by one gene and an error in the function of an enzyme is a result of an error on that gene.
ID103	A more modern understanding of the one gene one enzyme hypothesis is that this isn't always the case. Sickle cell anemia is an example that can be used to understand why this is not always the case.
ID101	He studied Alkaptonuria, which causes black urine. Archibald discovered that it had a Mendelian recessive trait. He postulated correctly that it was an error in the metabolic pathway.
ID101	The one gene one enzyme hypothesis is that the changing of one gene will change one enzyme, so a mutation would disable a single enzyme if the protein is not viable. This is not actually the case, for multiple polypeptide chains can make up a single enzyme, therefore, a better hypothesis is one gene one polypeptide chain.
ID101	The sickle cell anemia occurs when a mutation occurs and one of the sub units of the beta sheet where glutamic acid becomes a valine amino acid which prevents oxygen from correctly binding to hemoglobin.
ID091	Archibald Gerard used mendelian genetics and physical observation to suggest that genes control metabolism. He looked at a rare disease, alkaptonuria, that causes brown to black urine. He looked at families with the disease and concluded it was inherited recessively on an autosomal chromosome. So, it seemed that a gene was affecting the metabolism of the individuals.
ID091	George beadle used neurospora and e. coli to arrive at his hypothesis. He looked at the metabolic pathway of arginine. He saw that different mutants stopped at different points in the pathway.  This suggested that a specific enzyme was missing as a result of the different mutations.
ID091	Sickle cell anemia is a disease of hemoglobin. Hemoglobin has a tertiary structure with 4 subunits, 2 alpha domains and 2 beta domains. A mutation in different genes can cause sick cell anemia.  A healthy hemoglobin enzyme requires 4 separate polypeptides be produces correctly. So, multiple genes are required. So, it is more accurate to say one gene codes for one polypeptide.
ID081	Inborn errors accoutn for a large number of genetic diseases of metabolism. Often, errors are due to defects of single genes that code for enzymes which facilitate the conversion of various substances or substrates into products.
ID081	This hypothesis proposed that one gene directly codes for one enzyme within individual.
ID081	The genes for sickle cell anemia allow for an individual to be immune to malaria when they are heterozygous for the allele. These individual still have some syptoms of sickle cell anemia. As a result, there is more than one enzyme found to be related to only one gene. A better hypothesis is the one-gene-one-polypeptide.
ID031	This is talking about how gerard researched and found out that genes can act as a mechanism that sort of corrects internal errors of metabolism by coding for other things.
ID031	This is just talking about how for every enzyme their is specifically one gene that goes with and these genes work together to form chemical pathways.
ID031	This showed that the same enzyme was not associated with just one specific gene but two genes that were closely similar though.
ID105	Garrod studied alkaptonuria,which is a disorder that makes urine turn black. This is due to the presence of gentesate. He also noticed that it was prevalent among populations whose parents were closely related this inbreeding led to the errors in metabolism.
ID105	This was the idea that each individual gene coded for a single specific enzyme. This was eventually found to be over simplified and was changed. However, it was a very important step in the right direction.
ID105	Sickle cell anemia hemoglobin showed that variations due to just a single protein could be due to just a single polypeptide chain, leading to the one gene one polypeptide hypothesis.
ID023	Gerard knew that genes controlled metabolism errors that eventually led to the discovery of mutations which then led to further studies into the various categories of mutations and their effect. Concerning metabolism, in E. coli, if one metabolism pathway contains a mutation, the organism will not be able to metabolize other amino acids in sequence after the mutation.
ID023	The one gene one enzyme hypothesis states that each gene directly produces one enzyme. However, in actuality, not all proteins are enzymes and multiple codons can code for the same protein.
ID023	Sickle Cell anemia demonstrates that multiple causes and mutations can cause the disease; not just one. This demonstration directly rejects the one gene one enzyme hypothesis.
ID150	While studying the disease alkaptonuria, Garrod proposed that diseases are the result of genes not properly producing what is needed for the body's chemical pathways. This was the first proposal of a biochemical basis in genes.
ID150	The one gene-one enzyme hypothesis states that genes are responsible for the enzymatic activity in cells. Simply put, each gene has a sequence that encodes for one enzyme.
ID150	Studies of sickle cell showed that a mutation in a single polypeptide can change the function of the organism. What resulted was the one gene-one polypeptide hypothesis that states that each gene encodes for one polypeptide, which can function by itself or be paired in a multimeric compound.
ID147	Inborn erros of metabolism are genetic mutations and disorders that prevent the body from converting food into energy because of a distrubance in the metabolic pathway usually involving a disfunctional gene that encodes a dysfunctional enzyme.
ID147	Beadle proposed that each gene only encodes for one specific enzyme. He thought that for every enzyme in a metabolic pathway each had its own unique gene that did not encode for any other enzymes/poplypeptides.
ID147	Sickle cell anemia is a disorder of the Hemoglobin molecule's protein structure. Since it is made up of multiple polypeptides it can be seen that the one-gene one-enzyme relationship is an oversimplification. It is closer to a one-gene one-protein replationship. Heterozygous individuals are not affected since they produce both normal and sickle hemoglobin but homozgous recessive individuals are affected.
ID113	He was the first to suggest that genes dictate phenotype through enyzmes that catalyze specific chemical reactions in the cell
ID113	the theory that each gene directly produces a single enzyme, which consequently affects an individual step in a metabolic pathway
ID113	sickle-cell anemia was the first direct evidence that proved that genes specify proteins other than enzymes
ID134	Gerard was an english physician who noted that several inherited diseases showed harmful alterations in body
ID134	Genes are directly responsible for the synthesis of enzymes. Both he and Tatum showed through nutritional mutations in bread mold in Neurospora directly caused loss of enzymatic activity. Through this testing they were able to conclude that each gene code for a specific enzyme which has a specific purpose within an organism and therefore that mutations or other circumstances that affect those genes will obviously affect the synthesis of its particular enzyme.
ID134	The mutation what causes this is a change to a specific protein, while more than one gene change can produce that. The gene to enzyme hypothesis, however, underestimates this through its simplicity and not accounting for the complexity of mutations or the complexity of gene coding and protein production
ID035	Garrod discovered inborn errors of metabolism. He noticed that it was prevalent among populations whose parents were closely related.
ID035	The one gene-one enzyme hypothesis is the idea that genes act through the production of enzymes. Each gene responsible for producing a single enzyme that affects a single step in a metabolic pathway.
ID035	Ingram showed through protein fingerprinting that genetic variations in proteins could have differences in polypeptide chain in a multimeric protein leading to the, one gene-one polypeptide, hypothesis
ID087	Genes coded for the proteins needed to carry out certain steps within metabolism. If there is an error in the genes, then there is an error in metabolism.
ID087	He proposed that one gene coded for one enzyme.
ID087	One gene doesn't always code for one enzyme. However, a gene can code for several enzymes or none at all.
ID118	Gerard demonstrated that several inherited diseases showed harmful alternations in body chemistry. His hypothesis was based on black urine, genetically determined because families with alkaptonuria had several affected members. Alkaptonuria is much more common in 1st cousin marriages than marriages with unrelated partners. Alkapeonuria results from homogentisic acid (HA) in the urine. Garrod reasoned that normal people metabolize HA, but those with alkaptonuria do not because they lack the necessary enzyme. recessive on chromosome 3.
ID118	George Beadle proposed one gene-one enzyme hypothesis because enzymes are always made up of protein. The function of enzyme is described by the arrangement of polypeptide chain and the peptide fragments in the protein structure. One gene can impact the individual path of the protein and that can impact the metabolism pathway. But late on the hypothesis was changed to one gene-one polypeptide hypothesis as not all protein are enzymes. each gene would be responsible for the enzyme facilitating a single step in a metabolic process.
ID118	One gene one enzyme hypothesis was changed to one gene one polypeptide chain hypothesis because nearly all enzymes are proteins but not all proteins are enzymes and proteins have subunit structure with two or more polypeptide chains. Two molecules are designated for sickle cell anemia HbA and HbS they differ by single peptide fragment on number 6 position HbS has valine instead glucin which alters the entire function of the protein.
ID140	Inborn errors are things like congenital diseases. Garrod suggested that these were caused by single gene mutations that would change one substance to another.
ID140	His hypothesis was that each gene was responsible for producing one specific enzyme which would then further the metabolic pathway.
ID140	Sickle cell anemia is due to a change in polypeptide which would make the one-gene-one-enzyme theory more accurate if it was the one-gene-one-polypeptide theory.
ID124	Garrod was an MD who focused on rare and unusual birth defects like alkaptonuria.
ID124	One gene-one enzyme is a the theory where each gene makes an enzyme but some proteins are not enzymes. This was determined after his experiments with Neurospora mutants.
ID124	One gene-one polypeptide is more inclusive. It was show that differences in proteins could occur from a single polypeptide.
ID133	Gerard was studying the disorder alkaptonuria in which individuals have a disruption in an important metabolic pathway and cannot metabolize homogentisic acid. He determined that this disorder is much more common in 1st cousin marriages.  He postulated that these people cannot metabolize homogentisic acid because they lack a certain enzyme to do so.
ID133	Beadle showed nutritional mutations in Neurospora caused loss of enzymatic activity. It also appeared that a mutation could be found for nearly any enzymatically controlled reaction. His experiment led to his hypothesis that one gene specifies one enzyme.
ID133	Normal and sickle-cell hemoglobin chemically differ by two molecules: HbA and HbS.  The fingerprinting technique demonstrated HbA and HbS hemoglobins differ by a single peptide fragment.
ID083	While the rate of mutation can increase or decrease, mutation itself is random and does not occur in response to 'need'. Rather, mutation occurs and specific mutations may be selected for in a population if they are adaptive (provide a reproductive advantage)
ID083	While the rate of mutation can increase or decrease, mutation itself is random and does not occur in response to 'need'. Rather, mutation occurs and specific mutations may be selected for in a population if they are adaptive (provide a reproductive advantage)
ID083	While the rate of mutation can increase or decrease, mutation itself is random and does not occur in response to 'need'. Rather, mutation occurs and specific mutations may be selected for in a population if they are adaptive (provide a reproductive advantage)
ID098	Archibald Gerard noticed that some inherited diseases resulted in harmful alterations in biochemistry and suggested that genes played a role in these alterations. He made this suggestion in his 1923 book Inborn Errors of Metabolism, which was the first suggestion of how genes worked.
ID098	George Beadle was the first to show that genes are directly responsible for the synthesis of enzymes. Beadle observed that some mutations would render the mold Neurospora incapable of growing in a minimal medium, but the same Neurospora colony could grow in a medium that provided a specific enzyme. This led him to form the one gene, one enzyme hypothesis.
ID098	We have learned from sickle cell anemia that hemoglobin is not formed by a single protein from a single gene. Instead, it is formed by an alpha chain and a beta chain, which are encoded by different genes. Sickle cell anemia occurs in people with a mutation on the gene that codes for the beta chain. This means that one gene makes one polypeptide, but not necessarily one enzyme.
ID089	He showed that genes are responsible for what he called inborn errors of metabolism, what we now call genetic disorders. Because he was so early in the science of genetics, there is little data or discovery associated with his inborn errors of metabolism, just that genes are associated with errors.
ID089	Using neurospora, they determined that a single mutation results in a single deficiency or missing product.
ID089	One gene one enzyme hypothesis is not entirely correct because some enzymes are made up of multiple proteins and so there are multiple genes involved in the translation of proteins with multiple subunits.
ID095	Garrod formulated the hypothesis that hereditary information controls the chemical reactions in the body.  He observed that the disorder alkaptonuria (a rare trait) was more likely to be expressed among consanguineous  offspring.  Knowing that recessive heterozygous traits have a higher chance of being expressed as a result of inbreeding, he concluded that there had to be a heritable trait passed on to individuals to code for enzymatic processes.
ID095	Beadle's test with Neurospora proved that mutating a gene results in altering the functionality of the organism.  By isolating particular mutants with their complementary minimal media growth, he was able to determine that mutating a gene in an organism could render the organism unable to grow without having the supplement added to the medium.  This proved that a single gene coded for the organism to synthesize a particular supplement.
ID095	Vernon Ingram proved, through the use of fingerprinting technique, that valine was substituted for glutamic acid at a particular position in the Beta chain of hemoglobin.  This mutation is a recognized trait in sickle cell anemic patients, and this proved that a single mutation in the polypeptide chain of the Beta hemoglobin results from one gene alteration.
ID056	Archibald Gerard discovered the 'inborn errors of metabolism' by watching the development of alkaptonuria via the inability of metabolize homogentistic acid. Through his observations he was able to conclude that medical conditions were due to errors of mutant genes.
ID056	Beadle ran two different experiments to provide information in support of the one gene-one enzyme hypothesis. His first experiment showed that mutant genes altered eye color in Drosophila due to the loss of enzyme function. The second experiment performed involved the growth of Neurospora mutants on complete and minimal medium.
ID056	Sickle cell anemia was the basis for establishing that one-gene:one-protein hypothesis and further the one-gene:one-polypeptide chain hypothesis. This was shown through the alteration of red blood cells from the substitution of an amino acid that composes a polypeptide chain.
ID111	Gerald studied alkaptonuria by looking for patterns of inheritance. He eventually concluded that it was genetic in nature. Many of the cases of this disease were confined to only a few families. Often times the parents were unaffected, but were related as first cousins-consanguineous. These parents have a higher probability of of producing offspring with expressed recessive traits. Gerald concluded that this inheritance condition was the result of an alternative mode of metabolism, which implies that hereditary information controls chemical reactions in the body.
ID111	He studied Neurospora. His study concluded that each gene directly produces a single enzyme, which consequently affects an individual step in the metabolic pathway.
ID111	Vernon Ingram showed this through protein fingerprinting. This showed that genetic variations in proteins, such as sickle cell hemoglobin, could be limited to differences in just a single polypeptide chain in a multimeric protein, leading to a one gene-one polypeptide hypothesis instead.
ID077	Inborn errors or metabolism are genetic diseases involving congenital disorders of metabolism. The majority are  due to defects of single genes that code for enzymes that facilitate conversion of various substances into others.
ID077	Beadle provided the first convincing experimental evidence that genes are responsible for the synthesis of enzymes. He confirmed that mutant genes that altered the eye color of fruit flies could be linked to biochemical errors that involved the loss of enzyme function. He then went and investigated nutritional mutations in the pink bread mold Neurospora crassa which lead to the one-gene:one-enzyme hypothesis.
ID077	Modification of the one-gene:one-enzyme hypothesis came about when it was recognized that nearly all enzymes are proteins, not all proteins are enzymes. Also, proteins often have a subunit structure consisting of two or more polypeptide chain which lead to one-gene:one-polypeptide hypothesis.
ID090	In 1923, Gerard proposed a theory that certain diseases were due to hereditary errors in pathways that control chemical reactions in the body. While the terms “genes” and “enzymes” were not yet in use, the idea was the same: Gerard believed that certain diseases were due to defects in a pathway that would convert various substances into other products necessary for normal function.
ID090	The work of Gerard was the basis for the Beadle’s proposal of the one gene-one enzyme hypothesis. The hypothesis is based on the concept that genes are able to act through enzymes and that each gene codes for the translation of one enzyme. The enzyme produced is then used in a step of a metabolic pathway.
ID090	While the one gene-one enzyme hypothesis is very nearly accurate, it is not completely correct. Through studies of sickle cell anemia, it was able to be concluded that there were 2 main errors with the one gene-one enzyme hypothesis. First, while all enzymes are proteins not all proteins are enzymes. Second, proteins often consist of many polypeptides, and each polypeptide is coded for by a different gene. This led to the establishment of the more accurate one gene-one polypeptide hypothesis.
ID020	One example of an inborn error is alkaptonuira where rine turns black; contains alkapton. It was hypothesized a missing enzyme for breaking down alkapton. Another example would be phenylketonuria. This disorder can lead to mental retardation and is transmitted as an autosomal recessive disease.
ID020	The hypothesis of one gene-one enzyme is the idea that hat genes act through the production of enzymes, with each gene responsible for producing a single enzyme that in turn affects a single step in a metabolic pathway. This was studied with the mutation in the pink bread mold Neurospora crassa.
ID020	Sickle cell anemia lead the one-gene-one-enzyme hypothesis is be changed to one-gene:one-polypeptide chain hypothesis because of the hemoglobin structure in individuals afflicted with sickle-cell anemia.
ID027	He discovered that organisms had mutations that disrupted different levels of the organism's synthesis of required molecules. He didn't entirely understand it well enough to describe in further than errors of metabolism.
ID027	Beadle suggested that one gene codes for one enzyme because enzymes are usually not able to be synthesized due to a mutation of a single gene. His prediction was mostly correct.
ID027	Sickle cell anemia shows that a single gene is responsible for a single protein rather than a single enzyme. This was because many proteins make up a single enzyme, and some genes can code for multiple proteins due to alternate splicing.
ID058	Alkaptonuria results from homogentisic acid (HA) in the urine. Garrod reasoned that normal people metabolize HA, but those with alkaptonuria do not
ID058	When studying Neurospora, they figured out that genes are definitely responsible for synthesis of enzymes and came up with the one gene-one enzyme hypothesis.
ID058	The hypothesis was proven wrong and changed to one gene-one polypeptide because some proteins are made up of more than one polypeptide such as hemoglobin.
ID141	Before Archibald Gerard looked into inherited metabolic disorders, diseases were not looked at as something that is genetic in nature, but was always assumed to be the effect of viruses/parasites.  However, after looking into some specific situations, Gerard suggested that genes controlled inborn errors of metabolism by analyzing how certain reactions in a pathway can cause diseases if blocked.
ID141	George Beadle and Edward Tatum proposed this one gene-one enzyme hypothesis studying the bread mold Neurospora crassa.  They grew the mold on complete medium and if it would not grow, they knew that it must have a nutritional mutation.  They then by trial and error plated the mutated strain on different mediums with different supplements until the deficiency was targeted.  Whatever this deficiency was the specific thing that the mold could not synthesize.
ID141	The one gene-one enzyme hypothesis changed to the one gene-one polypeptide chain hypothesis due to two factors.  One because nearly all enzymes are proteins, and two because proteins have a subunit structure that are at least 2 polypeptide chains.
ID108	Inborn errors of metabolism are defects in single genes that encode for important enzymes that convert particular substances in the body into a different form. If a non-converted substance builds up in the body, it could be toxic to the person. Garrod studied alkaptonuria which was a disease caused by the buildup of phenylalanine and tyrosine which produces black urine and darkened cartilaginous areas on the affected person.
ID108	One-gene-one-enzyme is the theory that genes act through the enzyme produced, and each gene is responsible for making a single enzyme. Each enzyme affects a step in the metabolic pathway. Beadle studied Neurospora crassa to show that there were failures in the metabolic pathways due to errors in specific enzymes. This led to the thought that genes code for specific proteins like enzymes.
ID108	After the one-gene-one-enzyme hypothesis, while looking at the genetic basis of sickle cell this hypothesis was found to be too vague. Protein fingerprinting was used to show that in sickle cell hemoglobin there were differenences in the single  polypeptide chain of the protein. This led to the one-gene-one-polypeptide hypothesis.
ID063	This is a discovery of inborn errors in metabolism. He used epistaxis to explain his discovery. He did this studying alkaptonuria.
ID063	Genes code for enzymes. Therefore, each gene has the necessary codes for the required enzymes.
ID063	This discovery led to the idea of one gene one polypeptide. The anemia was found to not be caused by a one gene one enzyme.
ID127	He discovered the rare case of alkaptonria which has to do with a defect in the metabolism. This is an example of inherited disorders of the metabolism. The most common symptom of this disorder is the blackening of the urine and is usually seen in the diapers of infants. The mutation is in a gene encoding for an enzyme that helps the metabolism.
ID127	After the discovery of alkapotonria, George Beadle studied the pigments in the eyes of fruit flies and hypothesized that one gene was only responsible for one enzyme. This was because he believed that each gene was responsible for affecting an enzyme in the fruit fly metabolism.
ID127	Protein finger printing make the one enzyme-one polypeptide hypothesis not likely to be correct. This is because there was variation in a single proteins which means one gene was more likely responsible for one polypeptide instead of enzyme. Sickle cell anemia is where the variations in proteins were observed.
ID123	Inborn errors in metabolism are rare genetic disorders in which the body cannot breakdown food into useful energy.  This is due to the defects in the specific proteins or enzymes which metabolize food.  These proteins are coded for by specific genes, thus genes control the inborn errors of metabolism.
ID123	The one gene-one enzyme hypothesis is a theory which explains that one gene produces one enzyme that affects a step in the metabolic pathway.  Beadle proposed this hypothesis by demonstrating that one gene in a fruit fly controlled one metabolic reaction which was also controlled by a specific enzyme.
ID123	The genetic basis of sickle cell anemia demonstrates that the one-gene-one enzyme hypothesis is not quite correct because the difference in the protein that causes sickle cell anemia is in the polypeptide.  It is the result of a change in a single polypeptide of the sickle cell anemia hemoglobin.  This shows that the one gene one enzyme hypothesis is not right.  It shows that one gene can code for one polypeptide.  All enzymes are proteins, but not all protein are polypeptides.
ID080	He was the first person to consider these metabolic disorders to be genetic based instead of another type of disease. It made the way they were dealt with and thought of different.
ID080	He believed that each gene controlled one specific enzyme but was later proven to be one gene one protein
ID080	It showed that a certain gene only effected one little component of the enzyme and not the whole entire thing.
ID151	Archibald Gerard was the first to suggest a possible linkage between errors in ones genetic makeup and the possibility of disorders. He claimed that each gene contains some function to ones phenotype; furthermore, if a gene is nonfunctional in some manner, it could result in errors within a metabolic pathway thus resulting in disorders. He also determined that due to the fact that the genetic code is passed down from generation to generation, so can certain diseases depending on the location of the mutated cell.
ID151	This hypothesis that was created by Beadle was the first evidence of the Central Dogma, but this statement was revised later to be more specific which is given below.
ID151	The phrase one gene-one enzyme was changed to one gene-one polypeptide once hemoglobin was studied. Our blood, hemoglobin, is a quarterly structure composed of two alpha chains, two beta chains, and heme groups.
ID142	Gerard had observed that due to a change in gene sequence, as a consequence, humans could have defective metabolism processes.
ID142	Beadle was studying nutritional mutations in pink bread mold (Neurospora) when he hypothesized the one gene-one enzyme hypothesis in which one gene in DNA would produce one enzyme via transcription and then translation.
ID142	Studies done on sickle cell anemia furthered scientist knowledge of Beadle's original hypothesis only to change it slightly to one gene- one protein (one polypeptide) because enzymes are a type of protein and proteins complete many needs in cells.
ID092	Genes control these errors because the mitochondria is responsible for metabolism thus when genes are transcribed wrong, there are problems with the mitochondria.  These problems with mitochondria are the "inborn errors" Gerald was talking about.
ID092	Beadle proposed this hypothesis because at the time, they thought one gene only coded for one enzyme.  However, as technology advanced, scientists soon realized that many genes can code for the same enzyme, thus the theory is better known as one gene-one polypeptide.
ID092	Sickle cell anemia demonstrates Beadle's error because if one gene-one enzyme was correct, then there would only be one source of sickle cell anemia.  However, we now know that multiple genes can be the basis of sickle cell.
ID088	When Garrod studied alkaptonuria, a disorder that makes urine quickly turn black due to the presence of gentesate, he noticed that it was prevalent among populations whose parents were closely related. This showed how closely related people reproducing could result in errors.
ID088	Beadle and Tatum showed nutritional mutations in bread mold Neurospora caused loss of enzymatic activity. This led to the one gene-one enzyme hypothesis.
ID088	Nearly all enzymes are proteins, but not all proteins are enzymes. Proteins have subunit structure with two or more polypeptide chains. Analysis demonstrated that HbS and HbA hemoglobin differ by single peptide fragment.
ID145	The "inborn errors of metabolism" referred to inherited mutations that affected the body's chemistry.
ID145	The hypothesis was proposed based off studies of neurospora that showed that genes are directly responsible for the synthesis of enzymes.
ID145	The hypothesis was changed to one-gene-one-polypeptide because the difference in the two designated hemoglobins of sickle cell were only different by one polypeptide.
ID069	Upon noting that multiple inherited diseases displayed harmful alterations in body chemistry, Archibald Gerard suggested that inherited disease was caused by an "inborn error of metabolism," thus first suggesting how genes work. Gerard studied families with alkaptonuria (causes black urine as a result of homogentisic acid in the urine) of whom affected members were more common as a result of first cousin marriages than unrelated marriages. He reasoned that healthy people metabolize homogentisic acid, but people afflicted with alkaptonuria lack the necessary enzyme and thus cannot metabolize HA.
ID069	George Beadle's experimented with nutritional mutations in bread mold and showed that genes are directly responsible for the synthesis of enzymes. He added radiation to conidia, then monitored its growth in the presence of amino supplements. Because mutant cells only grew when tyrosine was added, he determined that the mutation affected the synthesis of tyrosine.
ID069	While nearly all enzymes are proteins, not all proteins are enzymes. This, in addition to the fact that proteins have subunit structures consisting of two or more polypeptide chains, revamped the one gene-one enzyme hypothesis into the one gene-one polypeptide hypothesis. In regards to sickle cell anemia, chemical differences exist between normal and sickle-cell hemoglobin, and fingerprinting showed that these hemoglobins differed by a single peptide fragment.
ID002	Gerard's suggestions suggested that errors in genes affected metabolism so that certain individuals cannot metabolize certain substances. Certain diseases are a result of this phenomena
ID002	Beadle proposed the theory that each gene directly produces a single enzyme, which consequently affects an individual step in a metabolic pathway
ID002	The one-gene-one enzyme hypothesis does not hold quite true with proteins in that they can differ just by a single polypeptide fragment, such as that in sickle cell anemia. The single difference in the peptides changes the complex of the cell
ID016	The one gene one enzyme hypothesis states that a single gene produces a single enzyme only, and that enzyme affects a metabolic pathway. This hypothesis is too simple considering the complex level of regulation that occurs in the regulation of eukaryotic and prokaryotic gene expression.
ID016	Genes can code for polypeptides. Sickle cell anemia demonstrates that a single mutation can cause a drastic disease. Instead of an enzyme being the product of the central dogma, a polypeptide is the product. A single gene provides information for a polypeptide chain.
ID017	The majority are due to defects of single genes that code for enzymes that facilitate conversion of various substances
ID017	Each gene directly produces a single enzyme, which consequently affects an individual step in a metabolic pathway.
ID017	Sickle cell anemia is when red blood cells are misshapen and broken down, and this slightly disproves the one-gene-one enzyme hypothesis because no enzymatic process is affected as well as any of the metabolic pathways. Although respiration is difficult for people with sickle.
ID008	When there is an error in metabolism, the body can not make food into energy properly.
ID008	He hypothesized that genes work through enzymes. Each genes produces its' own enzyme.
ID008	variations in proteins causes differences in each polypeptide chain which verified that the one-gene-one enzyme hypothesis was false.
ID078	used black urine to determine the cause was hereditary and that a different mode of metabolism was responsible
ID078	this hypothesis arose from causing a mutation in neurospora and observing the lack of a functional protein.
ID078	one gene encodes for a single polypeptide, which can be edited to make a protein, or form a complex with other polypeptides to make an enzyme
ID038	Since genes code for all proteins found in the body, metabolism errors arise due to defects in genes. Metabolism is normally controlled by proteins and if the genes are defective the proteins will be non functional and there will be errors in metabolism
ID038	He observed yeast cells and upon mutation of a gene he saw that the protein corresponding to that gene was defective. he thus concluded that each gene coded for a protein.
ID038	Only one mutation on a gene causes the diseases. However, the protein is made up of several polypeptides and there are no mutations in other proteins. Thus the correct hypothesis is one gene one polypeptide and not protein. protein are made up of many polypeptides.
ID009	Through his investigation of alkaptonuria, a disease that involves a disorder in an important metabolic pathway and is persistent, but not serious, throughout life. His study helped him to decide that it was a hereditary disease, most of the cases came from a few certain families. This leads to the chemical reactions in the body having to adjust to suit the disruption in metabolism. Therefore, this inherited trait resulted in transformations in the body.
ID009	He and Tatum experimented with Neurospora since a lot was known about its biochemistry compared to other things at the time. After doing numerous tests on mediums, their results suggested that per every gene mutation, there was a reaction regulated by enzymes.
ID068	Gerard noticed that inherited diseases caused chemical changes in the body. He said this was caused by "inborn errors of metabolism". These errors are mutations affecting processes of metabolism. One example is Phenylketonuria, which phenylalanine cannot be converted to tyrosine and can cause severe health problems.
ID068	George Beadle discovered that genes are directly responsible for the formation of proteins. He believes that one gene would encode for one full/complete protein/enzyme. He did this by studying Neurospora and their biosynthetic pathway. His hypothesis was later corrected to what we now know as one gene one polypeptide.
ID068	The one gene one enzyme hypothesis was disproved and shown to be that one gene one polypeptide is true. Many proteins/enzymes are made up of more than one polypeptide so the first hypothesis could not work. In studying sickle cell anemia, 2 different molecules of hemoglobin were discovered and only differed by one peptide fragment. This one fragment made the different between normal RBC or sickle cell.
ID028	The inborn errors of metabolism present problems within a cell. An example is in alkaptonuria victims. People affected with this genetic disease cannot metabolize homogentisic acid and their urine is blackened and around cartilaginous areas.
ID028	The one gene one enzyme hypothesis was proposed by George Beadle in his work with Neurospora. The hypothesis suggests that any biochemical pathway involving a single gene will result in a single enzyme product.
ID028	Instead of the one gene one enzyme hypothesis, studies about the genetic basis of sickle cell anemia showed that it is instead a one gene one polypeptide hypothesis.
ID059	genetic diseases that involves congenital disorders of metabolism. The defets occur due to the function of single genes that code or enzymes that are used for a variety of other substances into other products.
ID059	George Beadle proposal of the one gene-one enzyme hypothesis is due to the process of genes producing enzymes. Genes is responsible for making enzymes and then turn other functions into single metabolic pathways.
ID059	They were able to discover the fact that one gene-one enzyme was in fact not the exact cause of sickle cell anemia. It was indeed the cause of one mutation changing one amino acid which had a direct relation with a DNA molecule and a peptide chain.
ID039	Gerald had worked as a physician and he saw that inherited diseases displayed harmful changes in body chemistry. He did more research and In 1923, he published his findings and noted that it was due to inborn errorsof metabolism. His work was the first to suggest how genes might work
ID039	Beadle came up with the concept of one gene one enzyme. Beadle worked with neurospora and demonstrated that for the synthesis of each enzyme there was one gene that controlled the process and was directly responsible for enzyme synthesis
ID039	Sickle cell anemia demonstrated that the one gene one enzyme hypothesis was not correct. sickle cell anemia showed that it was instead one gene one polypeptide
ID048	An explanation for epistasis is an important manifestation of Gerard's research, albeit indirectly. When Gerard studied alkaptonuria, a disorder that makes urine quickly turn black due to the presence of gentesate, he noticed that it was prevalent among populations whose parents were closely related.
ID048	genes work through enzymes, each gene produces a single enzyme that affects a single metabolic pathway.
ID048	One-gene:one-protein hypothesis changed to
ID075	In the early 1900’s, the English physician,Archibald Gerard, noted that several inherited diseases showed harmful alterations in body
ID075	George Beadle showed genes are directly
ID075	Sickle-cell anemia is a recessive genetic disease. The afflicted individuals are homozygous for HbS
ID064	Gerard was an English doctor who noticed that patients with metabolic deficiencies had disorders (PKU, alkaptonuria, etc) characterized by a lack of something, and these conditions were genetic, hence the "inborn".
ID064	Beadle did experiments with Neurospora that showed mutants only grew when whatever they were deficient in was provided them in the medium. Since Neurospora has a fairly linear biochemical pathway to produce arginine, the examination of mutants in the pathway led Beadle to propose that one gene made one enzyme product.
ID064	Sickle cell anemia is caused by a defective hemoglobin protein, which is defective because of a missense mutation caused by an incorrect amino acid in the peptide chain. This disproved the one-gene-one-enzyme hypothesis because there was not a lack of enzyme produced, but an issue with a polypeptide chain. Therefore the theory was modified to say one gene codes for one polypeptide.
ID050	He hypothesized that the breakdown of the production of a certain enzyme could lead to the entire metabolic pathway to fail and then thereby causing all sort of issues with the human body.
ID050	This hypothesis referred to the notion that a certain enzyme was only produced by a certain gene and that each enzyme was needed to activate a certain step in the metabolic pathway.
ID050	Sickle cell lead to the understanding it may not be a one gene one enzyme pathway but rather a one gene one substrate pathway. This discovery came about through the understanding that all proteins were enzymes.
ID137	He called inborn errors inherited diseases, and if a person had an inherited disease, then they would be unable to synthesize specific enzymes.
ID137	He hypothesized that every gene each synthesized a specific enzyme that would would be part of one step in a metabolic process.
ID137	Through protein fingerprinting, it was discovered that genetic variation in proteins, for example the single base pair mutation of sickle cell, would have differences in only one polypeptide chain. So it became the one-gene-one polypeptide hypothesis.
ID057	Gerard discovered that there were inborn errors of the metabolism. He discovered this by studying alkaptonuria, this disorder causes urine to turn black. He noted this disease was prevalent in parents who were cousins or closely related.
ID057	This hypothesis stated that genes act through the production of enzymes, which can affect various metabolic pathways. George developed this hypothesis by observing genetics mutations in mold,  Neurospora crassa. Although this hypothesis helped our understanding of molecular biology this hypothesis wound up being an over simplification of what was actually occurring.
ID057	When observing the sickle cell mutation in hemoglobin they needed to know if the mutations occurred in the alpha or beta chain. At the time the amino acid sequences of these chains had not been determined. Eventually it was found out that the mutation occurred in the beta chain. This lead to the revision of Beadles one gene one enzyme hypothesis to one gene one peptide chain. A single mutation of changing one amino acid greatly strengthened the concept of a relationship between DNA and the protein polypeptide.
ID114	Garrod was an MD focused on rare and unusual birth defects, particularly inborn errors of metabolism. For example, alkaptonuria
ID114	The theory that each gene directly produces a single enzyme, which consequently affects an individual step in a metabolic pathway. In 1941, Beadle demonstrated that one gene in a fruit fly controlled a single, specific chemical reaction in the fruit fly, which one enzyme controlled.
ID114	Sickle-cell disease and related forms of anemia result only from mutations in the beta chain of hemoglobin, but never from mutations in the alpha chain. All mutations affecting oxygen-transporting properties of hemoglobin are associated with changes in the electrophoretic properties of hemoglobin. The sickle-cell trait results from mutations in genes coding for the alpha subunit, whereas individuals with sickle-cell disease bear mutations in the gene coding for the beta subunit.
ID100	Archibald Gerard is the individual who discovered that genes control inborn errors of metabolism through his studies of alkaptonuria. Alkaptonuria is a disorder that makes an individuals urine to turn black. He figured out that this disorder along with others seem to be inherited. The cause of alkaptonuria is a malfunctioning enzyme that cannot metablolize the alkapton 2,5-dihydroxyphenylacetic acid, homogentisic acid. When Homogentisic acid accumulates in the body, it is oxidized to produce a black product that is excreted in the urine. Gerard referred to this malfunctioning enzyme, which is controlled by genes, as an "inborn error of metabolism."
ID100	The one gene-one enzyme hypothesis proposed by George Beadle was the first to provide a convincing experimental evidence proving genes are directly responsible for the synthesis of enzymes. Beadle conducted two experiements the first involving Drosophila and the second invovling Neurospora crassa which lead to his one-gen one enzyme hypothesis. In the drosophila study he figured out that the mutant genes that cause different color eyes could be linked to malfunctioning enzymes. The Neurospora crassa study looked at a nutritional mutation which was caused by the inability for the organism to carry out an important chemical conversion reaction. Together these lead to the one gene-one enzyme hypothesis, which was later proven false.
ID100	Sickle cell anemia, a disorder characterized by mutant hemoglobin. In individuals with sickle cell anemia when oxygen levels are low, hemoglobin become elongated and aggregate together depriving tissues of oxygen. The study of this diseases helped disprove the one gene-one enzyme hypothesis. These investigations proved that a single gene codes for a single polypeptide chain and that a mutation can affect an individual's phenotype by causing a single amino acid substitution.
ID146	Garrod's hypothesis stems from an investigation into families exhibiting a disease known as alkaptonuria, which is an organic acid metabolism abnormality that causes urine to darken from yellow to brown once exposed to air. Later in their life, individuals with the disease develop arthritis with brown pigmentation in the effected joint cartilage/connective tissue. Garrod followed the inheritance of the disease, observing that it followed an autosomal recessive pattern, and postulated that it was caused by a mutation in a gene encoding an enzyme involved in the metabolism of a class of compounds called alkapton. Over the next decade he developed an understanding of the possible nature of inherited diseases of metabolism, specializing in the nature of recessive inheritance of enzyme defects. Garrod expanded his metabolic studies to cover cystinuria, pentosuria, and albinism, and in 1923 he summarized these studies in an expanded edition of his best known work.
ID146	Beadle and Tatum's key experiments involved exposing the bread mold Neurospora crassa to x-rays, causing mutations. In a series of experiments, they showed that these mutations caused changes in specific enzymes involved in metabolic pathways. The one gene-one enzyme hypothesis is the idea that genes act through the production of enzymes, with each gene responsible for producing a single enzyme that in turn affects a single step in a metabolic pathway.
ID146	The one gene-one enzyme hypothesis was further modified in 1962 by Vernon Ingram, and from it, the one gene-one polypeptide hypothesis was born. In 1949, it was proposed that sickling was caused by a single gene mutation, which was heterozygous in sickle cell trait individuals and homozygous in individuals with full sickle cell anemia. Simultaneously, it was also noted that the hemoglobin from normal individuals and that from sickle cell anemic individuals migrated differently on an electrophoresis plate, illustrating that there was a physical difference in the hemoglobin types and supporting the single gene mutation. Ingram analyzed the different polypeptide chains in hemoglobin molecules using a modified form of Frederick Sanger's protein analysis. This technique gave a fingerprint of the different hemoglobin types; this showed that the differences between the two types of hemoglobin could be found in one peptide section of eight amino acids. When this section was isolated and analyzed, the only difference was one amino acid (glutamic acid in normal and valine in sickle cell hemoglobin). The difference between these amino acids was one base in the triplet codon. Further analysis showed that amino acid changes in one chain were independent of changes in the other chain, suggesting that the genes determining the alpha and beta chains were located at different loci and show independent assortment. From this, it can be seen that hemoglobin is composed of two independent gene products, each of which is a separate polypeptide. The gene is a section of DNA that determines the amino acid sequence of a polypeptide. One gene codes for one polypeptide and several polypeptides may be required for a functional protein or enzyme.
ID117	Archibald Gerard studied many traits like albinism, but he was particularly interested in alkaptonuria. This condition, among others, seemed to be inherited from the parent. This disorder is a disruption in an important metabolic pathway that inhibits an individual from metabolizing homogentisic acid.In his studies, he found that 19 cases were contained to seven families, with one having four infected individuals. He concluded that genes played a role in the presence of this disorder and others.
ID117	George Beadle was the first scientist to find common ground between biochemistry and genetics with the use of cultured mutations in Neurospora genes. He found that each mutated gene  affected the synthesis of a different enzyme, which led him to propose the one gene-one enzyme hypothesis.
ID117	The study of hemoglobin altered the h=one gene:one enzyme hypothesis. Two factors led to this: the first being that while nearly all enzymes are proteins, not all proteins are enzymes. Also, proteins usually have subunit polypeptide structures. These led to the more specific one gene:one polypeptide hypothesis. However, the basis of both hypotheses were relatively similar. The presence of sickle cell anemia was found to be a change in an amino acid in a polypeptide chain, which led to the formation of this hypothesis as well.
ID073	Archibald Gerard referred to inherited diseases as inborn errors of metabolism. In 1989 Archibald Gerrod was the first to suggest that jeans dictate phenotype through enzymes that catalyze specific chemical reactions in the cell. Archibald Gerard suggest said that the symptoms of an inherited disease reflect a person's inability to synthesize a particular enzyme.
ID073	The one gene–one enzyme hypothesis, proposed by George Wells Beadle in 1941, is the theory that each gene directly produces a single enzyme, which consequently affects an individual step in a metabolic pathway. In 1941, Beadle demonstrated that one gene in a fruit fly controlled a single, specific chemical reaction in the fruit fly, which one enzyme controlled.
ID073	By the early 1950s, advances in biochemical genetics spurred by the original hypothesis, made the one gene-one enzyme hypothesis seem very unlikely. Beginning in 1957, Vernon Ingram and others showed through protein fingerprinting that genetic variations in proteins such as sickle cell anemia could be limited to differences in just a single polypeptide chain in a multimeric protein, leading to a one gene-one polypeptide hypothesis instead.
ID085	This hyothesis was important because it showed that genetic diseases could come from defects in enzymes, instead of just from mutations in chromosomes.
ID085	The one-gene-one-enzyme hypothesis was a theory that one gene is responsible for only producing one enzyme.
ID085	Because more than one gene codes for the enzyme responsible for sickle-cell anemia, it was discovered that only the Beta-subunits of that enzyme needed to be defective in order for the individual to have sickle cell anemia.
ID128	Before this survey I didn't remember this topic from class so I looked it up in our textbook. What I found was that Garrod studied alkaptonuria and studied families to understand how heredity worked. Ultimately he discovered that this inherited condition was a result of an alternative mode of metabolism which showed that heredity controlled the chemical reactions occuring in the body in a time before genes and enzymes were even common terms. However, chapter 14 didn't seem to mention the year 1923 at all.
ID128	The one-gene-one-enzyme hypothesis is an idea that was developed by Beadle and Tatum through their work with Neurospora that stated that each gene specified its own enzyme. This idea built on Garrod's work. However, as seen with sickle cell anemia, the hypothesis was not perfect.
ID128	The genetic basis of sickle cell anemia demonstrates that the one-gene-one enzyme hypothesis isn't entirely correct because with sickle cell multiple mutation have been seen that lead to the sickling of red blood cells.
ID072	Garrod concluded that this inherited condition,alkaptonuria, was the result of an alternative movde of metabolism, thus implying that hereditary information controls chemical reactions in the body.
ID072	George beadle provided the first convincing experimental evidence that genes are directly responsible for the synthesis of enzymes.
ID072	The sickle cell anemia was the first direct evidence that genes specify proteins other than enzymes.
ID047	Gerard studied families with alkapturnia which is a a disease within families involving organic acid metabolism. He realized it was an autosomal recessive pattern of inheritance and that it was a mutation in a gene encoding n enzyme.
ID047	Beadle proposed this hypothesis meaning that each gene directly produces a single enzyme which in turn affects a single metabolic step.
ID047	Through protein fingerprinting genetic variations in proteins limited to a difference in a single polypeptide chain lead to the one gene-one polypeptide hypothesis
ID138	I do not know too much about this. I know that basically, inborn errors of metabolism are problems that are associated with breaking down nutrients and acquiring what the body needs. Archibald Gerard was the first person to say that this could be controlled by genes.
ID138	This is the idea that a specific gene codes for a specific enzyme. We know now that this is not true and that the more proper way of looking at it is one-gene-one-polypeptide.
ID138	The genetics of sickle cell anemia are such that they do not support the argument for one-gene-one-enzyme.
ID036	Inborn errors of metabolism form a large class of genetic diseases involving congenital disorders of metabolism. The majority are due to defects of single genes that code for enzymes that facilitate conversion of various substances (substrates) into others (products).
ID036	gene controls the synthesis or activity of a single enzyme. The concept, which united the fields of genetics and biochemistry, was proposed by American geneticist George Wells Beadle and American biochemist Edward L. Tatum, who conducted their studies in the mold Neurospora crassa. T
ID036	Vernon Ingram and others saw  protein fingerprinting there was genetic differences hemoglobin. Some of these genetic differences caused sickle cell.In which  a single polypeptide chain in a multimeric protein is different.
ID084	This was the first suggestion of how genes worked. Gerard believed that "inborn errors of metabolism" could be inherited from the parents.
ID084	The one gene - one enzyme hypothesis stated that each enzyme was created or correlated to just one gene.
ID084	We know the one gene - one enzyme hypothesis is not quite correct, rather we know that one gene codes for one protein. Enzymes can be made up of several proteins; therefore, are coded by several genes.
ID119	In 1923, Archibald Gerard was the first person to come up with the idea that genes were responsible for the errors of metabolism.
ID119	George Beadle was the person who came up with the "one gene-one enzyme hypothesis."
ID119	The one gene-one enzyme hypothesis states that each gene is responsible for the creation of one enzyme. The theory is not quite right and was discovered so by studying the genetic basis of sickle cell anemia.
ID003	This showed that the mutations  actually occur on the chromosome to effect the DNA.
ID003	I do not know much information about this topic.
ID003	There are more genes than enzymes. So there can not just be one enzyme for every gene
ID104	Gerard laid the ground work for study of genetic inheritance and how genetic diseases were translated to the offspring
ID104	according to George Beadle for every enzyme there was a specific one gene that coded for it in the DNA code.
ID104	With the single base mutation that occurs in sickle cell anemia it shows us that a gene can actually result in a different enzyme. it suggests that the idea is one gene-one amino acid.
ID107	Gerard's study of inborn errors of metabolism helped to establish that proteins did play a role in heredity. He noticed that some inherited diseases caused harmful alterations in body chemistry. He studied families with alkaptonuria and reasoned that they lacked the proper enzyme to metabolize homogentisic acid. He established that it was a genetic trait because families had several affected members.
ID107	George Beadle studied Neurospora to establish that genes are directly responsible for the synthesis of enzymes. He showed that nutritional mutations in bread mold caused the loss of enzymatic activity. Beadle targeted the metabolic pathway in Neurospora that synthesized arginine. He studied 7 auxotroph mutant strains that were unable to convert citrulline to arginine.
ID107	We now know that the one-gene-one-enzyme should really be one-gene-one polypeptide because proteins are made up of more than one polypeptide. A sickle cell and a normal cell only differ by one peptide fragment, yet they have many chemical differences.
ID126	The inborn errors is where the body cannot probably turn food into energy.
ID126	This is the idea that genes act through the production of enzyme, each gene is responsible for one enzyme.
ID126	Sickle cell anemia shows that there are more than one enzyme to one gene not exactly a one to one ratio.
ID130	inborn errors of metabolism are genetic diseases that are usually single gene mutations in genes that code for metabolic enzymes.
ID130	George Beadle proposed that each individual gene coded for an individual enzyme which effects a specific step in the metabolic pathway.
ID130	Sickle cell demonstrates more of a one gene- one polypeptide idea.  Because the hemoglobin are made up of multiple proteins and still effected by the mutation.
ID110	generally a single gene defect causing congenital disorders of metabolism.. this includes albinism: melanin deficiency as well as kidneys production of kidney stone.
ID110	the idea that one gene codes specifically for one enzyme, this hypothesis is false, one enzyme is coded by many genes
ID110	the genetic basis of sickle cell is described as a single amino acid alteration (mutation) in the genetic code, and heterozygotes have a complimentary advantage over malaria. it demonstrates that the one gene one enzyme hypothesis was incorrect because heterozygosity shows more genes to be involved
ID074	Archibald Garrod discovered inborn errors of metabolism. An explanation for epistasis is an important manifestation of Garrod’s research, albeit indirectly. When Garrod studied alkaptonuria, a disorder that makes urine quickly turn black due to the presence of gentesate, he noticed that it was prevalent among populations whose parents were closely related
ID074	The one gene-one enzyme hypothesis is the idea that genes act through the production of enzymes, with each gene responsible for producing a single enzyme that in turn affects a single step in a metabolic pathway. The concept was proposed by George Beadle and Edward Tatum in an influential 1941 paper on genetic mutations in the mold Neurospora crassa, and subsequently was dubbed the "one gene-one enzyme hypothesis" by their collaborator Norman Horowitz. In 2004 Norman Horowitz reminisced that “these experiments founded the science of what Beadle and Tatum called ‘biochemical genetics.’ In actuality they proved to be the opening gun in what became molecular genetics and all the developments that have followed from that.” The development of the one gene-one enzyme hypothesis is often considered the first significant result in what came to be called molecular biology. Although it has been extremely influential, the hypothesis was recognized soon after its proposal to be an oversimplification. Even the subsequent reformulation of the "one gene-one polypeptide" hypothesis is now considered too simple to describe the relationship between genes and proteins.
ID074	By the early 1950s, advances in biochemical genetics—spurred in part by the original hypothesis—made the one gene-one enzyme hypothesis seem very unlikely (at least in its original form). Beginning in 1957, Vernon Ingram and others showed through protein fingerprinting that genetic variations in proteins (such as sickle cell hemoglobin) could be limited to differences in just a single polypeptide chain in a multimeric protein, leading to a "one gene-one polypeptide" hypothesis instead.According to geneticist Rowland H. Davis, "By 1958 – indeed, even by 1948 – one gene, one enzyme was no longer a hypothesis to be resolutely defended; it was simply the name of a research program."
ID097	Gerard's study lead to the discovery of many other metabolic disorders.
ID097	Each gene is able to produce a single enzyme.
ID097	Sickle cell anemia helps combat Malaria therefore this mutation can be helpful and this enzyme has been coded from multiple genes.
ID046	These errors block off paths in the metabolic pathway, which inable the body to produce certain chemicals it needs. An example is PKU, in which the body is unable to produce Phenalalanine due to errors in metabolism.
ID046	This hypothesis proposes that one gene codes for one enzyme. He drew conclusions from the metobolic pathway.
ID046	The one gene one enzyme hypothesis has a loophole, in which some nucleotides would mutate one base pair (sickle cel). A better theory would be "one gene one nucleotide"
ID052	While studying a disease, Archibald Gerald was able to find a link to epistasis, which was an important point to conclude that genes were responsible for the regulation of "inborn errors of metabolism.'
ID052	Through George Beadle's experimentation, he concluded that one specific gene was responsible for the coding of one specific enzyme.
ID052	Sickle cell anemia is an example which disproves Beadle's one-gene enzyme hypothesis.
ID120	The genes that regulated a certain way are responsible for the viability of the offspring. If these genes are not working correctly then, mutations may have caused these problems resulting in inborn errors.
ID120	This proposed that each gene read for one particular enzyme as 1 to 1 ratio.
ID120	Actually there are a number of genes that are responsible for an enzyme. A gene makes a protein but enzymes can be made up of multiple proteins.
ID094	Gene regulation in eukaryotes involves more factors that operate in a more complex manner than what occurs in prokaryotes.
ID094	swi-snf is a nuceleosome modeling complex that controls how DNA is folded and package in yeast. It can also alter the position on nucleosomes.
ID094	Acetylation and deacetylation of nucleosome tails controls how DNA is expressed; this process can either package/unpackage dna from a nucleosome complex which can cause it to be expressed.
ID095	I know that eukaryotes have an acetylation mechanism in the histones that control gene regulation.  I know that eukaryotes have many more mechanisms to regulate genes than prokaryotes.  I still need to review this material.
ID095	The swi-snf mutant results in proteins that affect the ATP function of the histones and DNA.
ID095	Acetylation and deacetylation control gene function in eukaryotes. During acetylation, the histones interactions are relaxed and euchromatin forms, allowing the DNA to be transcribed.  In deacetylation, the reverse takes place, and the genes are silenced.
ID099	eukaryotic gene regulation is much more complicated due to the in depth nature of more complex organisms. Nucleosomes and histone tails apply a large range of complexity in gene silencing/activation
ID099	lack of the re-positioning factors leads to indeterminate cell proliferation and cancer due to the genes never being re-arranged in the inactive transcription state
ID099	acetylation of nucleosome tails helps to signal for active transcription and if either acetylation or de-acetylation falters the organism could suffer
ID046	The genome of eukaryotes is significantly larger for one reason, another is that eukaryotes have introns.
ID046	The swi/snf protein complex is normally used to remodel DNA. However, this mutant is deficient in this process.
ID046	In these processes, lysine is aectlyated and deacetylated in the N terminus tail.
ID056	The gene regulation in eukaryotes is more highly specified than the process that occurs in prokaryotes. Not only does eukaryotes have more DNA than prokaryotes, but the DNA has histones and other protein complexes that help affect the regulation of gene expression. Eukaryotic mRNA also has an immense process that it must pass through.
ID056	The normal SWI/SNF works by rearranging the components that make up a nucleosome. Obviously a mutant of the SWI/SNF is going to cause an error in the repositioning and attachment of the nucleosome. In the example, the mutant eukaryote was not able to ferment sucrose.
ID056	Hyperacetylation (profound amounts of acetyl groups added) of nucleosome tails promotes gene activity and deacetylation or hypoacetylation (lack of acetyl groups) promotes gene inactivity. This occurs through proteins HAT (Histone acetyltransferase gene) and HDAC (Histone deacetylation gene).
ID036	Gene regulation in Eukaryotes is more complex than prokaryotes. Prokaryotic.Prokaryotic gene expression is primarily controlled at the level of transcription.
ID036	Loosens attachment between histone and DNA
ID036	The trp operon is regulated primarily by a repressor protein and secondarily by attenuation.
ID026	Gene regulation is very important to a cell and the chromosomes inside of it. However, in prokaryotes it only occurs during transcription. In eukaryotes it happens in both transcription and translation. This added timeframe makes it both more common and complex in eukaryotes.
ID026	This mutant in the Saccharomyces is not as able to reposition its nucleosomes as well as the normal swi-snf's. This can result in structural differences that can lead to bigger problems with the organism.
ID026	The acetylation and deacetylation of nucleosome tails can make it harder for chromosomes to condense and fold in certain ways which can lead to structural problems as well. These situations can lead to bigger problems with the organism.
ID109	Gene regulation in eukaryotes is more complex than regulation in prokaryotes. This is because eukaryotes are more complex organisms than prokaryotes. Prokaryotes have more DNA that needs to be replicated. Prokaryotes also have more organelles to copy. Further, gene regulation in prokaryotes happens mainly in transcription. However, in eukaryotes, it happens in transcription and also post transcription modification. Overall, eukaryotes are more complex organisms than prokaryotes, so gene regulation is also more complex.
ID109	The swi-snf mutant in the eukaryote Saccharomyces is deficient in nucleosome repositioning. If loop occurs in template strand during replication, DNA polymerase misses looped out nucleotides, and small insertions or deletions occur. Replication slippage is common in repeat sequences or hot spots. These contribute to hereditary diseases and they are also responsible for genetic anticipation in these diseases, for example, Fragile-X and Huntington disease.
ID109	Acetylation and deacetylation of nucleosome tails affects chromosome condensation and gene silencing. This is because histone acetylation and deacetylation is the process where the lysine residue in the N-terminal tail from the histone core of the nucleosome is acetylated and deacetylated as part of gene silencing. They donate alkyl groups to amino or keto group and alter base-paring affinity, which causes transition mutations, for example, mustard gas.
ID059	Gene regulation in eukaryotes are more complex than in prokaryotes because the central dogma stage undergoes several steps whereas in prokaryotes everything happens in one process. For example we have transcription, mRNA, translation, and then the protein making.
ID059	The swi-snf mutant occurs when the nucleosomes are mapped by DNA footprinting and protein cross linking. This mutant is a result of the disruption and mobilization of chromatin in ATP-dependent manner.
ID059	Acetylation and deacetylation are essential parts of the gene regulation. the acetylation of the nucleosome tail is the process when the acetyl end of a functional group is transferred from one molecule to another. and deacetylation is the exact opposite where the acetyl funtional group is removed from the nucleosome tails which causes the process of gene silencing.
ID039	Gene regulation is more complex in eukaryotes than prokaryotes. In prokaryotes everything takes place in one location, but with eukaryotes things happen inside and outside of the nucleus. In eukaryotes,it is more complex because it involves more processes and more things. For instance, there is a larger amount of DNA to account for and mRNA goes through more processes before it can be transported from the nucleus.
ID039	Normally, wild type Saccharomyces is able to switch its mating type, but the swi-snf mutant can not because it can not reposition nucleosomes. Being able to not reposition nucleosomes means that areas that would normally be easily to access  by things like RNA polymerase and transcription activators are no longer available to reach these sites making Saccharomyces unable to switch mating types.
ID039	Acetylation allows the chromosome to be decondense and less compacted which allows transcription to take place. Deacetylation does the opposite and prevents/blocks transcription.
ID082	Gene regulation in eukaryotes is more complex than regulation in prokaryotes in a variety of ways. Many of these differences are due to the fact that eukaryotic DNA is much large than prokaryotic DNA, and because of this, eukaryotic organisms have had to develop mechanisms to  compensate or work with their large DNA. One difference between gene regulation in eukaryotes and prokaryotes is that in eukaryotes, RNA polymerase doesn’t recognize the promoter sequence itself, but rather the transcription factor proteins that recognize and bind the promoter sequence (TATA box) recruit the RNA polymerase after they have bound to the sequence. Another aspect of the differences between gene regulation in prokaryotes and eukaryotes is that prokaryotic systems are more adept at responding to the environment around them. Unlike in eukaryotes which have stable mechanisms of maintaining homeostasis, prokaryotes are sensitive to their environment and respond more quickly and directly to outside stimuli.
ID082	The swi-snf mutant was found to be deficient in the protein that is responsible for moving histones. These proteins usually can physically move the histones, reorganize and remodel the chromosome from tightly packed to loosely packed to allow gene expression. Under various circumstances, there are two genes that determine the sex in the Saccharomyces, and depending on what nucleosomes are unpacked and flipped, one of the two alleles for mating type will be in the correct position and will thus determine the sex of the Saccharomyce.
ID082	The nucleosomes have tails that hang out of the nucleosome and are free to be acted on by other enzymes in the cells. These tails are critical to gene regulation, gene expression, and chromatin remodeling. Using the mechanism of acetylation and deacetylation, enzymes in the cell can act on these tails, which induce conformational change, either winding the histone/nucleosome complex even tighter, or relaxing the histone/nucleosome complex, allowing gene transcription and gene expression. A chromosome is considered “silenced” when it is packed in such a way that transcription macherinery cannot get to the gene sequences in the DNA wrapped around the histones, and this wrapping and packing is regulated by enzymes that can either use Acetylation or deacetylation of nucleosome tails to silence or turn back on a gene.
ID000	Gene regulation in Eukaryotes is more complex because eukaryotes have a nuclear membrane, DNAs are associated with nucleosome and mRNAs are capped, adenylated and introns are removed.  Gene expression is regulated by chromatin modification, transport through the nuclear membrane, half-lives of mRNA, regulation of translation and protein modifications.
ID000	The swi (switch) mutant screen identified mutants that were deficient in ability to switch mating types.  The snf (sugar non-fermenting) mutant screen identified mutants that could not utilize sucrose as a carbon source.  One mutant from both screens was the same and was shown to be responsible to re-positioning nucleosomes.
ID000	Modification of nucleosome histone tails by HAT (histone acetyltransferase)  is associated with transcriptionally active genes.  Deacetylation of nucleosome histone tails is associated with chromosome condensation and gene silencing.
ID052	Prokaryotic gene expression is mainly regulated during transcription. However, eukaryotic gene expression has many more complex mechanisms that take part in regulation. Epigenetics, transcription, post-transcriptional modifications, translation, and post-translational modifications must all be taken into account.
ID052	The swi-snf mutant in this eukaryotic model organism indicates that the cis and trans factors can be studied to further determine how histones modifications occur.
ID052	Acetylation and deacetylation serves as an important epigenetic regulation mechanism. The addition or deletion of an Acetyl-CoA tail may cause that particular DNA strand to become euchromatin or heterochromatin.
ID027	Gene regulation in eukaryotes does not rely on operators. It requires enhancers/silencers/transcription factors/histone rearrangement/etc. It is much more complex.
ID027	My knowledge on this topic is limited, but I assume this is responsible for binding to the nucleosomes and moving them into areas where they need to be.
ID027	The acetylating a histone tail causes the gene to become readable since the histones will open up in order for the gene to be read. The deacetylation does the opposite.
ID037	The reason is because eukaryotes have many types of specialized cells that have to express different genes at different times. The human body alone has something like 200 different cell types--each performing a different function. This function is dictated by what genes are being expressed.
ID037	Histone acetylation and deacetylation are the processes by which the lysine residues within the N-terminal tail protruding from the histone core of the nucleosome are acetylated and deacetylated as part of gene regulation.
ID085	In eukaryotes, regulation is more complex because of histones, and because transcription occurs in the nucleus, meaning there are many more possibilities for regulation.
ID085	Because swi-snf is deficient in nucleosome repositioning, it can provide regulation by remodeling the chromatin.
ID085	Acetylation and deacetylation are two histone modifications; acetylation decreases the + charge on histones, opening up the chromatin. Deacetylation does the opposite.
ID101	In Prokaryotes the transcription and translation occur at the same time right next to the DNA. However, the Eukaryotes transcription occurs in the nucleus, and the mRNA are transported to the ribosomes outside the nucleus.
ID101	The swi snf mutant prevents the Saccharomyces from importing lactose into the gene. If the lactose is died blue, then only the non blue colonies will have the insertion of the gene.
ID101	Acetylation promotes transcription of the DNA, and it helps activate the silenced DNA that are condensed and wrapped around the histones.
ID144	Eukaryotes have many types of specialized cells that have to express different genes at different times. Eukaryotes tend to be much larger than prokaryotes. This entails the need for more complex regulatory mechanisms involving hormones and other signaling molecules that must travel long distances quickly.
ID144	Loosens attachment between histone and DNA. Loosens DNA strand from nucleosome core. Causes reorganization of internal nucleosome.
ID144	The histone tails insert themselves in the minor grooves of the DNA and extend through the double helix, which leaves them open for modifications involved in transcriptional activation. Acetylation has been closely associated with increases in transcriptional activation while deacetylation has been linked with transcriptional deactivation. These reactions occur post-translation
ID077	Gene regulation in eukaryotes is more complex than prokaryotes for a number of reasons. A major reason gene reuglation in eukaryotes is more complex is because eukaryotic DNA is wrapped around histones to form nucleosomes and the genome of eukaryotes is significantly larger than prokaryotes.
ID077	Swi-snf is a very well studied remodeling complex that loosens attachments between histones and DNA in nucleosomes. This loosens DNA strands from the nucleosome core to cause reorganization of internal nucleosome components. The swi-snf mutant does not allow for the nucleosome to be repositioned.
ID077	Acetylation and deacetylation are essential parts of gene regulation. Acetylation is the process where an acetyl functional group is transferred from one moleculeto another and deacetylation is the reverse action where an acetyl group is removed from a molecule. Acetylation leads to greater levels of gene transcription and deacetylation leads to decreased levels of transription.
ID061	Eukaryotic cells contain much more DNA than prokaryotic cells, this DNA is associated with histones and other proteins to form highly compact chromatin structures with an enclosed nucleus. In terms of regulation, the mRNAs of the majority of eukaryotic genes have to go through posttranscriptional regulation in which the strands are spliced, capped, and polydentylated before they can leave the nucelus. In addition, translation rates can be modulated, as can the way that proteins are processed, modified, and degraded. With the histone structures and nucleosomes, transcription can be controlled by acetylation and methylation of nucleosome tails.
ID061	The SWI/SNF serves to remodel chromatin. It can loosen the attachment between histones aand DNA, resulting in the nucleosome sliding along the DNA and exposing regulatory regions. It can also loosen the DNA strand from the nucelosome core, or reorganize the internal nucelosome components. If there is a mutation in the gene, then the nucelosome can not be repositioned and the DNA in it will not be exposed to association with transcription factors and RNA polymerases. If this is the case, the genes cannot be transcribed and the proteins they code for cannot be produced.
ID061	Acetylation of a nucelosome tail will decrease the positive charge on histones by adding acetate groups, which results in a reduced affinity of the histone for DNA. This helps the chromatin conformations open up and allows for the binding of transcription regulatory proteins to DNA. Meanwhile, deacetylation would do the opposite. Histone deacetylase enzymes remove acetate groups from histone tails which will increase the negative charge on the histone and lead to increased affinity of the histone for the DNA. This makes the chromatin recondense and be unavailable for transcription.
ID118	Gene regulation in eukaryotes is more complex than regulation in prokaryotes as greater amount of DNA that is associated with histones and other proteins. mRNAs must be spliced, capped , and polyadenylated prior to transport from nucleus. Genes on numerous chromosomes are enclosed in a double membrane nucleus. mRNAs have wide range of half-lives. Modulation of mRNA translation, protein processing, modification, and degradation. This unique characteristic of gene regulation differs prokaryotes and eukaryotes.
ID118	Mutant were isolated that were defective in ability to switch mating type. Yeast which is saccharomyces can "switch" their mating type under certain environmental conditions. Mutants were given the designation sugar non fermenting=snf. One mutant that could not ferment sucrose or switch mating type is swi-snf. This mutant was part of a multilane complex than repositioned nucleosomes.
ID118	Acetylated histones may recruit remodeling complexes that decidedness chromatin for transcription. Deacetylated histones may recruit other protein complexes that repress transcription. Deacetylation can turn off gene transcription in the eukaryotes.Methylation is one way to deacetylation as it add methyl group that prevents genes to condense and remove nucleosomes. acetylation and methylation can change over time based on the environmental effect. This is not a genetic change but the impact of epigenetic on the gene regulation.
ID105	Eukaryotes have a greater amount of DNA that is
ID105	This mutant was given the name swi-snf. This mutant was part of a multigene complex that repositioned nucleosomes.
ID105	Acetylated histones may recruit remodeling complexes that decondense chromatin for transcription. Deacetylated histones may recruit other protein complexes that repress transcription
ID134	Eukaryotic gene regulation is more complex than that in
ID134	The SWI-SNF complex is one of the best-studied remodeling complexes. It loosens attachment between histone and DNA, and loosens the DNA strand from nucleosome core. Lastly, it causes reorganization of internal nucleosome components. Nucleosome re-positioning makes room for transcription regulatory proteins, transcription activators, and RNA polymerase II.
ID134	Acetylated histones may recruit remodeling complexes that decondense chromatin for transcription.
ID058	Prokaryotes often use operons to group genes together and enact control over them. In eukaryotes, gene regulation can occur at any step from DNA to protein. Cells can use histone modification or chromatin remodeling by SWI/SNF. They also have enhancers and silencers and various transcription factors. Finally they have posttranscriptional modification.
ID058	This mutant was found to be unable to ferment sucrose or switch mating type. It was part of a complex that repositioned nucleosomes as part of chromatin remodeling.
ID058	Acetylation makes the DNA hyperacetylated which makes the genes active and allows them to be transcribed. Deacetylation makes the DNA hypoacetylated and turns off the genes which means they will not be transcribed.
ID111	Eukaryotes employ a wide range of mechanisms for altering the expression of genes. Prokaryotes typically have gene regulation that occurs primarily at the level of transcription. In eukaryotes it can occur at many levels. Eukaryotes contain much more DNA in the cells than prokaryotes do. Eukaryotic genes are located on many different chromosomes rather than just one and these chromosomes are enclosed within a double membrane bound nucleus. Eukaryotic mRNAs can have a wide range of half lives while prokaryotic mRNA tend to decay rapidly.
ID111	Swi-snf is a remodeling complex. These move or rearrange nucleosomes along the DNA. Repositioned nucleosomes make regions of the chromosomes accessible to transcription regulatory proteins. SWI-SNF works in multiple ways. The DNA histone contacts may be loosened, allowing the nucleosomes to slide along the DNA, exposing DNA regulatory regions. The path of the DNA around a nucleosome core particle may be altered. Components of the core nucleosome particle may be rearranged, resulting in a modified structure. If it is deficient in nucleosome repositioning, these will not occur.
ID111	Acetylation decreases the positive charge on histones, resulting in a reduced affinity of the histone for DNA. This may assist in the formation of open chromatin conformations which would allow binding of transcription regulatory proteins to DNA. Histone acetylation is catalyzed by HAT, which are recruited to genes by the presence of certain transcription activator proteins that bind to transcription regulatory regions. Histone deacetylases remove acetate groups from histone tails. Histone deacetylases can be recruited to genes by the presence of certain repressor proteins on regulatory regions. Acetylation affects condensation while deacetylation affects silencing.
ID065	Gene regulation is more complex in eukaryotes in comparison to prokaryotes because eukaryotes have linear chromosomes that require modification before they can leave the nucleus and be translated into proteins. Eukaryotes also lac operons, which are the more simple derivative of gene regulation. There are also much more transcription factors involved in eukaryotic gene regulation than in prokaryotic gene regulation.
ID065	This mutation occurs due to a deficiency in nucleosome repositioning. nucleosomes are responsible for wrapping and condensing chromosomes in eukaryotes, and their ability to unfold and relax plays a major role in gene expression and repression. This type of modification in swi-snf mutants result in mutants because they lack the ability to express certain genes that code for certain enzymes.
ID065	Acetylation of nucleosome tails results in the DNA wrapped around the nucleosomes to relax and ultimately decondense. This decondensation allows for polymerases to enter and initiate transcription. Deacetylation does the opposite and ultimately turns off transcription by causing the chromosomes to condense.
ID083	Regulation of some operons in E. coli are leaky. To compensate, a second system called attenuation has evolved. Attenuation involves a short leader sequence with areas of complementary base pairing. When sufficient gene product (amino acids and thus tRNAs for those amino acids) is available, ribosomes read through the leader sequence which allows formation of an intrinsic terminator and releases RNA polymerase from the DNA. When insufficient gene product is available, the ribosomes stall, a different secondary pairing of mRNA occurs and formation of an intrinsic terminator is prevented. RNA polymerase reads through the operon structural genes.
ID083	Regulation of some operons in E. coli are leaky. To compensate, a second system called attenuation has evolved. Attenuation involves a short leader sequence with areas of complementary base pairing. When sufficient gene product (amino acids and thus tRNAs for those amino acids) is available, ribosomes read through the leader sequence which allows formation of an intrinsic terminator and releases RNA polymerase from the DNA. When insufficient gene product is available, the ribosomes stall, a different secondary pairing of mRNA occurs and formation of an intrinsic terminator is prevented. RNA polymerase reads through the operon structural genes.
ID083	Regulation of some operons in E. coli are leaky. To compensate, a second system called attenuation has evolved. Attenuation involves a short leader sequence with areas of complementary base pairing. When sufficient gene product (amino acids and thus tRNAs for those amino acids) is available, ribosomes read through the leader sequence which allows formation of an intrinsic terminator and releases RNA polymerase from the DNA. When insufficient gene product is available, the ribosomes stall, a different secondary pairing of mRNA occurs and formation of an intrinsic terminator is prevented. RNA polymerase reads through the operon structural genes.
ID103	In eukaryotes there are many transcprtion factors that must be recruited for RNA polymerase to bind and proceed with copying.
ID103	The swi-snf complex is responsible for altering the position of nucleosomes so that different genes can be accessed. If the swi-snf genes are damaged the proteins associated with them will not function correctly.
ID103	I know that the acetylation and deacetylation affects the charge of the histone complex and  determines how tightly the DNA is bound to it.  This determines which genes are exposed to be transcribed.
ID115	For one, there is more DNA in eukaryotes than prokaryotes. Then, mRNA needs to undergo processing before it can be translated. Also, histones wrap around chromosomes, making them inaccessible unless they are deacetylated and decondensed for DNA replication.
ID115	Because of its ability to decondense DNA by destabilizing nucleosomes, swi-snf is capable of using ATP to destabilize the DNA but is not efficient in its mechanism, as it is much simpler in eukaryotes and does not have an efficient way of repositioning by the way of loops that affect the stability of chromosomal DNA and nucleosomes.
ID115	If nucleosome tails are acetylated, gene expression is more likely as it decondenses the chromosome by removing a positive charge on the histones and relaxing the chromosome's structure, allowing for transcription. Deacetylation, on the other hand, discourages gene expression and if bound tightly enough, can silence a gene by making it unaccesssible to RNA polymerase for transcription.
ID064	Eukaryotic DNA is linear, unlike circular prokaryotic DNA, and eukaryotic cells also have more DNA. To densely pack this DNA, eukaryotes have chromatin complexes. These histones and coiling of DNA complicate transcription and translation, in addition to the fact that eukaryotic DNA is found inside the nucleus. Also, eukaryotes can control gene regulation at many different steps, starting at transcription, through translation, and beyond.
ID064	A yeast that is mutant in both components of the swi/snf remodeling complex cannot reposition nucleosomes. Swi and snf are two genes that make up this complex of proteins that affect gene expression by binding to DNA and repositioning the nucleosomes.
ID064	Lysine residues can either acetylate or methylate nucleosomes. This affects how tightly the chromatin is wound around the histones, and therefore how easily the DNA can be transcribed. Altering the accessibility of the DNA in this manner can turn a gene "on" or "off." It creates many possibilities for histone variants.
ID079	Eukaryotic cells are more complex than prokaryotic cells, and therefore, gene regulation is more complex. Eukaryotic gene regulation occurs in various organelles within the cell, while prokaryotic gene regulation occurs in the cytoplasm because a nucleus is not found in prokaryotic cells.
ID079	Yeast can switch mating types (A or alpha will be expressed depending on which gene is at a certain location) and can ferment sucrose. Swi-snf mutants can do neither.
ID079	Acetylation is controlled by HAT and results in the opening of chromosome structure, which makes the gene accessible. Deacetylation is controlled by HDAC, which closes the chromatin structure and prevents the gene from being accessible.
ID014	Gene regulation is far more complex in eukaryotes considering that DNA is wrapped around histone proteins. Gene regulation can happen at any step between that from reading the code to producing the protein.Gene regulation in prokaryotes can be a response to environmental conditions and usually there is an operon type of regulation responsible for gene regulation in prokaryotes.
ID014	The mutants that were obtained from both screens and one mutant did not have the ability to ferment sucrose or switch mating type. They later found that this mutant was apart of a nucleosome repositioning chromosome. Considering that the nucleosomes were not repositioned then the conclusion was reached that there was a mutatoin in the swi-snf.
ID014	If the gene is active then it will be rich in acetyl groups but if the genes are inactive then the area will be poor in acetyl groups. HAT and HDAC are responsible for the addition or removal of acetyl groups respectively.
ID098	Like almost any other aspect of cell activity, gene regulation in eukaryotes is more complex than gene regulation in prokaryotes. In prokaryotes, regulation is carried out predominantly at the level of increasing or decreasing transcription. In eukaryotes, other mechanisms can act to regulate gene expression, like chromatin remodeling, alternate mRNA splicing, transportation of mRNA to the cytoplasmic ribosomes, etc. Additionally, eukaryotes have enhancers and silencers in addition to the promoters and repressors found in prokaryotes.
ID098	Swi-snf is a well-studied chromatin remodeling complex. It loosens attachments between the histones, thus allowing the DNA to move away from the nucleosome core. Organisms with this mutation are unable to remodel the chromatin, meaning they are unable to switch genders (“swi” portion) and unable to ferment sugar (“snf” portion).
ID098	Active genes have nucleosomes that are rich in acetyl groups, while inactive genes have histones that are poor in acetyl groups. This happens because acetylated histones help recruit remodeling complexes that loosen chromatin for transcription. Likewise, deacetylated histones recruit complexes that can repress transcription.
ID022	Prokaryotic gene expression is simple and it primarily is controlled at the level of transcription. However, the more complex eukaryotic gene expression is controlled at the level of transcription, post-transcription, translation, and post-translational. Prokaryotic gene expression both transcription and translation occurs only at cytoplasm since there is no nucleus. However, in eukaryotes the transcription occurs in nucleus and translation occurs in cytoplasm.
ID022	Swi-snf is a nucleosome remodeling complex found in both eukaryotes and prokaryotes. it is a group of proteins that associate to remodel the way DNA is packaged. It is composed of several proteins  and polypeptides. It possesses a DNA stimulated ATPase activity and can destabilize histone and DNA interactions in nucleosomes. However, when mutation occurs though deletion, insertion, etc the protein complex undergoes chemical and physical changes. Due to this the swi-snf complex is not able to bind to the cis factor of the AT rich content, also due to the mutation the polyA tail would cause problem for swi-snf complex for nucleosome remodeling.
ID022	The acetylation and deacetylation of nucleosome is done for gene regulation. it is the process through which lysin residue in the N-terminal tail in histone of the nucleosome are acetylated and deacetylated. Acetylation is simply adding an acetyl group on the nucleosome and deacetylation is removing the acetyl group. Acetylation takes off the positive charge on the histones and in doing so it decreases the interaction of the N terminial of histones with the negatively charged phosphate groups of DNA. Due to this the condensed chromatin becomes more relaxed and allows for greater levels of gene transcription. However, the acetylation of the nucleosome makes the histone more compact and thus allows for reduced level of gene transcription.
ID081	Gene regulation in eukaryotes occurs at the levels of epigenetics, transcription, post-transcription, translation and post-translation, while prokaryotic gene regulation primarily occurs at the transcription level. Gene regulation also occurs in the cytoplasm in prokaryotes, while it occurs in a more complex nucleus in eukaryotes.
ID081	The swi-snf is a mutation that causes the protiens that associate with DNA to remodel how it is packaged. This remodeling of packaging has to do with nucleosome repositioning.
ID081	Acetylation of nucleosome causes the DNA to be active and uncondensed. On the other hand, deactylation causes DNA to be condensed and become inactive.
ID033	Prokaryotic gene regulation is more simple than eukaryotic gene regulation because there aren't any introns or exons in Prokaryotes.
ID033	Haven't read about this yet
ID033	Acetylation and deactylation of the nucleosomes causes them to wind or unwind which expresses the genes so that they can be translated
ID084	Prokaryotes tend to regulate gene expression by the amount of transcription allowed to occur. Eukaryotes are much more complex, as they practice gene regulation through a combination of epigenetics, transcription, post-transcription, translation, and post-translation. The post steps are big factors in fixing mutations, and thus regulating genes.
ID024	There are many differences in eukaryotic and prokaryotic regulation. Eukaryotic regulation is more complex, so eukaryotes can be much more diverse than proaryotes.
ID024	The swi-snf mutant is one of the best-studied remodeling complexes. It loosens the attachment between histones and DNA, thus exposing the DNA so the genes can be expressed.This loosening happens between the DNA and the nucleosome core. It causes reorganization of internal nucleosome
ID024	When nucleosome tails are acetylated, they uncoil and the adjacent genes can now be expressed, but if the tails are deacetylated, the nucleosomes grab the DNA even tighter, making gene expression in that region impossible.
ID015	Eukaryotic gene expression occurs in both the nucleus (transcription) and cytoplasm (translation).
ID015	It possesses a DNA-stimulated ATPase activity and can destabilise histone-DNA interactions in reconstituted nucleosomes in an ATP-dependent manner, though the exact nature of this structural change is unknown.
ID015	Acetylation is the process where an acetyl functional group is transferred from one molecule to another. Deacetylation is simply the reverse reaction where an acetyl group is removed from a molecule.
ID018	Because of the structure of a eukaryote is much more complex than a prokaryote and eukaryotes require more, gene regulation is more complex. Eukaryotes are usually parts of a multicellular organism, which makes them need more regulation than a unicellular organism.
ID018	I do not understand this concept. I will have to reread it before our test.
ID018	Acetylation recruits modeling complexes to decondense chromstin for transcription
ID035	In prokaryotic cells, the control of gene expression is mostly at the transcriptional level. In eukaryotic cells, the DNA is contained inside the cell's nucleus where it is transcribed into RNA. The newly-synthesized RNA is then transported out of the nucleus into the cytoplasm where ribosomes translate the RNA into protein.
ID035	SWI/SNF is capable of altering the position of nucleosomes along DNA.[5] Two mechanisms for nucleosome remodeling by SWI/SNF have been proposed.[6] The first model contends that a unidirectional diffusion of a twist defect within the nucleosomal DNA results in a corkscrew-like propagation of DNA over the octamer surface that initiates at the DNA entry site of the nucleosome.
ID035	Acetylation and deacetylation are essential parts of gene regulation. These reactions are typically catalysed by enzymes with histone acetyltransferase activity. Acetylation is the process where an acetyl functional group is transferred from one molecule  to another. Deacetylation is simply the reverse reaction where an acetyl group is removed from a molecule.
ID040	Prokaryotic gene regulation is primarily controlled at the level of transcription. Eukaryotic gene regulation is controlled in multiple areas, including transcription, post-transcriptionally, translation, and post-translationally. Eukaryotic gene expression must also take care of the nucleosomes that potentially block a gene from being read. Changes can be made to the mRNA chain, or to proteins that have just been made by a ribosome.
ID040	The swi-snf remodeling complex loosens attachments between a histone and the DNA and loosens the DNA from the nucleosome core. This causes reorganization of internal nucleosome components. The swi-snf mutant is unable to ferment sucrose and also unable to switch mating types unlike normal yeast. This is a result of the protein complexes being unable to move so that different DNA is able to be read.
ID040	Hyperacetylation of nucleosome tails occurs in active genes and make the nucleosome rich in acetyl groups. Inactive genes have histones that are low in acetyl groups and are therefor hypoacetylated. Acetylated histones may recruit remodeling complexes that decondense chromatin for transcription. Deacetylated histones may recruit other protein complexes that repress transcription.
ID090	Gene regulation is more complicated in eukaryotes than in prokaryotes. This is due to several factors. Prokaryotes only have gene regulation at transcription, but eukaryotes have gene regulation at the DNA template, translation, through mRNA and protein modifications and stability as well as transcription.
ID090	The wild type gene of swi/snf codes for a chromatin remodeling complex that can move nucleosomes to one side so that the DNA is exposed and can be transcribed. Mutants of this gene are unable to move the nucleosomes off of the TATA box, so the genes on the DNA are unable to be transcribed.
ID090	Histone tails are often not packaged into the nucleosome and are thus key targets for gene regulation. Acetylation adds acetyl groups on to the histone tail, opening the chromatin and allowing genes to be transcribed. This turns genes ON. Deacetylation, however, removes the acetyl groups causing the chromatin to condense and thus silencing the genes.
ID075	Gene regulation in eukaryotes is more complex than regulation in prokaryotes because eukaryotes have greater amount of DNA because they have histones and other proteins. Some things that make them more complex include things such as the fact that mRNAs have to be spliced, capped and polyadenylated before transport from nucleus.
ID075	Mutants that were defective in ability to switch mating type were isolated. Yeast can "switch" their mating type under certain environmental conditions. Mutants were given the designation sugar non fermenting=snf. One mutant that could not ferment sucrse or sitch mating type is swi-snf. This mutant was part of a multilane complex than repositioned nucleosomes.
ID075	The acetylated histones may recruit remodeling complexes that decidedness chromatin for transcription. The deactylated histones may recruit other protein complexes that repress transcription. Deacetylation turns off gene transcription in eukaryotes.
ID108	In prokaryotes the regulation of transcription is the main way to control the type and amount of protein needed for the cell. Thus, in prokaryotic cells gene expression is mostly at a transcriptional level. Because Eukaryotic cells have organelles, the DNA in the nucleus is transcribed to RNA, then the RNA is transported to the cytosplasm where the ribosome turns it into protein. The regulation of gene expression can happen at any stage, from the DNA uncoiling and binding to transcription factors, at a transcriptional level, posttranscriptional level, translation level, or a post translational level. Thus, creating a more complex regulation in eukaryotes.
ID108	The swi-snf mutant in eukaryotic Saccharomyces is deficient because repositioned nucleosomes make chromsoomes accessible to transcription regulatory proteins, transcription activators, and RNAP II. The mutants were defective in its ability to switch mating type.
ID108	Acetylated histones decondense chromatin for transcription and deacetylated histones repress transcription. Hyperacetylated  groups have active genes, while hypoacetylated groups have inactive genes.
ID140	Gene regulation is more complex in eukaryotes, potentially because they are generally larger organisms with generally more complex needs. Examples of more complexity would include needing attachment sites in eukaryotes whereas those are not necessary in prokaryotes.
ID140	Saccaramoyces is a yeast that can switch sex type based on need. It was here that SWI-SNF was first discovered.
ID140	Acetylation causes chromosome condensation with then leads to increased gene expression. Deacetylation also causes chromosome condensation which leads to gene silencing.
ID089	Gene regulation in Prokaryotes consists largely of transcriptional regulation where eukaryotic regulation consists of regulation at all stages of the process of DNA>Product. Eukaryotes have nucleosomes that can be methylated or acetylated. The mRNA transcripts can be modified in many ways including the addition of a 5 prime cap or a poly A tail. Additionally, mRNA transcripts can be spliced in many different ways, increasing the number of possible gene products. There are even more methods beyond this.
ID089	The swi/snf product is involved in nucleosome and histone remodeling complex. Some types of yeast can use this product to switch their mating types in response to certain factors.
ID089	Acetylation is associated with increased gene expression while deacetylation is associated with deacreased gene expression. The acetyl groups neutralize the positive charge on the histone groups of the nucleosomes, resulting in a more accessible molecule to polymerases.
ID116	Prokaryotic gene regulation is only controlled at transcription, while eukaryotic gene regulation is controlled with epigenetics, transcription, post-transcription, translation, and post-translation. Furthermore, prokaryotic gene expression occurs only in the cytoplasm and DNA moves about freely as there is no defined nucleus. In eukaryotic cells, gene expression occurs in both the nucleus and cytoplasm.
ID116	Nucleosomes that have been positioned often affect the positioning of nearby nucleosomes.  The two cis factors affect barrier positioning.
ID116	Acetylation leads to increased gene expression with transcription being activated. Deacetylation makes it harder for transcription to begin because transcription factors cannot bind to DNA. This is called gene silencing.
ID023	Given the overall complexity of Eukaryotes compared to prokaryotes, there is no surprise that gene regulation would be more complex. Eukaryotes have gene regulation in almost every stage in transcription from inside the nucleus to the production of proteins. Eukaryotes also have different binding sites and different transcription factors and enzymes that are incorporated in the process of transcription as well.
ID023	The presence of nucleosomes makes the transcription process difficult in some cases and next to impossible in others. the SWI/SNF complex takes part in chromatin remodeling and loosens the histones from the DNA strand; thus allowing for transcription. Without this complex, the DNA strand would not be able to  be transcribed because of the presence of histones, or only sections of DNA would be transcribed (which could lead to a nonfunctional protein)
ID023	Acetylation and deacetylation can cause the chromatin to tighten around histones and become silenced or loosen around histones and become active. This is why cells on our tongue are different from cells on our liver. The tighter certain sections of chromosomes are, the more inactive and vice versa. This topic is crucial when it comes to understanding the human body.
ID008	The genome being regulated is much larger and eukaryotes are more multilayered with more gene to gene expression
ID008	They are capable of altering the DNA positions in nucleosomes.
ID008	When deacetylating the histone tails, the DNA becomes more tightly wrapped around the histone cores, making it harder for transcription factors to bind to the DNA.
ID031	This requires many more initiators and other enhancers throughout the process than prokaryotes do making it more complicated.
ID031	The does not posses the ability to re-position nucleosomes which is a critical part of the process so many things can go wrong because of this.
ID031	The tails of nuccleosomes can either be added by actetylation or they can be removeed by the deacytlyation and this woulf affect the way that chromsomes would then condense after this.
ID147	Eukaryotic gene regulation happens at all stages of epigenetics. From transcriptional to post-transnational eukaryotic regulation is much more in depth that the mostly transnational control exhibited in prokaryotes.
ID147	Since the swi-snf mutatnts are missing the SWI/SNF complex which alters the position of nucleosomes. Since they cannot produce the complex they then cannot properly regulate histone movement that is characteristic of nucleosome repositioning.
ID147	Histones with few acetyl groups tend to be inactivated or silenced genes. Also deacetylated histones are much highly condensed when compared to highyl acetylated regions.
ID087	Gene regulation occurs in the nucleus of the eukaryotic cell, where in prokaryotes there is no nucleus. Prokaryotes do not have histone to regulate how much a gene is regulated, however they do have repressor proteins.
ID087	Without the swi-snf gene, the cell cannot alter the position of nucleosomes. Without this altering, genes cannot be regulated.
ID087	Acetylation and deacetylation loosen and tighten the DNA around the nucleosomes. With this loosening and tightening, it affects how easily RNA Polymerase can have access to the genes that need to be transcribed or not transcribed. The more space, the more transcription.
ID072	In eukaryotes, gene expression is tightly controlled in order to express the required levels of gene products at specific times, in specific cell types.
ID072	Repositioned nucleosomes make regions of the chromosome accessible to transcription regulatory proteins such as transcription activators and RNAP II.
ID072	Acetylation decreases  the positive charge on histones, resulting in a reduced affinity of the histone for DNA. In turn, this may assist the formation of open chromatin conformations which would allow the binding of transcription regulatory proteins to DNA.
ID038	Many regulators control eukaryotic gene such as methylation and acetylation. However, this is different in prokaryotes because the DNA is subjected the cytoplasmic environment and thus can be directly regulated. Eukaryotes therefore require more gene regulators.
ID038	The mutation changes the structure of protein which makes it non-functional. As a result, the new protein cannot reposition nucleosomes effectively in DNA.
ID038	Acetylation makes the DNA bulky and keeps out the transcription enzymes which leads to gene silencing. Also, acetyl groups are bulky which do not allow DNA to condense properly either.
ID060	The regulation in eukaryotes is far more complex due to its massive amounts of information needed to be regulated. There are far more genes in eukaryotes since there are a lot more chromosomes. Prokaryotes only have 1 chromosome.
ID060	The swi-snf mutant can reposition nucleosomes.
ID060	The acetylation and deacetylation of nucleosome tails can alter the rate of expression of the genes.
ID092	Gene regulation is more complex for a lot of reasons.  I have not had time to spend on the material that I should have so I do not have any good answers for these questions.  I plan on Studying on Thanksgiving break.
ID092	See First
ID092	See First
ID002	Eukaryotic gene regulation is more complex than regulation in prokaryotes because there is a greater amount of DNA that is associated with histones and other proteins, mRNAs must be spliced, capped, and a poly A tail must be added prior to leaving the nucleus, gene on numerous chromosomes are enclosed in a double membrane nucleus, and there is a modulation process for translation
ID002	The SWI/SNF in eukaryotes is deficient in nucleosome repositioning in that it loosens attachment between histones and DNA, loosens DNA strand from nucleosome core, and causes reorganization of internal nucleosome components
ID002	Acetylation decreases the positive charge on histones, resluting in a reduced affinity of the histone for DNA; this may assist the formation of open chromatin confromations which would allow the binding of transcription regulatory proteins to DNA. Acetylated histones recruit remodeling complexes that de-condense chromatin for transcription. De-acetylated chromatin recruit protein complexes that repress transcription
ID055	Many components make gene regulation in eukaryotes than in prokaryotes. Some of these include greater number of DNA associated with proteins and histones, mRNA's have to be capped, spliced and polyadenylated before transported to the nuclease to be translated, and that is just a few things.
ID055	SWI/SNF mutant allows for reorganization of nucleosomes by moving and loosening the nucleosomes so they are able to be rearranged. If the eukaryote Saccaromyces is deficient in nucleosome repositioning then then it has the SWI/SNF mutant that which is part of the repositioning nucleosome.
ID055	Nucleosomes that are rich in acetyl groups are active genes and those that are poop in acetyl groups are inactive gens. HAT allows genes to uncoil and available for transcription. HDAC coils the gene and the gene becomes inactive making in unavailable for expression and copying.
ID047	Eukaryotes are more complex and therefore have different set of regulatory controls. Eukaryotes are not regulated by operons. Each gene has its own promoter and is transcribed independently. They also have many more transcriptional activators and repressors. Mechanisms also differ between prokaryotes and eukaryotes because transcription and translation are physically separated.
ID047	The swi-snf is a nucleosome remodeling complex. Without the efficiency of swi-snf access to DNA for transcription and gene regulation
ID047	Acetylation relaxes chromatin condensation and exposes DNA for TF being essential for gene regulation. deacetylation is the reverse and an acetyl group is removed.
ID069	Gene regulation is more complex in eukaryotes compared to prokaryotes for numerous reasons. For one, eukaryotes contain a greater amount of DNA, which is associated with histones and other proteins. Additionally, in eukaryotes have to be regulated, undergoing splicing, capping, and polyadenylation, before being transported out of the nucleus. mRNAs also have a wide range of half-lives, and can be modulating during translation, protein processing, modification, and degradation. Also, genes on numerous eukaryotic chromosomes are enclosed in a double membrane nucleus.
ID069	The SWI/SNF complex is a remodeling complex that can act in several different ways to cause DNA to be exposed from the nucleosome and transcribable. It was discovered that this complex aids in nucleosome repositioning after the effects of mutant Saccharomyces SWI/SNFs were observed. SNI mutants are defective in the ability to switch mating type, and SNF mutants are defective in the ability to ferment sucrose (sugar-nonfermenting). Mutants possessing both SWI/SNF are deficient in nucleosome repositioning, indicating that SWI/SNF plays a crucial role as a remodeling complex.
ID069	Histones may be modified by acetylation, whereby the enzyme histone acetyltransferase (HAT) adds an acetyl group to the positively charged amino group. Doing so neutralizes the positive charge of the histones and negates their attraction to DNA. This allows the chromatin to open up and become available for transcription. On the other hand, deacetylation occurs when histone deacetylases (HDACs) remove acetate groups from histone tails, causing the chromatin to close back up around the histones. Deacetylation causes the chromosome to once again condense and effectively silences those genes for the time being.
ID093	The presence of a nucleus means that mRNA can be regulated before it enters the cytosol. The mRNA must be spliced and capped before it can leave the nucleus.
ID093	SWI/SNF is a nucleosome remodeling complex that repositions nucleosomes so that transcription may occur.
ID093	Acyl groups at the end of histone tails will cause the nucleosomes to be closer together than if the tails were methylated. This prevents expression and silences the gene.
ID078	In the simplest terms, prokaryotic gene regulation is controlled mainly at the transcription phase while eukaryotic regulation occurs in all phases. Since prokaryotes don't have a defined nucleus, this happens in the cytoplasm, but eukaryotes are able to regulate genes in both the cytoplasm and nucleus.
ID078	Not covered in class and haven't read about this in the book yet.
ID078	Acetylation adds an acetyl functional group to a molecule while Deacetylation removes an acetyl functional group from a molecule. Acetylation decreases reaction with the N termini of DNA and therefore letting the structure of the DNA condense and relax in order to create a structure that is more agreeable to transcription. Deacetylation makes DNA wrap tightly around the histone cores, leading to gene silencing.
ID088	Prokaryotic gene expression is primarily controlled at the level of transcription. On the other hand, eukaryotic gene expression is controlled at the levels of epigenetics, transcription, post-transcription, translation, and post-translation.
ID088	It is a group of proteins that associate to remodel the way DNA is packaged. Therefore, cannot switch mating type and also cannot ferment sucrose.
ID088	Deacetylated histones may recruit other protein complexes that repress transcription. Acetylated histones may recruit remodeling complexes that de-condense chromatin for transcription.
ID007	Eukaryotic regulation involves a larger variety of proteins and mechanisms.  The fact that unlike prokaryotes they have transcription factors and the histone code involved in regulation accounts for much of this.
ID007	Without being able to move the nucleosome proteins out of the way to access the coding DNA, these mutants cannot transcribe mRNA for this protein.
ID007	Hyperacetylated nucleosome tails are found in active genes, whereas hypoacetylated tails are found in condensed, inactive genes.
ID127	Gene regulation in eukaryotes is more complex because there are transcription factors instead of operons. The transcription factors help induce or reduce the amount of regulation in the gene. For example, enhancers will enhance the regulation of genes and may increase gene expression. Activators will activated the transcription, and repressors will stop or pause transcription. For prokaryotes, there are operons. The operons have different structural genes that will produce different enzymes which will determine whether or not beta galactosidase.
ID127	The mutant gene causes the disruption of nucleosomes which makes the yeast deficient in nucleosome repositioning. Some genes cause defects in the complex which alters its function.
ID127	Acetylation adds an acetyl group which exposed the DNA and increases gene expression. Deaceylation is the removal of an acetyl group which tightens the nucleosome and DNA is not exposed therefore gene expression is decreased.
ID107	There are a greater amount of DNA associated with histones and other proteins in euakaryotes than prokaryotes. in eukaryotes, mRNA must be spliced, capped, and polyadenylated before leaving the nucleus. Genes from many chromosomes are enclosed in a double membrane nucleus in eukaryotes. Eukaryotic mRNAs have a wide range of half lives.
ID107	Swi/Snf is a remodeling complex that loosens the attachment between the histone and DNA, loosens DNA strand from nucleosome core, and causes reorganization of internal nucleosome components. Swi/snf mutants were unable to switch mating types and ferment sucrose.
ID107	Active genes are hyperacetylated (have histone groups rich in acetyl groups) and inactive genes are hypoacetylated (have histone groups poor in acetyl groups). Acetylated histones recruit remodeling complexes that decondense chromatin for remodeling and deacetylated histones recruit other protein complexes that repress transcription.
ID049	There are many more mechanisms for gene regulation in eukaryotes: histones, activators, enhancers, and inhibitors. Prokaryotes are affected by their environment and operons
ID049	main function is nucleosome repositioning for gene expression mutant leads to dysfunction
ID049	addition and subtraction of functional groups change the structure of chromosomes and can cause different parts to be tightly wound or become relaxed.
ID011	Eukaryotic gene regulation is greatly influenced by the relation between histones and DNA.
ID011	Swi-snf complex is key to the remodeling between histones and DNA and thus gene expression.
ID011	The modification of histone tails with acetyl groups influences gene expression in chromosomes.
ID044	Gene regulation is different between eukaryotes and prokaryotes.  Prokaryotic gene expression is mainly controlled at the transcription level.  Eukaryotic gene expression is controlled at many different places like epigenetics, transcription, post-transcription, translation, and post-translation. Prokaryotic regulation  occurs within the cytoplasm because there is a lack of a nucleus. Eukaryotic gene expression occurs in both the nucleus and cytoplasm. Regulation can occur at all stages of the process, like when the DNA is uncoiled and loosened from nucleosomes to bind transcription factors.  It can also be regulated when the RNA is transcribed or post-transcription, when RNA is made into proteins, or after the proteins have been made.
ID044	The Swi-Snf is a complex found in both eukaryotes and prokaryotes and is a group of proteins that remodel the way DNA is package and can destabilise histone-DNA interactions. the swi-snf allows yeast to change mating types so the mutation is deficient in repositioning required to switch types.
ID044	Acetylation and deacetylation changes the amount of chromosome condensation, which then affects gene silencing.  These reactions are catalysed by enzymes histone acetyltransferase (HAT) and histone deacetylase (HDAC).  Adding or removing acetyl groups to histone tails condenses/ de-condenses the chromosomes, which then regulates transcription of genes.
ID068	In Eukaryotes, DNA is highly used with histones and nucleosomes unlike prokaryotes. There is also many types of post transcription modifications involved such as capping, splicing, and polyadenylation. mRNA also has a wide range of half lives and modifications with translation.
ID068	SWI/SNF deals with chromatin remodeling by repositioning of nucleosomes. This remodeling makes genes more available for transcription factors and polymerases. SWI is mutants that are able to switch their mating type. SNF (sugar non fermenting) is mutants that could not ferment sucrose.These mutants were studied in their ability to reposition nucleosomes.
ID068	Active genes on a chromosome are rich in acetyl groups and inactive genes are poor in acetyl groups. Histones that are acetylated have remodeling complexes that decondense chromatin and allow transcription. Deacetylated histones have other complexes that will not allow or repress transcription.
ID143	Gene regulation in eukaryotes is generally more complex than regulation in prokaryotes because these two types of cells differ in their structure and function in various ways.  First of all, eukaryotes have many more genes than prokaryotes, and these genes are spread among many chromosomes within the organism, while prokaryotes do not have as many genes to spread across multiple chromosomes.  Furthermore, the genes of a prokaryote are all contained within an individual chromosome.  In prokaryotes, groups of genes that produce similarly-functioning proteins are organized into operons, while these genes are not organized in operons of eukaryotes.  Ultimately, the greater complexity of eukaryotic cells signifies that these organisms require a more complex system of gene regulation.
ID143	The swi-snf mutant in the eukaryote Saccharomyces is a complex system of various subunits that requires the energy of ATP to control gene expression within organisms.  This mutant is typically responsible for repositioning nucleosomes, which aids in the process of modulating gene expression within the organism.  However, in Saccharomyces, this ability is deficient, leading to lack of control of gene expression within the organism.
ID143	Acetylation is the process in which an acetyl group is added to a molecule; deacetylation is simply the reverse process.  When an acetyl group is added or removed from a nucleosome tail, chromosome condensation and gene silencing can be affected because this process often causes the DNA to be more tightly or loosely packed together in the long run.
ID028	Gene regulation goes through more steps in eukaryotes than in prokaryotes. For example, in order for a eukaryotic cell to be able to express a gene, it must not be wound up tightly into nucleosomes, and must be euchromatin. Prokaryotic cells do not even have histone proteins to exhibit this kind of gene regulation.
ID028	Because of this deficiency in nucleosome repositioning, yeast cannot switch their mating type under certain environmental conditions.
ID028	Acetylation and deacetylation of nucleosome tails causes some genes to get turned on because the DNA is not tightly packed into heterochromatin. Genes can also be turned off by the assembly of these nucleosome complexes.
ID043	It is more complex due to multiple factors
ID043	I didn't get to review this topic
ID043	This has to do with histones
ID123	Gene regulation in eukaryotes is much more complex than in prokaryotes for a number of reasons.  There is a greater amount of DNA that is associated with histones and other proteins.  mRNAs must be spliced, capped, and polyadenylated before being transported to the cytoplasm.  Genes on numerous chromosomes are enclosed in a double membrane nucleus.  mRNAs have a wide range of half lifes. Modulation of mRNA translation, protein processing, modifications, and degradation are much more complex.
ID123	The swo-snf mutant in the eukaryote Saccharomyces is deficient in nucleosome repositioning, which leads to the yeast not being able to ferment sucrose or switch mating types.  This because the mutation does not allow for the chromatin to be remodeled by allowing the DNA to separate from the histones and nucleosomes.
ID123	Acetylation of nucleosome tails recruits remodeling complexes that decondenses chromatin for transcription.  This renders the gene active.  Deacetylation of DNA recruits other protein complexes that repress transcription.  This condenses chromatin and inactivates genes.
ID009	Eukaryotes are more complex in general, however, gene regulation is more complex in eukaryotes because they are much larger. This means that they have more going on and many different areas of the body. Eukaryotes need mechanisms in the body that can travel quickly throughout it.
ID009	Swi-snfs typically use ATP hydrolysis to rearrange nucleosomes and regulate transcription. When swi-snf is taken from Saccharomyces its structure is altered. Their job is to promote the switching of positions of nuclesomes and to help regulate some initiation.
ID009	Both of these are important parts of gene regulation. Deacetylation undoes acetylation- which transfers acetyl group between molecules.
ID006	Prokaryotic gene expression is primarily controlled at the level of transcription.
ID006	The DNA of eukaryotic cells is spooled around large histone protein complexes, forming nucleosomes that make up the basis for a high-order packaging structure called chromatin. Compared to naked DNA, nucleosomal DNA is less accessible to regulatory proteins and regulatory processes. The exact positions of nucleosomes therefore influence several cellular processes, including gene expression.
ID006	egulation. These reactions are typically catalysed by enzymes with "histone acetyltransferase" (HAT) or "histone deacetylase" (HDAC) activity. Acetylation is the process where an acetyl functional group is transferred from one molecule (in this case, Acetyl-Coenzyme A) to another. Deacetylation is simply the reverse reaction where an acetyl group is removed from a molecule.
ID048	Prokaryotic gene expression is primarily controlled at the level of transcription. Eukaryotic gene expression is controlled at the levels of epigenetics, transcription, post-transcription, translation, and post-translation. These are key differences.
ID048	s a nucleosome remodeling complex found in both eukaryotes and prokaryotes. In simpler terms, it is a group of proteins that associate to remodel the way DNA is packaged.
ID048	are the processes by which the lysine residues within the N-terminal tail protruding from the histone core of the nucleosome are acetylated and deacetylated as part of gene regulation. Acetylation is the process where an acetyl functional group is transferred from one molecule to another. Deacetylation is simply the reverse reaction where an acetyl group is removed from a molecule.
ID062	Prokaryotic transcription and translation occur simultaneously in the cytoplasm, and regulation occurs at the transcriptional level. Eukaryotic gene expression is regulated during transcription and RNA processing, which take place in the nucleus, and during protein translation, which takes place in the cytoplasm. Further regulation may occur through post-translational modifications of proteins. so one is not more difficult than the other, just different.
ID062	The primary function of the SWIB protein domain is to aid gene expression. In yeast, it expresses certain genes, in particular BADH2, GAL1, GAL4, and SUC2. It works by increasing transcription. It has ATPase activity, which means it breaks down ATP, the basic unit of energy currency. This destabilises the interaction between DNA and histones. This disrupts chromatin and opens up the transcription-binding domains. Transcription factors can bind to this site, leading to an increase in transcription.
ID062	enzymatic modifications acetylation, and methylation, primarily occur at N-terminal histone tails. Such modifications affect the binding affinity between histones and DNA, and thus loosening and tightening the condensed DNA wrapped around histones, e.g., Methylation of specific lysine residues in H3 and H4 causes further condensation of DNA around histones, and thereby preventing binding of transcription factors to the DNA leading to gene repression. On contrary, histone acetylation relaxes chromatin condensation and exposes DNA for TF binding, leading to increase gene expression
ID050	Eukaryotic gene expression is regulated by multiple factors including epigenetics, transcription, post-transcription, translation, and post-translation. While prokaryotic expression is only controlled at the transcription level.
ID050	When the swi-snf mutant does not allow for nucleosome reposition thus not allowing for different gene expression.
ID050	The acetylation and deacteylation affects nucleosome tails by causing certain chromosomes to be deterred from presenting as well as the other chromosomes and thereby affecting the way genes express.
ID057	Eukaryotes gene expression occurs in the nucleus, transcription, and cytoplasm, translation. Prokaryotes gene expression happens only within the cytoplasm due to the DNA being free floating in the cell. Further regulation may occur through post translational modifications in proteins.
ID057	The swi-snf is a remodeling complex and utilizes energy stored up at ATP to reposition the nucleosomes. Therefore, if there is a mutation in the swi-snf saccharomyces the nucleosome repositioning could be affected in the gene expression.
ID057	Acetylation is the process where an acetyl functional group is transferred from one molecule ) to another. Deacetylation is simply the reverse reaction where an acetyl group is removed from a molecule. This determines what genes are active and what genes are deactivated.
ID119	Eukaryotes have many differenent variants of specialized cells and they are larger and more complex in general, which leads to gene regulation being more complex as a result in eukaryotes as compared to prokaryotes.
ID119	The swi-snf mutation in eukaryotic cells causes deficiency in the repositioning of the nucleosome.
ID119	Chromosome condesnation and gene silencing are simply affected by the acetylation and deacetylation of nuclesome tails.
ID137	Eukaryotic gene regulation occurs due to epigenetics, transcription, post-transcription, translation, and post-translation whereas prokaryotic gene regulation only occurs due to transcription.
ID137	swi is a mutant that is unable to ferment sugar and snf is a mutant that cannot switch mating type. The mutant swi-snf refers to part of a remodeling complex that helps to reorganize nucleosomes.
ID137	Acetylation is the addition of an acetyl group to a molecule and deacetylation is the removal of an acetyl group from a molecule. Deacetylation brings about condensation and gene silencing.
ID102	Instead of the gene regulation being controlled at a single level (transcription), eukaryotes are much more complex. They are controlled at multiple levels (epigenetics, transcription, post transcription, etc).
ID102	I'm unclear about this concept. Very confused about it.
ID102	These are reactions catalyzed by HAT and HDAC activity. Acetylation is when there is a transfer of one molecule to another, in this case acetyl to acetyl-coenzyme A. Thus, deacetylation is just the reverse of this process.
ID129	Gene regulation is more complex in eukaryotes because eukaryotes are larger. Prokaryotes are less complex; they have to have fast response.
ID129	Nucleosome positioning is critical for gene expressioning.
ID129	Acetylation is when an acetyl group is introduced into a compound. Methylation occurs when a methyl group is introduced into a chemical compound.
ID117	Gene expression is more complex in eukaryotes because eukaryotic cells are often much more specialized--such as in humans--so they require more specific mechanisms of regulation. Also, multicellular eukaryotes require complex mechanisms to carry stimuli over long distances, through multiple cells, very quickly.
ID117	SWI/SNF mutants are cells that are unable to ferment sucrose to fulfill chromatin remodeling and the repositioning of nucleosomes. The eukaryotic Saccharomyces mutants were unable to loosen chromatin strands and allow for nucleosomes to reposition themselves because they were unable to ferment sucrose.
ID117	Deacetylation of nucleosomes has a direct affect on silencing genes, because it tightens coils and does not allow for transcription. Acetylation loosens nucleosomes to allow for transcription.
ID034	eukaryotic cells contain a much greater amount of DNA than do prokaryotic cells
ID034	the DNA is left transiently exposed to association with transcription factors and RNA polymerase
ID034	acetylation decreases the positive charge on pistons, resulting in a reduced affinity of the histone for DNA
ID100	Gene regulation in eukaryotes has been proven to be more complex than regulation in prokaryotes. For starters prokaryotic gene expression is controlled mainly at the transcription level, whereas eukaryotic gene expression includes transcriptional regulation, epigentics, post-transcriptional regulation, and translational, and post-translational regulation. Basically, eukaryotic cells can regulation gene expression at almost any point within the central dogma. Also, prokaryotic gene expression occurs in the cytoplasma whereas eukaryotic gene expression occurs within the nucleus. This difference allows prokaryotic cells to begin translation before trascription has been completed. In eukaryotic cells, transcription is temporally and spatially seperated. Transcription occurs in the nucleus and translation occurs in the cytoplasm at the ribosome. Also, transcription in eukaryotes has to be completed before translation can begin.
ID100	The swi-snf complex in eukaryotic Saccharomyces is an ATP-dependent chromatin remodeling complex used by cells to stimulate or repress gene transcription. The swi-snfi complex binds to both the DNA and the nucleosome but does not do so in a site-specific fashion. When swi-snf is absent, as seen in the mutant eukaryote, mRNA levels are significantly decreased. Basically, the swi-snf complex remodeles chromosomes making genes more accessible for transcription by loosening DNA strands or by reorganizing internal nuclosome components. When this complex is absent, this form of gene regulation is also absent making DNA less capable of being transcribed.
ID100	Acetylation and deacetylation of nucleosome tails in histones can affect chromosome condensation and gene silencing. This process involves the addition or removal of an acetyl group to the lysine residues within the N-terminal tails of the nucleosomes that make up histones. This process helps regulate gene expression. Depending on how acetylated or deactylated the histones in a chromosome are. The enzyme responsible for the acetylation of these proteins is HAT, or histone acetyltransferase, and the enzyme responsible for deacetylating these proteins is HDAC, histon deacetylase. Acetylation relaxes and uncondenses chromatin whereas deacetylation recondenses chromatin. When the chromatin is relaxed DNA binding proteins are better able to interact with the genes activating gene transcription.
ID130	There are many factors complicating the gene regulation of eukaryotes.  For starters, eukaryotes have two sets of each gene, making a competition for which gene is read.
ID130	The position of nucleosomes in the chromosomes is what determines the active genes.  If swi-snf is deficient then the genes that need to be active are not active and genes that don't need to be expressed are expressed.
ID130	Acetylated genes are condensed and and silent.  Non-acetylated genes are expressed.
ID073	Transcription control in eukaryotes is more complex than in prokaryotes, with more gene-gene
ID073	There are several classes of chromatin remodeling complexes, including the SWI/SNF (switch/sucrose-nonfermentable), ISWI (imitation switch), INO80 (inositol-requiring), and CHD (chromo-helicase/ATPase-DNA-binding) families. All complexes possess a conserved ATPase subunit that provides free energy to facilitate DNA translocation, but different subfamilies exhibit divergent remodeling activities.
ID073	Histone acetylation and deacetylation are the processes by which the lysine residues within the N-terminal tail protruding from the histone core of the nucleosome are acetylated and deacetylated as part of gene regulation. Histone acetylation and deacetylation are essential parts of gene regulation.
ID120	Gene regulation is eukaryotes requires a wider range of enzymes to carry on the process of transcription.
ID120	If the nucleosomes aren't repositioned then they will not be able to transcribed when genes are needed.
ID120	Acetylation decondenses the nucleosomes and allows for transcription to proceed. Deacetylation will prevent the ability for transcription.
ID150	Prokaryotes only regulate genes on the transcriptional level, while eukaryotes modify genes with epigenetics, during and after transcription, and during and after translation. Prokaryotes only perform regulation in the cytoplasm, while eukaryotes do in a variety of environments.
ID150	The swi/snf mutation causes the yeast cells to be unable to produce working proteins that modify chromatin. Nucleosome repositioning is the methylation of portions of genes in order to deactivate them.
ID150	Acetylation of tails removes the positive charge. Without a positive charge, the tails are less reactive and loosen up. Once loosened, the DNA strand can be transcribed.
ID029	Eukaryotic gene regulation involves more regulatory regions, like enhancers and silencers, than prokaryotic gene regulation.
ID029	This mutant can insert transpose genes to change its sex.
ID029	The acetylation and deacetylation is responsible for loosening and tightening the chromatin.
ID066	Gene regulation in eukaryotes is more complex and requires more protein structures and enzymes in order to be completed when compared to prokaryotes
ID066	Saccharomyces is a yeast which contains mutants that make the transcription process deficient in nucleosome re repositioning.
ID066	Adding of acetate groups to nucleosome tails of transcribing DNA causes the transctiption proteins to function incorrectly.
ID003	Eukaryotes actually have a nucleus and
ID003	Do not know much information about this.
ID003	do not know much information about this.
ID012	Prokaryotes occurs only in the cytoplasm. Eukaryotes occur in the nucleus and the cytoplasm.
ID012	It causes ATP-dependent disruption of nucleosome structure and this causes an increase in the binding of transcription factors to their sites on nucleosomal templates
ID012	Acetylation causes the hustings to have negative charges causing the chromatin to relax and spread out. Deacetylation does the opposite by making the chromatin get tighter and condensed.
ID131	The regulation of genes in eukaryotes is much more complex thanks to epigenetics having large effects on gene expression. There are also translational differences as well as post transcriptional modifications.
ID131	The swi-snf is a protein complex that changes the nucleosome and allows the nucleosome to be unwound. The mutant causes this to not be allowed to occur.
ID131	The acetylation and deacetylation of nucleosome tails would be main effectors in the epigenome. These changes to the nucleosome regulate the state of a chromosome as well the expression of many genes throughout the body.
ID030	As would be expected, a more complicated type of cell indeed as a more complex gene regulation process. Firstly, gene regulation in eukaryotes can occur at any of the steps between DNA and protein product. The wide range of regulation techniques allows the amount of gene product produced to be precisely controlled. This includes regulation elements such as DNA methylation, promoters, enhancers, etc.
ID030	The swi/snf complex functions to remodel the packaging of DNA. This can cause changes in the expression of some genes, as genes become either available or unavailable to be read. In Saccharomyces, it was found that the swi-snf complex could actually cause a change in the mating type of the organism.
ID030	This is another valuable form of gene regulation that cell use to control gene product. When the N-terminal tail of a histone is acetlyated, it removes the positive change from the histone. This causes the interaction between the histone and DNA to become weakened, causing DNA to become less condensed. This then allows DNA to be transcribed and translated into gene product.
ID070	Gene regulation is significantly more complex in eukaryotes than in prokaryotes for a number of reasons. First, the genome being regulated is significantly larger. The E. coli genome consists of a single, circular chromosome containing 4.6 Mb. This genome encodes approximately 2000 proteins.
ID070	In molecular biology, SWI/SNF (SWItc, is a nucleosome remodeling complex found in both eukaryotes and prokaryotes. In simpler terms, it is a group of proteins that associate to remodel the way DNA is packaged. It is composed of several proteins – products of the SWI and SNF genes as well as other polypeptides.It possesses a DNA-stimulated ATPase activity and can destabilise histone-DNA interactions in reconstituted nucleosomes in an ATP-dependent manner, though the exact nature of this structural change is unknown.
ID070	Histone acetylation and deacetylation are the processes by which the lysine residues within the N-terminal tail protruding from the histone core of the nucleosome are acetylated and deacetylated as part of gene regulation.
ID091	Prokaryotes have circular, double stranded DNA, while eukaryotes have DNA organized around histones into nucleosomes. So, prokaryotes have a much more simple gene regulation. Eukaryotes can alter the structure of their DNA while prokaryotes rely solely on operons.
ID091	Swi-snf mutants can not ferment sucrose or switch mating types. The gene mutation occurred within a multi-gene region that coded for a complex that allowed for the loosing of histones around DNA, allowing exterior DNA to interact with interior DNA.
ID091	Acetylation works by neutralizing the positive charge on lysine residues of the histone. The histones relax the DNA, and allow for mRNA transcription to occur. Hypoacetylation can silence genes as the DNA becomes very condensed.
ID063	More DNA is required for eukarytoes. mRNAs need to be spliced and have polyA added before transportation can begin. The genes of chromosomes are in a nucleus that has a double membrane.
ID063	The function of the Swi/Snf is to loosen that histone from the DNA through the nucleosome core. Mutants are defective where they cannot change their mating type.
ID063	Acetylation allows fro the chromosome tails to grow where deacetylation will terminate the growth of the tails allowing for chromosome condensation.
ID141	Gene regulation is more complex in eukaryotes compared to prokaryotes due to many differences between the two cell types in general.  Eukaryotes have more DNA which is complexed with histones.  Also, messenger RNA must be spliced, capped, and have a tail added before it can be sent out of the nucleus in eukaryotes.  Finally, many modifications can be had once the messenger RNA has been translated, giving the wide array of proteins possible from an entire power less of specific genes.
ID141	This is true because the swi-snf non mutant is a remodeling complex which exposes different parts of the DNA to transcription factors and RNA polymerase.
ID141	Acetylation assists in the opening up of the chromatin which allows the binding of trascription factors to DNA.  Acetylation is catalyzed by HAT's.  To close the chromatin and make it not available to transcriptions factors, there are HDAC's which remove acetyl groups.
ID125	Eukaryotes are more complex creatures
ID125	the swi-snf mutant can not reposition the nucleosome
ID125	the poly-a tail is very important to gene expression
ID013	In prokaryotes, the gene regulation system consists of the lac operon and the trp operon. In these systems, structural genes are linked by one regulatory gene and promoter and operator sequences. When these systems fail, the system of attenuation is used. The eukaryotic gene regulation system is more complex because it can occur in transcription and anytime after as in during translation, mRNA transport, ribosome regulation, or protein folding and synthesis.
ID013	The swi-snf mutant shows that this complex is involved in chromatin remodeling. It involves a single protein loosening the DNA strand from the nucleosome to reorganize the chromosome for replication or transcription. In Saccharomyces, the swi-snf mutant is manipulated to study this complex.
ID013	Active genes are hyper or over acetylated while inactive genes are hypo or under acetylated. These processes of acetylation and deacetylation of nucleosome tails is regulated by HAT and HDAC enzymes, respectively. When HDAC removes the acetyl group, it can silence genes while acetylated histone tails can talk to proteins that condense chromatin.
ID110	in prokaryotes: it utilizes simple regulatory pathways, lacks long term control of gene expression, single short term regulation sequence is present all along the DNA
ID110	SWI/SNF
ID021	Eukaroytes are more complex than prokaryotes, and therefore must regulate their genes in a more complex fashion. Eukaroytes have a greater amount of DNA that is
ID021	The swi-snf is a nucleosome regulator that causes the DNA to be loosened from it so that transcription can be increased. In swi-snf mutants of yeast, the yeast cannot perform this act and cannot switch mating types.
ID021	Active genes have nucleosomes that are highly acetylated. Inactive genes have histones that are lowly acetylated. Hypoacetylation causes the histone to recruit proteins that cause the chromosome to condense and transcription to be physically repressed in that gene.
ID133	Gene regulation is more complex in eukaryotes than prokaryotes because of the presence of a nucleus in eukaryotes.  Transcription and translation occurs at different times and in different places rather than a single action in prokaryotes. Also there are modifications to DNA in eukaryotes with the presence of histones.
ID133	The swi-snf complex loosens the attachment between histones and DNA,  loosens DNA strand from nucleosome core, or causes reorganization of internal nucleosome component. A mutation in this complex would prevent the repositioning of nucleosomes.
ID133	Acetylated histones may recruit remodeling complexes that decondense chromatin for transcription.  Deacetylated histones may recruit other protein complexes that repress transcription.
ID020	Eukaryotic transcription initiation requires the assembly of transcription regulatory proteins at cis-acting DNA sites known as promoters, enhancer, and silencer. Eukaryotic gene expression is also regulated by multiple posttranscriptional mechanisms including alternative splicing, control of mRNA stability, translation regulation and posttranslational processing. There are many factors that go into eukaryotic gene regulation vs prokaryotic.
ID020	Swi-Snf is a nucleosome remodeling complex found both in eukaryotes and prokaryotes. This complex may loosen the attachment between histones and DNA, resulting in the nucleosome sliding along the DNA and exposing regulatory regions. In the eukaryote Saccharomyces there is a mutant in Swi-SNf causing a deficient in nucleosome repositioning.
ID020	Nucleosome make up histones. Acetylation decreases the positive  charge on histone, resulting in a reduced affinity of the histone of DNA. By deacetylating the histone tails, the DNA becomes more tightly wrapped around the histone cores, making it harder for transcription factors to bind to the DNA. This leads to decreased levels of gene expression and is known as gene silencing.
ID151	There are various reasons as to why gene regulation is much more complex in an eukaryotic organism than a prokaryotic organism. For instance due to the fact that eukaryotes are multi-cellular organisms, they have more DNA in comparison to the prokaryotic or unicellular organisms which requires more extensive packaging of that DNA. Hence, eukaryotic gene regulation deals with removing and re-attaching the histone proteins that allow condensation of the DNA; processes such as acetylation, deacteylation, and methylation are all processes that affect the amount of histone proteins that are wrapped inside DNA thus affecting gene expression. Another difference that results in the complexity of gene regulation in eukaryotes is the fact that transcription and translation do not occur simultaneously since there is a nucleus that holds the chromosomal DNA  (transcription occurs in the nucleus while translation occurs in the cytoplasm).  This causes another level of gene regulation in eukaryotes that is not present in prokaryotes.Therefore, these physiological differences in between the eukaryotes and prokaryotes results in the high level of complexity of gene regulation in eukaryotes.
ID151	I currently do not know much information about this topic, but I do know that the SWI-SNF mutants got their name since the mutants are unable to to switch between mating types depending upon the given environmental conditions (SWI) and they are also unable to ferment sucrose ( SNF= sucrose non-fermenting). Furthermore, the since the mutants are deficient in nucleosome repositioning, I am assuming there is either a possibility of hyper-expression or hypo-expression of a particular gene sequence. I do plan on learning more about this topic soon especially since I am highly unaware about the experiment associated with these mutants and the effect the mutations has on nucleosome repositioning.
ID151	The processes of acetylation and deacetylation enable the chromosomes to decondense and re-condense thus allowing and increase and a decrease in gene expression, respectively. Acetylation is the process of adding an acetyl group to a histone through the enzyme called histone acetyltransferase or HAT. The addition of an acetyl group (negatively charged) to the positively charged histone protein results in the histone protein to become neutral thus losing its affinity for the negatively charged DNA (Phosphate backbone)and thus allows the unraveling or uncoiling of some of the genetic material from its associated proteins. This process results in an increase in gene expression since there are no histone proteins that condense the genetic material to that extreme as before. On the other hand, deacetylation is a process in which decreases the expression of a gene through the enzyme called Histone deacteyltransferase or Hdac which is the reverse process of what is discussed above. This enzyme removes acetyl groups that are previously attached and allows the re-condensation of the genetic material which then suppresses gene expression and enables the silencing of a gene sequence. In conclusion, the attachment and removal of acetyl groups enables the variation in gene expression of certain gene sequences within the genome of the organism.
ID004	how do i explain this sufficiently to even get full credit, so why even try
ID004	I know absolutely nothing on this concept
ID004	I do not get what this is saying
ID113	multicellular eukaryotes control cell differentiation through more complex and precise temporal and spatial regulation of gene expression
ID113	the swi-snf mutant is an ATP-dependent chromatin remodeling complex that can activate or repress transcription
ID113	Histone acetylation and histone deacetylation involve the addition or removal of an acetyl group on lysine residues in the N-terminal tail and on the surface of the nucelosome core of histone proteins
ID053	Gene regulation is more complex in eukaryotes compared to prokaryotes for numerous reasons. For one, eukaryotes contain a greater amount of DNA, which is associated with histones and other proteins. Additionally, in eukaryotes have to be regulated, undergoing splicing, capping, and polyadenylation, before being transported out of the nucleus. mRNAs also have a wide range of half-lives, and can be modulating during translation, protein processing, modification, and degradation. Also, genes on numerous eukaryotic chromosomes are enclosed in a double membrane nucleus
ID053	The SWI/SNF complex is a remodeling complex that can act in several different ways to cause DNA to be exposed from the nucleosome and transcribable. It was discovered that this complex aids in nucleosome repositioning after the effects of mutant Saccharomyces SWI/SNFs were observed. SNI mutants are defective in the ability to switch mating type, and SNF mutants are defective in the ability to ferment sucrose (sugar-nonfermenting). Mutants possessing both SWI/SNF are deficient in nucleosome repositioning, indicating that SWI/SNF plays a crucial role as a remodeling complex.
ID053	Histones may be modified by acetylation, whereby the enzyme histone acetyltransferase (HAT) adds an acetyl group to the positively charged amino group. Doing so neutralizes the positive charge of the histones and negates their attraction to DNA. This allows the chromatin to open up and become available for transcription. On the other hand, deacetylation occurs when histone deacetylases (HDACs) remove acetate groups from histone tails, causing the chromatin to close back up around the histones. Deacetylation causes the chromosome to once again condense and effectively silences those genes for the time being.
ID128	As we've seen with most topics this semester, eukaryotes are more complex in gene regulation than prokaryotes just as they are more complex in just about every other way. For example, the greater complexity of eukaryotic cells can be seen in the fact that each structural gene has its own promoter and is transcribed independently. Since prokaryotes are single cell, they don't have cell specific gene expression.
ID128	The swi-snf remodeler acts in several different ways. It can loosen the attachment between histones and DNA. They can also loosen the DNA from strand from the nucleosome core. Therefore, a mutation in these causes a deficiency in nucleosome repositioning.
ID128	HATs catalyze histone acetylation while HDACs remove acetate groups to accomplish the opposite. Both of these can be recruited by transcription activator proteins while certain activator proteins can actually carry out HAT activity on their own without the need of these extra enzymes.
ID114	Prokaryotic gene expression is primarily controlled at the level of transcription.
ID114	The SWI/SNF complex (in yeast) is capable of altering the position of nucleosomes along DNA.
ID114	Acetylation removes the positive charge on the histones, thereby decreasing the interaction of the N termini of histones with the negatively charged phosphate groups of DNA. As a consequence, the condensed chromatin is transformed into a more relaxed structure that is associated with greater levels of gene transcription.
ID011	DNA sequencing allows us to not only observe the effects of certain genes as both normal and mutated, but is also used for identifying species and genetic disease research.
ID011	Restriction enzymes are able to "cut" apart sections of DNA, usually palindromic sequences, and insert them into vectors.
ID011	Transcription factors can act as repressors, activators, alter histone code, and assist RNA polymerase II.
ID115	Because of different techniques established from the original Sanger technique, sequencing has revolutionized genetics by helping identify genetic diseases and the functions of certain genes.
ID115	restriction enzymes defend bacterial cells by lysing and cutting viral DNA into pieces to prevent infection. They can also be used for DNA sequencing by cutting certain pieces of DNA into gene fragments that can then be cloned for sequencing.
ID115	Transcription factors can enhance or inhibit transcription by allowing or inhibiting binding of RNA polymerase to the promoter. These transcription factors are not located on the same chromosome and will be called to transcription based on the amount of substrate or substance present in a bacterial cell.
ID094	DNA sequencing has allowed scientists to create databases of genomic code and compare functionality of common sequences.
ID094	These enzymes are used by their native cells to degrade foriegn DNA.
ID094	These proteins act within a genome to either activate or deactivate a gene's expression, and are a key element of gene regulation.
ID137	DNA sequencing is the process of determining the order of nucleotides on a strand of DNA. These would be the four bases of Adenine, Guanine, Cytosine, and Thymine - using recombinant DNA technology can help determine this order.
ID137	Restriction enzymes cut DNA into fragments based on the location of restriction sites.
ID137	Eukaryotic gene expression requires many transcription factors in order to function properly. This includes trans-acting factors that bid to cis-acting factors and RNA polymerase in order to stabilize the whole complex.
ID039	DNA sequencing has revolutionized genetics because it has allowed many advancements. It has allowed for sequencing of genomes, genetic engineering, and medical advancements.
ID039	Restriction enzymes are important to bacterial defense mechanisms. They are important defense mechanisms because restriction enzymes cut up viral DNA.
ID039	Transcription factors are trans-acting proteins with specific DNA binding motifs. They are trans-acting because this conformation allows for genes to be expressed more.
ID026	DNA sequencing has revolutionized genetics and has now become one of the most important pieces of information in the field. By knowing an organism's specific DNA sequence we are able to find and locate genes that code for many different things in the organism. Without this information we would not be able to analyze and experiment with genes like we are able to now.
ID026	Restriction enzymes are important proteins that are used as bacterial defense mechanisms. They are able to cut DNA into many pieces by recognizing a particular sequence so that bacteriophage DNA is not able to be replicated with the cell's normal DNA.
ID026	Transcription factors are proteins that are able to bind to DNA in order to control the rate of transcription of a sequence by either promoting or repressing a sequence and its use of RNA polymerase. These factors are key components of gene expression.
ID038	DNA sequencing lets us know how the nucleotides are arranged in DNA. Comparing this sequence to other organism has helped us understand how different organism share the same sequence of nucleotides for a given protein with minor variations.
ID038	Restriction enzymes recognize a specific sequence of DNA nucleotides and then cleave the DNA at a particular site. Since bacteria's own DNA has a unique methylation pattern, it is not digested but other DNA can be and thus it is a defense mechanism.
ID038	Since Transcription factors are just proteins, they can bind to either strand of the DNA which has a particular sequence of nucleotides. Thus they are trans-acting proteins.
ID015	DNA sequencing is the process of determining the precise order of nucleotides within a DNA molecule. It includes any method or technology that is used to determine the order of the four bases—adenine, guanine, cytosine, and thymine—in a strand of DNA.
ID015	A restriction enzyme or restriction endonuclease is an enzyme that cuts DNA at or near specific recognition nucleotide sequences known as restriction sites.
ID015	They are usually considered to be proteins, that bind to the cis-acting sequences to control gene expression
ID084	jflaksjd;fo dpofasdfnc msof ifj ;odfj cadskfn; isdnf
ID084	slkdfn9 ej ;fj'dpvjsvs ,faadsf
ID084	dflsjdf u[09u4r 2qvkfdjsi uf09q 'klcdnslzkh;dufb lfe0 9dsihwf
ID077	DNA sequencing has the potential to sequence entire genomes in days but it is expensive and still has problems with short reads and accuracy. DNA sequencing is the ultimate way to characterize DNA structure at the molecular level.
ID077	Restriction enzymes are produced by bacteria as a defense mechanism against bacteriophages. Restriction enzymes are DNA cutting enzymes that recognize specific sequences to cut. There are over 3000 known restriction enzymes.
ID077	Transcription factors contain 2 or more domains, one for binding to DNA and one for activating transcription. Some transcription factors act by altering the histone code. Transcription factors recognize not  only a specific sequence but also a specific histone code.
ID131	Genetics is an evolving field of study, and the discovery of DNA along with base pairs. DNA sequencing along with the human genome project has allowed us to know more about heritable traits than seems imaginable. It can improve medicine and has served as entire new field of medical research.
ID131	Restriction enzymes are defense mechanisms against viruses. This is because restriction enzymes are able to cleave DNA strands which could prevent an invasive virus DNA from harming the bacteria.
ID131	These proteins bind to sequences in a gene on different regions and either promote or inhibit the binding of RNA polymerase and the production mRNA.
ID090	DNA sequencing is the process of determining the order of nucleotides within DNA. This can be done through Sanger, PCR, and Pyrosequencing for example. These methods allow geneticists to determine the sequence of nucleotides in a gene and as well as determine the function.
ID090	Restriction enzymes are used in cloning to recognize foreign DNA and then cut it so that it cannot harm the bacteria. They do not harm the bacterial DNA since the DNA of the bacteria is typically methylated. Restriction enzymes can be used to cut DNA to be put in vectors.
ID090	Transcription factors are trans acting (meaning that they do not necessarily act on the same chromosome on which they are coded). They are capable of movement and bind specific DNA sequences to aid in transcription.
ID022	If we know the sequence for a particular protein then we can know how that protein is being made, how it is suppose to fold and how i is suppose to react for normal behavior.  Therefore, based on this information if there is a change in the sequence i.e mutation then we can find out how to fix it in the future and also can understand how a person gets new disease like Cystic Fibrosis which is caused due to mutation in chromosome 7. This can be done basically for various amino acids and also to find out how other diseases are caused and to cure them.
ID022	The restriction enzymes are one of the best bacterial defense because they recongonize the foreign DNA and then cleaves it off so that the bacteria does not get affected with that virus or foreign material and can survive. Because of this genomic cleavage the foreign DNA or RNA is not able to replicate itself and thus no disease is caused.
ID022	Transcription factors are trans-acting proteins with specific DNA binding motifs. They recognize specific sequences on the DNA and bind to that site.  In eukaryotes, transcription cannot initiate unless the transcription factors are present. They interact together and eventually create the PIC and then the RNA polymerase is able to bind and initiate.
ID065	DNA sequence has revolutionized genetics because it has allowed us to understand the specific sequences of DNA associated with diseases, mutations, and other genetic interests. Having a greater understanding of these sequences allows for the possibility for more cures and more prevention with respect to these genetic anomalies. Also, further understanding these sequences allows for scientists to be able to manipulate enzymes and sequences in order to make new discoveries and obtain a further understanding of genetic specificities.
ID065	Restriction enzymes make cuts in DNA. Cutting foreign DNA in a bacteria from, for example, a virus, would destroy the function of the DNA. It is obvious how this is a form of a defense mechanism. Restriction enzymes also recognize specific sequences, and if the bacterial cells' restriction enzymes recognize foreign DNA, it can cut it before it even has the possibility to be potentially infectious to the cell.
ID065	Transcription factors acting as trans-acting proteins means that they can function on strands of DNA during transcription and are not limited to a single strand. Being able to attach to more than the strand of origin is helpful because one transcription factor can be used on chromosome A even if it was made on chromosome B.
ID130	allows us to know which genes are where and code for what
ID130	restriction enzymes allow bacteria to  produce restriction fragments which break up dna
ID130	they bind to the dna and begin the transcription process, very specific to a promoter region
ID151	Advancements in genetics such as DNA Sequencing has better enabled researchers to learn more about an organism's genome and the functions within each gene. Furthermore, sequencing an individual's genome also allows one to determine particular diseases he/she may have as well as how that disorder is affected by the lack or excess production of a gene product, a protein. For instance, researchers could use methods such as PCR to amplify a particular gene of interest that may be the root cause of particular symptoms. DNA Sequencing and particular machinery can also assist geneticists in determining the possibility of a child developing a particular hereditary disease if the parents are carriers or even if the parent(s) have the disease.
ID151	Restriction Enzymes were encountered in bacteria such as E. coli when the bacterial cell is being invaded by virus, specifically a bacteriophage. These enzymes can recognize foreign DNA that has entered the cell by the absence of methylation in the foreign DNA/RNA since bacterial cells are known to have methylated DNA. The restriction enzymes are then recruited and cut up the viral DNA thus digesting the foreign material before infection can arise inside the bacterial cell. Once the function of restriction enzymes were noticed by researchers and became a great tool for creating recombinant DNA.
ID151	Transcription factors are known to be trans-acting proteins since the proteins can affect other gene sequences that may not be a part of the gene sequence that codes for the transcription factors. For instance, a tanscription factor TBP or TATA binding protein recognizes the TATA box and binds to that region of the promoter sequence. Furthermore, the transcription factors not only recognize a particular gene sequence but they also recognize specific histone modification patterns as well. These transcription factors can then bind to their corresponding sequences of modifications even if the recognized gene is a part of a different chromosome.
ID082	The ability to sequence DNA has given geneticists and researchers a chance to investigate and pin point the exactly cause of genetic diseases. Knowing the where and the what of the mutation, medical researchers can then assess the damage and work to find a way to reverse or ameliorate the defective DNA. DNA sequencing paved the way for the Human Genome Project, through which researchers from around the world collaborated and worked together to sequence all of human DNA. DNA sequencing has allowed for both genomic and complementary libraries, which are pivotal tools in genetics research. A genomic library has one copy of all the sequences of the genome that the researcher is investigating. Complementary DNA libraries contain the complementary DNA copies made from mRNAs, meaning these libraries contain the DNA of genes that are active transcriptionally (no introns).
ID082	Restriction enzymes are a special kind of enzyme that cuts DNA. These enzymes do more than just cut the DNA at random, rather, they bind and cut the DNA at a specific place in the sequence, giving various fragments of the DNA strand. When this technique is used on a specific sample of DNA, that DNA sample is said to be digested. By using restriction enzymes in research, the fragments that result from the digest by restriction enzymes can be analyzed for their size. Using known restriction enzymes (over 3000 options), geneticists can determine whether or not an individual has a mutation at a specific restriction site because of the presence or absence of a fragment in the gel (that they use to tell how big the fragments are).
ID082	Trans-acting proteins are proteins that can “float around” the chromosomes, and can act on many different chromosomes and influence a variety of genes. Transcription factors are trans-acting because they are recruited to numerous promoter regions in eukaryotic gene transcription to begin the process. These transcription factors must be trans-acting so they can be recruited from anywhere near by in the nucleolus. Trans-factors allow a more efficient process of transcription initiation, since they can act on more than just the gene they are right next to. However, these transcription factors are regulated because it would be very bad if they could float to any gene at any time and turn it on. The way that transcription factors are regulated is by their recognition of specific DNA binding motifs. These motifs signal to the transcription factors that this is where they need to bind in the genome.
ID081	DNA sequencing has allowed for many advances in genetics to occur. With multiple sequences of cloned DNA, it much easier to study the mutations of genes and functions of genes.
ID081	Restriction enzymes cut DNA at specific sites within DNA. They also allow inserts of DNA to occur within plasmids that are used as vectors.
ID081	Transcription factors can act on any DNA or chromosome within the cell. It does not have to stay on only one chromosome where it is currently at.
ID016	DNA sequencing allows geneticists to determine the sequence of bases in DNA. This allows scientists to determine when certain genes are mutated as well as the relatedness of certain species. Also, knowing the code of a gene can allow for the manufacturing of that gene product.
ID016	Restriction enzymes are used to ensure that bacterial cells are free from the DNA of bacteriophages. These restriction enzymes degrade DNA at certain sequences of nucleotides, rendering the single DNA strand into many pieces. This is useful for scientists, however, since restriction enzymes can be used to modify DNA at known sequences.
ID016	Transcription factors allow initiation of eukaryotic transcription. These are specific in the sense that they only bind to the certain promoters of genes, allowing transcription initiation to start at the desired time. These factors may enhance or repress the levels of transcription in response to the need or excess of the desired gene product.
ID035	DNA seq and the number of developments with the networks help keep us healthy or making us sick, but in many respects it has played the leading role in this process.
ID035	These enzyme cut DNA at or near specific recognition nucleotide sequences known as restriction sites.
ID035	Transcription initiation complex is composed of promoter sequences and DNA binding proteins. These two components of transcription are normally described as cis-acting elements and trans-acting factors.
ID109	DNA sequencing has revolutionized genetics because it provides a link between genotype and phenotype.  We can now identify low frequency mutants and variations, which allows us to look at a certain lotus or tiny region in the gene. Because of DNA sequencing we can easily compare the genomes of organisms and species, which allowed us to see just how closely organisms were related to each other.
ID109	Restriction enzymes are bacterial defense mechanisms because the enzyme cuts the DNA at a certain location, the restriction site. This is a defense mechanism because the enzyme cuts up foreign DNA. The restriction enzyme most likely evolved from a common ancestor and grew through horizontal gene transfer. Restriction enzymes also produce a double strand of DNA. There are four groups of restriction enzymes.
ID109	Transcription factors are trans-acting proteins with specific DNA binding motifs because the increase the level of gene expression. The flexible protein domains in activators connect DNA-binding domains to the activation domains. This may explain why alterations in spacing between control elements is tolerated in the  eukaryotic control regions. When the DNA-binding domains of nearby transcription factors are moved in their relative positions on the DNA, their activation domains can still interact because they are attached to the DNA-binding domains through flexible protein regions.
ID129	DNA sequencing has allowed genetics to use technology such as gel electrophoresis, which uses an electric current to pull different DNA fragments to the end of the gel and determine how many base pairs each DNA fragment is. The polymerase chain reaction (PCR) is used to amplify a single copy of DNA to generate millions of copies of DNA.
ID129	Restriction enzymes in bacteria serve as a defense mechanism against viruses. They cut up foreign DNA while protecting host DNA with a modification enzyme.
ID129	trans-acting factors have two domains. one domain is required for the factor to DNA, the other one is required for the activation of transcription.
ID101	The sequencing of DNA has revealed information stored by the cell, and from it.
ID101	The restriction enzymes are used to cut DNA in defense of viruses. These are only done at specific sequences.
ID101	The trans acting proteins function across two genes. They are necessary for initiating transcription.
ID070	DNA sequencing has revolutionized biological and medical research, and is poised to have a similar impact in medicine. This tool is just one of a number of developments in our capability to identify, quantitate and functionally characterize the components of the biological networks keeping us healthy or making us sick, but in many respects it has played the leading role in this process.
ID070	A restriction enzyme or restriction endonuclease is an enzyme that cuts DNA at or near specific recognition nucleotide sequences known as restriction sites. Restriction enzymes are commonly classified into three types, which differ in their structure and whether they cut their DNA substrate at their recognition site, or if the recognition and cleavage sites are separate from one another. To cut DNA, all restriction enzymes make two incisions, once through each sugar-phosphate backbone of the DNA double helix.
ID070	a transcription factor is a protein that binds to specific DNA sequences, thereby controlling the rate of transcription of genetic information from DNA to messenger RNA.
ID020	DNA sequencing is the best way to characterize DNA structure at the molecular level. The ability to sequence DNA has greatly enhanced our understanding of genome organization and increased our knowledge of gene structure, function, and mechanisms of regulation. The most commonly used method of DNA sequencing is the Sanger sequencing.
ID020	Restriction enzymes are produced by bacteria as a defense mechanism against phages. They restrict or prevent viral infection by degrading the DNA of invading viruses. Restriction enzymes bind to DNA at specific recognition sequence (restriction site) and cleaves DNA to produce restriction fragments.These enzymes cleaves both strands of DNA.
ID020	Some transcription factors increase levels of transcription initiation and some reduce levels. Some transcription factors may be present in only certain types of cells thereby regulating their target genes for tissue-specific levels of expression. Each transcription factor has a characteristic motif. These motifs are short structures which comprise only a small portion of the protein. These motifs are used to define a specific class of trans-acting factors
ID119	The ability to determine the order of nucleotides within DNA molecules was one of the biggest accomplishments in the history of genetics and has revolutionized the study as a whole.
ID119	Restriction enzymes has several functions, but one of the roles of restriction enzymes is to fight off bacteriophages.
ID119	Trans-acting factors are proteins that bind to the cis-acting sequences in order to regulate gene expression. The also have specific binding mechanisms.
ID044	DNA sequencing has made genetics more understandable.  Techniques like PCR have opened doors to understanding and knowledge.  It has made genetic sequencing quicker and easier. Something that took weeks and hard work now takes a few days and is done by a computer.
ID044	Restriction enzymes are endonuclease are derived from bacteria, which protect host cells from viral attack by degrading invading, foreign DNA. The host DNA is usually protected by methylation.
ID044	Transcription factos are trans-acting and act on specific cis-acting factors like TATA box, CAAT, and GC regions.  These bind to specific binding motifs and act to regulate transcription by regulating gene expression, either negatively or positively.
ID138	DNA sequencing has made it much easier and faster for geneticists to identify specific gene markers. This is crucial for the study of genetic diseases and the affects that different gene mutations can have on a person.
ID138	Restriction enzymes are enzymes produced by bacteria that break apart DNA. They are used as a defense mechanism because they are able to break down the viral DNA that is transmitted into the bacteria from viruses.
ID138	Transcription factors that are trans-acting are able to bind to other strands of DNA and express themselves on the other strands to make them express (or repress depending on the factor). This is important because it advances our understanding of gene expression.
ID027	Being able to read an organism's genome has allowed geneticists to look at things in a new light. They don't have to guess how the bases line up and can get a more accurate view of what's going on. This also allows them to make predictions based on genetic markers.
ID027	Restriction enzymes allow the bacteria to survive by cutting unmethylated DNA. This has helped the organism survive things like bacteriaphages. This enzyme has been adapted for use in human experiments and things like insulin manufacturing.
ID027	Transcription factors don't have to work on the chromosome they came from. They can float freely in the nucleus and bind to the specific segments they need to attach to. They have a range of functions from initiating to enhancing RNA production.
ID046	DNA sequencing is implemented by several methods. PCR, libraries, cDNA, probes, and blots.
ID046	Restriction enzymes cut foreign DNA in prokaryotes and are used to separate and cut certain sections of DNA.
ID046	Transcription factors are proteins that bind to the cis acting promoters.
ID087	Prior to DNA sequencing the time it took to figure out the sequence of a single gene or the sequence of an organisms whole genome would take forever. Now with PCR and BLAST sequencing the process takes much less time. This allows the comparison of organisms, connections via evolution and markers for diseases.
ID087	Restriction enzymes are found in bacteria and they defend against invading viral DNA by "cutting up" the DNA at specific sites.
ID087	These proteins bind to specific sequences of DNA regulating the transcription of genes. They can be turned on or off by regulation factors that bind to the transcription factors.
ID066	DNA sequencing has allowed scientists to understand genetic principles better than they ever had before. With recent DNA sequencing technology it has become possible to document specific genomes and find out more about DNA organization and function.
ID066	A restriction enzyme or restriction endonuclease is an enzyme that cuts DNA at or near specific recognition nucleotide sequences known as restriction sites. Restriction enzymes are commonly classified into three types, which differ in their structure and whether they cut their DNA substrate at their recognition site, or if the recognition and cleavage sites are separate from one another. To cut DNA, all restriction enzymes make two incisions, once through each sugar-phosphate backbone (i.e. each strand) of the DNA double helix.
ID066	Transcription factors are proteins involved in the process of converting, or transcribing, DNA into RNA. Transcription factors include a wide number of proteins, excluding RNA polymerase, that initiate and regulate the transcription of genes. One distinct feature of transcription factors is that they have DNA-binding domains that give them the ability to bind to specific sequences of DNA called enhancer or promoter sequences. Some transcription factors bind to a DNA promoter sequence near the transcription start site and help form the transcription initiation complex. Other transcription factors bind to regulatory sequences, such as enhancer sequences, and can either stimulate or repress transcription of the related gene. These regulatory sequences can be thousands of base pairs upstream or downstream from the gene being transcribed. Regulation of transcription is the most common form of gene control. The action of transcription factors allows for unique expression of each gene in different cell types and during development.
ID069	DNA sequencing has allowed for a more conclusive, universal understanding of genetics. It enables such analyses as forensic analysis and can shed light on the genetic relationships between species. The latter analysis may also reveal evolutionary patterns of species. Perhaps most importantly, DNA sequencing has created a venue for more intensive therapy of genetic disorders. As genetics continues to be better understood, the impact of its knowledge will continue to ameliorate the lives of all.
ID069	Restriction enzymes extract DNA from one organism and enzymatically fragment it. They protect host cells from viral attack by degrading invading DNA, while the host DNA is usually protected by methylation of its own DNA. Type I recognizes a specific site and cuts the DNA at random at a distance away from the site, and type II produces a double-stranded break precisely within the sequence. Type II recognizes palindromic sequences and may make a blunt end or staggered cut.
ID069	Transcription factors are transcription regulatory proteins that target cis-acting sites of genes regulating expression. They contain one domain for binding to DNA and one for activating transcription. Multiple transcription factors may bind to several different enhancer and promoter elements to fine-tune the level of transcription initiation.
ID060	The ability to sequence DNA has allowed experts to acquire information about our genome and how to treat it.
ID060	Restriction enzymes cut foreign body genome as a defense mechanism.
ID060	Transcription factors bind to specific DNA thereby controlling the rate of transcription
ID058	A very useful example of DNA sequencing is Sanger sequencing. It uses ddNTPS which lack an O on their 3' carbon. This results in nucleotides being unable to be added to it and so transcription halts. This results in multiple fragments with colored tags that can then be read by a computer. This process allows whole sequences to be quickly determined.
ID058	Restriction enzymes are a bacteria's defense mechanism against invasion by bacteriophages. It slices up the viral DNA so it cannot replicate itself and causes the bacteria to lyse.
ID058	Transcription factors can have 3 different binding motifs. One is helix-turn-helix which is very common and can be seen in the repressors of the lac and trp operons. Zinc fingers and leucine zippers are other binding motifs.
ID034	innovation approaches to genome sequencing that are providing data for range of new research and clinical application
ID034	restriction enzyme help bacteria break down DNA from outside, prevent transformation,
ID034	transcriptions factors are regulatory agents which are not part of the regulated genes
ID098	The ability to sequence DNA through methods like Sanger sequencing has had a profound effect on the field of genetics because now researchers can catalog the exact nucleotide sequence of a gene or regulatory region, making it easier to identify mutations. This, in turn, has given us a far better understanding of genetic disorders and just a better understanding of various genomes.
ID098	Restriction enzymes likely evolved as a defense mechanism for bacteria, so they can protect themselves from invading viruses. The primary function of restriction enzymes is to sever DNA at specific sequences, rendering that strand of DNA completely useless by breaking it into many fragments. When viral DNA inserts itself into a bacterium, the bacterium responds this way, which keeps its own DNA and cell body safe from further invasion and lysis.
ID098	Transcription factors are molecules that are necessary for transcription initiation. They are trans-acting and bind to transcription elements, which are specific sequences of DNA in the promoter region of a gene. Transcription factors essentially provide a landing platform for RNA polymerase II, and transcription could not occur without them.
ID093	Through DNA sequencing, such as the Sanger chain method, it is possible to identify unknown organisms quickly. Online databases store thousands of known sequences allowing for the classification of many organisms.
ID093	Restriction endonucleases and exonucleases cleave DNA at specific sites. These enzymes are utilized in bacterial defense mechanisms against invading DNA (viruses). The CRISPR-Cas system is a "bacterial immune system" of sorts. Short repeating sequences specific to a virus are stored within the bacterial genome. The cell can identify and inhibit viral infection.
ID093	Trans-acting transcription factors directly binding to DNA and altering transcription. Some transcription factors function by altering the histone code. Histone deacetylation turns off genes while hyperacetylated genes are turned on.
ID116	DNA sequencing determines the number of nucleotides within a strand of DNA and then determines the order in which cystine, adenine, guanine, and thymine are present on the strand. This has been important for genetics discovery, as well as developments in medicine and forensic biology.
ID116	Restriction enzymes cut DNA sequences at restriction sites, which are specific recognition nucleotide sequences. Two incisions are made, one at each strand sugar-phosphate backbone. In bacteria these act as a defense mechanism against viruses by cutting up foreign DNA with restriction enzymes.
ID116	Transcription factors are responsible for acting on target sequences of DNA and for interacting with other transcription factors. These are trans-acting factors that control gene expression. Their characteristic motif is a short structure that only comprises a small amount of the protein. Some examples are steroid receptors and zinc finger proteins.
ID024	DNA sequencing, which determines how the nucleotides are paired in DNA, allows scientists to map DNA. With this advancement in science, we can now see where genetic diseases arise and we can now look into how to prevent and possibly correct genetic errors.
ID024	Restriction enzymes are enzymes that can cut DNA into certain pieces. These pieces can then be incorporated into the DNA of another bacterium.
ID024	Transcription factors aid in the process of transcribing DNA into RNA. They include proteins, RNA polymerase, and all the other things that have a part in transcription. They are trans acting because they can act on different chromosomes. They are not limited to one chromosome, like other cis-acting proteins.
ID033	Because scientists have sequenced the DNA, geneticists don't have to do as much guess work and probability for crossovers and mutations. It's more direct.
ID033	Restriction enzymes can cut DNA into smaller fragments so if a virus inserts its DNA then restriction enzymes can protect the bacteria by cutting the viral DNA.
ID033	Transcription factors are trans acting proteins that initiate transcription of DNA into RNA.
ID000	DNA sequencing has provided a molecular basis for classical patterns of inheritance, provided information about how genes are regulated in prokaryotes and eukaryotes, elucidated evolutionary patterns and provided information about genomes and their organization. The two methods discussed in class were Sanger sequencing and the use of pyrosequencing to sequence hundreds of thousands of genes at a time.
ID000	Restriction enzymes were initially identified from bacteria and are a defense mechanism to cope with invading viral DNA.  Type II restriction enzymes digest DNAs at palindromic sites leaving either blunt or sticky (cohesive) ends. They were adapted for use in creating recombinant DNA molecules, often plasmid or viral DNA with an insert from a different organism.
ID000	Numerous transcription factors must bind to promoter and enhancer regions upstream from a structural gene.  Some of these will increase transcription (activators) and some will decrease transcription (repressors).  The common DNA-binding motifs are Zinc fingers, helix-turn-helix and leucine zippers.
ID056	DNA sequencing allows one to examine the genetic structure at the molecular level. This process allows one to see what nucleotide bases are specifically contributing towards the genes of the individual being studied. Sequencing techniques covered in class include the Sanger Protocol and pyrosequencing.
ID056	The purpose of having restriction enzymes is for the cutting of DNA in specific areas. The defense mechanism includes protection from the effects of bacteriophages. They also recognize and bind to restriction sites to produce restriction fragments.
ID056	This statement is misleading because we studied how transcription factors act upon cis-acting regulatory genes. These transcription regulatory proteins are found in several motifs such as Helix-turn-Helix (HTH), Zinc-fingers, and basic leucine zippers (bZIP). These differing structures allow for specific functions of the transcription factors to take place such as binding to different enhancers and promoters.
ID104	With the help of knowing the DNA sequence behind many genes we have be able to completely change the world of genetics and how we see our genetic material. DNA sequencing has allowed us to understand why certain genes act the way the do and what they code for.
ID104	Restriction enzymes allow bacteria to cleave out unwanted sections of DNA that could be from bacteriophages in order to protect the cell from unwanted mutations. It can also be used to insert desired strands of DNA into a host for a desired phenotypic expression
ID104	Transcription factors are used to promote or silence the transcription of certain sections of DNA. They are usually trans-acting and only bind to a certain specific section.
ID089	DNA sequencing has allowed researchers to know the sequences of bases in a DNA molecule and thus the genes, location of genes, and has allowed for the development of increasingly important and revolutionary genomic research methods. This includes transgenic organisms, the ability to knock out certain traits, and to study the affect of each gene on the organism.
ID089	Restriction enzymes work to defend bacteria from the DNA of viruses. The restriction enzymes cleave DNA at certain sequences in order to make that gene product nonfunctional. Restriction enzymes are also used to cut DNA in a cell and then insert a gene using the sticky ends on the DNA and on the target gene.
ID089	Transcription factors bind to the operator region of the DNA and serve to activate or repress transcription of the genes. Coactivators or corepressors can bind to the transcription factor to cause a change in structure and therefor function. This can induce or repress the activator or repressor, respectively.
ID125	DNA sequencing has revolutionized biological and medical research, and is poised to have a similar impact in medicine. This tool is just one of a number of developments in our capability to identify, quantitate and functionally characterize the components of the biological networks keeping us healthy or making us sick,
ID125	A restriction enzyme or restriction endonuclease is an enzyme that cuts DNA at or near specific recognition nucleotide sequences known as restriction sites.Restriction enzymes are commonly classified into three types, which differ in their structure and whether they cut their DNA substrate at their recognition site, or if the recognition and cleavage sites are separate from one another.
ID125	transcription-control elements found in eukaryotic DNA are binding sites for trans-acting regulatory proteins.
ID118	DNA sequencing is used to find the exact location of specific gene on the chromosome and alters based on the technology. one of the sequencing that is used very common is the Sangar's protocol which is unidirectional PCR reaction that is randomly terminated when a dideoxynuclotide is incorporated instead of normal nucleotide. Dideoxy nucleotides lack an oxygen on the 3' carbon and cannot add another nucleotide at that position and tied with fluorescent die. The PCR reaction creates millions of partial copies of a DNA sequence, each one terminated with ddNTPS replects that DNA sequence at that point. another way to sequence the DNA is pyrosequencing it is done by synthesis.
ID118	Restriction enzyme is required to clone the DNA and for recombinant DNA. it is used in the first step in recombinant DNA is to cut DNA with restriction enzyme, the same enzyme on the vector and also at the donor DNA. Restriction enzyme is the defense system used by bacteria against bacteriophages. It is used to cut the DNA on the both strand leaving sticky or blunt ends. which under certain condition be hybridized making recombinant DNA in the vector. The vector can be plasmid or bacteriophage which is then inserted into bacteria to make library.
ID118	Transcription factors are also transcription regulatory proteins are activators which increase transcription initiation, repressors that decrease transcription initiation. multiple transcription factors binds to several different enhancers and promoter elements and fine tune the level of transcription initiation. general transcription factors are proteins that are required at promoter to initiate basal or enhanced levels of transcription, assembly of proteins in specific order forms pre-initiation complex, PIC provides platform for RNAP II to recognize transcription start sites. There are also coactivators and enhanceosome that repressor proteins at silencer elements decrease rate of PIC assembly and RNA pol II release.
ID072	DNA sequencing was developed by Fred Sanger and his colleagues. A double stranded DNA molecule whose sequence is to be determined is converted to single strands that are used as a template for synthesizing a series of complementary strands.
ID072	Restriction enzymes are found in bacteria and archaea and provide a defense mechanism against invading viruses.
ID072	Transcription factors, usually considered to be proteins, that bind to the cis-acting sequences to control gene expression
ID103	DNA sequencing allows for the locating of genes on a chromosome. It is the first step to opening up all the mysteries and the promise genetics has to offer.
ID103	Restriction enzymes are used by bacteria to cleave viral DNA that threatens the cell. They are useful in research because they cleave DNA at specific locations.
ID103	Transcriptions factors are proteins that regulate the activity of rna polymerase by binding to DNA. The can bind to the chromosome where the gene was that created the protein or to a different chromosome.
ID008	It has the capability to identify, quantitate and functionally characterize the components of the biological networks.
ID008	They can be found in bacteria and archaea and provide a defense mechanism against viruses.
ID008	They bind to the cis-acting sequences to control gene expression. They also bind to RNA Polymerase to stabilize the initiation complex
ID085	The ultimate way to characterize DNA is by looking at it on a molecular level. Because of DNA sequencing, developed by Fred Sanger, we can now do this, and the cost of doing so is becoming more and more affordable every year.
ID085	Restriction enzymes were created in bacteria as a defense against bacteriophage. In order to prevent viral infection, they degraded the viral DNA.
ID085	Transcription factors are trans-acting, meaning that they attach to DNA from above. This also means that they can bind to multiple chromosomes, not just the one they first attach to. Because of their specific motifs, a specific TF will promote a specific region.
ID048	DNA sequencing has changed the way genetics is studied and many discoveries have been made since DNA sequencing.
ID048	They are produced by bacteria in order to provide defense against bacteriophage. They are DNA-cutting enzymes which bind to DNA at specific recognition sequence (restriction site) and cleaves DNA to produce restriction fragments
ID048	factors, usually considered to be proteins, that bind to the cis-acting sequences to control gene expression.These motifs are short structures which comprise only a small portion of the protein. These motifs are used to define a specific class of trans-acting factors. The following are specific examples of trans-acting factors, each with a defined and specific motif.
ID010	DNA sequencing is used today to determine the order of nucleotides along a DNA molecule. Different technological methods are used to determine the arrangement of the 4 bases: adenine, guanine, cytosine, and thymine.
ID010	Restriction enzymes allowed for a self-defense mechanism against invading bacteriophages. They are also known as restriction endonucleases. The restriction enzyme acts on a DNA sequence in the form of a recognition sequence, protecting it from bacteriophages and also protecting the DNA sequence itself.
ID010	Transcription factors are involved in converting DNA to RNA. It is one of the main points of the Central Dogma. They include a large number of proteins that help with the process, not including RNA polymerase, which initiates and regulates gene transcription.
ID055	With DNA sequencing, we can identify a specific gene on a chromosome. It has allowed scientist to characterize DNA structure at the molecular level.
ID055	Restriction enzymes act against bacteriophages. Also, they are a DNA-cutting enzyme. Restriction enzyme binds to a specific region and cleaves DNA to give restriction fragments.
ID055	Trans-acting proteins known as transcription factors help to regulate a particular gene.
ID036	DNA sequencing is a efficient and cheap way to amplify many genes. You can do this by PCR with Sanger protocol automated sequencing, and the Pyrosequencing.sequencing by synthesis  ssDNA is incubated with a base plus DNA polymerase.  If the base is incorporated, pyrophosphate is released which triggers a series of reactions creating a flash of light.  The light flash is recorded
ID036	Restriction enzymes are used as a defense mechanisms by bacteria from bacterophages.They restrict viral infection by degrading viral DNA
ID036	A repressor is a trans acting protein. That has Helix- turn helix that interacts with specific Nucleotide seqences.
ID110	has given us the ability to locate markers and tags necessary for further understanding of genetic transmission from parent to offspring
ID120	DNA sequencing allows gene patterns to be discovered in which similarities can be used to better understand the genome. Shows what mechanism are unique to which genes.
ID120	Restriction enzymes cleave DNA at specific regions. If a bacteria needs a particular gene it can be transformed and inserted due to the restriction enzymes.
ID120	Transcription factors can work on all chromosomes, but the right conditions must be available for the transcription factors to bind.
ID139	Once we discovered the basic elements of DNA sequencing it allowed us to learn and study so much more about how the DNA is coded and functions.
ID139	These enzymes filter what can and cannot pass; preventing harmful bacteria from entering.
ID050	DNA sequencing has changed genetics because it has allowed scientists a look into how certain areas of DNA control certain portions and how one change can lead to certain diseases.
ID050	Restriction enzymes are the defense mechanism because they block viruses from reaching the actual DNA and cuts it into useless pieces.
ID050	Transcription is trans acting in certain DNA binding motifs because then it can change different pathways and unblock the ability to not reach certain pathways.
ID099	with DNA sequencing it is easier to compare mechanisms in different organisms
ID099	The ability  to cut out sections of DNA can be doubled as a Genetic Engineering tool
ID099	These factors are somewhat omni-present and essential in Transcription
ID013	DNA sequencing techniques involve Sanger sequencing and pyrosequencing. These sequencing abilities have allowed scientists to identify museum specimens, analyze human mitochondrial DNA, identify victims in forensics, and identify symbiosis relationships like the mycorrhizal associations. Sanger sequencing uses PCR but it includes dideoxy nucleotides that lack an oxygen on the 3 prime carbon thus can't add a nucleotide. These dideoxy nucleotides are tagged with fluorescent dyes and once they are terminated, they can be used to identify the DNA sequence up to that point. Pyrosequencing is sequencing by synthesis, which is carried out in wells and a light flashes when a base is incorporated.
ID013	Restriction enzymes are enzymes that bacteria produce to cut up viral DNA as defenses against viral infections. These enzymes recognize specific recognition sequences on the DNA. They recognize palindromic sequences, which are read the same on both strands. The enzymes can cut these sequences in a staggered fashion, leaving cohesive or sticky ends, or they can cut in a straight fashion leaving blunt, double-stranded ends.
ID013	Transcription factors bind to DNA to turn on transcription (activators) or turn off transcription (repressors). They can be three different binding motifs: helix-turn-helix, zing finger, or leucine zipper. The HTH are shown in the lac operon repressor while the zinc finger depends on zinc residues with cytosine cross-links and the leucine zipper depends on a pair of scissors that fits over DNA with leucine.
ID136	With DNA sequencing we now can detect and resolve any problems arising from DNA in the genetics world.
ID136	Restriction enzymes can block bacteria that can harm us by not letting them be read through.
ID136	These proteins bind specific DNA together to do certain jobs.
ID108	DNA sequencing is the revolutionary way to characterize DNA because it looks at it a molecular level. An example is pyrosequencing which uses chemical light producing enzymatic reactions to trigger a molecular recognition event. Whenever A,C,G or T is added into the growing chain a reaction occurs and a light signal is produced.
ID108	Restriction enzymes are produced by bacteria as defense mechanisms against bacteriophages. They are DNA cutting enzymes, restriction endonucleases and bind to DNA at a specific recognition sequence called the restriction site. It then cleaves DNA to produce restriction fragments and the enzyme cleaves both strands of DNA, digesting it. By recognizing particular sequences, especially palindromic, restriction enzymes can protect against viruses.
ID108	Trans-factors are usually proteins that bind to cis-acting sequences to control gene expression. Each transcription factor has a particular motif which are short structures that are a very small part of the protein. For example, in zinc finger proteins Zinc binds at a specific site and in leucine zipper proteins, Leucine is at every 7 amino acids in a protein region.
ID144	DNA sequencing is basically a way of identifying genes based on their DNA sequences.
ID144	A restriction enzyme is an enzyme that cuts DNA at or near specific recognition nucleotide sequences known as restriction sites.To cut DNA, all restriction enzymes make two incisions, once through each sugar-phosphate backbone of the DNA double helix. These enzymes are found in bacteria and provide a defense mechanism against invading viruses.
ID144	A trans-acting element is usually a DNA sequence that contains a gene. This gene codes for a protein that will be used in the regulation of another target gene. The trans-acting gene may be on the same chromosome as the target gene, but the activity is via the intermediary protein or RNA that it encodes.
ID021	DNA sequencing has allowed researchers to reach a golden era in this field. We can now easily and quickly figure out the sequence of the DNA of organisms with just a tiny bit of DNA. DNA sequencing is completed most commonly using the Sanger automated method.
ID021	Restriction enzymes are used in bacteria to cut apart invading DNA that is a potential threat to the livelihood of the bacterial cell. For example, restriction enzymes can identify viral genetic material in the bacterial cell and cut it up so that the virus cannot hurt the cell and cannot replicate.
ID021	Trans acting proteins like transcription factors act on any gene in the genome. Transcription factor proteins recognize specific DNA sequences that signal for transcription, like a promoter region TATA box or CAAT box.
ID014	There are many methods to DNA sequencing. From the Sanger sequencing to the pyrosequencing, technology is advancing towards increasing sequencing of genes. We have been able to synthesize the entire human genome which has helped us to find a lot of diseases and chromosomal disorders.
ID014	Restriction enzymes are able to cut DNA and RNA fragments at certain points on the DNA. Restriction enzymes are also used to cut DNA to insert into plasmids. The restriction enzymes can either make sticky ends or blunt ends.
ID014	Transcription factors are able to bind to the promoter region to help increase the amount of transcription in eukaryotes. There are many factors that constitute the TBP, which allow DNA to fold on itself.
ID003	DNA sequencing shows how certain mutations
ID003	Restriction enzymes are a defense mechanism because
ID003	Transcription factors can go on any
ID113	DNA sequencing is the process of determining the precise order of nucleotides within a DNA molecule
ID113	it's an enzyme that cuts DNA at or near specific recognition nucleotide sequences known as restriction sites and is used against invading viruses
ID113	a trans-acting gene may be on the same chromosome as the target gene, but the activity is via the intermediary protein or RNA that it encodes
ID091	DNA sequencing has allowed the specific sequence of genes and entire genomes. This allows us to analyze a person's genome for possible traits.  It also lets us see how certain species differentiated along with numerous other applications.
ID091	Restriction enzymes cut DNA at specific sites (sometimes different than recognition or using a guide RNA). The cut in DNA makes the DNA easy to decompose and unable to produce gene product. Viral DNA is cut and unable to take over the host cell. So, restriction enzymes are taken from bacterial cells and applied to genetics.
ID091	Transcription factors act in a variety of ways, however they all act on DNA. So, transcription factors usually fall into a motif such as helix turn helix, zinc finger, leucine zipper, or some other conformation. This structure allows the factors to bind to DNA and RNA polymerase.
ID126	DNA sequencing is the process of determining the precise order of nucleotides within a DNA molecule.
ID126	DNA sequencing is the determination of  the precise order of nucleotides within a DNA molecule and are utilize in the pharmaceutical drugs.
ID126	Transcription factors are proteins involved in the process of converting, or transcribing, DNA into RNA. Trans-actin may be on the same chromosome and still affect.
ID040	DNA sequencing is a field that has expanded rapidly in the past few decades. Sequencing of entire genomes
ID068	DNA sequencing allows geneticists to see the exact sequences of DNA strands. By being able to read sequences we can discover the sequences associated with genes and what sequence changes can cause mutations/diseases. There are many methods such as pyrosequencing which releases a flash of light when a nucleotide is matched to the chain.
ID068	Restriction enzymes cut bacteriophages in bacteria. These enzymes can be used to cut DNA into restriction fragments. They cut certain palindromic sequences with either sticky or blunt ends. The fragments can then be combined with other DNA for cloning.
ID068	Transcription factors are trans acting which mean they can act on chromosomes other than the one they are on. There are activators that increase transcription and repressors that decrease transcription. TF proteins can bind to enhancers or promoters to influence the level of transcription.
ID107	DNA sequencing is the process of determining the exact order of nucleotides within a DNA molecule. There are many different techniques you can use to sequence genes. DNA sequencing can be used to identify mutations within a genome and even help to find markers of genetic disease.
ID107	Restriction enzymes are DNA-cutting enzymes that bind to DNA at specific recognition sequence
ID107	Transcription factors target the cis-acting sites of genes that regulate gene expression. Transcription factors can activate or repress transcription initiation.
ID142	Sequencing DNA has opened many doors within the field of genetics. Understanding the sequence of even just a gene allows for a closer look at how mutations in those genes are affecting genetic diseases. Many people thought that mapping the Human genome would be the key to solving many diseases and it has helped, but also has led to the field of epigenetics and how certain DNA sequences are expressed based of off cell environment and different body signals.
ID142	Restriction enzymes cleave DNA at restriction sites which are usually palindromic sequences. These enzymes are particularly important to bacteria because it also specific, harmful, inserted DNA to be removed. However, restriction enzymes have also become extremely important in genetic research specific to recombinant DNA.
ID142	transcription factors are trans-acting which means they are on the DNA strand opposite the strand with the gene on it. These TF are recruited to specific sites coded by genes. They bind to these sites in the DNA and then the TF can interact with (recruit) the RNA polymerase at the promoter site.
ID088	By studying the sequence of DNA, researchers are able to determine what codes for each gene. This will enable us to pinpoint genetic defects.
ID088	Restriction enzymes cut DNA at specific points. These enzymes are found in bacteria and archaea and provide a defense mechanism against invading viruses.
ID088	These are factors, usually considered to be proteins, that bind to the cis-acting sequences to control gene expression. They are essential to transcription initiation.
ID080	It has changed the understanding of how transcription is used to translate. It has revolutionized how genetics are looked at and studied. It has allowed us to find errors in the sequence and try to fix them.
ID080	They are used to fight against certain viruses that would otherwise kill the bacteria. This is done by making it impossible it interfere with the DNA structure.
ID080	Transcription factors are vital to the initiation of transcription for eukaryotic organisms. There are three different major motifs.
ID063	Knowing the background of the DNA including its structure and chemical process has allowed for the sequencing. Through sequencing, the basis of disease can be determined as well as mutation identification.
ID063	A bacteria will activate its defense system when a foreign invader is recognized. The restriction enzymes will be released to target and attack the invaders genome turing it into a nonfunctional genetic code.
ID063	Transcription factors act on both strands of the DNA. They have characteristics that allow it to act upon specific nucleic acids on both strands.
ID078	knowing the order of DNA makes it easier to manipulate it
ID078	These cut the DNA of foreign genes to protect the bacteria they are in
ID078	They're trans acting because they aren't actually part of the same chromosome
ID023	I understand that being able to sequence DNA has provided scientists the ability to know intricate details about specific organisms and how we are related to them as well as to produce medicine or treatment in order to treat certain disorders.
ID023	Bacterial enzymes are used to cut invading bacteriophage DNA in order to allow the bacteria to survive the invasion. I also understand that these enzymes are involved with recombinat DNA research.
ID023	Transcription factors are involved in gene regulation and are one of the reasons that our cells in our stomach are different from the cells within out heart.
ID123	Genetics has been revolutionized by genetics because of the many different process devised to sequence the genomes.  These process have brought to light many different characteristic s of the genome which were not known before.  They have shed light on different mutations and also evolutionary mechanisms.
ID123	Restriction enzymes can act as defense mechanisms in bacteria by fragmenting the invading viral DNA.  Once the viral DNA has been broken down, it can no longer be incorporated into the bacterial DNA.  This allows the bacteria to remain intact.
ID123	Transcription factors allow the binding of different molecules to begin transcription.  They include three different binding motifs.  The helix-turn-helix, zinc finger, and leucine zipper are the binding motifs which allow for DNA binding.
ID149	DNA sequencing (specifically the Human Genome Project) has allowed the field of genetics to advance incredibly. Without it, we wouldn't know the base pairs that make a specific gene.
ID149	Restriction enzymes cut DNA near (or at) a specific nucleotide sequence. Research has shown they provide a defense mechanism against viruses.
ID149	I'm not entirely sure what a trans-acting protein really is.
ID053	DNA sequencing has allowed for a more conclusive, universal understanding of genetics. It enables such analyses as forensic analysis and can shed light on the genetic relationships between species. The latter analysis may also reveal evolutionary patterns of species. Perhaps most importantly, DNA sequencing has created a venue for more intensive therapy of genetic disorders. As genetics continues to be better understood, the impact of its knowledge will continue to ameliorate the lives of all.
ID053	Restriction enzymes extract DNA from one organism and enzymatically fragment it. They protect host cells from viral attack by degrading invading DNA, while the host DNA is usually protected by methylation of its own DNA. Type I recognizes a specific site and cuts the DNA at random at a distance away from the site, and type II produces a double-stranded break precisely within the sequence. Type II recognizes palindromic sequences and may make a blunt end or staggered cut.
ID053	Transcription factors are transcription regulatory proteins that target cis-acting sites of genes regulating expression. They contain one domain for binding to DNA and one for activating transcription. Multiple transcription factors may bind to several different enhancer and promoter elements to fine-tune the level of transcription initiation.
ID059	DNA sequencing has revolutionized genetics because it gives us the ability to get a better view of whats really causing health problems in our genotype or phenotype as well as ways to prevent these problems. It has become a major deal in the idea of creating medications as well.
ID059	Restriction enzymes are defense mechanisms for bacteria because theycut at specific regions og the DNA sequence to prevent foreign virus from invading the prokaryotic cell. This important mechanisms help the bacteria cell stay alive and function properly without the interruption of harmful outside organisms.
ID059	Transcription factors are trans -acting proteins that allows RNA polymerase to transcribe genes. It has several roles depending on the process. For example transcription factors can act to regulate the binding of transcriptional apparatus to the core promoter or it can act as a controlling method to switch from initiation to elongation.
ID140	DNA sequencing has allowed us to better understand genetics and be able to use it to learn more.
ID140	Restriction enzymes allow for bacteria to regulate production of necessary gene products.
ID140	Ttanscription factors can act on other chromosomes to help with DNA binding.
ID117	The ability to sequence DNA has revolutionized genetics in a variety of different ways. DNA sequencing is the way in which geneticists characterize DNA at the structural level with nucleotides. The use of sequencing has allowed for advancements in our understanding of DNA structure, pairing, and genes on a strand of DNA.
ID117	Restriction enzymes are produced in bacteria as a defense mechanism against infection by bacteriophage DNA. These enzymes cut strands of DNA, making either blunt or sticky ends. The restriction enzymes degrade the infecting DNA.
ID117	Certain transcription factors have specific motifs when binding to different enhancers or promoters on a strand of DNA. The helix-turn-helix, zing finger, or leucine zipper are three examples. Transcription factors are trans-acting, meaning that they have influence on chromosomes adjacent to them.
ID141	Genetics would be nowhere near the advancements it has discovered had it not been for the methods of DNA sequencing.  From medicine to farming, DNA sequencing is currently and has in the past made better crops, and helped diagnose patients after sequencing their DNA.
ID141	Restriction enzymes defend the bacteria against viruses by degrading the DNA of invading viruses.
ID141	The 3 common transcription factors are helix turn helix proteins, leucine zippers, and zinc fingers.  General transcription factors are required at promoter for enhanced transcription to occur.  They make up the pre initiation complex in which RNA polymerase II can land on.  I like to think of the pic as a landing pad.
ID145	By sequencing genes you can find what genes cause certain diseases.
ID145	These can be used in cloning.
ID145	Because they are trans they do not only work on the gene they are on the same chromosome with.
ID006	DNA sequencing has revolutionized biological and medical research, and is poised to have a similar impact in medicine. This tool is just one of a number of developments in our capability to identify, quantitate and functionally characterize the components of the biological networks keeping us healthy or making us sick, but in many respects it has played the leading role in this process.
ID006	Restriction enzyme - an enzyme that cuts DNA at internal phosphodiester bonds; different types exist and the most useful ones for molecular biology (Type II) are those which cleave at a specific DNA sequence
ID006	the transcription initiation complex is composed of promoter sequences and DNA binding proteins. These two components of transcription are normally described as cis-acting elements and trans-acting factors.
ID083	Promoters in Eukaryotes can be focused (a single TATA box) or diffuse (several potential transcription start sites). Eukaryotic promoters consist of proximal promoter elements (TATA, CAT and GC-rich regions), Enhancers are cis-acting sequences that up-regulate gene expression and can be found at some distance from the gene itself. Chromosome looping brings enhancers and the proteins that are bond to them into contact with promoter proximal elements to activate transcription.
ID083	Promoters in Eukaryotes can be focused (a single TATA box) or diffuse (several potential transcription start sites). Eukaryotic promoters consist of proximal promoter elements (TATA, CAT and GC-rich regions), Enhancers are cis-acting sequences that up-regulate gene expression and can be found at some distance from the gene itself. Chromosome looping brings enhancers and the proteins that are bond to them into contact with promoter proximal elements to activate transcription.
ID083	Promoters in Eukaryotes can be focused (a single TATA box) or diffuse (several potential transcription start sites). Eukaryotic promoters consist of proximal promoter elements (TATA, CAT and GC-rich regions), Enhancers are cis-acting sequences that up-regulate gene expression and can be found at some distance from the gene itself. Chromosome looping brings enhancers and the proteins that are bond to them into contact with promoter proximal elements to activate transcription.
